---
layout: full_article
title: "Genetic Structure in Long Tailed Macaque Populations in The Region of East Java: Diversity of Mithochondrial DNA in Alas Purwo and  Baluran Population"
author: "I Nengah Wandia, I Gusti Agung Arta Putra, I Gede Soma"
categories: jikh
canonical_url: https://jurnal.harianregional.com/jikh/full-20513 
citation_abstract_html_url: "https://jurnal.harianregional.com/jikh/id-20513"
citation_pdf_url: "https://jurnal.harianregional.com/jikh/full-20513"  
comments: true
---

<p><span class="font5">Jurnal Ilmu dan Kesehatan Hewan, Februari 2015</span></p>
<p><span class="font5">Vol 3 No 1: 35-40</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font7" style="font-weight:bold;"><a name="bookmark1"></a>Genetic Structure in Long Tailed Macaque Populations in The Region of East Java: Diversity of Mithochondrial DNA in Alas Purwo and Baluran Population</span></h1>
<h2><a name="bookmark2"></a><span class="font7" style="font-style:italic;"><a name="bookmark3"></a>Struktur Genetik Populasi Monyet Ekor Panjang di Kawasan Jawa Timur: Diversitas DNA Mitokondria pada Populasi Alas Purwo dan Baluran</span></h2>
<h3><a name="bookmark4"></a><span class="font5" style="font-weight:bold;"><a name="bookmark5"></a>I Nengah wandia<sup>1,2,*</sup>, I Gusti Agung Arta Putra<sup>1,3</sup>, I Gede Soma<sup>1,2</sup></span></h3>
<p><span class="font5"><sup>1</sup>Primate Research Center, Udayana University, Kampus Bukit-Jimbaran Bali <sup>2</sup>Faculty of Veterinary Medicine, Udayana University, Jl. PB Sudirman, Denpasar Bali <sup>3</sup>Faculty of Animal Husbandry, Udayana University Kampus Bukit-Jimbaran Bali *corresponding author: </span><a href="mailto:wandia@unud.ac.id"><span class="font5">wandia@unud.ac.id</span></a></p>
<p><span class="font4" style="font-weight:bold;">ABSTRAK</span></p>
<p><span class="font4">Monyet ekor panjang (</span><span class="font4" style="font-style:italic;">Macaca fascicularis</span><span class="font4">) telah menyebar secara luas hampir di seluruh kawasan di Indonesia, namun, kini, keberadaannya terpisah-pisah menjadi populasi lokal di dalam suatu pulau. Penelitian ini ditujukan untuk mengeksplorasi diversitas genetik populasi lokal monyet ekor panjang yang menempati Kawasan Jawa Timur. Sebagai penelitian pendahuluan, diversitas genetik populasi dikaji dengan marka molekuer </span><span class="font4" style="font-style:italic;">d-loop region</span><span class="font4"> mtDNA menggunakan sepasang primer khusus. Sejumlah 29 sampel darah (15 dari populasi Alas Purwo dan 14 dari populasi Baluran) diekstraksi untuk mendapatkan DNA total. </span><span class="font4" style="font-style:italic;">d-loop region</span><span class="font4"> mtDNA diamplifikasi melalui teknik PCR. Produk PCR disekuen dengan metode </span><span class="font4" style="font-style:italic;">dideoxynucleotide chain-termination</span><span class="font4"> pada Macrogen Inc. Korea. Hasil penelitian menunjukkan bahwa sejumlah 6 haplotipe ditemukan dengan sebaran 3 jenis haplotipe di masing-masing populasi. Diversitas haplotipe berkisar 0,257 – 0,275 dan diversitas nukleotida sebesar 0,0012 – 0,0025. Diferensiasi genetik antar populasi adalah F<sub>ST</sub> = 44,1%. Rekonstruksi filogeni menunjukkan bahwa seluruh haplotipe/individu membentuk dua kelompok (</span><span class="font4" style="font-style:italic;">haplogroup</span><span class="font4">) yang berbeda dan tidak ditemukan adanya </span><span class="font4" style="font-style:italic;">shared</span><span class="font4"> haplotipe antar populasi. Hasil penelitian dapat disimpulkan bahwa kedua populasi ( Alas Purwo dan Baluran) bersifat unik dan memiliki diversitas genetik rendah.</span></p>
<p><span class="font4">Kata kunci: monyet ekor panjang, diversitas genetik, </span><span class="font4" style="font-style:italic;">d-loop region</span><span class="font4" style="font-weight:bold;">, </span><span class="font4">mtDNA</span></p>
<p><span class="font4" style="font-weight:bold;">ABSTRACT</span></p>
<p><span class="font4">Long tailed macaque (</span><span class="font4" style="font-style:italic;">Macaca fascicularis</span><span class="font4">) has distributed throughout the regions in Indonesia, however, they, now, live separately to many local populations within an island. This research aimed to explore the genetic diversity within and between populations of long tailed macaque in the Region of East Java. The genetic diversity was examined by molecular marker of d-loop region mtDNA using a pair of specific primers. A total of 29 blood samples (15 samples from Alas Purwo population and 14 samples from Baluran population) was extracted to find out a total DNA. The d-loop region mtDNA was amplified through PCR tecnique. The PCR products were sequenced using dideoxynucleotide chain termination method at Macogen Inc. Korea. The research found 6 haplotypes, in which 3 haplotypes were found in each population. Haplotype and nucleotide diversities varied from 0.257 to 0.275 and 0.0012 to 0.0025 repectively. Genetic differentiation between populatioon was F</span><span class="font3">ST</span><span class="font4">=44.1%. Phylogenetic reconstruction showed that all haplotypes constructed two different groups (haplogroups) and there was no shared haplotype betwen populations. It can be concluded that both populations (Alas Purwo and Baluran) are uique and have low genetic diversities.</span></p>
<p><span class="font4">Key words: long tailed macaque, genetic diversity, d-loop region, mtDNA</span></p>
<h3><a name="bookmark6"></a><span class="font5" style="font-weight:bold;"><a name="bookmark7"></a>INTRODUCTION</span></h3>
<p><span class="font5">Long tailed macaque (</span><span class="font5" style="font-style:italic;">Macaca fascicularis</span><span class="font5">) lives in female phylipatric social groups (Jolly, 1985). The females are being a</span></p>
<p><span class="font5">permanent member in a social group as long as they live in thier natal groups, in contrary, the males will go out from the natal group and migrate to others (Napier &amp;&nbsp;Napier, 1985; Mitchell &amp;&nbsp;Erwin, 1986). The social group of</span></p>
<p><span class="font5">long tailed macaques belongs to multimales-multifemales group, ie, many adult males and adult females and thier offsprings in a group (Napier &amp;&nbsp;Napier, 1985; Mitchell &amp;&nbsp;Erwin 1986; Bennett </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1995), and it is also frequent that many subgroups were found within a group (Rowe, 1996).</span></p>
<p><span class="font5">A high adaptibility of the long tailed macaques (Napier &amp;&nbsp;Napier, 1985) makes them spread broadly in the world (Mitchell &amp;&nbsp;Erwin, 1986; Bennett </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1995). Their ability to conquer a new habitat is also supported by thier ability to change the strategy of resource utilisation for life (Brotcorne </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2014). In Indonesia, this macaques are found at almost islands in South Archipelago of Indonesia (Fooden, 1995).</span></p>
<p><span class="font5">Primatologist believes that the long tailed macaques in South Archipelago of Indonesia migrate from west to east, in which the Java as the origin of population. Migration of this animal from Java to Bali Island occurred many times when Java-Bali uniting in Sunda Shelf (Eudey, 1980; Fooden, 1995). To day, the long tailed macaques live in local poplation with limited habitat (Kawamoto </span><span class="font5" style="font-style:italic;">et al.,</span><span class="font5"> 1984; Wandia, 2000; Wandia </span><span class="font5" style="font-style:italic;">et al.,</span><span class="font5"> 2004). In such condition, it is important to examine the genetic diversity within and between populations for getting a genenically consideration in making conservation strategy for the future life.</span></p>
<p><span class="font5">Macaque populations in the Region of East Java play an important role in the history of their distributions in South Archipelago of Indonesia. The pulation in Alas Purwo National Park and Baluran National Park dwell the most closely to Bali Island, so these populations have a high potency to migrate to Bali Island. Exploring the geneic diversity of the populations and comparing them to those of Bali populations can enlighten the distribution history of the macaques. This preliminary research explored the genetic diversity of long tailed macaque populations at Alas Purwo National Park and Baluran National Park, in which the population genetic diversities were examined using a d-loop region of mitochondrial DNA (mtDNA) as molecular marker.</span></p>
<p><span class="font5" style="font-weight:bold;">MATERIALS AND METHODS</span></p>
<h3><a name="bookmark8"></a><span class="font5" style="font-weight:bold;"><a name="bookmark9"></a>Blood Samples</span></h3>
<p><span class="font5">A total of 29 blood samples was collected from two populations namely 15 samples from Alas Purwo National Park and 14 samples from Baluran National Park. The blood samples were taken after the macaques were anesthetized with Ketamine (dose 10 mg/kg body weight) combined Xylazine at ratio 5:1 using blow pipe. A 5-10 ml blood was taken through femoral vein using 10 ml syringe that formerly added by 0.4 ml 10% EDTA as anticoagulant. All the blood samples have been collected in 2007 and deposed at Genetic and Tissue Culture Laboratory, Primate Research Center-Udayana Universiy.</span></p>
<h3><a name="bookmark10"></a><span class="font5" style="font-weight:bold;"><a name="bookmark11"></a>Total DNA Extraction and D-Loop Region mtDNA Amplification</span></h3>
<p><span class="font5">Total DNA was extracted using QIAmp DNA Blood Kits produced by Qiagen. The steps of extraction is in accordant to company protocol (Qiagen, 2007). The d-loop region mtDNA was amplified by PCR using a pair of specific sequence of primers, namely 5’-ATCACGGGTCTATCACCCTA-3’ (F) dan 5’-GGCCAGGACCAAGCCTATTT-3’ (R) The volume of the PCR unit was 50 </span><span class="font6">µ</span><span class="font5">L which contain 1x PCR buffer, 3 mM MgCl2, 0.2 mM dNTP (mix), 0.2 µM each primer, and 0.625 U DNA polymerase, and 2 </span><span class="font6">µ</span><span class="font5">L DNA template. The PCR was carried out in Applied Biosystems 2720 Thermal Cycler, repeated by 40 cycles with 58<sup>O</sup>C of annealing temperature. The PCR products was electrophoresed at 50 V in 1.5% agarose gel for 30 minutes. The fragment was developed by ethidium bromide and its length was compared to 100 bp ladder standard marker. The positive PCR products were, then, sequenced with dideoxynucleotide chain termination method at the public company of Macrogen Inc. Korea.</span></p>
<h3><a name="bookmark12"></a><span class="font5" style="font-weight:bold;"><a name="bookmark13"></a>Data Analysis</span></h3>
<p><span class="font5">Editing and aligning the nucleotide sequences used MEGA 6 software (Tamura </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2013). Haplotype and its ditribution were</span></p>
<p><span class="font5">analyzed using dnaSP ver. 5.10.01 software (Rozas </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., &nbsp;2010). Genetic diversities</span></p>
<p><span class="font5">within population (at the level of nucleotide and haplotype) and the genetic distance (F<sub>ST</sub>) were calculated with Arlequin ver 3.5.1.2 software (Excoffier </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2005). MEGA 6 software (Tamura </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2013) was also used to reconstruct a phylogenetic tree with the substitusion model of Kimura 2 parameters and the statistic mehtod of maximum likelihood.</span></p>
<p><span class="font5" style="font-weight:bold;">RESULTS AND DISCUSSION</span></p>
<p><span class="font5" style="font-weight:bold;">Results</span></p>
<p><span class="font5" style="font-weight:bold;">Haplotype Distribution</span></p>
<p><span class="font5">Analysis of 29 samples of d-loop region mtDNA resulted 6 haplotypes (Table 1). In general, haplotype length was 574 nucleotides which composed of 31.22% (A), 23.76% (T), 33.76% (C), and 11.27% (G). There were 16 variable sites across haplotypes, in which haplotype hap_ap1 had 2 nucleotides deletion (Table 2). Totally, transition/tranversion mutation ratio was R=0.51.</span></p>
<p><span class="font5">There was no cosmopolitan haplotype. Three unique haplotypes were found in each population (Table 2). Alas Purwo population was dominated by haplotype hap_ap2 (wild type), on the other hand, Baluran population was dominated by haplotype hap_bl1 (wild type).</span></p>
<table border="1">
<tr><td style="vertical-align:top;"></td><td style="vertical-align:bottom;">
<p><span class="font5">Table 1. Haplotypes of long tailed macaques in The Region of East Java</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">No</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Haplotype &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Nucleotide sequences*</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">1</span></p>
<p><span class="font4">2</span></p>
<p><span class="font4">3</span></p>
<p><span class="font4">4</span></p>
<p><span class="font4">5</span></p>
<p><span class="font4">6</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">hap_ap1 T C A T G A G G - A - T C T A G hap_ap2 . . . C . . . . A . A . A . . T hap_ap3 . A . C . . . . A . A . A . G T hap_bl1 . . . C . G . . A . A . A . . T hap_bl2 . . T C A C A C A C A C A A . A hap_bl3 G . . C . G . . A . A . A . . T</span></p></td></tr>
</table>
<p><span class="font4">*Nucleotides provided are the variable nucleotides (16 nucleotides) that differentiate the haplotypes. The number of variable sites is not showed. Haplotypes are named arbitrarily.</span></p>
<h3><a name="bookmark14"></a><span class="font5" style="font-weight:bold;"><a name="bookmark15"></a>Genetic Diversities Within Population</span></h3>
<p><span class="font5">Haplotype diversities (hd) and nucleotide diversities (π) are figured on Table 3. The haplotype diversities express the probability of two haplotypes to be differ at a time of consent, and the nucleotide diversities imply the proportion of nucleotide differentiation between two sequences (Kawamot </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2013). Alas Purwo population and Baluran population had hd and π relatively low.</span></p>
<p><span class="font5">Table 2. Distribution of Haplotypes within</span></p>
<p><span class="font5">Populations</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:top;">
<p><span class="font4">No</span></p></td><td rowspan="2" style="vertical-align:top;">
<p><span class="font4">Haplotipe</span></p></td><td colspan="2" style="vertical-align:middle;">
<p><span class="font4">Distribution</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Alas Purwo (n=15)</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Baluran (n=14)</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">hap_ap1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">0.07</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">2</span></p></td><td style="vertical-align:top;">
<p><span class="font4">hap_ap2</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.86</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">hap_ap3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.07</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;">
<p><span class="font4">hap_bl1</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font4">0.86</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">5</span></p></td><td style="vertical-align:top;">
<p><span class="font4">hap_bl2</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font4">0.07</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">6</span></p></td><td style="vertical-align:top;">
<p><span class="font4">hap_bl3</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font4">0.07</span></p></td></tr>
</table>
<table border="1">
<tr><td style="vertical-align:top;">
<p><span class="font5">Table</span></p></td><td style="vertical-align:top;">
<p><span class="font5">3. &nbsp;&nbsp;Genetic</span></p></td><td style="vertical-align:top;">
<p><span class="font5">Diversities</span></p></td><td style="vertical-align:top;">
<p><span class="font5">within</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td colspan="2" style="vertical-align:bottom;">
<p><span class="font5">Populations</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">No</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Parameter</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Alas Purwo</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Baluran</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Number of sample</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">15</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">14</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Number of haplotype</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Number of polymorphic sites</span></p></td><td style="vertical-align:top;">
<p><span class="font4">5</span></p></td><td style="vertical-align:top;">
<p><span class="font4">10</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Haplotype</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.257</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.275</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:bottom;">
<p><span class="font4">diversity (hd)</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">(0.142)</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">(0.148)</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">5</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Number of</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.667</span></p></td><td style="vertical-align:top;">
<p><span class="font4">1.429</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:bottom;">
<p><span class="font4">nucleotide differentiation between haplotypes</span></p></td><td style="vertical-align:top;">
<p><span class="font4">(0.541)</span></p></td><td style="vertical-align:top;">
<p><span class="font4">(0.926)</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">6</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Nuclotide</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.0012</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.0025</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:bottom;">
<p><span class="font4">diversity (π)</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">(0.0010)</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">(0.0018)</span></p></td></tr>
</table>
<p><span class="font4">Number in bracket: standard deviation</span></p>
<h3><a name="bookmark16"></a><span class="font5" style="font-weight:bold;"><a name="bookmark17"></a>Genetic Distance and Phylogenetic Tree</span></h3>
<p><span class="font5">The genetic distance was calculated by F<sub>ST</sub> which is good for figuring out the short divergent time of two populatons (Excoffier </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2005). The genetic distance between Alas</span></p>
<p><span class="font5">Puwo population and Baluran population was F<sub>ST</sub>=0.441 (P=0.00..; 110 permutations). The genetic distance between populations was very significant. Phylogenetic recunstruction depicted that all haplotypes composed 2 groups (haplogroups), arbitrarily called Alas Purwo clade and Baluran clade. Each haplogroup had a unique haplotype (Figure 1).</span></p>
<p><span class="font0">AP12</span></p>
<p><span class="font0">AP13</span></p>
<p><span class="font0">AP11</span></p>
<p><span class="font0">AP10</span></p>
<p><span class="font0">AP9</span></p>
<p><span class="font0">AP8</span></p>
<div>
<p><span class="font0">AP7</span></p>
</div><br clear="all">
<div>
<p><span class="font0"><sup>34</sup> AP6</span></p>
</div><br clear="all">
<div>
<p><span class="font2">ALAS PURWO</span></p>
</div><br clear="all">
<p><span class="font0">AP5</span></p>
<p><span class="font0">AP4</span></p>
<p><span class="font0">AP3</span></p>
<p><span class="font0">Ap14</span></p>
<p><span class="font0">AP2</span></p>
<p><span class="font0">AP15</span></p>
<p><span class="font0">AP1</span></p>
<p><span class="font0">BL1</span></p>
<p><span class="font0">BL2</span></p>
<p><span class="font0">BL3</span></p>
<p><span class="font0">BL5</span></p>
<p><span class="font0">BL6</span></p>
<p><span class="font0">BL7</span></p>
<p><span class="font0">BL8</span></p>
<p><span class="font0">BL9</span></p>
<p><span class="font0">BL10</span></p>
<div>
<p><span class="font2">BALURAN</span></p>
</div><br clear="all">
<p><span class="font0">BL11</span></p>
<p><span class="font0">BL12</span></p>
<p><span class="font0">BL13</span></p>
<p><span class="font0">BL14</span></p>
<p><span class="font0">BL4 </span><span class="font2">3 BALURAN</span></p>
<p><span class="font0">0.002</span></p>
<p><span class="font4">Figure 1. Phylogenetic reconstruction of Long Tailed Macaques in the Region of East Java. AP Alas Purwo, BL Baluran</span></p>
<h3><a name="bookmark18"></a><span class="font5" style="font-weight:bold;"><a name="bookmark19"></a>Discussion</span></h3>
<p><span class="font5">The long tailed macaque populations are very cohesive (Kawamota </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1984; Evans </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2010). Likewise, the long tailed macaque populations in the Region of East Java are very cohesive. This bonding is demonstrated by no cosmopolitan haplotype or no shared haplotype between population. Female migration is not supported by the data found, or if there is a migration between populations it would be very small number per generation. The present study showed that each population had a different wild type of haplotype. The haplotype hap_ap2 and hap_bl1 were the wild type haplotypes of Alas Purwo population and Baluran population consecutively. This finding strengthens the two populations develop an evolutionary history of live independently. The uniqueness of haplotype especially the most common haplotype can be used for a haplotype marker of population identity. Such population can be rendered in a different management unit to maintain its uniqueness</span></p>
<p><span class="font5">of the genetic variation within population (Frankham </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2004).</span></p>
<p><span class="font5">Mitochondrial DNA (mtDNA) is a strictly maternally inherited molecular marker that frequently used to examine the genetic variation within population because of having a high mutation rate (Wallace </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1999; Dimauro &amp;&nbsp;Davidzon, 2005; Evans </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2010). However, variation resources of mtDNA one fourth of the nuclear genome make a variant of mtDNA will reach a fixation state quicker than those of the nuclear genome (Perwitasari-Farajallah </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1999). The present study showed that Alas Purwo population and Baluran population had hd and π relatively low. These may associated to a small number of different haplotypes found in each population and also a small number of nucleotide differentation between sequences within population. Comparing to those, hd and π of 7 bonobo popuations in Congo were 0.694 – 0.980 and 0.0166 – 0.0256 consecutively (Kawamoto </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2013). The low genetic diversities within population found in present study may reflect a small or even no gene flow between populations. In this case, although mtDNA has a high mutation rate, on the other hand, it has one fourth of resource of variation and there is no female migration between popuations showed by the data, as the final result, the population will have a low genetic variation. Many other factors can attribute to that condition, these are a bottle neck effect in the past, a founder effect, a selection, and a genetic drift (Avise, 1994; Nozawa </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1996; Hartl &amp;&nbsp;Clark, 1997; Wallace </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1999). A further study is needed to better understand factors that associate to a low genetic diversities in long talied macaque populatios in the Region of East Java.</span></p>
<p><span class="font5">A gene flow which is expressed by migration of indivudual inter populations is not only a new genetic resource within a population but also it homogenizes the genetic variation between populations (Nozawa </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 1996; Hartl &amp;&nbsp;Clark, 1997). The genetic distance between Alas Puwo population and Baluran population was F<sub>ST</sub>=0.441, in which this genetic distance was very significant</span></p>
<p><span class="font5">(P=0.00..; 110 permutations). This fact indicates that there is a very small or even no gene flow between populations. Each population looks like having its own evolutionary process. This fenomenon was also found at long tailed populations in Bali Island (Wandia </span><span class="font5" style="font-style:italic;">et al</span><span class="font5">., 2014). The phylogenetic reconstruction also supports this event that all haplotypes were splitted to two defferent groups (haplogroups) without shared haplotype between populations (Figure 1).</span></p>
<h3><a name="bookmark20"></a><span class="font5" style="font-weight:bold;"><a name="bookmark21"></a>CONCLUSION</span></h3>
<p><span class="font5">Genetic Structure of long tailed macaque populaton in Alas Purwo and Baluran based on d-loop region mtDNA was unique. Haplotype diversity within population was low and genetic differentiation between populations was high.</span></p>
<h3><a name="bookmark22"></a><span class="font5" style="font-weight:bold;"><a name="bookmark23"></a>ACKNOWLEDGEMENT</span></h3>
<p><span class="font5">Thank to our president of Udayana University who has been funding this researh through &nbsp;&nbsp;&nbsp;&nbsp;the &nbsp;&nbsp;&nbsp;&nbsp;Contract &nbsp;&nbsp;&nbsp;&nbsp;Number:</span></p>
<p><span class="font5">104.16/UN14.2/PNL.01.03.00/2014, March 5, 2014.</span></p>
<h3><a name="bookmark24"></a><span class="font5" style="font-weight:bold;"><a name="bookmark25"></a>REFERENCES</span></h3>
<p><span class="font4">Avise JC. 1994. Molecular Markers, Natural History, and Evolution. Chapman and Hall Inc. New York.</span></p>
<p><span class="font4">Bennett BT, Abee CR, Hendrickson R. 1995. </span><span class="font4" style="font-style:italic;">Nonhuman Primates in Biomedical Research. Biology and Management</span><span class="font4">. Academic Press Inc. San Diego, California.</span></p>
<p><span class="font4">Brotcorne F, Maslarov C, Wandia I N, Fuentes A,. Beudels‐ Jamar RC, and Huynen M-C. 2014. The Role of Anthropic, Ecological, and Social Factors in Sleeping Site Choice by Long‐Tailed &nbsp;&nbsp;&nbsp;&nbsp;Macaques &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(</span><span class="font4" style="font-style:italic;">Macaca</span></p>
<p><span class="font4" style="font-style:italic;">fascicularis</span><span class="font4">). &nbsp;&nbsp;&nbsp;</span><span class="font4" style="font-style:italic;">American Journal of</span></p>
<p><span class="font4" style="font-style:italic;">Primatology</span><span class="font4">. DOI: &nbsp;&nbsp;&nbsp;10.1002/ajp.22299.</span></p>
<p><span class="font4">Published online XX Month Year in Wiley Online Library (wileyonlinelibrary.com).</span></p>
<p><span class="font4">Dimauro S, Davidzon G. 2005. Mitochondrial DNA and disease. </span><span class="font4" style="font-style:italic;">Annals of Medicine</span><span class="font4">. 2005; 37: 222–232</span></p>
<p><span class="font4">Eudey AA. 1980. Pleistocene glacial phenomena and the evolution of Asian macaques. In </span><span class="font4" style="font-style:italic;">The Macaques</span><span class="font4">. </span><span class="font4" style="font-style:italic;">Studies in Ecology, Behavior and Evolution</span><span class="font4">. Edited by D.G. Lindburg. :52-83.</span></p>
<p><span class="font4">Evans BJ, Pin L, Melnick DJ, Wright SI. 2010. Sex-Linked Inheritance in Macaque Monkeys: &nbsp;&nbsp;Implications for Effective</span></p>
<p><span class="font4">Population Size and Dispersal to Sulawesi. </span><span class="font4" style="font-style:italic;">Genetics</span><span class="font4"> 185: 923–937 ( July 2010). DOI: 10.1534/genetics.110.116228.</span></p>
<p><span class="font4">Excoffier L, Laval G, and Schneider S. 2005. Arlequin ver. 3.0: An integrated software package for population genetics data analysis. </span><span class="font4" style="font-style:italic;">Evolutionary Bioinformatics</span><span class="font4"> Online </span><span class="font4" style="font-weight:bold;">1</span><span class="font4">:47-50.</span></p>
<p><span class="font4">Fooden J. 1995. </span><span class="font4" style="font-style:italic;">FIELDIANA. Zoology</span><span class="font4">. New Series No. 81. Systematic Review of Southeast Asian Longtail Macaques, </span><span class="font4" style="font-style:italic;">Macaca fascicularis</span><span class="font4"> (Raffles, [1821]). Published by Field Museum of Natural History. USA.</span></p>
<p><span class="font4">Frankham R, Ballou JD, Briscoe DA. 2004. </span><span class="font4" style="font-style:italic;">A Primier of Conservation &nbsp;&nbsp;&nbsp;Genetics</span><span class="font4">.</span></p>
<p><span class="font4">Cambridge University Press. New York.</span></p>
<p><span class="font4">Hartl DL, Clark AG. 1997. </span><span class="font4" style="font-style:italic;">Principles of Population &nbsp;Genetics</span><span class="font4">. Third &nbsp;ed. Snauer</span></p>
<p><span class="font4">Associates, &nbsp;Inc. Publishers. &nbsp;Sunderland,</span></p>
<p><span class="font4">Massachusetts</span></p>
<p><span class="font4">Jolly A. 1985. </span><span class="font4" style="font-style:italic;">The Evolution of Primate Behavior</span><span class="font4">. 2<sup>nd</sup> Ed. Macmillan Publishing Company. New York.</span></p>
<p><span class="font4">Kawamoto Y, Ischak TM, Supriatna J. 1984. Genetic variation within and between troops of the crab-eating macaque (</span><span class="font4" style="font-style:italic;">Macaca fascicularis</span><span class="font4">) on Sumatra, Jawa, Bali, Lombok and Sumbawa, Indonesia. </span><span class="font4" style="font-style:italic;">Primates</span><span class="font4">, 25(2):131-159.</span></p>
<p><span class="font4">Kawamoto Y, Takemoto H, Higuchi S, Sakamaki T, Hart JA,. Hart TB, Tokuyama N, Reinartz GE, Guislain P, Dupain J, Cobden AK, Mulavwa MN, Yangozene K, Darroze S, Devos C, Furuichi T. 2013. Genetic Structure of Wild Bonobo Populations: Diversity of Mitochondrial DNA and Geographical Distribution. </span><span class="font4" style="font-style:italic;">PLoS ONE</span><span class="font4"> 8(3): &nbsp;e59660.</span></p>
<p><span class="font4">doi:10.1371/journal.pone.0059660.</span></p>
<p><span class="font4">Mitchell G &amp;&nbsp;Erwin J. 1986. </span><span class="font4" style="font-style:italic;">Behavior, Cognition, and Motivation</span><span class="font4">. Comparative Primate Biology. Volume 2, Part A. Alan R. Liss, Inc., New York.</span></p>
<p><span class="font4">Napier JR &amp;&nbsp;Napier PH. 1985. </span><span class="font4" style="font-style:italic;">The Natural History of the Primates</span><span class="font4">. The British Museum (Natural History). Cromwell, London.</span></p>
<p><span class="font4">Nozawa K, Shotake T, Minezawa M, Kawamoto Y, Kawamoto K, Kawamoto S. 1996. Population-genetic studies of the Javanese macaque, </span><span class="font4" style="font-style:italic;">Macaca fuscata</span><span class="font4">. In: </span><span class="font4" style="font-style:italic;">Variations in the Asian Macaques</span><span class="font4">. T. Shotake and K. Wada (eds.). Tokai University Press. Tokyo, Japan.: 1-36</span></p>
<p><span class="font4">Perwitasari-Farajallah D, Kawamoto Y, Suryobroto B. &nbsp;1999. Variation in blood</span></p>
<p><span class="font4">proteins and mitochondrial DNA within and between local populations of longtailed macaques, </span><span class="font4" style="font-style:italic;">Macaca fascicularis</span><span class="font4">, on the island</span></p>
<p><span class="font4">of Jawa, Indonesia. </span><span class="font4" style="font-style:italic;">Primates</span><span class="font4">. 40(4): 581</span></p>
<p><span class="font4">595.</span></p>
<p><span class="font4">Qiagen 2007. QIAamp DNA Mini and Blood Mini Hanbook. 2<sup>nd</sup> Eds. November 2007. Pp 2729.</span></p>
<p><span class="font4">Rowe N. 1996. </span><span class="font4" style="font-style:italic;">The Pictorial Guide to the Living Primates</span><span class="font4">. Pogonias Press. New York.</span></p>
<p><span class="font4">Rozas, J dan Librado P. Librado (2009). DnaSP v5: a Software for comprehensive analysis for DNA polymorphism data. </span><span class="font4" style="font-style:italic;">Bioimformatics </span><span class="font4">25: 1451-1452.</span></p>
<p><span class="font4">Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. 2013. MEGA6. Molecular Evolutionary Genettics Analusis version 6. Molecular Biology and Evolution: &nbsp;30</span></p>
<p><span class="font4">2725:2729.</span></p>
<p><span class="font4">Wallace DC, Brown MD, Lott MT. 1999. Mitochondrial DNA variation in human evolution and disease. </span><span class="font4" style="font-style:italic;">Gene</span><span class="font4"> 238 ( 1999)</span></p>
<p><span class="font4">211±230</span></p>
<p><span class="font4">Wandia I N. 2003. Mikrosatelit sebagai Penanda molekul untuk mengukur polimorfisme genetik monyet ekor panjang di Sangeh, Bali. </span><span class="font4" style="font-style:italic;">J. Vet.</span><span class="font4"> 4(3):93-100.</span></p>
<p><span class="font4">Wandia I N, Supraptini-Mansjoer S, Suryobroto B. 2004. Polimorfisme genetik monyet ekor panjang di daerah pariwisata Uluwatu, Bali. </span><span class="font4" style="font-style:italic;">J. Vet.</span><span class="font4"> 5(2):67-76.</span></p>
<p><span class="font4">Wandia IN, Arta Putra IGA, Soma IG. 2014. Diversitas genetik monyet ekor panjang menggunakan marka </span><span class="font4" style="font-style:italic;">d-loop region mtDNA </span><span class="font4">di Kawasan Bali Utara. </span><span class="font4" style="font-style:italic;">Prosiding Senastek 2014</span><span class="font4">: 696-704.</span></p>
<p><span class="font1">40</span></p>