---
layout: full_article
title: "The Stability of the Bali’s Rabies Virus Molecular Marker for the Development of Diagnostic Method"
author: "Putu Dimas Abiyoga, Gusti Ayu Yuniati Kencana, I Nyoman Dibia"
categories: jikh
canonical_url: https://jurnal.harianregional.com/jikh/full-20530 
citation_abstract_html_url: "https://jurnal.harianregional.com/jikh/id-20530"
citation_pdf_url: "https://jurnal.harianregional.com/jikh/full-20530"  
comments: true
---

<p><span class="font4">Jurnal Ilmu dan Kesehatan Hewan, Agustus 2015</span></p>
<p><span class="font4">Vol 3 No 2: 45-50</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font6" style="font-weight:bold;"><a name="bookmark1"></a>The Stability of the Bali’s Rabies Virus Molecular Marker for the Development of Diagnostic Method</span></h1>
<h2><a name="bookmark2"></a><span class="font6" style="font-style:italic;"><a name="bookmark3"></a>Stabilitas Penanda Molekuler Virus Rabies Bali untuk Pengembangan Metode Diagnosa</span></h2>
<p><span class="font4" style="font-weight:bold;">Putu Dimas Abiyoga<sup>1,2*</sup>, Gusti Ayu Yuniati Kencana<sup>1</sup>, I Nyoman Dibia<sup>2 </sup></span><span class="font4"><sup>1</sup>Magisterial Program of Veterinary Medicine, Udayana University Jl. PB Sudirman, Denpasar <sup>2</sup>Laboratory of Virology, Animal Disease Investigation Centre, Denpasar Corresponding author: </span><a href="mailto:abiyogadimas@yahoo.co.id"><span class="font4">abiyogadimas@yahoo.co.id</span></a></p>
<p><span class="font3" style="font-weight:bold;">ABSTRAK</span></p>
<p><span class="font3">Penelitian ini bertujuan untuk membuktikan penanda molekuler virus rabies Bali masih tetap </span><span class="font3" style="font-style:italic;">conserve</span><span class="font3"> dan untuk pengembangan metode diagnosa rabies berbasis penanda molekuler. Tiga puluh sampel otak anjing yang terinfeksi rabies pada tahun 2014 dan 2015 digunakan pada penelitian. Sekuen nukleotida yang yang didapatkan dari penelitin dan yang diakses di </span><span class="font3" style="font-style:italic;">GenBank</span><span class="font3"> dianalisis dengan program MEGA 5.2. Hasil analisis membuktikan bahwa asam amino spesifik Isoleusin pada posisi 308 (</span><span class="font3" style="font-style:italic;">open reading frame</span><span class="font3">) sebagai penanda molekuler virus rabies Bali masih </span><span class="font3" style="font-style:italic;">conserve</span><span class="font3">. Gen N yang diamplifikasi dengan teknik RT-PCR yang menggunakan primer spesifik menunjukkan bahwa masing-masing isolate virus rabies memiliki pita khas , dapat membedakan isolat Bali dengan isolat lain di Indonesia.</span></p>
<p><span class="font3">Kata kunci: Bali, rabies, penanda molekuler, diagnosa, RT-PCR.</span></p>
<p><span class="font3" style="font-weight:bold;">ABSTRACT</span></p>
<p><span class="font3">The objectives of this study were to prove that the molecular marker of Bali’s rabies virus is still conserve and to develop a diagnostic method based on molecular marker. Thirty brain samples of dog that had been infected by a rabies virus from 2014 and 2015 were used for this research. The sequences of nucleotide which were obtained and the sequences of nucleotide accessed in GenBank were analyzed using MEGA 5.2 software. The result provided that the specific amino acid (isoleusin) at position 308 (open reading frame) as a molecular marker of Bali’s rabies virus was still conserve. Fragment of N gene amplified by Reverse RT-PCR method with a specifically designed primers showed that every isolate of rabies virus had it’s own typical band, and could distinguish Bali isolate from the others in Indonesia.</span></p>
<p><span class="font3">Keywords: Bali, rabies, molecular marker, diagnose, RT-PCR</span></p>
<h3><a name="bookmark4"></a><span class="font4" style="font-weight:bold;"><a name="bookmark5"></a>INTRODUCTION</span></h3>
<p><span class="font4">Based on the Decree of the Minister of Agriculture No. 4026/Kpts /OT.140/3/2013, rabies is grouped into Strategic Animal Diseases (PHMS) and given priority in the prevention, control and eradication. Cases of rabies in Bali confirmed in the laboratory first time in November 2008. The disease raises serious public health problem in all districts/municipalities in the Province of Bali. Efforts to control and eradicate rabies have been implemented over five years through a mass vaccination program in Bali, but until now the cases of rabies in animals (especially dogs) still occur. Outbreaks of</span></p>
<p><span class="font4">rabies occurred in Bali is very likely due to the inclusion of rabies infected animals.</span></p>
<p><span class="font4">Rabies is a fatal zoonotic disease that is caused by the rabies virus. As a member of the RNA virus, rabies virus is often characterized by excessive genetic variation by mutation during replication. This occurrence is due to the absence of a mechanism for proofreading and error correction after replication by RNA polymerase enzyme. The most common mutation occurred in rabies virus is a mutation of a single nucleotide in the genome of a virus called point mutations. The occurrence of point mutations often happen naturally (Ratam, 2005). Point mutations can</span></p>
<p><span class="font4">be synonymous or non-synonymous substitutions. In synonymous nucleotide substitution, nucleotide change is not followed by changing in the amino acids of the protein expressed. According to Nagarajan </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">. (2006) changing to a new specific amino acids that occurred from non-synonymous substitutions in a gene of rabies virus can produce a new molecular marker. Dibia </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">. (2014), stated that the rabies virus of Bali isolates has a molecular marker composed of amino acid isoleucine at position 308 (open reading frame). That molecular markers can be used as markers to investigate the dynamic and distribution of rabies virus epidemiologically. Therefore, the molecular characterization of the rabies virus isolates that keep in nature needs to be done. This research aimed to prove that the Bali rabies virus is still conserve throughout the years using a specific molecular marker.</span></p>
<h3><a name="bookmark6"></a><span class="font4" style="font-weight:bold;"><a name="bookmark7"></a>MATERIALS AND METHODS</span></h3>
<p><span class="font4">Thirty samples of brains of animals were used in this study. The samples were collected from dogs infected by rabies in 2014 and 2015 from several areas in Bali. For the comparation, the research also used each one brain of dog from Sumatra, Java, Kalimantan, Sulawesi and Flores that detected positive for rabies.</span></p>
<h3><a name="bookmark8"></a><span class="font4" style="font-weight:bold;"><a name="bookmark9"></a>Flourescent Antibody Test (FAT)</span></h3>
<p><span class="font4">Flourescent Antibody Test was done by making smears and fixed with acetone at 20°C for 30 minutes. After drying at room temperature, preparations were imbued with anti-rabies nucleocapsid conjugate. Incubation was done inside the incubator at 37 ° C for 30 minutes. Then, preparations were washed with a solution of PBS pH 7.2 for 3 times and followed by adding aqueous mounting media and closed with a cover slip. Furthermore, the examination was conducted under a fluorescent microscope and neuron cells infected with the rabies virus will be marked by a fluorescent green color (OIE, 2008; Trimarchi and Nadin-Davis, 2007; Direktorat Kesehatan Hewan, 2009).</span></p>
<h3><a name="bookmark10"></a><span class="font4" style="font-weight:bold;"><a name="bookmark11"></a>RNA Isolation Rabies</span></h3>
<p><span class="font4">Brains of animals that positive for the rabies virus based on FAT, were made into 20% suspension in PBS pH 7.4 and inactivated by SDS (450 mL of 20% brain suspension was added 50 mL of 10% SDS). RNA isolation was done by using Trizol®reagent (Invitrogen), according to the manufacturer's instructions. A total of 250 mL and 750 mL suspension brain Trizol incorporated into 1.5 ml eppendorf tube. Vorteks mixture for one minute. After incubation at room temperature (20-25 ° C) for 5 minutes, to the mixture was added chloroform 200 mL, then vortex 15 seconds and incubated at room temperature for 15 minutes. Next, the tube was centrifuged at a speed of 12,000 RCF for 15 minutes. Aquaeus part was taken and put into a sterile 1.5 ml eppendorf tube. It was added to 500 mL of isopropyl alcohol and mixed until homogeneous. The mixture was incubated for 10 min at room temperature. Furthermore eppendorf tube which already contains the mixture was centrifuged at 12,000 RCF for 10 min, the supernatant was discarded, and added alcohol 70% 1000 mL. After vortex and centrifuged at 7500 RCF for 5 min, the supernatant was discarded, while the sediment was dried at a temperature of 55°C in incubator and resuspended in ultrapure aquabidest (Otsu-WI, Otsuka). Subsequently, the RNA of the rabies virus was sequenced at Genetika Science, Jakarta.</span></p>
<h3><a name="bookmark12"></a><span class="font4" style="font-weight:bold;"><a name="bookmark13"></a>Sequence Data Analysis</span></h3>
<p><span class="font4">Data were analyzed using molecularbased MEGA 5.2 to determine the stability of specific amino acids as molecular markers of the Bali rabies virus. The nucleotide sequence obtained by this research was confirmed by the program of Basic Local Alignment Search Tool (BLAST) </span><a href="http://blast/"><span class="font4">(http://blast.</span></a><span class="font4">ncbi. nlm.nih.gov/Blast.cgi) (Kumar </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2004) to identify the typical sequence of nucleoprotein gene fragment (N) of rabies virus. The sequence of nucleotides obtained in this study was then lined with other nucleotide sequences of the rabies virus in Indonesia and in various countries that were accessed from</span></p>
<p><span class="font4">the GenBank </span><a href="http://www.ncbi.nlm.nih.gov/%20genomes"><span class="font4">(http://www.ncbi.nlm.nih.gov/</span></a><span class="font4"> </span><a href="http://www.ncbi.nlm.nih.gov/%20genomes"><span class="font4">genomes)</span></a><span class="font4"> using ClustalW alignment. GenBank access number of the rabies virus nucleotide sequences that were analyzed was displayed in analysis results. Determination of the presence of rabies virus genetic variation was based on the sequences of genes that encoding the nucleoprotein (N) and analyzed using MEGA 5.2 program (Tamura </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2007).</span></p>
<h3><a name="bookmark14"></a><span class="font4" style="font-weight:bold;"><a name="bookmark15"></a>Development of RT-PCR Method</span></h3>
<p><span class="font4">Thirty samples of brains of dogs that positive for rabies by DFAT were amplified using RT-PCR with a general and specific primers. On the reverse transcription (RT), complementary DNA (cDNA) ofthe rabies virus genome was synthesized using NF36 (C/T) (forward primer) (5'-TCAGGTGGTC TCYTTGAAGCC-3') position 36-56 nucleotides and NR1251 (reverse primer) (5'-TTAGTCGACCTCCGTTCA-3') positions 1232-1251 nucleotides. The primers is belong</span></p>
<p><span class="font4">to a general primer. Then, the primer pair NF898 (S) (forward primer) (5'-TACTCATCTAATGCAGTTGGTCACA-3 ') positions 898-922 nucleotides and NR1349 (reverse primer) (5'-CGAGTCACTAGAA TACGTCTTGTT-3') position 1326-1349 nucleotides, are a primary partner that specifically flank 451 base pairs.</span></p>
<p><span class="font4" style="font-weight:bold;">RESULTS AND DISCUSSION</span></p>
<h3><a name="bookmark16"></a><span class="font4" style="font-weight:bold;"><a name="bookmark17"></a>Results</span></h3>
<p><span class="font4">Detection of rabies virus antigen using methods DFAT has been carried out on 30 samples of brains of dogs that were diagnosed rabies from various regions in Bali and the samples from Sumatra, Kalimantan, Sulawesi, Java, and Flores showed rabies positive. A specific aggregate (nucleocapsid protein) can be detected by luminescence apple green or greenish yellow with a dark background (Figure 1).</span></p>
<div><img src="https://jurnal.harianregional.com/media/20530-1.jpg" alt="" style="width:387pt;height:135pt;">
<p><span class="font2" style="font-weight:bold;">Samples were positive FAT shown in Figure 1 CA.). In winch brain samples are aggregates of nucleocapsid protein are apple green or greenish yellow with a Harle background Whereas Figure 1 (B) brain samples HiH not indicate the presence of nucleocapsiH protein are apple green or greenish yellow.</span></p>
</div><br clear="all">
<h3><a name="bookmark18"></a><span class="font4" style="font-weight:bold;"><a name="bookmark19"></a>Molecular Marker Analysis</span></h3>
<p><span class="font4">The amino acid sequences of the Bali rabies virus nucleoprortein had a typical amino acid at position 308. This typical amino acid was isoleucine, which was different from others, the valin amino acid (Figure 2). The amino acid sequences of the Bali rabies virus nucleoprortein were obtained from the translation of nucleotide sequences using MEGA software and the other sequences got from on line searching using the program of Basic Local Alignment Search Tool (BLAST)</span></p>
<p><span class="font4">(</span><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi"><span class="font4">http://blast.ncbi.nlm.nih.gov/Blast.cgi</span></a><span class="font4">) (Kumar </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2004), and aligned by the program of ClustalW.</span></p>
<h3><a name="bookmark20"></a><span class="font4" style="font-weight:bold;"><a name="bookmark21"></a>Development of RT-PCR</span></h3>
<p><span class="font4">Thirty samples of brains of dogs that were positive for rabies by DFAT successfully amplified using RT-PCR with the general and specific primers. The general primer produced the 1251 bp of band length (Figure 3), while the specific primer yielded 451 bp of band length (Figure 4).</span></p><img src="https://jurnal.harianregional.com/media/20530-2.jpg" alt="" style="width:192pt;height:213pt;">
<p><span class="font1">figured Alignment of amino acid sequences Bali rabies vims nucleoprotein and rabies viruses that are accessed on the GeneBank. The same amino acid is indicated by dots. Specific substitution at position 308 indicated by the arrow in the top of the box.</span></p><img src="https://jurnal.harianregional.com/media/20530-3.jpg" alt="" style="width:205pt;height:90pt;">
<p><span class="font1" style="text-decoration:underline;">Figure. 3</span><span class="font1"><sub>:</sub> Results of electrophoresis OfRT-PCRproducts<sub>7</sub> M=marker (1 OObp ladder DNA) (+) =positive control, (-) =negative control, Lane 1-10= SampleBali rabies virus, Lane 11-15 = Samples viruses out Bali. The arrows indicate product-specific RT-PCR at position 1215 bp.</span></p><img src="https://jurnal.harianregional.com/media/20530-4.jpg" alt="" style="width:204pt;height:91pt;">
<p><span class="font1" style="text-decoration:underline;">Eig⅛re,.⅜,</span><span class="font1"> Results of electrophoresis ofRT-PCRproducts, M=marker (IOObpladder DNA) (+) =positive control, (-)=negative control,Lane 1-10=samplesBali rabies virus, Lane 11-15 = Samples viruses out Bali. The arrows indicate product-specific RT-PCR at position 451 bp.</span></p>
<h3><a name="bookmark22"></a><span class="font4" style="font-weight:bold;"><a name="bookmark23"></a>Discussion</span><br><br><span class="font4" style="font-weight:bold;"><a name="bookmark24"></a>Detection of Rabies Virus Using FAT</span></h3>
<p><span class="font4">According to Dean </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">. &nbsp;(1996);</span></p>
<p><span class="font4">Trimarchi and Nadin-Davis (2007) test results using the FAT has been in accordance with the standard requirements. The FAT is one of the test to detect rabies antigen in the network, with a sensitivity of 98-100% of</span></p>
<p><span class="font4">cases of rabies (McColl and Lunt., 2003; Tepsumethanon </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2004; OIE, 2008). The sensitivity of the test which is owned by FAT makes this test becomes a recommendation of World Health Organization as a standard test for rabies diagnose (OIE, 2008; Wacharapluesadee </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2008).</span></p>
<p><span class="font4">However, the sensitivity of the FAT test is influenced also by the quality of the samples, conjugate, equipment and capabilities of diagnostic tools (McElhinney </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2008). The test results which are efficient, fast and accurate are essential for the management of rabies carrier animal bitten patients (Akoso, 2007). But according to David </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">. (2002), this technique has the disadvantage of not being able to detect viral antigens in brain specimens that had rotted. Besides, the FAT is unable to distinguish the specific strains of rabies virus. Therefore, RT-PCR can be used as a complementary diagnostic tool for the FAT testing, mainly for animal brain samples taken after 8 hours of death.</span></p>
<h3><a name="bookmark25"></a><span class="font4" style="font-weight:bold;"><a name="bookmark26"></a>Stability of Molecular Markers of Bali Rabies Virus</span></h3>
<p><span class="font4">Results of the analysis of amino acid sequences derived from the nucleoprotein gene showed that molecular markers of Bali rabies virus isolates is still conserve. Dibia </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">. (2014) reported that Bali rabies virus has a substitution of non synonym unique amino acid isoleucine at position 308 (Open Reading Frame), while all of the rabies virus in Indonesia and in other countries in the world as well as some of the vaccine strains have amino acid valin in the same position (Figure 2).</span></p>
<p><span class="font4">Among the genes that are owned by the rabies virus, nucleoprotein gene is a gene that is highly conserve with 99% homology (Ito </span><span class="font4" style="font-style:italic;">et al.,</span><span class="font4"> 2001). Previous researchers, Konznetzoff </span><span class="font4" style="font-style:italic;">et al.</span><span class="font4"> (1998) identified that the N gene regions that most conserve locate at amino acid position 298-352.</span></p>
<p><span class="font4">Mutational events that occur in viral isolates of rabies in Bali has shown its character as a member of RNA viruses (Dibia </span><span class="font4" style="font-style:italic;">et al.</span><span class="font4">, 2014b). While, Murphy </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">. (2007)</span></p>
<p><span class="font4">stated that RNA viruses are characterized by a high rate of mutation during replication in the absence of proofreading and repair mechanisms mistake after replication by RNA polymerase enzyme. This study confirms that the the nucleoprotein of Bali rabies virus has a typical amino acid, isoleucine, at position 308, , which is not owned by the isolates of rabies in Indonesia and a few countries in the world. In addition, specific amino acids (isoleucine) is the part that most conserve in the N genes of rabies virus, so it can be said that the isoleucine at position 308 (open reading frame) of the nucleoprotein gene is a molecular marker for Bali rabies virus and it can be used as epidemiological markers to investigate the dynamics and distribution of the virus.</span></p>
<h3><a name="bookmark27"></a><span class="font4" style="font-weight:bold;"><a name="bookmark28"></a>Development of RT-PCR</span></h3>
<p><span class="font4">The results of RT-PCR indicate that the primers used are specific and stick to the expected position. In addition, the primer that is specifically designed in this study is able to detect both the N gene of rabies virus in animals of some infected areas in Bali. These results also indicate that the RT-PCR that developed have a high agreement with FAT method, and is able to distinguish the rabies virus of Bali isolates with rabies virus isolated from outside Bali, so this test can be used as a reliable confirmatory diagnostic tool.</span></p>
<p><span class="font4">The results support a similar study conducted by Benedictis </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., (2011) which revealed the superiority of RT-PCR. Furthermore, Benedictis </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">. (2011) reported that one-step RT-PCR showed relatively high specificity of 98.94% (CI: 97.55 to 99.65), the sensitivity of 99.71% (CI: 98.40 to 99.99) and accuracy 98.90% compared with the values obtained by DFAT used as the standard method of diagnose of rabies. Moreover, Benedictis </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2011 stated that the agreement between the test method of one step RT-PCR and the standard method of FAT has an almost perfect agreement with Cohen's Kappa coefficient of 0.977. In addition, the advantage of using RT-PCR is that the method is still able to detect viral antigens in brain specimens that had rotted</span></p>
<p><span class="font4">which was not detected using FAT (David </span><span class="font4" style="font-style:italic;">et al</span><span class="font4">., 2002).</span></p>
<h3><a name="bookmark29"></a><span class="font4" style="font-weight:bold;"><a name="bookmark30"></a>CONCLUSION</span></h3>
<ul style="list-style:none;"><li>
<p><span class="font5">1.</span><span class="font4"> &nbsp;&nbsp;&nbsp;Molecular markers </span><span class="font5">of </span><span class="font4">Bali rabies virus at position 308 &nbsp;(</span><span class="font5">open </span><span class="font4">reading frame</span><span class="font5">)</span></p></li></ul>
<p><span class="font4">remained stable</span><span class="font5">.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font5">2. &nbsp;&nbsp;&nbsp;The </span><span class="font4">method of RT-</span><span class="font5">PCR </span><span class="font4">using specific primers </span><span class="font5">of </span><span class="font4">Bali rabies virus strain </span><span class="font5">is able to </span><span class="font4">distinguish between the Bali an</span><span class="font5">d </span><span class="font4">no</span><span class="font5">n </span><span class="font4">Bali rabies virus.</span></p></li></ul>
<h3><a name="bookmark31"></a><span class="font4" style="font-weight:bold;"><a name="bookmark32"></a>ACKNOWLEMENT</span></h3>
<p><span class="font4">Author to convey gratitude to the Head of Center for Veterinary, Denpasar for giving research permission and facilities.</span></p>
<h3><a name="bookmark33"></a><span class="font4" style="font-weight:bold;"><a name="bookmark34"></a>REFERENCES</span></h3>
<p><span class="font3">Akoso BT. 2007. Pencegahan dan Pengendalian Rabies.Yogyakarta: Penerbit Kanisius.</span></p>
<p><span class="font3">Benedictics PD, Battisti CD, Dacheux L, Marciano S, Ormelli S, Salomoni A, Caenaszzo ST, Lapelletier A, Bourhy H, Capua I, Cattoli G. 2011. Lyssavirus Detection and Typing Using Pyrosequencing. </span><span class="font3" style="font-style:italic;">J Clin Microbiol </span><span class="font3">49(5) : 1932-1938.</span></p>
<p><span class="font3">David D, Yakobson B, Rotenberg D, Dveres N, Davidson I, Stram Y. 2002. Rabies Virus Detection by RT-PCR in Decompoosed Naturally Infected Brains. </span><span class="font3" style="font-style:italic;">Vet Microbiol </span><span class="font3">87(2) : 111-118.</span></p>
<p><span class="font3">Dean DJ, Abelseth MK, Anatasiu P. 1996. The Fluorescent Antibody Technique.In: Meslin, F. X., Kaplan, M. M., Koprowski, H., (ed). </span><span class="font3" style="font-style:italic;">Laboratory techniques in rabies, 4<sup>th</sup> ed. </span><span class="font3">WHO: 88-95.</span></p>
<p><span class="font3">Dibia N, Sumiarto B, Susetya H, Putra AAG, Mahardika IGNK, Scott-Orr H. 2014. Diagnosis and Molecular Marker Analysis of Bali Rabies Virus Isolates. </span><span class="font3" style="font-style:italic;">J. Vet.</span><span class="font3"> Vol.15(3): 288-297.</span></p>
<p><span class="font3">Direktorat Kesehatan Hewan, 2009. Standar Nasional Metode Diagnosa Rabies. Jakarta: Direktorat Jenderal Peternakan, Departemen Pertanian. Jakarta.</span></p>
<p><span class="font3">Ito N, Kakemizu M, Ito KA, Yamamoto A, Yoshida Y, Sugiyama M, Minamoto N. 2001. A comparison of complete genom sequences of the attenuated RC-HL strain of rabies virus used for production of animal vaccine in Japan, and the parentral Nishigahara strain. </span><span class="font3" style="font-style:italic;">Microbiol.Immunol.</span><span class="font3"> 45: 51-58.</span></p>
<p><span class="font3">Kumar S, Tamura K, Nei M. 2004. MEGA3: Integrated software for Molecular Evolutionary Genetic Analysis and sequence alignment. </span><span class="font3" style="font-style:italic;">Brief. Bioinfor.</span><span class="font3"> 5(2): 150-163.</span></p>
<p><span class="font3">Kouznetzoff A, Buckle M, Tordo N. 1998. Identification of a region of the rabies virus N protein involved indirect binding to the viral RNA. </span><span class="font3" style="font-style:italic;">J Gen Virol</span><span class="font3"> 79 : 1005-1013.</span></p>
<p><span class="font3">McColl KA, Lunt RA. 2003. Australia and New Zealand Standard Diagnostic Procedures. Victoria: CSIRO Australian Animal Health Laboratory.</span></p>
<p><span class="font3">McElhinney LM, Fooks AR, Radford AD. 2008. Diagnostic tools for the detection of rabies virus.</span><span class="font3" style="font-style:italic;">European Journal Of Companion Animal Practice</span><span class="font3"> 3: 224-231.</span></p>
<p><span class="font3">Murphy FA, Gibbs EPJ, Horzinek MC, Studdert MJ. 2007. Veterinary Virology.Third edition. USA: Elsevier, Academic Press.</span></p>
<p><span class="font3">Nagarajan T, Mohanasubramanian B, Seshagiri EV, Nagendrakumar SB, Saseendranath MR, Satyanarayana ML, Thiagarajan D, Rangarajan PN, &nbsp;Srinivasan VA, &nbsp;2006.</span></p>
<p><span class="font3">Molecular Epidemiology of Rabies Virus Isolates in India.</span><span class="font3" style="font-style:italic;">J. Clin. Microbiol</span><span class="font3">. &nbsp;44(9):</span></p>
<p><span class="font3">3218-3224.</span></p>
<p><span class="font3">OIE (Office International des Epizooties). 2008. Manual of Standards for Diagnostic tests and Vaccines. Chapter 2.1.13: 304-323.</span></p>
<p><span class="font3">Ratam FA. 2005. Virologi. Cetakan pertama. Surabaya: Airlangga University Press.</span></p>
<p><span class="font3">Tamura K, Dudley J, Nei M, Kumar S. 2007. MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) Software Version 4.0. </span><span class="font3" style="font-style:italic;">Mol. Biol. Evol.</span><span class="font3"> 42(8): 1596-1599.</span></p>
<p><span class="font3">Tepsumethanon V, &nbsp;&nbsp;&nbsp;&nbsp;Lumlertdacha &nbsp;&nbsp;&nbsp;&nbsp;B,</span></p>
<p><span class="font3">Mitmoonpitak C, Sitprija V, Meslin FX, and Wilde H. 2004. Survival of naturally infected rabid dogs and cats.</span><span class="font3" style="font-style:italic;">Clin. Infect. Dis.</span><span class="font3"> 39: 278280.</span></p>
<p><span class="font3">Trimarchi CV., and Nadin-Davis SA., 2007. Diagnostic Evaluation. In: Jackson AC., and Wunner WH., (ed.). </span><span class="font3" style="font-style:italic;">Rabies</span><span class="font3">. Second edition. USA: Elsevier, Inc.: 411-470.</span></p>
<p><span class="font3">Wacharapluesadee &nbsp;&nbsp;&nbsp;&nbsp;S., &nbsp;&nbsp;&nbsp;&nbsp;Sutipanya &nbsp;&nbsp;&nbsp;&nbsp;J.,</span></p>
<p><span class="font3">Damrongwatanapokin S., Phumesin P., Chamnanpood P., Leowijuk &nbsp;&nbsp;&nbsp;C., and</span></p>
<p><span class="font3">Hemachudha T. 2008. Development of a TaqMan real-time RT-PCR assay for the detection of rabies virus.</span><span class="font3" style="font-style:italic;">J. Virol. Methods </span><span class="font3">151: 317-320.</span></p>
<p><span class="font0">50</span></p>