---
layout: full_article
title: "Prediction of Wave-induced Liquefaction using Artificial Neural Network and Wide Genetic Algorithm"
author: "Dwi Kristianto, Chastine Fatichah, Bilqis Amaliah, Kriyo Sambodho"
categories: lontar
canonical_url: https://jurnal.harianregional.com/lontar/full-25789 
citation_abstract_html_url: "https://jurnal.harianregional.com/lontar/id-25789"
citation_pdf_url: "https://jurnal.harianregional.com/lontar/full-25789"  
comments: true
---

<p><span class="font0" style="font-weight:bold;">LONTAR KOMPUTER VOL. 8, NO.1, APRIL 2017</span></p>
<p><span class="font0" style="font-weight:bold;">DOI : 10.24843/LKJITI.2017.v08.i01.p01</span></p>
<p><span class="font0" style="font-weight:bold;">p-ISSN 2088-1541</span></p>
<p><span class="font0" style="font-weight:bold;">e-ISSN 2541-5832</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font1" style="font-weight:bold;"><a name="bookmark1"></a>Predicon of Wave-induced Liquefaction using Artificial Neural Network and Wide Genetic Algorithm</span></h1>
<p><span class="font0">Dwi Kristianto<sup>a1</sup>, Chastine Fatichah<sup>b2</sup>, Bilqis Amaliah<sup>b3</sup>, Kriyo Sambodho<sup>b4</sup></span></p>
<p><span class="font0"><sup>a</sup>Postgraduate Program of Informatics Engineering Institut Teknologi Sepuluh Nopember Surabaya, Indonesia </span><a href="mailto:1dwi15@mhs.if.its.ac.id"><span class="font0">1dwi15@mhs.if.its.ac.id</span></a></p>
<p><span class="font0"><sup>b</sup>Institut Teknologi Sepuluh Nopember Surabaya, Indonesia Kampus ITS Keputih, Sukolilo, Surabaya, 60111, Jawa Timur </span><a href="mailto:2chastine@cs.its.ac.id"><span class="font0" style="text-decoration:underline;"><sup>2</sup>chastine@cs.its.ac.id</span></a></p>
<p><a href="mailto:3bilqis@cs.its.ac.id"><span class="font0" style="text-decoration:underline;"><sup>3</sup>bilqis@cs.its.ac.id</span></a><span class="font0" style="text-decoration:underline;"> </span><a href="mailto:4k_sambodho@oe.its.ac.id"><span class="font0"><sup>4</sup>k_sambodho@oe.its.ac.id</span></a></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Abstrak</span></p>
<p><span class="font0" style="font-style:italic;">Kerumitan solusi analitis dan numerik untuk pemodelan likuafaksi, pengujian laboratorium yang berulang dan observasi lapangan yang mahal, telah membuka peluang untuk mengembangkan pemodelan prediksi likuafaksi induksi gelombang yang sederhana, praktis, murah dan valid. Dalam studi ini, pemodelan regresi Artificial Neural Network (ANN) digunakan untuk memprediksi kedalaman likuafaksi. Pelatihan ANN menggunakan Genetic Algorithm (GA) yang telah dimodifikasi yang disebut sebagai Wide GA (WGA). WGA bertujuan untuk meningkatkan akurasi prediksi ANN dan terhindar dari kelemahan Back Propagation seperti konvergensi dini dan local optimum. WGA juga bertujuan untuk menghindari kelemahan GA yaitu keragaman populasi yang rendah dan cakupan pencarian sempit. Operasi kunci WGA yang Wide Tournament Selection, Multi-Parent BLX-α Crossover, Agregate Mate Pool Mutation dan Direct Fresh MutationCrossover. Akurasi prediksi ANN diukur dengan Median APE (MdAPE). Solusi global optimum WGA adalah konfigurasi bobot koneksi ANN yang terbaik dengan MdAPE terkecil.</span></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Kata kunci: </span><span class="font0" style="font-style:italic;">Likuafaksi akibat Gelombang, Prediksi Likuafaksi Tanah, Algoritma Genetika Lebar, Jaringan Saraf Tiruan, Propagasi Balik.</span></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Abstract</span></p>
<p><span class="font0" style="font-style:italic;">The hassle of analytical and numerical solution for liquefaction modeling, repetitive laboratory testing and expensive field observations, have opened opportunities to develop simple, practical, inexpensive and valid prediction of wave-induced liquefaction. In this study, Artificial Neural Network (ANN) regression modeling is used to predict the depth of liquefaction. Despite of using Back Propagation (BP) to train ANN, a modified Genetic Algorithm (called as Wide GA, WGA) is used as ANN training method to improve ANN prediction accuracy and to overcome BP weaknesses such as premature convergence and local optimum. WGA also aim to avoid conventional GA weaknesses such as low population diversity and narrow search coverage. Key WGA operations are Wide Tournament Selection, Multi-Parent BLX-α Crossover, Aggregate Mate Pool Mutation and Direct Fresh Mutation-Crossover. ANN prediction accuracy measured by Median APE (MdAPE). Global optimum solution of WGA is best ANN connections weights configuration with smallest MdAPE.</span></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Keywords: </span><span class="font0" style="font-style:italic;">Wave-induced Liquefaction, Prediction of Soil Liquefaction, Wide Genetic Algorithm, Artificial Neural Network, Back Propagation.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark2"></a><span class="font0" style="font-weight:bold;"><a name="bookmark3"></a>1. &nbsp;&nbsp;&nbsp;Introduction</span></h3></li></ul>
<p><span class="font0">Since 1970, the research works related with liquefaction prediction have been started. If Soil Liquefaction occurs, soil may undergo settlement then the structures lay above saturated soil</span></p>
<p><span class="font0">ground may experience foundation failure. Numerical model deals with liquefaction is quite complicated, requires validation with laboratory testing and need expensive field inspections. A new model has been proposed to predict Liquefaction with Artificial Neural Network (ANN) and Simple Genetic Algorithm (SGA) [1] [2].</span></p>
<p><span class="font0">ANN is selected among many other Artificial Inteligence (AI) methods, because ANN able to model the interaction of parallel computing on the brain through the learning process of the data [3]. Back Propagation (BP) is one of widely used training method for ANN but it has some weaknesses which is BP can be trapped into local optimum (local minima) problem and premature convergence [4]. Most BP training process using very small learning rate constant which define the step of solution search. BP global optimum can only be achieved when the starting point is near to global minima which is rare (See Fig.1). Genetic Algorithm (GA) is an optimization method that mimics evolution process in nature [5]. GA has been used in numerous field of science and also has been used in ANN training process [1]. GA used in this study has been modified to improve ANN prediction accuracy.</span></p>
<div><img src="https://jurnal.harianregional.com/media/25789-1.jpg" alt="" style="width:167pt;height:156pt;">
<p><span class="font0" style="font-weight:bold;">Figure 1. </span><span class="font0">Back Propagation may trapped into local optimum</span></p>
</div><br clear="all">
<p><span class="font0">This paper contains five parts with introduction and literature reviews is in part one and part two, respectively. Part three discuss about the proposed new methodology. Part four is focused on discussion about empirical study related with selected topic and finally, part five is the conclusion of the findings.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0" style="font-weight:bold;">2. &nbsp;&nbsp;&nbsp;Literature Review</span></p>
<ul style="list-style:none;">
<li>
<p><span class="font0" style="font-weight:bold;">2.1. &nbsp;&nbsp;&nbsp;Wave-induced Liquefaction</span></p></li></ul></li></ul>
<p><span class="font0">Phenomena of wave-induced liquefaction is marked by the increasing incidence of pressure load on the soil ground that exceeds the capacity of the soil particles pressure due to cyclic loading waves in coastal waters. When liquefaction occurred, the soil lost its strength. The soil strength really depend on the capability of soil to withstand cyclic loading during earthquakes or wave actions. Liquefaction condition can be formulated as Safety Factor </span><span class="font10" style="font-style:italic;">(</span><span class="font11" style="font-style:italic;">F<sub>s</sub></span><span class="font10" style="font-style:italic;">)</span><span class="font0"> below [6] :</span></p>
<h2><a name="bookmark4"></a><span class="font9" style="font-style:italic;"><a name="bookmark5"></a>C &nbsp;&nbsp;</span><span class="font12" style="font-style:italic;font-variant:small-caps;"><sup>crr</sup></span></h2>
<div>
<p><span class="font0">(1)</span></p>
</div><br clear="all">
<p><span class="font11" style="font-style:italic;">F</span><span class="font0"> =--</span></p>
<p><span class="font10" style="font-style:italic;"><sup>s</sup> &nbsp;&nbsp;</span><span class="font10" style="font-style:italic;font-variant:small-caps;">csr</span></p>
<p><span class="font0">CRR (Cyclic Resistance Ratio) or liquefaction resistance is the soil capacity due to cyclic load. CSR (Cyclic Stress Ratio) is the loads on the ground, that could lead to liquefaction. CRR and CSR formulated as follow [7] :</span></p>
<div>
<p><span class="font11" style="font-style:italic;">CRR = <sup>τ</sup>-^ =</span><span class="font11"> 0.0019 </span><span class="font11" style="font-style:italic;">D<sub>r </sub></span><span class="font10" style="font-style:italic;"><sup>σ</sup></span><span class="font9" style="font-style:italic;">v</span></p>
</div><br clear="all">
<div>
<p><span class="font8">1+2 K<sub>0</sub></span></p>
<p><span class="font8">3</span></p>
</div><br clear="all">
<div>
<p><span class="font11" style="font-style:italic;">CSR</span><span class="font11"> =</span></p>
</div><br clear="all">
<div>
<p><span class="font10" style="font-style:italic;text-decoration:underline;">τ<sub>v</sub></span><span class="font9" style="font-style:italic;text-decoration:underline;">h</span></p>
</div><br clear="all">
<div>
<p><span class="font10" style="font-style:italic;text-decoration:underline;">2<sup>π</sup></span><span class="font8"> ^</span><span class="font7">0 &nbsp;</span><span class="font8">— 2π</span><span class="font7">y</span></p>
<p><span class="font11" style="font-style:italic;">p'<sub>β</sub> </span><span class="font10" style="font-style:italic;">L <sup>e &nbsp;&nbsp;&nbsp;L</sup></span></p>
</div><br clear="all">
<div>
<ul style="list-style:none;"><li>
<p><span class="font11">(</span><span class="font0">2</span><span class="font11">)</span></p></li>
<li>
<p><span class="font11">(</span><span class="font0">3</span><span class="font11">)</span></p></li></ul>
</div><br clear="all">
<p><span class="font0">Liquefaction will be happened when </span><span class="font11" style="font-style:italic;">F<sub>s</sub> &lt;=</span><span class="font11"> 1</span><span class="font0">, then </span><span class="font0" style="font-style:italic;">z</span><span class="font0"> can be computed as :</span></p>
<div>
<p><span class="font11" style="font-style:italic;">z =</span></p>
</div><br clear="all">
<div>
<p><span class="font11">— ln(∙</span></p>
</div><br clear="all">
<div>
<p><span class="font8">0.0019 </span><span class="font10" style="font-style:italic;">D<sub>r</sub> </span><span class="font10" style="text-decoration:line-through;"><sup>1+2 </sup></span><span class="font10" style="font-style:italic;text-decoration:line-through;"><sup>Ko </sup></span><span class="font10" style="font-style:italic;text-decoration:underline;"><sup>r</sup></span><span class="font7" style="text-decoration:underline;"> 3</span></p>
</div><br clear="all">
<div>
<p><span class="font9" style="font-style:italic;">2π Po p's </span><span class="font9" style="font-style:italic;font-variant:small-caps;"><sup>l</sup></span></p>
</div><br clear="all">
<div>
<p><span class="font0"><sup>1</sup>) —</span></p>
<p><span class="font10" style="font-style:italic;">2π</span></p>
</div><br clear="all">
<div>
<p><span class="font11">(</span><span class="font0">4</span><span class="font11">)</span></p>
</div><br clear="all">
<div>
<h3><a name="bookmark6"></a><span class="font0" style="font-weight:bold;"><a name="bookmark7"></a>LONTAR KOMPUTER VOL. 8, NO.1, APRIL 2017</span></h3>
</div><br clear="all">
<div>
<h3><a name="bookmark8"></a><span class="font0" style="font-weight:bold;"><a name="bookmark9"></a>DOI : 10.24843/LKJITI.2017.v08.i01.p01</span></h3>
</div><br clear="all">
<div>
<h3><a name="bookmark10"></a><span class="font0" style="font-weight:bold;"><a name="bookmark11"></a>p-ISSN 2088-1541</span></h3>
</div><br clear="all">
<div>
<h3><a name="bookmark12"></a><span class="font0" style="font-weight:bold;"><a name="bookmark13"></a>e-ISSN 2541-5832</span></h3>
</div><br clear="all">
<div>
<p><span class="font11">r&gt; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font8">H 1</span></p>
<p><span class="font10" style="font-style:italic;"><sup>p</sup>0 </span><span class="font11" style="font-style:italic;">= </span><span class="font10" style="font-style:italic;"><sup>p</sup>w<sup>9</sup> 2C0^</span></p>
<p><span class="font11" style="font-style:italic;">L = </span><span class="font10" style="font-style:italic;"><sup>g</sup>^</span><span class="font9" style="font-style:italic;">-</span><span class="font11" style="font-style:italic;">tanh (—)</span></p>
<p><span class="font10" style="font-style:italic;">2π &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;L L J</span></p>
</div><br clear="all">
<div>
<p><span class="font11">(</span><span class="font0">5</span><span class="font11">)</span></p>
<p><span class="font11">(</span><span class="font0">6</span><span class="font11">)</span></p>
</div><br clear="all">
<div>
<p><span class="font0">Where </span><span class="font11" style="font-style:italic;">τ<sub>vhl</sub> =</span><span class="font0"> cyclic shear stress or cyclic mobility, </span><span class="font11" style="font-style:italic;">D<sub>r</sub> =</span><span class="font0"> relative density constant, </span><span class="font11" style="font-style:italic;">K<sub>0</sub></span><span class="font11"> = </span><span class="font0">earth pressure at rest constants, </span><span class="font11" style="font-style:italic;">τ<sub>υh</sub></span><span class="font11"> = </span><span class="font0">soil shear stress, </span><span class="font11" style="font-style:italic;">σ<sub>r</sub></span><span class="font11"> = </span><span class="font0">vertical effective overburden pressure, </span><span class="font11" style="font-style:italic;">z=</span><span class="font0"> soil liquefaction depth, </span><span class="font11" style="font-style:italic;">g=</span><span class="font0"> gravity acceleration, </span><span class="font11" style="font-style:italic;">P<sub>0</sub>=</span><span class="font0"> pressure at sea bottom, </span><span class="font11" style="font-style:italic;">p'= </span><span class="font0">submerged unit mass of soil, </span><span class="font11" style="font-style:italic;">p<sub>w</sub> =</span><span class="font0"> water unit mass, </span><span class="font11" style="font-style:italic;">L</span><span class="font11"> = </span><span class="font0">wave length, </span><span class="font11" style="font-style:italic;">H</span><span class="font11"> = </span><span class="font0">wave height, </span><span class="font11" style="font-style:italic;">d</span><span class="font11"> = </span><span class="font0">water depth and </span><span class="font11" style="font-style:italic;">T =</span><span class="font0"> wave periode. Sumer suggest Relative Density </span><span class="font10" style="font-style:italic;">(</span><span class="font11" style="font-style:italic;">D<sub>r</sub></span><span class="font0">) constants by soil category at Table 1 [8]. Jaky suggest Earth Pressure at Rest (</span><span class="font11">K<sub>0</sub></span><span class="font0">) constants by soil type at Table 2 [9].</span></p>
</div><br clear="all">
<div>
<table border="1">
<tr><td colspan="2" style="vertical-align:bottom;">
<p><span class="font0" style="font-weight:bold;">Table 1. </span><span class="font0">Relative Density (</span><span class="font11">D<sub>r</sub></span><span class="font0">) constants</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0" style="font-weight:bold;">Soil Category</span></p></td><td style="vertical-align:bottom;">
<p><span class="font11" style="font-style:italic;">D<sub>r</sub></span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">Very Loose</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.00 – 0.15</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Loose</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.15 – 0.35</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Medium</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.35 – 0.65</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Dense</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.65 – 0.85</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">Very Dense</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.85 – 1.00</span></p></td></tr>
</table>
</div><br clear="all">
<div>
<p><span class="font0" style="font-weight:bold;">T a</span><span class="font0" style="font-weight:bold;text-decoration:underline;">ble 2. </span><span class="font0" style="text-decoration:underline;">Earth Pressure at Rest (</span><span class="font11" style="text-decoration:underline;">K<sub>0</sub></span><span class="font0" style="text-decoration:underline;">) consta</span><span class="font0">nts</span></p>
<table border="1">
<tr><td style="vertical-align:bottom;">
<p><span class="font0" style="font-weight:bold;">Soil Type</span></p></td><td style="vertical-align:bottom;">
<p><span class="font11" style="font-style:italic;">K<sub>0</sub></span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">Sandy Clay</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.25 – 0.42</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Silt</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.42 – 0.54</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Sand Dense</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.25 – 0.67</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Sand Coarse</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.18</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">Sand Fine-grained</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.33</span></p></td></tr>
</table>
</div><br clear="all">
<ul style="list-style:none;"><li>
<h3><a name="bookmark14"></a><span class="font0" style="font-weight:bold;"><a name="bookmark15"></a>2.2. &nbsp;&nbsp;&nbsp;Artificial Neural Network</span></h3></li></ul>
<p><span class="font0">ANN defined as a nonlinear complex learning system occur in a network of neurons. Although Back Propagation (BP) is one of most widely used learning method for ANN, it can be trapped to a local optimum [10]. For each predicted results </span><span class="font11" style="font-style:italic;">y</span><span class="font0"> to </span><span class="font11" style="font-style:italic;">y</span><span class="font10" style="font-style:italic;">,</span><span class="font0"> there migth be a difference noted as </span><span class="font11" style="font-style:italic;">y - y </span><span class="font0">which is referred to as residual values, error estimation or prediction errors. Median Absolute Percentage Error (MdAPE) is used to determine the performance of the ANN prediction, using the formula as follows :</span></p>
<p><span class="font11">MdAPE </span><span class="font11" style="font-style:italic;">= median<sub>l</sub></span><span class="font10" style="font-style:italic;">=</span><span class="font11" style="font-style:italic;"><sub>1n</sub></span><span class="font11"> (|^^| </span><span class="font11" style="font-style:italic;">∙ 1θθ)</span></p>
<div>
<p><span class="font0">(7)</span></p>
</div><br clear="all">
<p><span class="font0">If the dataset has an output </span><span class="font11" style="font-style:italic;">y</span><span class="font0"> which its value near to zero, then MdAPE value become very large that will reduce the ANN performance. ANN performance may also decreased when each attribute in the dataset has value that is very much different in amplitude. MinMax normalization method is used to avoid these conditions and formulated as follows [11]:</span></p>
<div>
<p><span class="font11" style="font-style:italic;">S = </span><span class="font11" style="font-style:italic;font-variant:small-caps;">S<sub>low</sub></span><span class="font11" style="font-style:italic;"> + (S<sub>hlgh</sub></span></p>
</div><br clear="all">
<div>
<p><span class="font10" style="font-style:italic;"><sub>-</sub></span></p>
</div><br clear="all">
<div>
<p><span class="font10" style="font-style:italic;">c</span><span class="font7"> A &nbsp;&nbsp;&nbsp;</span><span class="font10" style="font-style:italic;text-decoration:underline;">^ ^</span><span class="font9" style="font-style:italic;text-decoration:underline;">min</span></p>
<p><span class="font10" style="font-style:italic;font-variant:small-caps;"><sup>s</sup>Iow</span><span class="font11" style="font-style:italic;font-variant:small-caps;">J</span><span class="font9" style="font-style:italic;"> × <sub>a</sub> _<sub>A</sub> . </span><span class="font10" style="font-style:italic;">^</span><span class="font9" style="font-style:italic;">max </span><span class="font10" style="font-style:italic;">^</span><span class="font9" style="font-style:italic;">min</span></p>
</div><br clear="all">
<div>
<p><span class="font10" style="font-style:italic;">A = A.. <sup>n n</sup> mm</span></p>
</div><br clear="all">
<div>
<p><span class="font10" style="font-style:italic;"><sup>+</sup> </span><span class="font11" style="font-style:italic;">(A</span><span class="font10" style="font-style:italic;">max <sup>-</sup> </span><span class="font11" style="font-style:italic;">A</span><span class="font10" style="font-style:italic;">min</span><span class="font11" style="font-style:italic;">)</span><span class="font0"> × &nbsp;---</span><span class="font10" style="font-style:italic;">χ7~</span></p>
<p><span class="font10" style="font-style:italic;"><sup>i</sup></span><span class="font9" style="font-style:italic;">high <sup>ii</sup>Iow</span></p>
</div><br clear="all">
<div>
<p><span class="font0">(8)</span></p>
<p><span class="font0">(9)</span></p>
</div><br clear="all">
<p><span class="font0">Where </span><span class="font11" style="font-style:italic;">A =</span><span class="font0"> original data, </span><span class="font11" style="font-style:italic;">A<sub>min</sub></span><span class="font11"> =</span><span class="font0">minimum value of original data, </span><span class="font11" style="font-style:italic;">A<sub>max</sub></span><span class="font11"> =</span><span class="font0">maximum value of original data, </span><span class="font11">S </span><span class="font11" style="font-style:italic;">=</span><span class="font0"> normalized data, </span><span class="font11" style="font-style:italic;">S<sub>high</sub> =</span><span class="font0"> highest value of normalized data, </span><span class="font11" style="font-style:italic;font-variant:small-caps;">S<sub>low</sub></span><span class="font11" style="font-style:italic;"> =</span><span class="font0"> lowest value of normalized data. De-normalization technique is used to restore the predicted values into the initial scale (Eq.9). In this study, value of </span><span class="font11" style="font-style:italic;font-variant:small-caps;">S<sub>low</sub></span><span class="font11" style="font-style:italic;"> = 0.4</span><span class="font0"> and </span><span class="font11" style="font-style:italic;">S<sub>high</sub> = 0.9</span><span class="font0"> will be used to avoid large MdAPE values.</span></p>
<p><span class="font0">Rectified Linear Units (ReLU) is used to overcome constant output of activation functions, formulated as follows [12]:</span></p>
<p><span class="font11" style="font-style:italic;">(x, x ≥ 0</span></p>
<div>
<p><span class="font0">(10)</span></p>
</div><br clear="all">
<p><span class="font10" style="font-style:italic;"><sup>y</sup></span><span class="font11"> t0, x &lt;&nbsp;0</span></p>
<p><span class="font11" style="font-style:italic;">x</span><span class="font0"> is input and </span><span class="font11" style="font-style:italic;">y</span><span class="font0"> is output of ReLU activation function. An algorithm is proposed by Dhiel to normalize all of the ANN connections weight to improve the performance of activation function being used above [13].</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark16"></a><span class="font0" style="font-weight:bold;"><a name="bookmark17"></a>2.3. &nbsp;&nbsp;&nbsp;Genetic Algorithm (GA)</span></h3></li></ul>
<p><span class="font0">GA begins by generating the initial population contains a number of genes [5]. Each gene represents a solution that has been encoded. GA is mainly composed of a selection, crossover and mutation operation. The GA evolution stop when it reaches certain number of iteration</span></p>
<p><span class="font0">(epoch) or achieving small error (convergence). GA has some constants to be determined in advance, i.e.: </span><span class="font11" style="font-style:italic;">P<sub>s</sub></span><span class="font0"> (population size), </span><span class="font11" style="font-style:italic;">P<sub>cr</sub></span><span class="font0"> (crossover probability [0,1]), and </span><span class="font11" style="font-style:italic;">P<sub>m</sub></span><span class="font0"> (mutation probability [0,1]). Darrell Whitley proposed Steady State GA [14], it has a small and fixed (</span><span class="font0" style="font-style:italic;">steady</span><span class="font0">) population number also it introduce replacement operation.</span></p>
<p><span class="font0">Tournament selection operation starts by selecting pair-wise genes randomly, compare the fitness values, choosing one best gene and put it into mate pool. It is repeated iV-times depend on crossover requirement. If </span><span class="font11">V = 2</span><span class="font0">, it’s called as a binary tournament. Some references recommended that the value of </span><span class="font11">V &gt;&nbsp;2 </span><span class="font0">to reduce the pressure of finding just the best genes alone [15].</span></p>
<p><span class="font0">Blend Alpha Crossover (BLX-α) is crossover operation between 2 parent. It starts by randomly select two parent </span><span class="font11">x<sup>1</sup> </span><span class="font0">and </span><span class="font11">x<sup>2</sup> </span><span class="font0">from the population, then </span><span class="font0" style="font-style:italic;">i<sup>th</sup></span><span class="font0"> offspring allele value is randomly chosen from uniformly distributed interval of [</span><span class="font11">X</span><span class="font8">1</span><span class="font11">, X<sub>i</sub><sup>2</sup></span><span class="font0">] where :</span></p>
<ul style="list-style:none;"><li>
<p><a href="#bookmark18"><span class="font11">X</span><span class="font8">1 </span><span class="font11">= min(χ<sub>1</sub><sup>1</sup>,χ<sub>1</sub><sup>2</sup>) </span><span class="font11" style="font-style:italic;">— a * d<sub>i</sub></span><span class="font0">(11)</span></a></p></li></ul>
<ul style="list-style:none;"><li>
<p><a href="#bookmark19"><span class="font11">X<sup>2</sup> = max(χ<sub>1</sub><sup>1</sup>,χ<sub>1</sub><sup>2</sup>) </span><span class="font11" style="font-style:italic;">— a * d<sub>i</sub></span><span class="font0">(12)</span></a></p></li></ul>
<p><a href="#bookmark20"><span class="font11" style="font-style:italic;">d<sub>i</sub>=</span><span class="font11"> </span><span class="font2">∖</span><span class="font11">χ<sub>1</sub><sup>1</sup>- χ<sub>1</sub><sup>2</sup></span><span class="font2">∣</span><span class="font11"></span></a></p>
<p><span class="font0" style="font-style:italic;">χ</span><span class="font10" style="font-style:italic;">1</span><span class="font0"> and </span><span class="font11">χ<sup>2</sup> </span><span class="font0">are </span><span class="font0" style="font-style:italic;">i<sup>th</sup></span><span class="font0"> allele value of parent </span><span class="font11">χ<sup>1</sup> </span><span class="font0">and </span><span class="font11">χ<sup>2</sup></span><span class="font0">, α is a positive parameter [0,1]. Picek suggested </span><span class="font11" style="font-style:italic;">a</span><span class="font11"> = 0.5 </span><span class="font0">[16]. If α = 0, crossover operation become more exploitative. If bigger α is used (</span><span class="font11">0 &lt;&nbsp;a &lt;&nbsp;1</span><span class="font0">), then crossover operation become more explorative (See Fig.2). Several previous studies indicate that crossover using multiple parent were able to improve the performance of GA [17] [18].</span></p>
<table border="1">
<tr><td style="vertical-align:top;">
<p><span class="font5">Exploration</span></p></td><td style="vertical-align:top;">
<p><span class="font5">Exploitation</span></p></td><td style="vertical-align:top;">
<p><span class="font5">Exploration</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font6">X</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">i<sup>1 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</sup></span><span class="font5"><sup>X</sup></span></p></td><td style="vertical-align:middle;">
<p><span class="font4">2</span></p></td></tr>
</table>
<p><span class="font5">α * d</span><span class="font3">i &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font5">d</span><span class="font3">i </span><span class="font5">= |X</span><span class="font3">i<sup>2</sup></span><span class="font5">-X</span><span class="font3">i<sup>1</sup></span><span class="font5">| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;α * d</span><span class="font3">i</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0" style="font-weight:bold;">Figure 2. </span><span class="font0">Exploitation and exploration of BLX-α crossover</span></p></li></ul>
<p><span class="font0">Yoon and Kim proposed Fine Mutation (FM), a mutation operation methods inspired by Gaussian mutation method, formulated as follows [19]:</span></p>
<p><span class="font11" style="font-style:italic;">z<sub>i</sub> = z<sub>i</sub>+Vd(0,</span><span class="font2" style="font-style:italic;">∖</span><span class="font11" style="font-style:italic;">χ<sub>i</sub>-y<sub>i</sub></span><span class="font2" style="font-style:italic;">∖</span><span class="font11" style="font-style:italic;">)</span><span class="font0"> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(14)</span></p>
<p><span class="font11" style="font-style:italic;">χ<sub>l</sub></span><span class="font0"> is </span><span class="font11" style="font-style:italic;">i<sup>t</sup></span><span class="font0" style="font-style:italic;"><sup>h</sup></span><span class="font0"> allele of parent </span><span class="font11">χ</span><span class="font0">, </span><span class="font11" style="font-style:italic;">y<sub>l</sub></span><span class="font0"> is </span><span class="font11" style="font-style:italic;">i<sup>t</sup></span><span class="font0" style="font-style:italic;"><sup>h</sup></span><span class="font0"> allele of parent </span><span class="font11" style="font-style:italic;">y</span><span class="font10" style="font-style:italic;">, </span><span class="font11" style="font-style:italic;">z<sub>l</sub></span><span class="font0"> is </span><span class="font0" style="font-style:italic;">i<sup>th</sup></span><span class="font0"> allele of offspring </span><span class="font11" style="font-style:italic;">z</span><span class="font0"> and </span><span class="font11" style="font-style:italic;">Vd</span><span class="font0"> is normal distribution with a standard deviation of </span><span class="font2">∖</span><span class="font11">χ<sub>1</sub> </span><span class="font11" style="font-style:italic;">-y<sub>i</sub></span><span class="font2" style="font-style:italic;">∖</span><span class="font10" style="font-style:italic;">.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark21"></a><span class="font0" style="font-weight:bold;"><a name="bookmark22"></a>2.4. &nbsp;&nbsp;&nbsp;Hybrid ANN-GA</span></h3></li></ul>
<p><span class="font0">There are enormous studies already conducted on the use of optimization methods in ANN training process replacing BP [20] [21] [22]. The weight of each ANN connections are encoded into a gene. Each connection weights are real numbers, so ANN-GA genes are also real numbers. ANN-GA hybridization objective is doing a global optimization of the ANN connection weights. GA have broader solution search coverage compared to BP, so it can avoid local optimum and premature convergence. GA may improve ANN performance which is maximizing accuracy and minimizing prediction error.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark23"></a><span class="font0" style="font-weight:bold;"><a name="bookmark24"></a>3. &nbsp;&nbsp;&nbsp;Proposed Methodology</span></h3></li></ul>
<p><span class="font0">The methodology proposed in this study will be called as Wide Genetic Algorithm (WGA) which is a modified version of GA that will be used to optimize the weight of each ANN connections. The WGA evolutionary iteration will stop when the ANN configuration with the lowest prediction error has been found. Fig.3 show the flowchart of ANN training process using WGA.</span></p><img src="https://jurnal.harianregional.com/media/25789-2.jpg" alt="" style="width:491pt;height:295pt;">
<p><span class="font0" style="font-weight:bold;">Figure 3. </span><span class="font0">ANN training using WGA</span></p>
<p><span class="font0" style="font-weight:bold;">3.1. Wide Tournament Selection</span></p>
<p><span class="font0">In general, the selection operation in conventional GA provides considerable big pressure to pick only the best genes, for example elitist selection method [15]. Too much selection pressure may decrease population diversity, narrowing the search coverage and can be trapped into local optimum and premature convergence. In the case of Tournament Selection, random selection of 2 genes can also increase the pressure selection. Based on the above conditions, this study proposed Wide Tournament Selection (WTS) which automatically include the worst gene from the population. WTS starts with ascending sorting based on gene fitness function then continue with choosing 4 genes randomly and adding 1 worst gene from the population (See Fig.4). Ascending sorting is needed to locate the worst gen.</span></p><img src="https://jurnal.harianregional.com/media/25789-3.jpg" alt="" style="width:436pt;height:156pt;">
<p><span class="font0" style="font-weight:bold;">Figure 4. </span><span class="font0">Wide Tournament selection flowchart</span></p>
<p><span class="font0" style="font-weight:bold;">3.2. Multi Parent BLX-α Crossover</span></p>
<p><span class="font0">Additional parent in the crossover operation can reduce the pressure in the selection process, increasing the diversity of the population and avoid incest [17] [18]. Multi Parent BLX-α crossover utilize worst gene which is selected by Wide Tournament selection operation above. The addition</span></p>
<p><span class="font0">of the worst gene aim to enhance exploration efforts to search in wider coverage. Eq.15-17 is modification of Eq.11-13 to include the worst gene.</span></p>
<p><a href="#bookmark25"><span class="font11" style="font-style:italic;">X</span><span class="font10" style="font-style:italic;"><sup>1</sup></span><span class="font11"> = min(⅛x<sup>2</sup>,χ</span><span class="font8">W</span><span class="font11">) - α<sub>c</sub> * </span><span class="font11" style="font-style:italic;">d<sub>l</sub></span><span class="font0">(15)</span></a></p>
<p><a href="#bookmark26"><span class="font11" style="font-style:italic;">Xi</span><span class="font11"> = </span><span class="font11" style="font-style:italic;">max(χ</span><span class="font10" style="font-style:italic;">1</span><span class="font11" style="font-style:italic;">, χ</span><span class="font10" style="font-style:italic;">2</span><span class="font11" style="font-style:italic;">, χ</span><span class="font10" style="font-style:italic;">W</span><span class="font11" style="font-style:italic;">)</span><span class="font11"> - α<sub>c</sub> * </span><span class="font11" style="font-style:italic;">d<sub>l</sub></span><span class="font0">(16)</span></a></p>
<p><a href="#bookmark27"><span class="font11" style="font-style:italic;">d<sub>i</sub> =</span><span class="font11"> </span><span class="font2">∣</span><span class="font11">max(⅛χ</span><span class="font8">2</span><span class="font11">,χ</span><span class="font8">W</span><span class="font11">) - min(χ</span><span class="font8">1</span><span class="font11">,χ</span><span class="font8">2</span><span class="font11">,χ</span><span class="font8">W</span><span class="font11">)</span><span class="font2">∣</span><span class="font11"></span></a></p>
<p><span class="font11">χ<sub>1</sub><sup>w</sup> </span><span class="font0">is </span><span class="font0" style="font-style:italic;">i<sup>th</sup></span><span class="font0"> allele value of worst gene </span><span class="font11">χ<sup>w</sup></span><span class="font0">. </span><span class="font11">α<sub>c</sub> </span><span class="font0">is constant [0,1], </span><span class="font11">α<sub>c</sub> = 0.2 </span><span class="font0">is used.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark28"></a><span class="font0" style="font-weight:bold;"><a name="bookmark29"></a>3.3. &nbsp;&nbsp;&nbsp;Aggregate Mate Pool Mutation</span></h3></li></ul>
<p><span class="font0">Conventional GA mutation use only an absolute constant value for all GA evolutions which may reduce its adaptability to iteration convergence. Aggregate Mate Pool Mutation (AMPM) aims to leverage the parents value range for adaptive mutation values. Inspired by Fine Mutation (FM) [19], AMPM using not just 2 reference genes, but 5 reference genes in mutation pool which is selected by Wide Tournament Selection operation (Fig.4). These factor may lead to better search exploration and avoid local optimum easily.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark30"></a><span class="font0" style="font-weight:bold;"><a name="bookmark31"></a>3.4. &nbsp;&nbsp;&nbsp;Direct Fresh Mutation-Crossover</span></h3></li></ul>
<p><span class="font0">In the conventional GA, crossover and mutation operations carried out with certain probabilities. These probabilities may lead to ineffectiveness of each evolution in GA, which may only execute recombination or mutation operation. It is very possible where a mutation operation can be followed by a process of recombination only. To overcome this, a new technique is proposed that executes the recombination-mutation and mutation-recombination operations in parallel. The purpose of this technique to improve the effectiveness of each iteration of WGA evolution.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark32"></a><span class="font0" style="font-weight:bold;"><a name="bookmark33"></a>4. &nbsp;&nbsp;&nbsp;Empirical Study</span></h3></li></ul>
<p><span class="font0">Empirical study has been done by conducted benchmark experiments to ANN-BP, ANN-GA and ANN-WGA. Each ANN training method tested with 5 independent experiments and 1000 training iterations for each independent experiments. ANN-BP tested with learning rate=0.01 and learning momentum=0.5 and K-folds=7. ANN-GA tested with population size=20, crossover probability=0.5 and mutation probability=0.2. ANN-WGA tested with population size=20. The data used in this study are gathered from previously published research [1] [2][1]. There are 7 syntetic generated dataset, its ranging from 1125 to 2520 rows of data. Each dataset divided into 2 big parts, first part is 75% of dataset used as training data and the rest 25% of dataset used as testing data. Some sample of dataset can be seen at Table 3.</span></p>
<p><span class="font0" style="font-weight:bold;">Table 3. </span><span class="font0">Dataset Sample</span></p>
<table border="1">
<tr><td colspan="5" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">Input</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">Output</span></p></td></tr>
<tr><td colspan="3" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">CSR</span></p></td><td colspan="2" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">CRR</span></p></td><td rowspan="2" style="vertical-align:top;">
<p><span class="font0" style="font-weight:bold;">z</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">T</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">d</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">H</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">K0</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">Dr</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">15</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">30</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">25</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.65</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">212.94</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">14</span></p></td><td style="vertical-align:top;">
<p><span class="font0">10</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.2</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.65</span></p></td><td style="vertical-align:top;">
<p><span class="font0">103.33</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">16</span></p></td><td style="vertical-align:top;">
<p><span class="font0">30</span></p></td><td style="vertical-align:top;">
<p><span class="font0">20</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.18</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.65</span></p></td><td style="vertical-align:top;">
<p><span class="font0">220.53</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">14</span></p></td><td style="vertical-align:top;">
<p><span class="font0">20</span></p></td><td style="vertical-align:top;">
<p><span class="font0">10</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.16</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.75</span></p></td><td style="vertical-align:top;">
<p><span class="font0">146.35</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">13</span></p></td><td style="vertical-align:top;">
<p><span class="font0">10</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.2</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.75</span></p></td><td style="vertical-align:top;">
<p><span class="font0">93.82</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">12</span></p></td><td style="vertical-align:top;">
<p><span class="font0">50</span></p></td><td style="vertical-align:top;">
<p><span class="font0">15</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.16</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.75</span></p></td><td style="vertical-align:top;">
<p><span class="font0">151.86</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">16</span></p></td><td style="vertical-align:top;">
<p><span class="font0">10</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.18</span></p></td><td style="vertical-align:top;">
<p><span class="font0">0.65</span></p></td><td style="vertical-align:top;">
<p><span class="font0">116.92</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">13</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">50</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">20</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.85</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">176.42</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">15</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">30</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">20</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.16</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.85</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">196.81</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">13</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">70</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">20</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.16</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.85</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">174.77</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">16</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">30</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">25</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.16</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">0.75</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">224.96</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font0">16</span></p></td><td style="vertical-align:middle;">
<p><span class="font0">70</span></p></td><td style="vertical-align:middle;">
<p><span class="font0">10</span></p></td><td style="vertical-align:middle;">
<p><span class="font0">0.2</span></p></td><td style="vertical-align:middle;">
<p><span class="font0">0.85</span></p></td><td style="vertical-align:middle;">
<p><span class="font0">205.81</span></p></td></tr>
</table>
<p><span class="font0">The ANN architecture that will be used in this study is dual hidden layers with single output node (later coded as 5:9:7:1), with has 5 input node, 9 hidden node + 1 bias at first hidden layer, 7</span></p>
<p><span class="font0">hidden node + 1 bias at second hidden layer and 1 output node. The ANN architecture can be found at Fig.5 below.</span></p><img src="https://jurnal.harianregional.com/media/25789-4.jpg" alt="" style="width:286pt;height:356pt;">
<p><span class="font0">Input &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hidden 1 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Hidden 2 Output</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0" style="font-weight:bold;">Figure 5. </span><span class="font0">ANN architecture</span></p></li></ul>
<p><span class="font0">Summary results of empirical study results can be seen at Table 4 and scattered diagrams between prediction and dataset can be seen at Fig.6. The experiments are conducted to show the performance of WGA in the training process of ANN.</span></p>
<p><span class="font0" style="font-weight:bold;">Table 4. </span><span class="font0">Summary Results of Empirical Study</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">Dataset</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">Total</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">Train</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">Test</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">ANN Arch</span></p></td><td colspan="3" style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">MdAPE</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">ANN-BP</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">ANN-GA</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">ANN-WGA</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font0">Liquefaction Dataset 1</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">1125</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">844</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">281</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">5:9:7:1</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">5.624</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">4.404</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">3.352</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Liquefaction Dataset 2</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1350</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1013</span></p></td><td style="vertical-align:top;">
<p><span class="font0">337</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5:9:7:1</span></p></td><td style="vertical-align:top;">
<p><span class="font0">4.880</span></p></td><td style="vertical-align:top;">
<p><span class="font0">4.880</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.487</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Liquefaction Dataset 3</span></p></td><td style="vertical-align:top;">
<p><span class="font0">2520</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1890</span></p></td><td style="vertical-align:top;">
<p><span class="font0">630</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5:9:7:1</span></p></td><td style="vertical-align:top;">
<p><span class="font0">4.377</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.085</span></p></td><td style="vertical-align:top;">
<p><span class="font0">2.289</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Liquefaction Dataset 4</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1200</span></p></td><td style="vertical-align:top;">
<p><span class="font0">900</span></p></td><td style="vertical-align:top;">
<p><span class="font0">300</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5:9:7:1</span></p></td><td style="vertical-align:top;">
<p><span class="font0">6.509</span></p></td><td style="vertical-align:top;">
<p><span class="font0">6.075</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.730</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Liquefaction Dataset 5</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1800</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1350</span></p></td><td style="vertical-align:top;">
<p><span class="font0">450</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5:9:7:1</span></p></td><td style="vertical-align:top;">
<p><span class="font0">4.161</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.625</span></p></td><td style="vertical-align:top;">
<p><span class="font0">2.733</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Liquefaction Dataset 6</span></p></td><td style="vertical-align:top;">
<p><span class="font0">2100</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1575</span></p></td><td style="vertical-align:top;">
<p><span class="font0">525</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5:9:7:1</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.945</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.100</span></p></td><td style="vertical-align:top;">
<p><span class="font0">2.080</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font0">Liquefaction Dataset 7</span></p></td><td style="vertical-align:top;">
<p><span class="font0">1152</span></p></td><td style="vertical-align:top;">
<p><span class="font0">864</span></p></td><td style="vertical-align:top;">
<p><span class="font0">288</span></p></td><td style="vertical-align:top;">
<p><span class="font0">5:9:7:1</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.327</span></p></td><td style="vertical-align:top;">
<p><span class="font0">3.527</span></p></td><td style="vertical-align:top;">
<p><span class="font0">2.707</span></p></td></tr>
<tr><td colspan="4" style="vertical-align:bottom;">
<p><span class="font0">Average MdAPE of All datasets</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:bottom;">
<p><span class="font0">4.689</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">4.100</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0">2.911</span></p></td></tr>
</table>
<p><span class="font0">Remark: Total, Train and Test are number of data rows.</span></p>
<p><span class="font0">From Table 4, the experiment results show that ANN-WGA can perform roughly 1.5 times better than other training methods. For all of 7 soil liquefaction dataset, ANN-WGA can produce MdAPE</span></p>
<p><span class="font0">below 4%. Liquefaction Dataset 6 and 3 are two largest dataset among others and the experiment results show that it has smallest ANN-WGA MdAPE. Liquefaction Dataset 4 and 1 are two dataset which has smallest number of data and the experiment results show that it has biggest ANN-WGA MdAPE. We can conclude that bigger number of data may reduce ANN-WGA MdAPE.</span></p>
<div><img src="https://jurnal.harianregional.com/media/25789-5.jpg" alt="" style="width:250pt;height:187pt;">
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/25789-6.jpg" alt="" style="width:249pt;height:187pt;">
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/25789-7.jpg" alt="" style="width:248pt;height:186pt;">
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/25789-8.jpg" alt="" style="width:248pt;height:186pt;">
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/25789-9.jpg" alt="" style="width:247pt;height:185pt;">
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/25789-10.jpg" alt="" style="width:246pt;height:184pt;">
</div><br clear="all"><img src="https://jurnal.harianregional.com/media/25789-11.jpg" alt="" style="width:260pt;height:195pt;">
<p><span class="font0" style="font-weight:bold;">Figure 6</span><span class="font0">. Scattered diagram of liquefaction prediction for dataset 1-7</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark34"></a><span class="font0" style="font-weight:bold;"><a name="bookmark35"></a>5. &nbsp;&nbsp;&nbsp;Conclusion</span></h3></li></ul>
<p><span class="font0">The empirical study shows that the proposed WGA which is modifications to conventional GA, i.e.: Wide Tournament Selection, Multi-Parent BLX-α Crossover, Aggregate Mate Pool Mutation and Direct Fresh Mutation-Crossover can improve ANN performance to achieve better ANN prediction accuracy. Compared BP and GA as ANN training method, WGA perform 1.5 times better and can produce MdAPE below 4% for all soil liquefaction dataset being used in the experiments. These performance achieved because WGA can explore broader search space, maintain population diversity, avoid premature convergence and escape from local optimum. WGA can find best global solution for ANN which is ANN connections weight configuration with lowest MdAPE prediction error.</span></p>
<p><span class="font0" style="font-weight:bold;">Acknowledgements</span></p>
<p><span class="font0">This research becomes possible through the financial support from Marine Reliability Availability Safety Laboratory, Marine Engineering Department, Faculty of Marine Technology, Institut Teknologi Sepuluh Nopember, Surabaya.</span></p>
<h3><a name="bookmark36"></a><span class="font0" style="font-weight:bold;"><a name="bookmark37"></a>References</span></h3>
<ul style="list-style:none;"><li>
<p><span class="font0">[1] &nbsp;&nbsp;&nbsp;D. H. Cha, H. Zhang, M. Blumenstein and D. S. Jeng, &quot;Accurate prediction of wave-</span></p></li></ul>
<p><span class="font0">induced seabed liquefaction at shallow daepths using multi-artificial neural networks,&quot; </span><span class="font0" style="font-style:italic;">Journal of Coastal Research,</span><span class="font0"> no. 56, 2009.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0">[2] &nbsp;&nbsp;&nbsp;C. C. Liao, Y. G. Z. Lin and D.-S. Jeng, &quot;Coupling model for waves propagating over a</span></p></li></ul>
<p><span class="font0">porous seabed,&quot; </span><span class="font0" style="font-style:italic;">Theoretical and Applied Mechanics Letters,</span><span class="font0"> vol. 5, no. 2, 2015.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0">[3] &nbsp;&nbsp;&nbsp;F. Gorunescu, Data Mining: Concepts, Models and Techniques, Berlin: Springer-Verlag,</span></p></li></ul>
<p><span class="font0">2010.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0">[4] &nbsp;&nbsp;&nbsp;J. Han and M. Kamber, Data Mining: Concepts and Techniques, Elsevier, 2006.</span></p></li>
<li>
<p><span class="font0">[5] &nbsp;&nbsp;&nbsp;J. H. Holland, &quot;Genetic algorithms,&quot; 2 &nbsp;&nbsp;10 &nbsp;&nbsp;2012. [Online]. Available:</span></p></li></ul>
<p><a href="http://www.scholarpedia.org/article/Genetic_algorithms"><span class="font0">http://www.scholarpedia.org/article/Genetic_algorithms</span></a><span class="font0">.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0">[6] &nbsp;&nbsp;&nbsp;S. C. Tuna and S. Altun, &quot;Modern Approaches in Soil Liquefaction Analysis,&quot; in </span><span class="font0" style="font-style:italic;">6th</span></p></li></ul>
<p><span class="font0" style="font-style:italic;">International Conference on Earthquake Geotechnical Engineering</span><span class="font0">, 2015.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0">[7] &nbsp;&nbsp;&nbsp;K. Ishihara and A. Yamazaki, &quot;Analysis of wave-induced liquefaction in seabed deposits</span></p></li></ul>
<p><span class="font0">of sand,&quot; </span><span class="font0" style="font-style:italic;">Japanese Society of Soil Mechanics and Foundation Engineering,</span><span class="font0"> vol. 24, no. 3, pp. 85-100, 1984.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0">[8] &nbsp;&nbsp;&nbsp;B. M. Sumer, Liquefaction Around Marine Structures, 2014.</span></p></li>
<li>
<p><span class="font0">[9] &nbsp;&nbsp;&nbsp;J. Jaky, &quot;The coefficient of earth pressure at rest,&quot; </span><span class="font0" style="font-style:italic;">Journal of the Society of Hungarian</span></p></li></ul>
<p><span class="font0" style="font-style:italic;">Architects and Engineers,</span><span class="font0"> vol. 78, no. 22, pp. 355-358, 1944.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font0">[10] &nbsp;&nbsp;&nbsp;M. V. Shcherbakov, A. Brebels, N. L. Shcherbakova, A. P. Tyukov, T. A. Janovsky and V. A. Kamaev, &quot;A survey of forecast error measures,&quot; </span><span class="font0" style="font-style:italic;">World Applied Sciences Journal,</span><span class="font0"> no. 24, pp. 171-176, 2013.</span></p></li>
<li>
<p><span class="font0">[11] &nbsp;&nbsp;&nbsp;E. Ogasawara, L. C. Martinez, D. D. Oliveira, G. Zimbrão, G. L. Pappa and M. Mattoso, &quot;Adaptive normalization: A novel data normalization approach for non-stationary time series,&quot; </span><span class="font0" style="font-style:italic;">International Joint Conference on Neural Networks,</span><span class="font0"> pp. 1-8, 2010.</span></p></li>
<li>
<p><span class="font0">[12] &nbsp;&nbsp;&nbsp;V. Nair and G. E. Hinton, &quot;Rectified Linear Units Improve Restricted Boltzmann Machines,&quot; </span><span class="font0" style="font-style:italic;">Proceedings of the 27th International Conference on Machine Learning (ICML-10),</span><span class="font0"> pp. 807-814, 2010.</span></p></li>
<li>
<p><span class="font0">[13] &nbsp;&nbsp;&nbsp;P. U. Diehl, D. Neil, J. Binas, M. Cook, S.-C. Liu and M. Pfeiffer, &quot;Fast-classifying, high-accuracy spiking deep networks through weight and threshold balancing,&quot; </span><span class="font0" style="font-style:italic;">International Joint Conference on Neural Networks,</span><span class="font0"> pp. 1-8, 2015.</span></p></li>
<li>
<p><span class="font0">[14] &nbsp;&nbsp;&nbsp;M. Lozano, F. Herrera and J. Ramón, &quot;Replacement strategies to preserve useful diversity in steady-state genetic algorithms,&quot; </span><span class="font0" style="font-style:italic;">Information Sciences,</span><span class="font0"> vol. 178, no. 23, 2008.</span></p></li>
<li>
<p><span class="font0">[15] &nbsp;&nbsp;&nbsp;D. Gupta and S. Ghafir, &quot;An overview of methods maintaining diversity in genetic algorithms,&quot; </span><span class="font0" style="font-style:italic;">International journal of emerging technology and advanced engineering,</span><span class="font0"> pp. 56-60, 2012.</span></p></li>
<li>
<p><span class="font0">[16] &nbsp;&nbsp;&nbsp;S. Picek, D. Jakobovic and M. Golub, &quot;On the recombination operator in the real-coded genetic algorithms,&quot; </span><span class="font0" style="font-style:italic;">Evolutionary Computation (CEC),</span><span class="font0"> 2013.</span></p></li>
<li>
<p><span class="font0">[17] &nbsp;&nbsp;&nbsp;S. Tsutsui and A. Ghosh, &quot;A study on the effect of multi-parent recombination in real coded genetic algorithms,&quot; </span><span class="font0" style="font-style:italic;">Evolutionary Computation Proceedings, 1998. IEEE World Congress on Computational Intelligence., The 1998 IEEE International Conference,</span><span class="font0"> 1998.</span></p></li>
<li>
<p><span class="font0">[18] &nbsp;&nbsp;&nbsp;A. Ariyarit and M. Kanazaki, &quot;Multi-modal distribution crossover method based on two crossing segments bounded by selected parents applied to multi-objective design optimization,&quot; </span><span class="font0" style="font-style:italic;">Journal of Mechanical Science and Technology,</span><span class="font0"> 2015.</span></p></li>
<li>
<p><span class="font0">[19] &nbsp;&nbsp;&nbsp;Y. Yoon and Y.-H. Kim, The roles of crossover and mutation in real-coded genetic algorithms, INTECH Open Access Publisher, 2012.</span></p></li>
<li>
<p><span class="font0">[20] &nbsp;&nbsp;&nbsp;Y. Zhang, X. Gao and S. Katayama, &quot;Weld appearance prediction with BP neural network improved by genetic algorithm during disk laser welding,&quot; </span><span class="font0" style="font-style:italic;">Journal of Manufacturing Systems,</span><span class="font0"> 2015.</span></p></li>
<li>
<p><span class="font0">[21] &nbsp;&nbsp;&nbsp;A. Jarndal, &quot;Combined genetic algorithm and neural network technique for transistor modeling,&quot; </span><span class="font0" style="font-style:italic;">ommunications, Signal Processing, and their Applications (ICCSPA), 2015 International Conference,</span><span class="font0"> 2015.</span></p></li>
<li>
<p><span class="font0">[22] &nbsp;&nbsp;&nbsp;M. B. Takahashi, R. J. Celso and E. G. F. Núñez, &quot;Optimization of artificial neural network by genetic algorithm for describing viral production from uniform design data,&quot; </span><span class="font0" style="font-style:italic;">Process Biochemistry,</span><span class="font0"> 2015.</span></p></li></ul>
<p><span class="font0">10</span></p>