---
layout: full_article
title: "PCR FINGERPRINTING OF DIVERSE GENOMES FROM BACTERIAL STRAINS USING UNIVERSAL RICE PRIMER (URP)"
author: "Hee Wan Kang"
categories: jbb
canonical_url: https://jurnal.harianregional.com/jbb/full-44266 
citation_abstract_html_url: "https://jurnal.harianregional.com/jbb/id-44266"
citation_pdf_url: "https://jurnal.harianregional.com/jbb/full-44266"  
comments: true
---

<p><span class="font0">INTERNATIONAL JOURNAL OF BIOSCIENCES AND BIOTECHNOLOGY • Vol. 6 No. 1 • September 2018</span></p>
<p><span class="font0">ISSN: 2303-3371</span></p>
<p><a href="https://doi.org/10.24843/IJBB.2018.v06.i01.p06"><span class="font0">https://doi.org/10.24843/IJBB.2018.v06.i01.p06</span></a></p>
<p><span class="font3" style="font-weight:bold;">PCR FINGERPRINTING OF DIVERSE GENOMES FROM BACTERIAL STRAINS USING UNIVERSAL RICE PRIMER (URP)</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font3" style="font-weight:bold;"><a name="bookmark1"></a>Hee Wan Kang<sup>1, 2, 3*</sup></span></h1>
<p><span class="font2"><sup>1</sup>Department of Horticultural life science, Hankyong National University, Ansung 17579, Korea <sup>2</sup>Institute of Genetic Engineering, Hankyong National University, Anseong 17579, Korea <sup>3</sup>JK BioTech.Co Ltd. Hankyong National University, Ansung 17579, Korea</span></p>
<p><span class="font2">*Corresponding author: </span><a href="mailto:kanghw2@hknu.ac.com"><span class="font2">kanghw2@hknu.ac.com</span></a></p>
<h1><a name="bookmark2"></a><span class="font3" style="font-weight:bold;"><a name="bookmark3"></a>ABSTRACT</span></h1>
<p><span class="font3">Twenty primers of 20 mer referred to universal rice primer (URP) were developed from a repetitive sequence of rice genome. URP-PCR protocol employed stringent PCR with high annealing temperature throughout the thermo-cycling reaction, giving high reproducibility. Under the PCR condition, each single URP primer produced characteristic fingerprints from diverse genomes of bacterial species. The universal application of URP-PCR was demonstrated by applying it to 24 strains from </span><span class="font3" style="font-style:italic;">Pectobacterium carotovoum</span><span class="font3"> subsp</span><span class="font3" style="font-style:italic;">. carotovorum,</span><span class="font3"> 41 </span><span class="font3" style="font-style:italic;">Agrobacterium vitis</span><span class="font3"> strains, 3 </span><span class="font3" style="font-style:italic;">Xanthomonas</span><span class="font3"> spp. 5 </span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> spp, </span><span class="font3" style="font-style:italic;">Rhizobium</span><span class="font3"> sp. plant pathogenic bacteria, human and animal pathogenic bacterial strains including 6 </span><span class="font3" style="font-style:italic;">Escherichia coli</span><span class="font3">, 4 </span><span class="font3" style="font-style:italic;">Salmonella</span><span class="font3"> spp., 7 </span><span class="font3" style="font-style:italic;">Mycobacterium</span><span class="font3"> spp and 3 </span><span class="font3" style="font-style:italic;">Blucella abortus </span><span class="font3">strains. In addition, thermophilic bacteria were randomly isolated form high temperature compost and their URP-PCR polymorphisms were characterized with genetic relatedness. PCR approach using URP primers will be useful for studying DNA diversity of diverse prokaryotic genomes, especially at inter- and intra species levels.</span></p>
<p><span class="font3" style="font-style:italic;">Keywords:</span><span class="font3"> Diverse bacterial strains, PCR polymorphism, Universal Rice Primers</span></p>
<h1><a name="bookmark4"></a><span class="font3" style="font-weight:bold;"><a name="bookmark5"></a>INTRODUCTION</span></h1>
<p><span class="font3">The classic taxonomical methods of bacteria, relies on fatty acids, nutritional composition, and biochemical properties and so on (Garrity et al., 2002). However, it has been pointed out that microorganisms tend to lead to other consequences due to environmental effects, and that their shorter</span></p>
<p><span class="font3">generation and higher rates of metamorphosis have been a limiting factor in the bacterial classification system. The sequence of 16S rDNA was adopted as a new classification tool of the bacterial class (Chang et al., 1997), providing an opportunity for the revision of bacterial taxonomy to take place. In particular, inter transcribed spacer (ITS)-16S</span></p>
<p><span class="font3">rDNA sequence region is widely used as a species-specific identification of bacterial species (Chang et al., 1997, Garrity et al., 2002 ). However, since these rDNA regions are highly preserved areas, there is a limit to find polymorphism at level of inter-species such as subspecies, pathovar and race strains within species. Various molecular typing methods such as extragenic palindromic (REP) sequence, 124-127 enterobacterial repetitive intergenic consensus (ERIC) sequence 154 bp and BOX that are derived from repetitive sequences located in bacterial genomes have widely been used for discrimination of various bacterial strains at inter species level (De Brujin, 1992, Hulton et al., 1991) Recently, in silico genomic fingerprints were devised to be produced by virtual hybridization of 191 fully sequenced bacterial genomes using a set of 15,264 13-mer probes specially designed to produce universal whole genome fingerprints(Jaimes-Diaz et al., 2011).</span></p>
<p><span class="font3">Multiple arbitrary amplicon profile</span></p>
<p><span class="font3">(MAAP) that can universally be used in different organisms including pants, animals and microorganisms was developed for the use of single or multiple arbitrary primers (Caetano-Anolles, 1994; Caetano-Anolles and Gresshoff, 1997), and includes the techniques of RAPD (Williams </span><span class="font3" style="font-style:italic;">et al.,</span><span class="font3"> 1990),</span></p>
<p><span class="font3">AP-PCR (Welsh </span><span class="font3" style="font-style:italic;">et al</span><span class="font3">., 1990), and DNA amplification fingerprinting (DAF) (Caetano-Anolles </span><span class="font3" style="font-style:italic;">et al.,</span><span class="font3"> 1992). Because of its simplicity, the RAPD method has been used widely in studies of genetic diversity and in examining phylogenic and taxonomic relationship of bacterial strains (Caetano-Anolles and Gresshoff, 1997). However, in PCR techniques using arbitrary and RAPD primers, low reproducibility that causes variable PCR results depending on PCR condition has been recognized as a disadvantage factor (Caetano-Anolles et al., 1992).</span></p>
<p><span class="font3">A repeated DNA fragment (pKRD) of 1,187 bp was isolated from a genomic library of weedy rice distributed in Korea. A homology search showed the pKRD sequence belonged to a putative transposable element, the CACTA-like </span><span class="font3" style="font-style:italic;">En/Spm</span><span class="font3"> family Furthermore, pKRD was used as a molecular marker to detect genetic variation of rice germplasm (Kang and Kang, 2008). Twenty primers consisting of 20 oligonucleotides were randomly designed from the repetitive</span></p>
<p><span class="font3">sequence, pKRD (Kang </span><span class="font3" style="font-style:italic;">et al</span><span class="font3">., 2002). Under the high stringent PCR condition, twenty primers of them can be applied in genomic fingerprinting of a variety of organisms including animals, other plants, and microorganisms, as well as rice and named as universal rice primer (URP). URP-PCR technology has been applied for accessing genetic diversity of various fungal and bacterial species. Numerous papers have demonstrated that URP-PCR profiles of fungal species are very useful for identifying microbial species at intra and inter species levels (Aggarwal et al., 2010. Aggarwal et al, 2010; Hong et al., 2008; Jana et al., 2005; Kang, 2012; Kang, 2001, Kang, 2002) and further the specific PCR polymorphic bands could be used for finding SCAR PCR primer for diagnosis of bacterial species or strain (Kang et al., 2003; Lim et al., 2007).</span></p>
<p><span class="font3">In this study, it was demonstrated that URP primers can universally be applied to PCR fingerprinting of diverse genomes from plant, human, animal and industry related bacterial strains at levels of intra and inter</span></p>
<p><span class="font3">species.</span></p>
<p><span class="font3" style="font-weight:bold;">MATERIALS AND METHODS</span></p>
<h1><a name="bookmark6"></a><span class="font3" style="font-weight:bold;"><a name="bookmark7"></a>Bacterial strains and genomic DNA extraction</span></h1>
<p><span class="font3">Fourty one </span><span class="font3" style="font-style:italic;">Agrobacterium vitis </span><span class="font3">strains were isolated from different grapevein varieties by Hankyong National University. Twenty four strains of </span><span class="font3" style="font-style:italic;">Pectobacterium carotovoum</span><span class="font3"> &nbsp;&nbsp;&nbsp;subsp</span><span class="font3" style="font-style:italic;">. &nbsp;&nbsp;&nbsp;carotovorum</span><span class="font3">, &nbsp;&nbsp;&nbsp;3</span></p>
<p><span class="font3" style="font-style:italic;">Xanthomonas</span><span class="font3"> spp. 5 </span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> spp, </span><span class="font3" style="font-style:italic;">Rhizobium</span><span class="font3"> sp. of plant pathogenic bacteria were provided by Korea Agricultural Culture Collection (KACC). Themophilic bacterial strains containing </span><span class="font3" style="font-style:italic;">Thermoactinomyces vulgaris</span><span class="font3"> that are randomly isolated form high temperature fermented compost were obtained from National Institute of Agricultural Sciences (NIAS), Rural Development Administration in Korea. DNA samples of human and animal related bacterial strains included 6 </span><span class="font3" style="font-style:italic;">Escherichia coli strains</span><span class="font3"> (ATCC11105, 111a111b, M15, DH5, and O157:H7), &nbsp;4 </span><span class="font3" style="font-style:italic;">Salmonella</span><span class="font3"> spp., 7</span></p>
<p><span class="font3" style="font-style:italic;">Mycobacterium</span><span class="font3"> spp. and 3 </span><span class="font3" style="font-style:italic;">Blucella abortus</span></p>
<p><span class="font3">strains were obtained from Medical School in Yeonsei University of Korea. Genomic DNA from gram negative and positive bacteria was extracted following the procedure described by Ausubel et al.(1987).</span></p>
<h1><a name="bookmark8"></a><span class="font3" style="font-weight:bold;"><a name="bookmark9"></a>PCR analysis</span></h1>
<p><span class="font3">URP with 20 mer were provided from JK BioTech, Co. Ltd., Ansung, Korea. PCR reactions were carried out in 50 ul reaction mixtures containing the DNA template (20 to 50 ng of purified DNA), 10 mM Tris-HCl, 50 mM KCl, 1.5 mM MgCl</span><span class="font1">2</span><span class="font3">, 0.01% gelatin, 200 uM of each dNTP, 200 ng primer and 2.5 unit </span><span class="font3" style="font-style:italic;">Taq</span><span class="font3"> DNA polymerase (Promega, USA). The reaction mixture was overlaid with a thin layer of sterile mineral oil to prevent evaporation. DNA amplification was performed in a programmable DNA thermal cycler (MJ Research, Inc., USA). The cycling parameters used were initial denaturation at 94<sup>o</sup>C for 4 min, followed by 35 cycles each consisting of 1 min at 94<sup>o</sup>C, 1 min at 55<sup>o</sup>C, and 2 min at 72<sup>o</sup>C. After the last cycle, the PCR tubes were incubated at 72C</span></p>
<p><span class="font3">for 7 min and were held at 4C. Amplified products were electrophoresed in a 1.5 % agarose gel in TAE buffer and visualized by staining with ethidium bromide.</span></p>
<h1><a name="bookmark10"></a><span class="font3" style="font-weight:bold;"><a name="bookmark11"></a>PCR polymorphic bands based genetic relationship</span></h1>
<p><span class="font3">PCR polymorphic bands were scored on their presence (value=1) or absence (value=0). The similarity coefficient was calculated by rearranging the scored bands of each rice variety. A dendrogram was constructed with the statistical program NTSYSpc (Rohlf, 2000) using the unweighted pair-group method with arithmetic mean (UPGMA).</span></p>
<h1><a name="bookmark12"></a><span class="font3" style="font-weight:bold;"><a name="bookmark13"></a>RESULTS AND DISCUSSIONS</span></h1>
<p><span class="font3">URP-PCR method was used to study the genetic diversity of 24 </span><span class="font3" style="font-style:italic;">P. carotovurum </span><span class="font3">subsp </span><span class="font3" style="font-style:italic;">carotovorum</span><span class="font3"> (Pcc) strains causing bacterial soft rot disease on different plant species and 9 additive plant pathogenic bacteria. Fig. 1 shows PCR fingerprinting profile of diverse bacteria including</span></p>
<p><span class="font3" style="font-style:italic;">Pectobacterium &nbsp;&nbsp;&nbsp;carotovorum</span><span class="font3"> &nbsp;&nbsp;&nbsp;&nbsp;subsp.</span></p>
<p><span class="font3" style="font-style:italic;">carotovorum</span><span class="font3"> &nbsp;&nbsp;&nbsp;&nbsp;(Pcc), &nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font3" style="font-style:italic;">Xanthomonas</span></p>
<p><span class="font3" style="font-style:italic;">axonopodis</span><span class="font3"> pv. </span><span class="font3" style="font-style:italic;">citri</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> spp., </span><span class="font3" style="font-style:italic;">Pseudomonas solanocearum</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Rhizobium leguminosarum</span><span class="font3">, &nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> &nbsp;&nbsp;&nbsp;&nbsp;pv</span></p>
<p><span class="font3" style="font-style:italic;">phaseoliala</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Pseudomonas &nbsp;&nbsp;marginalis</span><span class="font3">,</span></p>
<p><span class="font3" style="font-style:italic;">Xanthomonas campestris</span><span class="font3"> pv. </span><span class="font3" style="font-style:italic;">campestris</span><span class="font3"> and </span><span class="font3" style="font-style:italic;">Pseudomonas syrinage</span><span class="font3"> pv </span><span class="font3" style="font-style:italic;">syringae</span><span class="font3"> using URP 4. The PCR data revealed that the majority of the bacterial strains have multiple banding patterns showing their PCR polymorphisms. Furthermore, the results distinguished clearly difference among the Pcc strains, showing their high genetic diversity. In the case of </span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> species, </span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> pv </span><span class="font3" style="font-style:italic;">phaseoliala</span><span class="font3"> &nbsp;&nbsp;&nbsp;and</span></p>
<p><span class="font3" style="font-style:italic;">Pseudomonas marginalis</span><span class="font3"> were divided as closely related group showing analogous PCR pattern among the </span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> species. </span><span class="font3" style="font-style:italic;">Agrobacterium vitis</span><span class="font3">, a soilborn bacterium, is a causal agent of crown gall on grapevines. Crown gall disease of grapevine has rapidly become problematic in Korea because</span></p>
<p><span class="font3">tetraploid grapevine cultivars of </span><span class="font3" style="font-style:italic;">Vitis vinifera </span><span class="font3">(Kyohoand, Daebong), which are relatively sensitive to </span><span class="font3" style="font-style:italic;">A. vitis</span><span class="font3">, were intensively planted in farm fields (80%) of Ansung and Cheonan areas of Korea. Forty one strains of </span><span class="font3" style="font-style:italic;">Agrobacterium vitis</span><span class="font3">, the causal agent of crown-gall disease on grapevine (Fig. 2 A), originating from different geographical regions and 16 grapevine cultivars including 35 Kyoho cultivar of Korea, were characterized by PCR fingerprinting using URP 2 and URP 4(Fig. 2 B, C). </span><span class="font3" style="font-style:italic;">A. vitis </span><span class="font3">strains originated from Kyoho cultivar of grapevine showed relatively simple genetic diversity of the four PCR types, while the </span><span class="font3" style="font-style:italic;">A. vitis</span><span class="font3"> strains originated from other grapevine cultivars showed various genetic diversities with 8 types. Comprehensively, UPGMA analysis based genetic relationship using the URP-PCR polymorphic bands showed 41 </span><span class="font3" style="font-style:italic;">A. vitis</span><span class="font3"> strains are genetically clustered into large seven groups and 17 subgroups in the groups (Data not shown).</span></p>
<div>
<p><span class="font3" style="font-style:italic;">syringae</span></p>
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/44266-1.jpg" alt="" style="width:363pt;height:210pt;">
<p><span class="font3">Fig. 1. PCR fingerprinting of different plant pathogenic bacterial strains by URP 4. M: 1kb DNA ladder (Promega), 1-24: </span><span class="font3" style="font-style:italic;">Pectobacterium carotovoum</span><span class="font3"> subsp</span><span class="font3" style="font-style:italic;">. carotovorum,</span><span class="font3"> 25: </span><span class="font3" style="font-style:italic;">Xanthomonas axonopodis pv citri</span><span class="font3">, 26: </span><span class="font3" style="font-style:italic;">Pseudomonas</span><span class="font3"> sp. 27: </span><span class="font3" style="font-style:italic;">Pseudomonas solanocearum</span><span class="font3">, 28: </span><span class="font3" style="font-style:italic;">Rhizobium leguminosarum</span><span class="font3">, 29: </span><span class="font3" style="font-style:italic;">Pseudomonase</span><span class="font3"> pv </span><span class="font3" style="font-style:italic;">phaseoliala</span><span class="font3">, 30: </span><span class="font3" style="font-style:italic;">Pseudomonas marginalis</span><span class="font3">, 31: </span><span class="font3" style="font-style:italic;">Xanthomonas campestris</span><span class="font3"> pv. </span><span class="font3" style="font-style:italic;">campestris</span><span class="font3">, 32: </span><span class="font3" style="font-style:italic;">Pseudomonas syrinage</span><span class="font3"> pv</span></p>
</div><br clear="all">
<div>
<p><span class="font1" style="font-weight:bold;font-style:italic;">Agrobacteriuin vitis</span><span class="font1" style="font-weight:bold;"> strains</span></p><img src="https://jurnal.harianregional.com/media/44266-2.jpg" alt="" style="width:437pt;height:90pt;">
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/44266-3.jpg" alt="" style="width:437pt;height:111pt;">
<p><span class="font3">Fig. 2. URP-PCR polymorphism of </span><span class="font3" style="font-style:italic;">Agrobacterium vitis</span><span class="font3"> strains, causal agent of crown gall on grapevine (A) detected by URP 2 (B) and URP 4 (C)</span></p>
</div><br clear="all">
<p><span class="font3">PCR-based methods are cheaper, easier and provide faster results; these methods include Random Amplified Polymorphic DNA PCR (RAPD-PCR), Repetitive extragenic palindromic PCR (REP-PCR) and Enterobacterial Repetitive Intergenic Consensus PCR (ERIC-PCR). These methods have been successfully applied for typing many bacteria (Anita et al. 2005; Johnson and Bryan, 2000; Williams et al. 1990). RAPD. Nevertheless, AP-PCR have been pointed out as low reproducibility problems as they are applied with annealing temperatures below 40<sup>o</sup>C. On the other hand, the URP-PCR can achieve stable PCR results because it has a high annealing temperature</span></p>
<p><span class="font3">of 55C. REP-PCR and ERIC-PCR fingerprinting methods using repeated sequence DNA of bacteria are limited in bacterial species retaining the sequence, whereas URP-PCR can universally apply in genomes from diverse bacteria strains without the reasonable sequence information on bacteria in advance. In particular, species or strain specific PCR fragment generated from URP-PCR products could be used in developing SCAR primers, which useful for specifically detecting </span><span class="font3" style="font-style:italic;">Agrobacterium vitis </span><span class="font3">and </span><span class="font3" style="font-style:italic;">Pectobacterium carotovoum</span><span class="font3"> subsp</span><span class="font3" style="font-style:italic;">. carotovorum</span><span class="font3"> (Kang et al., 2003; Lim et al., 2007).</span></p>
<p><span class="font3">Table 1. Oligonucleotide sequences of URP primers and their PCR applicability</span></p>
<p><span class="font3">on bacterial strains</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:middle;">
<p><span class="font2" style="font-weight:bold;">Primers</span></p></td><td colspan="4" style="vertical-align:bottom;">
<p><span class="font2" style="font-weight:bold;">GC</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">Sequences (5’-3’)</span></p></td><td style="vertical-align:middle;">
<p><span class="font2" style="font-weight:bold;">Conte</span></p>
<p><span class="font2" style="font-weight:bold;">nt</span></p></td><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">Tm</span></p>
<p><span class="font2" style="font-weight:bold;">(<sup>o</sup>C)</span></p></td><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">Applicability</span></p>
<p><span class="font2" style="font-weight:bold;">on bacterial strains</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP1</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">ATCCAAGGTCCGAGACAACC</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">65</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">Yes</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP2</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">CCCAGCAACTGATCGCACAC</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">65</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">Yes</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP3</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">AGGACTCGATAACAGGCTCC</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">66</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">Yes</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP4</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">GGCAAGCTGGTGGGAGGTAC</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">65</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">Yes</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP5</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">ATGTGTGCGATCAGTTGCTG</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">67</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">Yes</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP6</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">TACATCGCAAGTGACACAGG</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">68</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">No</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP7</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">AATGTGGGCAAGCTGGTGGT</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">55</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">74</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">Yes</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP8</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">GATGTGTTCTTGGAGCCTGT</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">65</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">Yes</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP8</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">GGACAAGAAGAGGATGTGGA</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">65</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">ND</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP9</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">TACACGTCTCGATCTACAGG</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">65</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">No</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2">URP10</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">AAGAGGCATTCTACCACCAC</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">65</span></p></td><td style="vertical-align:middle;">
<p><span class="font2">No</span></p></td></tr>
</table>
<p><span class="font3">Genomic DNA of 225 strains in 45 bacterial species were used as templates for URP-PCR</span></p>
<p><span class="font3">fingerprinting. The applicability of URP-PCR represents Yes, No and ND (Not determined)</span></p>
<p><span class="font3">In the following experiment, URP-PCR was applied to identification between or within human and animal -related bacterial species. As representative bacterial materials, genomic DNAs extracted from </span><span class="font3" style="font-style:italic;">Mycobacterium</span><span class="font3"> spp</span><span class="font3" style="font-style:italic;">.</span><span class="font3"> including 4 </span><span class="font3" style="font-style:italic;">Mycobacterium intracellularae,</span><span class="font3"> strains</span><span class="font3" style="font-style:italic;">,</span><span class="font3"> 4 </span><span class="font3" style="font-style:italic;">M. avium strains,</span><span class="font3"> 3 </span><span class="font3" style="font-style:italic;">M. scrofllaceum</span><span class="font3"> strains</span><span class="font3" style="font-weight:bold;">, </span><span class="font3" style="font-style:italic;">M. fortuitum, M. gallinarum, M. canetti,</span><span class="font3"> and </span><span class="font3" style="font-style:italic;">M. smegmatis. M. intracellularae, M. avium</span><span class="font3"> and</span></p>
<p><span class="font3" style="font-style:italic;">M. scrofllaceum</span><span class="font3"> strains that are the most common cause of cervical</span><a href="https://en.wikipedia.org/wiki/Lymphadenopathy"><span class="font3"> lymphadenitis </span></a><span class="font3">in children</span><span class="font3" style="font-weight:bold;font-style:italic;">, </span><span class="font3" style="font-style:italic;">M. fortuitum</span><span class="font3"> is one of the many species of nontuberculosis mycobacteria (NTM) that are commonly found in the environment. </span><span class="font3" style="font-style:italic;">M. canettii</span><span class="font3">, a novel pathogenic taxon of the</span><a href="https://en.wikipedia.org/wiki/Mycobacterium_tuberculosis_complex"><span class="font3"> </span><span class="font3" style="font-style:italic;">Mycobacterium tuberculosis</span></a><span class="font3" style="font-style:italic;"> </span><a href="https://en.wikipedia.org/wiki/Mycobacterium_tuberculosis_complex"><span class="font3">complex </span></a><span class="font3">(MTBC) (Hirsh et al., 2004), </span><span class="font3" style="font-style:italic;">M. smegmatis</span><span class="font3"> is commonly used in work on the Mycobacterium genus due to its being a &quot;fast</span></p>
<p><span class="font3">grower&quot; and non-pathogenic (Simner et al., 2013).</span></p>
<div><img src="https://jurnal.harianregional.com/media/44266-4.jpg" alt="" style="width:424pt;height:204pt;">
<p><span class="font3">Fig. 3. PCR fingerprinting of human and animal related different bacterial species</span></p>
</div><br clear="all">
<p><span class="font3" style="font-style:italic;">Mycobacterium</span><span class="font3"> spp. (A), </span><span class="font3" style="font-style:italic;">Echerichia coli</span><span class="font3"> strains, </span><span class="font3" style="font-style:italic;">Salmonella</span><span class="font3"> spp. and Blucella (B) by URP2R.</span></p>
<p><span class="font3" style="font-style:italic;">Mycobacterium avium-intracellulare </span><span class="font3">infection (MAI) is an atypical mycobacterial infection, i.e. one with</span><a href="https://en.wikipedia.org/wiki/Nontuberculous_mycobacteria"><span class="font3"> nontuberculous</span></a><span class="font3"> </span><a href="https://en.wikipedia.org/wiki/Nontuberculous_mycobacteria"><span class="font3">mycobacteria </span></a><span class="font3">or NTM, caused by </span><a href="https://en.wikipedia.org/wiki/Mycobacterium_avium_complex"><span class="font3" style="font-style:italic;">Mycobacterium avium</span><span class="font3"> complex </span></a><span class="font3">(&quot;MAC&quot;), which is made of three mycobacteria species, </span><span class="font3" style="font-style:italic;">M. avium</span><span class="font3">, </span><span class="font3" style="font-style:italic;">M. intracellulare</span><span class="font3">, and </span><span class="font3" style="font-style:italic;">M. chimaera</span><span class="font3"> (Horsburgh.et al., 1985) This </span><a href="https://en.wikipedia.org/wiki/Infection"><span class="font3">infection </span></a><span class="font3">causes respiratory illness in birds, pigs, and humans, especially in </span><a href="https://en.wikipedia.org/wiki/Immunocompromised"><span class="font3">immunocompromised </span></a><span class="font3">people. As shown in Fig.3 (A), PCR polymorphism dependent on the strains was observed on PCR amplicons</span></p>
<p><span class="font3">of </span><span class="font3" style="font-style:italic;">Mycobacterium</span><span class="font3"> spp., suggesting URP-PCR is useful for strain typing within the bacterial species. In addition, several human animal pathogenic bacteria were tested for confirming usefulness of URP-PCR. Non pathogenic </span><span class="font3" style="font-style:italic;">Escherichia coli</span><span class="font3"> including strains ATCC11105, 111a111b, M15 and pathogenic </span><span class="font3" style="font-style:italic;">E. coli</span><span class="font3"> O157:H7 that is one of the Shiga toxin-producing type and is a cause of </span><a href="https://en.wikipedia.org/wiki/Disease"><span class="font3">disease,</span></a><span class="font3"> typically</span><a href="https://en.wikipedia.org/wiki/Foodborne_illness"><span class="font3"> foodborne illness </span></a><span class="font3">of the &quot;colonic escherichiosis&quot; type ( Furrer et al., 1990). </span><span class="font3" style="font-style:italic;">E. coli</span><span class="font3"> strains, </span><span class="font3" style="font-style:italic;">Salmonella</span><span class="font3"> spp. and </span><span class="font3" style="font-style:italic;">Blucella abortus</span><span class="font3"> strains were used as</span></p>
<p><span class="font3">template DNA and were amplified by primer URP-3(Fig. 3B). The </span><span class="font3" style="font-style:italic;">E. coli </span><span class="font3">O157:H7 specific PCR polymorphic bands were observed on size marker of 0.5 kb and other polymorphic bands were showed in reliance on </span><span class="font3" style="font-style:italic;">E. coli</span><span class="font3"> strains. As a result, six </span><span class="font3" style="font-style:italic;">E. coli</span><span class="font3"> strains were divided into four types. </span><span class="font3" style="font-style:italic;">Salmonella thompson</span><span class="font3">, </span><span class="font3" style="font-style:italic;">S. typhimurium</span><span class="font3">, </span><span class="font3" style="font-style:italic;">S. paratyphus</span><span class="font3"> A, and </span><span class="font3" style="font-style:italic;">S. paratyphus</span><span class="font3"> B that are causal agents of Salmonellosis</span><span class="font3" style="font-weight:bold;">, </span><span class="font3">food poisoning (McClelland et al., 2001; Bej et al., 1994), and animal and human pathogenic bacteria species such as </span><span class="font3" style="font-style:italic;">Brucella</span><span class="font3"> strains were</span></p>
<p><span class="font3">also examined. A polymorphic band of 0.8 kb was observed on only </span><span class="font3" style="font-style:italic;">S. paratyphus</span><span class="font3"> A, and </span><span class="font3" style="font-style:italic;">S. paratyphus</span><span class="font3"> B, showing its specific PCR fragment in the bacterial species. Nevertheless, URP-PCR polymorphism of </span><span class="font3" style="font-style:italic;">Blucella abortus</span><span class="font3"> strains was identical in the strains. Consequently, URP-PCR yielded distinct PCR profiles that permitted differentiation among them at interspecies level, and furthermore, ubiquitous strain types were observed within </span><span class="font3" style="font-style:italic;">E. coli</span><span class="font3"> strains and </span><span class="font3" style="font-style:italic;">Salmonella</span><span class="font3"> species.</span></p>
<div><img src="https://jurnal.harianregional.com/media/44266-5.jpg" alt="" style="width:360pt;height:240pt;">
<p><span class="font3">Fig. 4. URP-PCR profiles (A) and the PCR bands based UPGMA dendrogram (B)of</span></p>
</div><br clear="all">
<p><span class="font3">hemophilic bacterial strains isolated from compost fermented with high temperature</span></p>
<p><span class="font3">Finally, thermophilic bacteria that were randomly isolated from high temperature (45-55<sup>o</sup>C) fermented compost</span></p>
<p><span class="font3">was used for URP-PCR. Genomic DNAs were amplified by primers URP 3, URP 4 and URP 7 (Fig. 4 A) and resultant</span></p>
<p><span class="font3">polymorphic bands were scored to construct dendrogram. As showed in dendrogram data of Fig. 4 B, the bacteria were divided into two large groups containing culture collection, </span><span class="font3" style="font-style:italic;">Thermoactinomyces vulgaris</span><span class="font3"> and further five subgroups were formed within the bacterial strains, showing different genetic relationships.</span></p>
<p><span class="font3">The data obtained from studies of various bacteria suggests that URP-PCR is potentially a valuable tool in the characterization and grouping at inter- and intraspecific levels of various bacterial species associated with medical, agricultural, industrial, and environmental fields. In many experiments using additional bacterial species, it was found URP-PCR allows generating discrete DNA fragments that are species or strain -specific. The speciesspecific URP-PCR products are easily excised from gels and cloned or re-amplified with the same primer to be used as a probe, providing an alternative strategy in developing specific Sequence-characterized amplified region (SCAR) markers that detect target microbial species or strain. Previously, such strategy was successfully used to develop specific DNA probe that detects </span><span class="font3" style="font-style:italic;">Pectobacterium carotovorum</span><span class="font3"> subsp. </span><span class="font3" style="font-style:italic;">carotovorum</span><span class="font3"> and </span><span class="font3" style="font-style:italic;">A. vitis</span><span class="font3"> among other bacteria (Kang </span><span class="font3" style="font-style:italic;">et al.</span><span class="font3">, 2003; Lim et al., 2006).</span></p>
<h1><a name="bookmark14"></a><span class="font3" style="font-weight:bold;"><a name="bookmark15"></a>CONCLUSIONS</span></h1>
<p><span class="font3">PCR approach using URP primers offers a powerful tool for studying DNA diversity of prokaryotic genomes, with potential use in taxonomic and phylogenic analysis, as well as in genotypic screening of strains in species, especially at inter- and intra-species levels. The URP-PCR is clearly different to RAPD (Williams et al., 1990) and AP-PCR method (Welsh et al., 1990) using arbitrary selected-primers because of long primer and high annealing temperature. It was believed that such URP-PCR conditions result in highly reproducible amplification products from organisms. This result suggested universal applicability of URP-PCR to human, animal, plant and industry related bacteria strains. Because URP primers can be applied to a wide range of bacterial species, mass screen of bacterial resources and database construction of DNA profile can be very useful for various purposes such as identifying genetic characteristics of specific strains. However, it remains to elucidate how URP primers designed from</span></p>
<p><span class="font3">repetitive sequence of rice are able to produce DNA polymorphism on diverse bacteria genomes with high specificity at inter- and intra-species. Nevertheless, it is reasonably assumed that complementary sequences to URP primers are widely dispersed in the diverse bacterial genomes that may play a critical evolutionary role.</span></p>
<h1><a name="bookmark16"></a><span class="font3" style="font-weight:bold;"><a name="bookmark17"></a>REFERENCES</span></h1>
<p><span class="font3">Ausubel, F. M. T., Brent, R., Kingston, R. E., Moore, D. D., Seidman, J. G., Smith, J. A., Steria ruhl, K. (1987). Current protocols in molecular biology. Greene Publishing &nbsp;&nbsp;&nbsp;&nbsp;Associates &nbsp;&nbsp;&nbsp;&nbsp;Wiley</span></p>
<p><span class="font3">Interscience, New York.</span></p>
<p><span class="font3">Aggarwal, R., Singh, V., Shukla, R., Gurjar, M., &amp;&nbsp;Sharma, T. (2010). URP-based DNA fingerprinting of bipolaris sorokiniana isolates causing spot blotch of wheat. </span><span class="font3" style="font-style:italic;">J. Phytopath</span><span class="font3">, 158, 210-216.</span></p>
<p><span class="font3">Aggarwal, R., Tripathi, A., &amp;&nbsp;Yadav, A. (2010). Pathogenic and &nbsp;&nbsp;genetic</span></p>
<p><span class="font3">variability in Tilletia indica monosporidial culture lines using universal rice primer-PCR Eur. </span><span class="font3" style="font-style:italic;">J. Plant Pathol</span><span class="font3">, 128, 333-342</span></p>
<p><span class="font3">Bej, A. K., Mahbubani, M. H., Boyce, M. J., &amp;&nbsp;Atlas, R. M. (1994). Detection of Salmonella spp. In oysters by PCR. </span><span class="font3" style="font-style:italic;">Appl. Environ. Microbial</span><span class="font3">, 60, 368-373.</span></p>
<p><span class="font3">Caetano-Anolles, G. 1994. MAAP: &nbsp;a</span></p>
<p><span class="font3">versatile and universal tool for genome</span></p>
<p><span class="font3">analysis. </span><span class="font3" style="font-style:italic;">Plant Molecular Biology</span><span class="font3">, 25, 1011-1026.</span></p>
<p><span class="font3">Caetano-Anolles, G., &amp;&nbsp;Gresshoff, P. M.</span></p>
<p><span class="font3">(1997). DNA markers: protocols, Applications, and overviews. WILEY-VCH, N.Y</span></p>
<p><span class="font3">Caetano-Anolles, G., Bassam, G. J., &nbsp;&amp;</span></p>
<p><span class="font3">Gresshoff, P. M. (1992). Primertemplate interactions during DNA amplification fingerprinting with single arbitrary oligonucleotides. </span><span class="font3" style="font-style:italic;">Mol. Gen. Genet</span><span class="font3">, 235, 157- 165.</span></p>
<p><span class="font3">Chang, H. C., Wei, Y. F., Dijkshoorn, L., Vaneechoutte, M., Tang, C. T., et al. (2005). Species-level identification of isolates of the Acinetobacter calcoaceticus Acinetobacterbaumannii complex by sequence analysis of the 16S-23S rRNA gene spacer region. </span><span class="font3" style="font-style:italic;">J Clin Microbiol,</span><span class="font3"> 43, 1632–1639</span></p>
<p><span class="font3">Chang, H. R., Loo, L. H., Jeyaseelan, K., Earnest, L., &amp;&nbsp;Stackebrandt, E. (1997). Phylogenetic &nbsp;&nbsp;&nbsp;&nbsp;Relationships &nbsp;&nbsp;&nbsp;&nbsp;of</span></p>
<p><span class="font3" style="font-style:italic;">Salmonella typhi</span><span class="font3"> and </span><span class="font3" style="font-style:italic;">Salmonella typhimun'um</span><span class="font3"> Based on 16s rRNA Sequence Analysis. </span><span class="font3" style="font-style:italic;">International Journal of Systematic Bacteriology</span><span class="font3">, 47, 1253-1254.</span></p>
<p><span class="font3">De Brujin, F. J. (1992). Use of repetitive (repetitive extragenic pal-indromic and enterobacterial repetive intergenic consensus) sequences and the polymerase chain reaction to fingerprint the genomes of </span><span class="font3" style="font-style:italic;">Rhizobium meliloti</span><span class="font3"> isolates and other soil bacteria. </span><span class="font3" style="font-style:italic;">Appl. Environ. Microbiol</span><span class="font3">, 58, 21802187.</span></p>
<p><span class="font3">Jaimes-Diaz, H., Garcia-Chequer, A. J.,</span></p>
<p><span class="font3">Mendez-Tenorio, &nbsp;&nbsp;&nbsp;A., &nbsp;&nbsp;&nbsp;Santiago-</span></p>
<p><span class="font3">Hemadez, J. C., Maldonado-Rodriguez, R., &amp;&nbsp;Beattie, L. L. (2011). Bacterial classification &nbsp;&nbsp;&nbsp;&nbsp;using &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;genomic</span></p>
<p><span class="font3">fingerprints obtained by virtual hybridization.</span><a href="https://www.ncbi.nlm.nih.gov/pubmed/21906634"><span class="font3"> </span><span class="font3" style="font-style:italic;">J Microbiol Methods.</span><span class="font3"> </span></a><span class="font3">87, 286-94.</span></p>
<p><span class="font3">Furrer, B., Candrian, U., &amp;&nbsp;Luethy, J. (1990). Detection &nbsp;&nbsp;and identification of</span></p>
<p><span class="font3" style="font-style:italic;">Escherichia coli</span><span class="font3"> producing heat-labile enterotoxin type &nbsp;I &nbsp;by enzymatic</span></p>
<p><span class="font3">amplification of &nbsp;a &nbsp;specific DNA</span></p>
<p><span class="font3">fragment. </span><span class="font3" style="font-style:italic;">Lett. Appl. Microbial.</span><span class="font3"> 10, 3134.</span></p>
<p><span class="font3">Garrity, G. M., Johnson, K. L., Bell, J. A., &amp;&nbsp;Searles, D. B. (2002). Taxonomic Outline of the Procaryotes. Bergey's Manual of Systematic Bacteriology, Second Edition, Release 3.0., SpringerVerlag, New York. 365 pages.</span></p>
<p><span class="font3">Hirsh, A. E., Tsolaki, A. G., DeRiemer, K., Feldman, M. W., &amp;&nbsp;Small, P. M. (2004). Stable association between strains of Mycobacterium tuberculosis and their human host populations. </span><span class="font3" style="font-style:italic;">Proc Natl Acad Sci U S A,</span><span class="font3"> 101, 4871–4876.</span></p>
<p><span class="font3">Hong, S. K., Kim, W. K., Yun, H. G., &amp;&nbsp;Choi, &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;K. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;J. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2008).</span></p>
<p><span class="font3">Morphologicalvariations, &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;genetic</span></p>
<p><span class="font3">diversity &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;pathogenicity</span></p>
<p><span class="font3">ofColletotrichum species causing grape ripe rot in Korea. </span><span class="font3" style="font-style:italic;">Plant Pathol. J,</span><span class="font3"> 24, 269-278</span></p>
<p><span class="font3">Horsburgh, C. R, Mason, U. G., Farhi, D. C., &amp;&nbsp;Iseman, M. D. (1985). Disseminated infection with </span><span class="font3" style="font-style:italic;">Mycobacterium avium-intracellulare</span><span class="font3">. A report of 13 cases and a review of the literature. </span><span class="font3" style="font-style:italic;">Medicine,</span><span class="font3"> 64,</span></p>
<p><span class="font3">36-48.</span></p>
<p><span class="font3">Hulton, C. J. S., Higgins, C. F., &amp;&nbsp;Sharp, P. M. (1991). ERIC sequences: a novel family of repetitive elements in the genomes of </span><span class="font3" style="font-style:italic;">Escherichia coli</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Salmonella typimurium</span><span class="font3"> and other enterobacteria. </span><span class="font3" style="font-style:italic;">Molecular Microbiology, </span><span class="font3">5, 825-834.</span></p>
<p><span class="font3">Jana, T. K, Singh, N. K., Koundal, K. R., &amp;&nbsp;Sharma, T. R. (2005). Genetic differentiation of charcoal rot pathogen, Macrophomina phaseolina, into specific groups using URP-PCR. </span><span class="font3" style="font-style:italic;">Canadian J. Microbiol</span><span class="font3">, 51, 159-164.</span></p>
<p><span class="font3">Johnson, J. R., &amp;&nbsp;O’Bryan, T. T. (2000). Improved Repetitive-Element PCR fingerprinting for resolving pathogenic and nonpathogenic phylogenic groups within </span><span class="font3" style="font-style:italic;">Escherichia coli</span><span class="font3">. </span><span class="font3" style="font-style:italic;">Clin Diagn Lab Immunol</span><span class="font3">, </span><span class="font3" style="font-weight:bold;">7:</span><span class="font3">265-273</span></p>
<p><span class="font3">Kang, H. W. &nbsp;(2012). Genetic Divesity</span></p>
<p><span class="font3">Analysis &nbsp;of Fungal Species by</span></p>
<p><span class="font3">Universal Rice Primer (URP)-PCR.</span></p>
<p><span class="font3" style="font-style:italic;">The Korean Journal of Mycology</span><span class="font3">, 40, 78-85.</span></p>
<p><span class="font3">Kang, H. W., Kwon, S. W., &amp;&nbsp;Go, S. J. (2003). PCR-based specific and sensitive detection of </span><span class="font3" style="font-style:italic;">Pectobacterium carotovorum</span><span class="font3"> ssp. </span><span class="font3" style="font-style:italic;">carotovorum</span><span class="font3"> by primers generated from a URP-PCR fingerprinting-derived polymorphic band. </span><span class="font3" style="font-style:italic;">Plant pathology</span><span class="font3">, 52, 127-133.</span></p>
<p><span class="font3">Kang, H. W., Go, S. J., &amp;&nbsp;Eun, M. Y. (2002). Fingerprinting of diverse genomes using PCR with universal rice primers generated from repetitive sequence of Korean weedy rice. </span><span class="font3" style="font-style:italic;">Molecular Cells</span><span class="font3">, 13, 1-7.</span></p>
<p><span class="font3">Kang, H. W., Park, D. S., Park, Y. J., You, C. H., Lee, B. M. and Go, S. J. (2001). Genomic differentiation among oyster mushroom cultivars released in Korea by &nbsp;&nbsp;&nbsp;&nbsp;URP-PCR &nbsp;&nbsp;&nbsp;&nbsp;fingerprinting.</span></p>
<p><span class="font3" style="font-style:italic;">Mycobiology,</span><span class="font3"> 29, 85-89.</span></p>
<p><span class="font3">Kang, H. W., Park, D. S., Park, Y. J., Lee, B. M., Cho, S. M., Kim, K. T., Seo, K. S., &amp;&nbsp;Go, S. J. (2002). PCR based detection of </span><span class="font3" style="font-style:italic;">Phellinus linteus</span><span class="font3"> using specific primers generated from universal rice primer (URP) derived PCR polymorphic band. </span><span class="font3" style="font-style:italic;">Mycobiology</span><span class="font3">, 30, 202-207.</span></p>
<p><span class="font3">Kang, H. W. &amp;Kang, K. K. (2008). Genomic characterization of Oryza speciesspecific CACTA-like transposon element &nbsp;and its application for</span></p>
<p><span class="font3">genomic fingerprinting of rice varieties. </span><span class="font3" style="font-style:italic;">Molecular Breeding</span><span class="font3">, 21, 271-281.</span></p>
<p><span class="font3">Lim, S. H., Kim, J. K., &amp;&nbsp;Kang, H. W. (2007). Novel SCARprimers forspecific and sensitive detection of </span><span class="font3" style="font-style:italic;">Agrobacterium vitis</span><span class="font3"> strains. </span><span class="font3" style="font-style:italic;">Microbiological Research</span><span class="font3">, 164, 451-460.</span></p>
<p><span class="font3">McClelland, M., Sanderson, K. E., Spieth, J., Clifton, S. W., &amp;&nbsp;Latreille, P. (2001). Complete genome sequence of </span><span class="font3" style="font-style:italic;">Salmonella &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;enterica</span><span class="font3"> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;serovar</span></p>
<p><span class="font3" style="font-style:italic;">Typhimurium</span><span class="font3"> LT2. </span><span class="font3" style="font-style:italic;">Nature,</span><span class="font3"> 413, 852– 856.</span></p>
<p><span class="font3">Simner, P. J., Buckwalter, S. P., Uhl, J. R., &amp;&nbsp;Wengenack, N. &nbsp;&nbsp;&nbsp;L. &nbsp;&nbsp;&nbsp;&nbsp;(2013).</span></p>
<p><span class="font3">Identification of </span><span class="font3" style="font-style:italic;">Mycobacterium </span><span class="font3">Species &nbsp;&nbsp;&nbsp;&nbsp;and &nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font3" style="font-style:italic;">Mycobacterium</span></p>
<p><span class="font3" style="font-style:italic;">tuberculosis</span><span class="font3"> &nbsp;&nbsp;Complex &nbsp;&nbsp;Resistance</span></p>
<p><span class="font3">Determinants &nbsp;by Use &nbsp;of PCR-</span></p>
<p><span class="font3">Electrospray &nbsp;&nbsp;&nbsp;&nbsp;Ionization &nbsp;&nbsp;&nbsp;&nbsp;Mass</span></p>
<p><span class="font3">Spectrometry.</span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3889724/"><span class="font3"> </span><span class="font3" style="font-style:italic;">J Clin Microbiol</span><span class="font3">,</span></a><span class="font3"> 51,</span></p>
<p><span class="font3">3492–3498.</span></p>
<p><span class="font3">Welsh, J., Petersen, C., &amp;&nbsp;McClelland, C. (1990). Polymorphisms generated by arbitrarily primed PCR in the mouse: application to strain identification and genetic mapping. </span><span class="font3" style="font-style:italic;">Nucleic Acids Res</span><span class="font3">, 19: 303-306.</span></p>
<p><span class="font3">Williams, J. G. K., Kubelic, A. R., Livak, K. J., Rafalski, J. A., &amp;&nbsp;Tingey, S. V. (1990) </span><span class="font3" style="font-style:italic;">DNA polymorphisms amplified by arbitray primers are useful as genetic markers.</span><span class="font3"> Nucleic Acid Res</span><span class="font3" style="font-style:italic;">, 18, 6531-6535.</span></p>
<p><span class="font0">64 • FACULTY OF AGRICULTURE, UDAYANA UNIVERSITY IN COOPERATION WITH</span></p>
<p><span class="font0">ASIA OCEANIA BIOSCIENCE AND BIOTECHNOLOGY CONSORTIUM</span></p>