---
layout: full_article
title: "IN SILICO PRIMER DESIGN AND ANNEALING TEMPERATURE OPTIMIZATION TO AMPLIFY THE FRAGMENT OF gyrB GENE Mycobacterium tuberculosis ISOLATE P010 USING POLYMERASE CHAIN REACTION"
author: "Putu Irma Handayani"
categories: jhsm
canonical_url: https://jurnal.harianregional.com/jhsm/full-47788 
citation_abstract_html_url: "https://jurnal.harianregional.com/jhsm/id-47788"
citation_pdf_url: "https://jurnal.harianregional.com/jhsm/full-47788"  
comments: true
---

<p><span class="font2">Journal of Health Sciences and Medicine, Vol. 2 No. 1 February 2018</span></p>
<p><span class="font4" style="font-weight:bold;">IN SILICO PRIMER DESIGN AND ANNEALING TEMPERATURE OPTIMIZATION TO AMPLIFY THE FRAGMENT OF </span><span class="font4" style="font-weight:bold;font-style:italic;">gyr</span><span class="font4" style="font-weight:bold;">B GENE </span><span class="font4" style="font-weight:bold;font-style:italic;">Mycobacterium tuberculosis</span><span class="font4" style="font-weight:bold;"> ISOLATE P010 USING POLYMERASE CHAIN REACTION</span></p>
<p><span class="font3">Putu Irma handayani<sup>1</sup>, Dyah Subadrika Warma Dewi<sup>1</sup>, Ni Putu Eka Nata Sari<sup>1</sup>, Sang Ayu Made Rai Wulan Dewi<sup>1</sup>, Sagung Chandra Yowani<sup>1,3</sup>, Ni Nyoman Ayu Dewi<sup>2,3</sup>, Putu Sanna Yustiantara<sup>1,3</sup></span></p>
<p><span class="font2"><sup>1</sup>Department of Pharmacy, Faculty of Mathematics and Natural Science, Udayana University <sup>2</sup>Department of Medicine, Udayana University</span></p>
<p><span class="font2"><sup>3</sup>MDR-TB &amp;&nbsp;XDR-TB Research Group, Udayana University</span></p>
<p><span class="font2">Bali, Indonesia</span></p>
<p><span class="font2">E-mail : </span><a href="mailto:handayaniirma7@gmail.com"><span class="font2">handayaniirma7@gmail.com</span></a></p>
<p><span class="font2" style="font-weight:bold;">Abstract</span></p>
<p><span class="font2">One of the factors causing XDR-TB is due to mutations in the </span><span class="font2" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font2"> gene, one of them is in the </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene. Amplification of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene fragments from </span><span class="font2" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font2"> DNA using Polymerase Chain Reaction (PCR) method. The amplification process by the PCR method requires a pair of primers (forward and reverse) to limit the area to be amplified. The current study aims to obtain the best primer pair generated by in silico design using Clone Manager Suite 6 program while simultaneously optimizing the annealing temperature to amplify the fragment of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B </span><span class="font2" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font2">. The template used in designing the primer is the sequence of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B </span><span class="font2" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font2"> H37Rv isolate obtained from NCBI database of genbank code AL123456.3.</span></p>
<p><span class="font2">The current study obtained a pair of primer which respectively had 19 oligonucleotide length and the best annealing temperature of 56ºC. The primer is be able to do in silico amplification of the fragment of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B </span><span class="font2" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font2"> gene isolate P010 in the nucleotide area range from 1271-1755 bp with 485 bp fragment length.</span></p>
<p><span class="font1" style="font-weight:bold;">Key words: primer design, annealing temperature, </span><span class="font1" style="font-weight:bold;font-style:italic;">gyr</span><span class="font1" style="font-weight:bold;">B gene, PCR.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">I. &nbsp;&nbsp;&nbsp;INTRODUCTION</span></p></li></ul>
<p><span class="font2">Indonesia is included in 30 countries with a high burden of MDR-TB which reaches 32,000 new cases of MDR-TB from 480,000 global data on new cases of MDR-TB [8]. According to WHO data (2017) MDR-TB cases have grown by 6.2% to Extensively Drug Resistant Tuberculosis (XDR-TB). XDR-TB is a TB caused by bacteria resistant to INH and RIF (MDR-TB) moreover it is also resistant to at least Fluorokuinolone (FQ) and one of the second-line tuberculosis drugs in the form of injection [8]. In an tuberculosis drugs resistance treatment study, an examination has been performed on some second-line tuberculosis drugs. It is found out that Fluorokuinolone (FQ) is the most important drug of</span></p>
<p><span class="font2">the core MDR-TB regimen, this is due to the effectiveness of the therapy given and the low toxicity of the use of FQ compared to other second-line OAT.</span></p>
<p><span class="font2">The mechanism of development of FQ resistance in </span><span class="font2" style="font-style:italic;">M. tuberculosis</span><span class="font2"> is the presence of mutations in the Quinolone Resitance Determining Region (QRDR) </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene [5]. QRDR </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene is defined as the region at codons 500 to 540 [4].</span></p>
<p><span class="font2">The identification of mutations in </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene fragments can be done by PCR technique. In the process of amplification of gene fragments required a pair of primers (forward and reverse) is required. Primer serves as a limiting fragment of DNA to be amplified. The primer used must meet the criteria of a</span></p>
<p><span class="font2">good primer to amplify </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B fragments specifically. These criteria include: primary length,% GC, Tm, dimer and hairpins, primary stability, repeats, runs and false priming. Therefore the current study aims to design a pair of primer used to amplify </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene fragment as well as to find out the optimum annealing temperature to amplify </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B fragment by conducting annealing temperature optimization.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">II. &nbsp;&nbsp;&nbsp;METHODS</span></p>
<ul style="list-style:none;">
<li>
<p><span class="font2">2.1 &nbsp;&nbsp;&nbsp;Primer Design</span></p></li></ul></li></ul>
<p><span class="font2">Primer used to amplify </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene fragment in the current study was designed to utilize Clone Manager Suite 6 program. The first stage to be done was inputting the sequence nucleotide data of </span><span class="font2" style="font-style:italic;">gy</span><span class="font2">rB </span><span class="font2" style="font-style:italic;">M. tuberculosis</span><span class="font2"> data (Gen Bank AL123456.3), which was provided in the database at URL://www.ncbi.nlm.nih.gov, to Clone Manager Suite 6 program. A Pair of primer was designed by selecting menu bar primer design. The length of primer base produced was input and target area to amplified based on the mutation data literature was determined. Primers produced in the rank line was analyzed based on the primer criteria desired for </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene area.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">2.2 &nbsp;&nbsp;&nbsp;Annealing Temperature Optimization and Amplification of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B </span><span class="font2" style="font-style:italic;">M. tuberculosis</span><span class="font2"> Gene Fragment using PCR</span></p></li></ul>
<p><span class="font2">Determination of primer optimum temperature to be used to amplify </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B fragment on isolate H37Rv was done by conducting an experiment at 56°C, 58°C, 60°C</span></p>
<p><span class="font2">DNA template used for amplification was obtained from the isolation result of isolate P010. PCR was conducted using a pair of primer specially designed on the current study used Clone Manager Suite 6 where the amplification process started with predenaturation at 95°C for 15 minutes followed by 45 amplification cycle (denaturation at 94°C for 1 minute, annealing at optimization temperature at 56°C, 58°C, 60°C for 1 minute 20 seconds, extension at 72°C for 2 minutes and final extension at 72°C for 10 minutes).</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">2.3 &nbsp;&nbsp;&nbsp;PCR Product Detection</span></p></li></ul>
<p><span class="font2">PCR product was detected using electrophoresis agarose gel 1,5% b/v. The result of electrophoresis was then conducted visualization using UV Transilluminator.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">III. &nbsp;&nbsp;&nbsp;RESULT AND DISCUSSION</span></p>
<ul style="list-style:none;">
<li>
<p><span class="font2">3.1 &nbsp;&nbsp;&nbsp;Primer Design</span></p></li></ul></li></ul>
<p><span class="font2">The primer design performed using Clone Manager Suite 6 obtains the result of a pair of primer which are forward primer with a nucleotide base</span></p>
<p><span class="font2">sequence:5'-GAGAGTTGGTGCGGCGTAA-3' and reverse primer with a nucleotide base sequence:5'GCGGTCGGAGTATGCGAAT-3'.</span></p>
<p><span class="font2">The length of the forward and reverse primer of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B genes is 19 bases. Primers that have a length of less than 18 bases will have lower specificity and will easily cause mispriming [2].</span></p>
<p><span class="font2">Percent (%) of GC of forward and reverse primer of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene show the same amount that is 57%. The GC percentage has met the primer general requirements of 40% to 60% [7]. If a primer has a low GC level, the primer will not be able to attach effectively to the target so that there will be a decrease in the efficiency of the PCR process. In the other hand, the high level of GC will complic ate the separation of double helix chains on primers and templates [2].</span></p>
<p><span class="font2">Forward and reverse primers of the </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene have the same Tm (melting temperature) of 64 °C. Tm is the temperature at which 50% of the DNA's double helix has been separated. Tm in a pair of primer of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene has met the criteria ranging from 50-65 ° C and do not have a gap of Tm between one primer with the others [2]. A pair of primer should not have a high gap of Tm. The gap recommended is not more than 5°C, if it is higher, it will cause a decrease in the amplification process or even the amplification process can be performed. It also aims to facilitate the determination of annealing temperatures in the multiplex-PCR process [7].</span></p>
<p><span class="font2">The pair of primer generated from the design has a dimer at the tip of 3’ as much as 2 pb on each forward and reverse primer, while dimer-any presence as much as 2 pb on forward primer and 3 pb on reverse primer. It shows that primer generated from the design still meets the criteria of Clone Manager suite 6 program which has a tolerance limits of dimer as much as &lt;3 pb at the tip of 3’ and &lt;7pb other than 3’ area.</span></p>
<p><span class="font2">The designed primer has a stability of 2.5 kcals and 2.3 kcals for forward and reverse primers respectively. So the stability of the two primer design has met the criteria of Clone Manager Suite 6 that is more than or equal to 1.2 kcals. This figure indicates the stability of 5 bases (pentamer) of the tip of 5 ' is greater than at the tip of 3'.</span></p>
<p><span class="font2">Runs and repeats that are allowed in the Clone Manager Suite 6 program are less than 3. In the primer generated from the design, both primers meet the allowed runs and repeats criteria. Repetition of both nucleotide and dinucleotide bases that is more than 3 will increase the possibility of false priming [6].</span></p>
<p><span class="font2">Hairpins condition was not found in the primer design so it has met the criteria of a good primer based on Clone Manager Suite 6. Hairpins is a</span></p>
<p><span class="font2">condition which the tip of primers complement each other [3].</span></p>
<p><span class="font2">The last important criterion is false priming. When the presence possibility of false priming is not found in the pair of primer generated from the design, the primer design has met the Clone Manager Suite 6 criteria. False priming is not allowed to occur on primers. The existence of false priming will result in an amplification error beyond the annealing temperature so that the desired PCR product is not formed [1].</span></p>
<p><span class="font2">Based on literature and criteria reference of Clone Manager Suite 6, it can be concluded that a pair of primer design for </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene amplification has met the criteria of a good primer. Therefore, the primer can be used to amplify the </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene sequence so that it is able to produce the desired PCR product.</span></p>
<p><span class="font2">Table 1. The result of Primer Design of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene</span></p>
<p><span class="font2">based on the reference in </span><span class="font2" style="font-style:italic;">Clone Manager Suite</span><span class="font2"> 6</span></p>
<table border="1">
<tr><td style="vertical-align:top;">
<p><span class="font2">Criteria</span></p></td><td style="vertical-align:top;">
<p><span class="font2">Primer</span></p>
<p><span class="font2">For-ward</span></p></td><td style="vertical-align:top;">
<p><span class="font2">Primer</span></p>
<p><span class="font2">Reverse</span></p></td><td style="vertical-align:top;">
<p><span class="font2">References to Clone Manager Suite 6</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2">Long</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">19</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">19</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">18-22</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2">primer (basa) Percent</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">57</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">57</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">50-60</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2">(%) GC Tm (°C)</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">64</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">64</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">55-80</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font2">Dimer on</span></p></td><td style="vertical-align:top;">
<p><span class="font2">2</span></p></td><td style="vertical-align:top;">
<p><span class="font2">2</span></p></td><td style="vertical-align:top;">
<p><span class="font2">&lt;3</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2">tip 3’ Dimer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">3</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">&lt;7</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2">besides the tip 3’</span></p>
<p><span class="font2">Stabili-ty</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">2.5</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">2.3</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">≥1,2</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font2">(kcals) </span><span class="font2" style="font-style:italic;">Run</span><span class="font2">s</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">&lt;3</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font2">(nukleoti da) </span><span class="font2" style="font-style:italic;">Repeats</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">&lt;3</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font2">(dinukleo tida) </span><span class="font2" style="font-style:italic;">Hairpins</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">-</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2" style="font-style:italic;">False</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2">-</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2" style="font-style:italic;">priming</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
</table>
<ul style="list-style:none;"><li>
<p><span class="font2">3.2 &nbsp;&nbsp;&nbsp;Optimization of Annealing Temperature and Amplification of the Fragment of </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B </span><span class="font2" style="font-style:italic;">M. Tuberculosis</span><span class="font2"> Gene Isolate P010 Using PCR Technique</span></p></li></ul>
<p><span class="font2">Annealing temperature is temperature required for the primer to attach to the target DNA. The low</span></p>
<p><span class="font2">annealing temperature of the optimum temperature will result in false priming and if the annealing temperature used is higher than the optimum temperature, it will cause the primer to be unable to attach to the target DNA so that the expected PCR product will not be formed, in other words the PCR process is not successful [2]. Threfore, it is necessary to process annealing temperature optimization so that the process of PCR will work properly and produce the product which is appropriate to the target.</span></p>
<p><span class="font2">Optimization of PCR process on the annealing temperature in this study used 56ºC, 58 ºC dan 60 ºC. Temperature variations are performed with the aim of obtaining the most optimum temperature for the PCR process so that the amplification process will also work as it is planned. PCR in this study was conditioned at 95 ° C for 15 minutes for predenaturation, followed by 45 cycles of amplification with denaturation at 94 ºC for 1 minute, annealing temperature variation 56 ºC, 58 ºC dan 60 ºC for 1 minute 20 seconds, elongation at 72 ºC for 2 minutes and final elongation process at 72 ºC for 10 minutes. From annealing temperature optimization, it is obtained amplicon visualized by the process of electrophoresis agarose 1.5% w / v.</span></p>
<p><span class="font0">0</span><span class="font1">,5 kb</span></p><img src="https://jurnal.harianregional.com/media/47788-1.jpg" alt="" style="width:167pt;height:239pt;">
<p><span class="font2">Figure 1. Electroforegram amplification </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene fragment isolate P010 on UV- transluminator; </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B= product </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gen; M= Marker DNA </span><span class="font2" style="font-style:italic;">ladder</span><span class="font2"> 100 bp (0,1 kb)</span></p>
<p><span class="font2">3 bands with different optimization temperatures show the same result, just as bright and bold. The number of products produced is also</span></p>
<p><span class="font2">suitably seen on marker ±485 bp. If the resulting product is as good at two different temperatures, in this case there is no secondary hybridization and the DNA bands obtained are sufficiently thick, then a lower annealing temperature will be selected [1]. This consideration is caused by high annealing temperatures that can inhibit primary hybridization with DNA templates resulting in fewer PCR products so that temperatures 56 ºC can be used for </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene amplification with primer design results. So that the PCR process is conducted at 56 ºC annealing temperature according to the Clone Manager Suite 6 annealing temperature criteria is 56 ºC.</span></p>
<p><span class="font2">After optimizing the annealing temperature and amplification using the PCR method on the P010 isolate using a pair of primer design with Clone Manager Suite 6 that is a forward primer with a nucleotide base sequence:</span></p>
<p><span class="font2">5'-GAGAGTTGGTGCGGCGTAA-3' and reverse primer with nucleotide base sequence:</span></p>
<p><span class="font2">5'-GCGGTCGGAGTATGCGAAT-3' proved to be able to amplify the </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene in the P010 isolate used as a template.</span></p>
<p><span class="font2">It can be seen by detecting the PCR products performed using electrophoresis method on 1.5% agarose gel seen on UV Transilluminator. The electrophoresis results showed that PCR products formed correctly were products of ±485 bp in accordance with the size of the </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">IV. &nbsp;&nbsp;&nbsp;CONCLUSION</span></p></li></ul>
<p><span class="font2">In silico primer design produced primers which had good criteria as it had met the criteria such as primer length, %GC, Tm, </span><span class="font2" style="font-style:italic;">dimer</span><span class="font2">, </span><span class="font2" style="font-style:italic;">hairpins</span><span class="font2">, primer stability and </span><span class="font2" style="font-style:italic;">false priming</span><span class="font2">. A pair of primer obtained were primer forward: 5’-GAGAGTTGGTGCGGCGTAA-3’ and primer reverse:</span></p>
<p><span class="font2">5’-GAGAGTTGGTGCGGCGTAA-3’. It was proved that the pair of primer produced by in vitro were able to amplify </span><span class="font2" style="font-style:italic;">gyr</span><span class="font2">B gene fragment and produced amplicon with the size of ±485 bp after detecting using electrophoresis method and was seen using UV Transilluminator with temperature optimization results of 56 ºC.</span></p>
<p><span class="font2">ACKNOWLEDGEMENT</span></p>
<p><span class="font2">I would like to thank everyone who has helped me either directly or indirectly in order to conduct this study well.</span></p>
<p><span class="font2">REFERENCES</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">[1] &nbsp;&nbsp;&nbsp;Borah, P, 2011. Primer Designing for PCR. </span><span class="font2" style="font-style:italic;">Science Vision.</span><span class="font2"> 11(3): 134-136.</span></p></li>
<li>
<p><span class="font2">[2] &nbsp;&nbsp;&nbsp;Handoyo, D. dan A. Rudiretna, 2000. Prinsip Umum Dan Pelaksanaan Polymerase Chain Reaction </span><span class="font2" style="font-style:italic;">(PCR)</span><span class="font2"> (General Principles and Implementation Of Polymerase Chain Reaction). </span><span class="font2" style="font-style:italic;">Unitas</span><span class="font2">. 9 (1) : 17-29.</span></p></li>
<li>
<p><span class="font2">[3] &nbsp;&nbsp;&nbsp;Judelson, H. 2006. Guidelines For Designing Primers. </span><span class="font2" style="font-style:italic;">Primer Guidelines</span><span class="font2">. 10 (6) : 1-5.</span></p></li>
<li>
<p><span class="font2">[4] &nbsp;&nbsp;&nbsp;Li, J., X. Gao, T. Luo, J. Wu, G. Sun, Q, Liu, Y. Jiang, Y. Zhang, J. Mei and Q. Gao, 2014. Associaton of gyrA/B mutations and resistance levels to fluoroquinolones in clinical isolates of Mycobacterium tuberculosis. Emerging Microbes and Infections.</span></p></li>
<li>
<p><span class="font2">[5] &nbsp;&nbsp;&nbsp;Palomino, J. C. and A. Martin. 2014. Drug Resistance Mechanisms in </span><span class="font2" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font2">. </span><span class="font2" style="font-style:italic;">MDPI</span><span class="font2">. Vol 3 : 317-340.</span></p></li>
<li>
<p><span class="font2">[6] &nbsp;&nbsp;&nbsp;Premier Biosoft International (PBI). 2009. </span><span class="font2" style="font-style:italic;">Net Primer &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Manual</span><span class="font2">. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Available at:</span></p></li></ul>
<p><a href="http://www.PremierBiosoft.com"><span class="font2" style="text-decoration:underline;">http://www.PremierBiosoft.com</span></a><span class="font2">. (Cited December 18, 2014).</span></p>
<ul style="list-style:none;"><li>
<p><span class="font2">[7] &nbsp;&nbsp;&nbsp;Sasmito, D. E.K., Rahadian, K., Izzati, Muhimmah. 2014. </span><span class="font2" style="font-style:italic;">Karakteristik Primer pada Polymerase Chain Reaction (PCR) untuk Sekuensing DNA: Mini Review</span><span class="font2">, Seminar Nasional Informatika Medis (SNIMed) V: Yogyakarta.</span></p></li>
<li>
<p><span class="font2">[8] &nbsp;&nbsp;&nbsp;World Health Organization (WHO). 2016. </span><span class="font2" style="font-style:italic;">Global Tuberculosis Report 2016</span><span class="font2">. Perancis: World Health Organization.</span></p></li>
<li>
<p><span class="font2">[9] &nbsp;&nbsp;&nbsp;World Health Organization (WHO). 2017. </span><span class="font2" style="font-style:italic;">Global Tuberculosis &nbsp;&nbsp;&nbsp;&nbsp;Report</span><span class="font2">. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Available at:</span></p></li></ul>
<p><a href="http://www.who.int/tb/publications/global_report/"><span class="font2">http://www.who.int/tb/publications/global_report/</span></a><span class="font2"> en/. (Cited December 20, 2017).</span></p>
<p><span class="font3">8</span></p>