---
layout: full_article
title: "DNA Probe Design for Detection Mutation at Codon 315 In katG Gene of Mycobacterium Tuberculosis to Real-Time Polymerase Chain Reaction"
author: "I Gusti A. A. Santhi Rahmaryani, Ni Kadek Ariani, Dyah Subadrika Warma Dewi, Ni Komang Sasi Ani, Ade Ari Sundari, Kadek Widya Yuli Hartati, Sagung Chandra Yowani"
categories: jhsm
canonical_url: https://jurnal.harianregional.com/jhsm/full-36991 
citation_abstract_html_url: "https://jurnal.harianregional.com/jhsm/id-36991"
citation_pdf_url: "https://jurnal.harianregional.com/jhsm/full-36991"  
comments: true
---

<p><span class="font6">Journal of Health Sciences and Medicine, Vol. 1 No. 2, September 2017</span></p>
<p><span class="font6">31</span></p>
<p><span class="font7" style="font-weight:bold;">DNA Probe Design for Detection Mutation at Codon 315 In </span><span class="font7" style="font-weight:bold;font-style:italic;">katG Gene</span><span class="font7" style="font-weight:bold;"> of </span><span class="font7" style="font-weight:bold;font-style:italic;">Mycobacterium Tuberculosis</span><span class="font7" style="font-weight:bold;"> to Real-Time Polymerase Chain Reaction</span></p>
<p><span class="font6">I Gusti A. A. Santhi Rahmaryani<sup>1</sup>, Ni Kadek Ariani<sup>1</sup>, Dyah Subadrika Warma Dewi<sup>1</sup>, Ni Komang Sasi Ani<sup>1</sup>, Ade Ari Sundari<sup>1</sup>, Kadek Widya Yuli Hartati<sup>1</sup>, and Sagung Chandra Yowani<sup>2</sup></span></p>
<ul style="list-style:none;"><li>
<p><span class="font4"><sup>1</sup></span><span class="font6">Department of Pharmacy, Faculty of Mathematics and Natural Science, Udayana University, Bali, Indonesia Email: </span><a href="mailto:agayusanthi@gmail.com"><span class="font6">agayusanthi@gmail.com</span></a></p></li></ul>
<ul style="list-style:none;"><li>
<p><span class="font6"><sup>2</sup> MDR-TB &amp;&nbsp;XDR-TB Research Group</span></p></li></ul>
<p><span class="font6">Faculty of Mathematics and Natural Science, Udayana University, Bali, Indonesia Email : </span><a href="mailto:cyowani@gmail.com"><span class="font6">cyowani@gmail.com</span></a></p>
<p><span class="font6" style="font-weight:bold;font-style:italic;">Abstract:</span></p>
<p><span class="font6">High-level resistance to isoniazid as a first-line tuberculosis drugs can be caused by mutations in codon 315 </span><span class="font6" style="font-style:italic;">katG Mycobacterium tuberculosis</span><span class="font6">. Mutation at codon 315 is the most frequent mutation with the highest amino acid variation, compared to other codons in the </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis katG</span><span class="font6"> gene. Therefore, a specific </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> is required for rapid and proper detection of mutations at codon 315. In this study, the design of a nucleotide sequence probe with TaqMan labeling was performed using </span><span class="font6" style="font-style:italic;">Clone Manager Suite 6 software</span><span class="font6">. The mutant </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> obtained was analysed in two stages. The initial analysis is based on the length of the </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> (22-30 bases), Tm (70ºC), %GC (35-65%), not in </span><span class="font6" style="font-style:italic;">hairpin</span><span class="font6"> form, </span><span class="font6" style="font-style:italic;">dimer</span><span class="font6"> (&lt; 5 bases), </span><span class="font6" style="font-style:italic;">runs</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">repeat</span><span class="font6"> (≤ 4 for base A, T, C, and &lt;&nbsp;3 for base G). Furthermore the final analysis was carried out with no G base in 2 bases at the end of the 5’ </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> and the amount of base C ≥ G.</span></p>
<p><span class="font6">The study resulted in 260 probe mutants. After the initial analysis, 11 mutant probes were obtained to recognize mutations in the codon of 315 </span><span class="font6" style="font-style:italic;">katG Mycobacterioum tuberculosis</span><span class="font6"> genes. </span><span class="font6" style="font-style:italic;">The probe</span><span class="font6"> consists of 2 </span><span class="font6" style="font-style:italic;">probes</span><span class="font6"> for the S315T mutation, 6 </span><span class="font6" style="font-style:italic;">probes</span><span class="font6"> for S315N mutation, and 3 </span><span class="font6" style="font-style:italic;">probes</span><span class="font6"> for S315V mutation. The criteria of the 11 mutant </span><span class="font6" style="font-style:italic;">probes</span><span class="font6"> are 22-23 bases long, Tm 70ºC, % GC 56-63%, 4 </span><span class="font6" style="font-style:italic;">dimer</span><span class="font6">, 2 </span><span class="font6" style="font-style:italic;">runs</span><span class="font6">, and does not have </span><span class="font6" style="font-style:italic;">repeats</span><span class="font6"> and does not form </span><span class="font6" style="font-style:italic;">hairpin</span><span class="font6"> at a temperature of 56ºC. Based on the final analysis, 3 mutant </span><span class="font6" style="font-style:italic;">probes</span><span class="font6"> were obtained fulfilling the TaqMan probe labeling criteria, namely K315MT1 for specific detection of mutation S→T and K315MN5, then K315MN23 for specific detection of mutation S→N.</span></p>
<p><span class="font6">The conclusion of this study shows that the best mutant nucleotide sequence probes for the detection of mutations at codon of 315 KatG </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> genes are 5’-FAM-CC </span><span class="font6" style="text-decoration:underline;">ACC </span><span class="font6">GGC ATC GAG GTC GTA TG-TAMRA-3’; 5’FAM-ATC ACC </span><span class="font6" style="text-decoration:underline;">AAC</span><span class="font6"> GGC ATC GAG GTC G-TAMRA-3’; dan 5’FAM-C ACC </span><span class="font6" style="text-decoration:underline;">AAC</span><span class="font6"> GGC ATC GAG GTC GTA T-TAMRA-3’. The design of the mutant probe according to the TaqMan probe criterion for real-time PCR was obtained by 3 probes from the 11 selected mutant probes in the initial analysis.</span></p>
<p><span class="font6" style="font-weight:bold;font-style:italic;">Keywords: </span><span class="font6" style="font-style:italic;">Tuberculosis, gene katG mutation, in silico, TaqMan probe, real-time PCR</span><span class="font6">.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">I. &nbsp;&nbsp;&nbsp;INTRODUCTION</span></p></li></ul>
<p><span class="font6">According to </span><span class="font6" style="font-style:italic;">World Health Organization</span><span class="font6"> survey data in 2014, Tuberculosis (TB) is the second leading cause of death in the world after </span><span class="font6" style="font-style:italic;">Acquired Immuno Deficiency Syndrome</span><span class="font6"> (1). TB control is increasingly difficult with the onset of Multi-drug Resistant Tuberculosis (MDR-TB), which is defined as tuberculosis caused by </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> resistance to at least two first-line antituberculosis drugs (OAT), such as isoniazid and rifampicin (2). Resistance to isoniazid is found in high frequency. According to research by Lisdawati et al., (2015) the percentage of resistance to isoniazid in Indonesia is 28.6% (3).</span></p>
<p><span class="font6">Resistance to isoniazid can be caused by mutations in some </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> genes, such as </span><span class="font6" style="font-style:italic;">katG </span><span class="font6">genes, </span><span class="font6" style="font-style:italic;">inhA</span><span class="font6"> (4), </span><span class="font6" style="font-style:italic;">ahpC</span><span class="font6">, </span><span class="font6" style="font-style:italic;">kasA</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">ndh</span><span class="font6"> (5). Mutations in the </span><span class="font6" style="font-style:italic;">katG</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">inhA</span><span class="font6"> genes can be used as </span><span class="font6" style="font-style:italic;">predictive markers</span><span class="font6"> to predict resistance to isoniazid. KatG gene is the most commonly mutated gene with the percentage of 78% (6). Therefore, the katG gene is the focus of this study.</span></p>
<p><span class="font6">Mutations in the katG gene resulted in amino acid changes of the catalase-peroxidase enzyme which decreased the activity of the catalase-peroxidase enzyme in converting isoniazid into its active form, so that isoniazid resistance (7,8) occured. The codon 315 in the most commonly mutated katG genes and amino acids formed by mutations in</span></p>
<p><span class="font6">codon 315 has the greatest variation among the codon of other katG genes.</span></p>
<p><span class="font6">Two studies conducted on MDR-TB isolates in Bali with an amplified region of 2437-3160 pb showed that, the amino acid variation of codon 315 was due to mutations of S315T and S315V (9.8). Other mutations in this codon that cause variations in amino acids, was shown in the study of Bostanabad </span><span class="font6" style="font-style:italic;">et al.</span><span class="font6"> (2008) to isolates in Belarus which include S315R, S315N, and S315G mutations. Mutation S315T has the highest prevalence. The research of Rintiswati et al. (2011) showed that the S315T mutation was found in 27.05% of sputum samples obtained from several laboratories in Indonesia. The other studies also showed that the frequency S315T in clinical isolates that were resistant to isoniazid was very high at 46-85% (10,11,12).</span></p>
<p><span class="font6">Detection of </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> which is resistant to isoniazid can be done by molecular method; one of them is </span><span class="font6" style="font-style:italic;">Real-Time Polymerase Chain Reaction</span><span class="font6"> (</span><span class="font6" style="font-style:italic;">real-time </span><span class="font6">PCR). </span><span class="font6" style="font-style:italic;">Real-Time</span><span class="font6"> PCR is a combination of PCR chemical methods with fluorescence molecules to monitor the amplification products in each PCR cycle. The </span><span class="font6" style="font-style:italic;">real-time </span><span class="font6">PCR detection system is divided into 2 fluorescence molecules that are interconnected in a DNA double strand, and oligonucleotides is labeled fluorescence compound called DNA </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> (13). The </span><span class="font6" style="font-style:italic;">real-time</span><span class="font6"> PCR method with DNA </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> is capable of providing sensitivity and selectivity equivalent to conventional PCR analysis methods combined with </span><span class="font6" style="font-style:italic;">southern blot</span><span class="font6"> (14). In addition, the fluorescence signal generated by the </span><span class="font6" style="font-style:italic;">primer dimer</span><span class="font6"> would not be detected with the use of DNA </span><span class="font6" style="font-style:italic;">probes.</span><span class="font6"> Therefore, the DNA probe is more specific than the intercalated fluorescence molecule in the DNA double strand (15).</span></p>
<p><span class="font6">There are several kinds of DNA probe detection systems used for real-time PCRs, one of which is TaqMan probe (13). </span><span class="font6" style="font-style:italic;">TaqMan probe</span><span class="font6"> is a type of hydrolysis probe since it involves the activity of exonuclease from </span><span class="font6" style="font-style:italic;">Taq</span><span class="font6"> DNA polymerase. The sequence of the nucleotide probe with two kinds of hybridized fluorescence labels will be degraded by the exonuclease activity resulting in a fluorescence signal (16). The advantage of </span><span class="font6" style="font-style:italic;">TaqMan probe</span><span class="font6"> is safer than </span><span class="font6" style="font-style:italic;">radioactive probe</span><span class="font6"> (17), the detection result can be known quickly (18) and it is specific. </span><span class="font6" style="font-style:italic;">TaqMan probe</span><span class="font6"> has a good specificity because the sequence of the nucleotide probe will be hybridized with the target sequence on the amplification product (19).</span></p>
<p><span class="font6">The design of the DNA probe is a critical stage for all detection stages based on the hybridization process to ensure the stickiness specificity and detection efficiency. The good result of detection with </span><span class="font6" style="font-style:italic;">DNA probe</span><span class="font6"> depends on the design criteria of </span><span class="font6" style="font-style:italic;">DNA probe</span><span class="font6"> used. Some of the required criteria are </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> length, guanine and cytosine base composition (G-C), mutation position, </span><span class="font6" style="font-style:italic;">hairpin</span><span class="font6"> formation, and availability of a single base or dinucleotide over a long range exceeding 4 (16). The sequence of the DNA </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> must also be specific to the target DNA sequence. Therefore, DNA </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> design is required to avoid cross hybridization in the sequence that has many equations (20).</span></p>
<p><span class="font6">The use of software in the design of nucleotide probes sequence can dynamically optimize the design results compared to the manual design. The design of nucleotide probes sequence with methods based on </span><span class="font6" style="font-style:italic;">trial and error</span><span class="font6"> in the laboratory, the availability of physical needs such as tools, reagents and the number of target cloning or nontarget microbial groups will result in optimization of a nucleotide probes sequence getting more complicated. DNA probe designs made with software can be used as a reference to make experimental DNA probes in the laboratory. In addition, DNA probe design using the software will be able to minimize the time, and maximize the probability of successful proliferation of DNA probe with target DNA sequence as an attempt to detect mutations in KatG </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> gene (21,22).</span></p>
<p><span class="font6">The study is designed a DNA probe to detect mutations in codon 315 on </span><span class="font6" style="font-style:italic;">katG Mycobacterium tuberculosis</span><span class="font6"> genes. The best DNA probes will be selected from several design results using the </span><span class="font6" style="font-style:italic;">Clone Manager Suite 6 </span><span class="font6">software and the design criteria are also matched against DNA </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> criteria based on the literature. The selected DNA probes will be applied for rapid detection of resistance to isoniazid using </span><span class="font6" style="font-style:italic;">real-time</span><span class="font6"> PCR method.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">II. &nbsp;&nbsp;&nbsp;MATERIAL AND METHODS</span></p></li></ul><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font6" style="font-weight:bold;"><a name="bookmark1"></a>MATERIAL</span></h1>
<p><span class="font6">The probe design requires nucleotide sequence of </span><span class="font6" style="font-style:italic;">katG Mycobacterium tuberculosis</span><span class="font6"> H37Rv (</span><span class="font6" style="font-style:italic;">wild-type</span><span class="font6">) genes from database URL://www.ncbi.nlm.nih.gov (</span><span class="font6" style="font-style:italic;">accession number</span><span class="font6">: X68081.1). A </span><span class="font6" style="font-style:italic;">primer</span><span class="font6"> pair for probe analysis consists of a forward </span><span class="font6" style="font-style:italic;">primer</span><span class="font6"> with a sequence of 5’-GAAGTACGGCAAGAAGCTCTC-3’ and a </span><span class="font6" style="font-style:italic;">reverse</span><span class="font6"> primer a sequence of 5’-CGATCTATGAGCGGATCACG-3’. The </span><span class="font6" style="font-style:italic;">primer</span><span class="font6"> is the best design </span><span class="font6" style="font-style:italic;">in silico</span><span class="font6"> and has been experimentally optimized for the detection of mutations in the </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis katG</span><span class="font6"> gene. The experimental optimization results showed that at an </span><span class="font6" style="font-style:italic;">annealing</span><span class="font6"> temperature of 56ºC, the </span><span class="font6" style="font-style:italic;">primers</span><span class="font6"> can constantly produce thicker DNA bands and do not form </span><span class="font6" style="font-style:italic;">dimers </span><span class="font6">(Deniariasih, 2013; Suryadi, 2013). In this DNA </span><span class="font6" style="font-style:italic;">probe </span><span class="font6">design study, we used long data and Tm </span><span class="font6" style="font-style:italic;">primer in silico</span><span class="font6"> as reference to determine the design criteria of DNA </span><span class="font6" style="font-style:italic;">probe</span><span class="font6">.</span></p>
<h1><a name="bookmark2"></a><span class="font6" style="font-weight:bold;"><a name="bookmark3"></a>TOOLS</span></h1>
<p><span class="font6">The equipment to complete the research is a set of laptops with </span><span class="font6" style="font-style:italic;">Windows</span><span class="font6"> 10 32 </span><span class="font6" style="font-style:italic;">bit</span><span class="font6"> specifications, modems, and </span><span class="font6" style="font-style:italic;">Software Clone Manager Suite 6</span><span class="font6"> (</span><span class="font6" style="font-style:italic;">University of Groningen) </span><span class="font6">for designing and analyzing DNA </span><span class="font6" style="font-style:italic;">probes.</span></p>
<h1><a name="bookmark4"></a><span class="font6" style="font-weight:bold;"><a name="bookmark5"></a>STEPS</span></h1>
<h1><a name="bookmark6"></a><span class="font6" style="font-weight:bold;"><a name="bookmark7"></a>Sequence of Nucleotide Targets for Designing Probe DNA Probes</span></h1>
<p><span class="font6">The design of the </span><span class="font6" style="font-style:italic;">prob</span><span class="font6">e DNA uses the whole sequence of the genotype katG gene composed of 4810 pb with the starting codon at 1979 pb. DNA probes will be designed inside the amplification area at position 2437-3160 pb </span><span class="font6" style="font-style:italic;">katG Mycobacterium tuberculosis</span><span class="font6"> genes. In Figure 1, it</span></p>
<p><span class="font6">can be seen that the amplification region is a nucleotide sequence in the light blue line, numbered based on the total number of </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis katG</span><span class="font6"> genes. The amplification areas are limited by a pair of primers marked in green and include the position of mutations in the </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> katG gene based on Deniariasih's (2013) and Suryadi (2013) studies. One of the most commonly coded codons is the 315 codon (AGC). Therefore, codon 315 will serve as a specific design target of the mutant probe. The position of codon 315 is indicated by yellow in Figure 1.</span></p><img src="https://jurnal.harianregional.com/media/36991-1.jpg" alt="" style="width:204pt;height:133pt;">
<p><span class="font6">Figure 1. Nucleotide sequence of </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> katG gene targeted for DNA probe design (8).</span></p>
<p><span class="font6">Mutations in codon 315 are found in several variations, namely S315T, S315N, S315R, S315G, and S315V. This mutation results in a nucleotide change of the 315 codon. Therefore, the design of the DNA sequence of the mutant </span><span class="font6" style="font-style:italic;">probe</span><span class="font6"> refers to the nucleotide change of the codon 315. The nucleotide changes to codon 315 and its positions are listed in Table 1.</span></p>
<p><span class="font6">Table 1. Changes of Nukleotida Kodon 315 </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis katG Genes</span><span class="font6"> for DNA Probe Design Mutants</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:middle;">
<p><span class="font5">No.</span></p></td><td rowspan="2" style="vertical-align:top;">
<p><span class="font5" style="font-style:italic;">Probe Name</span></p></td><td rowspan="2" style="vertical-align:top;">
<p><span class="font5">Codon</span></p>
<p><span class="font5">315</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font5">Mutation</span></p></td><td colspan="2" style="vertical-align:top;">
<p><span class="font5">Nucleotides</span></p></td><td rowspan="2" style="vertical-align:top;">
<p><span class="font5">Position from codon </span><span class="font5" style="font-style:italic;">start</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font5" style="font-style:italic;">wild-iype</span></p></td><td style="vertical-align:top;">
<p><span class="font5">Jstoaa</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font5">1.</span></p></td><td style="vertical-align:top;">
<p><span class="font5">K315MT</span></p></td><td rowspan="5" style="vertical-align:middle;">
<p><span class="font5">AGC</span></p></td><td style="vertical-align:top;">
<p><span class="font5">S315T</span></p></td><td style="vertical-align:top;">
<p><span class="font5">G</span></p></td><td style="vertical-align:top;">
<p><span class="font5">C</span></p></td><td style="vertical-align:top;">
<p><span class="font5">944 ⅛</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font5">2.</span></p></td><td style="vertical-align:top;">
<p><span class="font5">K315MN</span></p></td><td style="vertical-align:top;">
<p><span class="font5">S315N</span></p></td><td style="vertical-align:top;">
<p><span class="font5">G</span></p></td><td style="vertical-align:top;">
<p><span class="font5">A</span></p></td><td style="vertical-align:top;">
<p><span class="font5">944 ⅛</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font5">3.</span></p></td><td style="vertical-align:top;">
<p><span class="font5">K315MR</span></p></td><td style="vertical-align:top;">
<p><span class="font5">S315R</span></p></td><td style="vertical-align:top;">
<p><span class="font5">C</span></p></td><td style="vertical-align:top;">
<p><span class="font5">G</span></p></td><td style="vertical-align:top;">
<p><span class="font5">94: ⅛</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font5">4.</span></p></td><td style="vertical-align:top;">
<p><span class="font5">K315MG</span></p></td><td style="vertical-align:top;">
<p><span class="font5">S315G</span></p></td><td style="vertical-align:top;">
<p><span class="font5">A</span></p></td><td style="vertical-align:top;">
<p><span class="font5">G</span></p></td><td style="vertical-align:top;">
<p><span class="font5">943 <sub>ε</sub>⅛</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font5">5.</span></p></td><td style="vertical-align:top;">
<p><span class="font5">K315MV</span></p></td><td style="vertical-align:top;">
<p><span class="font5">S315V</span></p></td><td style="vertical-align:top;">
<p><span class="font5">AG</span></p></td><td style="vertical-align:top;">
<p><span class="font5">GT</span></p></td><td style="vertical-align:top;">
<p><span class="font5">943, 944 Eh</span></p></td></tr>
</table>
<p><span class="font6">Information: </span><span class="font6" style="font-style:italic;">Probe</span><span class="font6"> name, e.g. K315MT: K = denotes </span><span class="font6" style="font-style:italic;">M. tuberculosis katG genes</span><span class="font6">; 315 = denotes the targen codon; M = denotes the type of probe design is a mutant probe; T = amino acids formed by mutations at codon 315</span></p>
<h1><a name="bookmark8"></a><span class="font6" style="font-weight:bold;"><a name="bookmark9"></a>Designing DNA </span><span class="font6" style="font-weight:bold;font-style:italic;">Probe</span></h1>
<p><span class="font6">The design of DNA probes was conducted in silico by software Clone Manager Suite 6. The first step is to download the nucleotide sequence of </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> H37Rv (wild-type) with accession number: X68081.1 in FASTA data format and saved in the </span><span class="font6" style="font-style:italic;">notepad </span><span class="font6">as a </span><span class="font6" style="font-style:italic;">template</span><span class="font6">. Next opened the Clone Manager Suite 6 program and clicked the primary menu, select the design, and then entered the data sequence nucleotide katG wildtype gene that has been stored in notepad and clicked OK. In the primary type, the probe is selected, then the length of the probe is varied from 22-30 bases in the length section and set the target range range including the mutation at codon</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">315, &nbsp;&nbsp;&nbsp;then clicked OK. Furthermore, some DNA probe designs will appear, then clicked primer report to display the nucleotide sequence of the probe. Nucleotide sequences that include the position of the mutations were screened. At this stage, some wild-type probe DNA designs will be produced. One base on the wild-type probe was subsequently replaced by mutation at codon 315 (Table 1) in order to obtain sequences of mutant probe nucleotide for the detection of mutations in the 315 codon </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis katG</span><span class="font6"> genes.</span></p></li></ul>
<ul style="list-style:none;"><li>
<p><span class="font6">III. &nbsp;&nbsp;&nbsp;RESULTS AND DISCUSSION</span></p></li></ul>
<h1><a name="bookmark10"></a><span class="font6" style="font-weight:bold;"><a name="bookmark11"></a>The Sequences of Nucleotide Target for Designing Probe DNA Probes</span></h1>
<p><span class="font6">The codon 315 was the target of the nucleotide sequence design of the mutant probes in this study. Mutations in the codon of 315 </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis </span><span class="font6">katG genes are one of the mutations that cause resistance to isoniazid. The resulting resistance level is high enough to cause the MIC of isoniazid to &gt;&nbsp;5 µg/ml (23). The prevalence and variation of amino acids from this codon mutation is also the highest compared to other codons in the </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> katG gene, such as codons 309, 316, and 357 (11). The prevalence of mutations at codon 315 reached 74% (12). Meanwhile, amino acid variations are formed due to several mutations at codon 315 with different prevalence, e.g. S315T 36-85%, S315N 1,9-5,7%, S315R 0,9-2,4%, S315G 1,1-2,4%, and S315V 1% (10, 11, 24, 9).</span></p>
<p><span class="font6">The nucleotide sequence used to design the mutant probe is the overall wild-type nucleotide sequence of the </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis katG</span><span class="font6"> genes. This wild-type nucleotide sequence is obtained online from genbank code X68081.1. </span><span class="font6" style="font-style:italic;">Genbank</span><span class="font6"> is a public database managed by the </span><span class="font6" style="font-style:italic;">National Center for Biotechnology Information (NCBI</span><span class="font6">). A number of online services to identify, align, and compare sequences are also provided by NCBI (25). In the previous study, the overall sequence of nucleotides of this katG gene has been used in the primary design for the detection of mutations in TB isolates in Bali with PCR. Based on these studies, two isolates were found to have mutations at codon 315 (8, 9).</span></p>
<h1><a name="bookmark12"></a><span class="font6" style="font-weight:bold;"><a name="bookmark13"></a>Results of DNA Probe Design</span></h1>
<p><span class="font6">The probe is a complementary nucleotide sequence with a nucleotide sequence on the target such as a primer. However, the probe does not limit the PCR amplification region and can not be prolonged as it has been phosphorylated at the 3 'end. In addition, the probe can also be used for detection because it is labeled with a pair of dyes in the form of a reporter and quencher at both ends and will be hydrolyzed when the elongation process proceeds (26).</span></p>
<p><span class="font6">The design of the mutant probe DNA is designed using Clone Manager Suite 6 Software. The probes can be generated with initial criteria, ie GC and Tm values, dimer, runs, repeats, and hairpin. Initial criteria software is not absolute, so it can be tailored to the needs. The design of</span></p>
<p><span class="font6">nucleotide sequence probe using this software has not been done. In previous study, this software has been used to design a pair of primers for the detection of mutations in the </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> katG gene (9, 8). Therefore, in this study, it is proved that Clone Manager Suite 6 Software can generate a TaqMan nucleotide probe sequence. This research is made easier because of the availability of this software along with the license code for software activation. Some other software for probe design such as </span><span class="font6" style="font-style:italic;">Alleleid, Beacon Designer, Oligo7,</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">Probe Maker</span><span class="font6"> are not used in this study because the license code of the software is inaccessible to the researcher, so it can not be used completely.</span></p>
<p><span class="font6">The designed mutant probe is a </span><span class="font6" style="font-style:italic;">TaqMan probe.</span><span class="font6"> This type of probe has a simple design and the fluorescence can be determined at the end of the extension phase (13). The TaqMan probe design is a linear oligonucleotide arrangement labeled by reporters at the 5 'end and quencher at the 3' end (17). TaqMan probe is specific so it can be used to detect a base change in the target nucleotide sequence (27). The fluorescence signal will increase in proportion to the increasing distance between the reporter and the quencher of the hybridized probe. The increase in the distance of the two labels occurs by probe hydrolysis by the 5 '→ 3' exonuclease activity of Taq polymerase enzyme during PCR (15).</span></p>
<p><span class="font6">The design of a mutant probe with Clone Manager Suite 6 software begins with the design of a wild-type probe. The resulting wild-type probe design results do not specifically include the desired target mutation area. Therefore, selection based on target area coverage was performed to obtain a wild-type probe design covering the range of mutation areas at 943, 944, and 945 pb position of the start codon (or position 2921, 2922, and 2923 pb of the whole katG gene). Screening of the mutation position against the wild-type probe design is also performed. The position of the mutation affects the hybridisation of the nucleotide sequence of the probe. The probe hybridization with mismacth in the center of the nucleotide sequence of the probe is more unstable than the probe hybridization with a mismatch near the probe tip 28. In the event of a mismatch between the probe and the target, the probe hybridization efficiency will decrease. In this condition, Taq polymerase enzyme will lengthen the probe target without degrading the probe, so the reporter fluorescence is undetectable (29, 30). The mismatch relationship with hybridization is shown in Figure 2. Detection of allele 1 (black mutation position) using a probe with a difference in one type of nucleotide (white mutation position in the center of the probe) resulted in a mismatch. The probe-target hybridization becomes unstable under mismatch conditions. Therefore, probe degradation cannot occur and FAM fluorescence signals (reporter) are not generated.</span></p><img src="https://jurnal.harianregional.com/media/36991-2.jpg" alt="" style="width:202pt;height:96pt;">
<p><span class="font6">Figure 2. Effect of Mutation Position on Probe Hybridization</span></p>
<p><span class="font6">Caption:(Part A) In the DNA amplification process, primers and probes will hybridize with their complementary nucleotide sequence on the target DNA; (Part B) The presence of a mismatch causes the probe not to be degraded so that the fluorescence signal is not formed (Anonymous d, 2001)</span></p>
<p><span class="font6">The selected wild-type probe design is then converted to mutant probe design via nucleotide base replacement in the direct entry menu. Replacement is made to obtain a specific mutant probe design for nucleotide change at codon 315. Each mutant probe made is marked by name. For example for mutant probe 1 with mutation S315T is named K315MT1. K denotes the KatG </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis </span><span class="font6">genes, 315 is the target codon, M denotes the mutant probe, T denotes the name of the amino acid due to mutation and the number 1 is the serial number of the mutant probe design. There are 260 probes mutant probes generated from the design process in silico. The design of the mutant probe consists of probes for several mutations at codon 315, such as the S315T, S315N, S315R, S315G, and S315V mutation.</span></p>
<h1><a name="bookmark14"></a><span class="font6" style="font-weight:bold;"><a name="bookmark15"></a>Result of Mutable Probe Design Analysis Initial Analysis Result of Mutan Probe</span></h1>
<p><span class="font6">The analysis process was conducted in two stages consisting of initial analysis and final analysis. Gradual analysis is intended to maximize the selection process of probe designs in silico. An initial analysis was performed to obtain a mutant probe design that meets the general criteria of the probe. Analyze process used direct entry menu. Designs that meet all general criteria of the probe will be displayed with a check mark (√) on the software, whereas if any of the criteria does not match, then the results of the analysis will be displayed with a cross (×). An example of the analysis results was shown with the software in Figure 3. Figure A shows the results of initial analysis of K315MN23 probe met probe criteria, whereas in figure B the results of K315MN30 probe analysis did not meet the probe Tm criteria. In addition to the results also displayed the sequence of nucleotides of each mutant probe.</span></p><img src="https://jurnal.harianregional.com/media/36991-3.jpg" alt="" style="width:224pt;height:99pt;">
<p><span class="font6">Figure 3. Example display of early stage analysis results in software</span></p>
<p><span class="font6">Caption: (A) The nucleotide sequence of the K315MN23 probe meets the criteria of the initial probe analysis of the software; (B) The nucleotide sequence of the K315MN30 probe does not meet the Tm criteria of the software</span></p>
<p><span class="font6">Some of the criteria for initial analysis of the probe are based on the primary criterion used (15). The primary criteria used as a reference in this study are the length and value of primary Tm to determine the length and value of Tm probe. This is necessary to prevent non-specific attachment of primers or probes as they are in the same reaction during PCR (31). The initial criteria of the analysis included the length of the 22-30 base probe, the Tm 70ºC value, the GC content of 35-65%, the dimer less than 5 bases, the hairpin was not formed, and the number of runs and repeats was not more than 4 for bases other than guanine base. Special runs and guanine base repeats should be less than 3 (15, 16).</span></p>
<p><span class="font6">The results of the initial analysis of the mutant probe obtained were 11 mutant probes for mutation detection at codon 315. The criteria and sequence of mutant probes were shown in Table 2. All the mutant probes (Table 2) had variations in lengths from 22-23 bases. The overall design meets the length of TaqMan probe criteria. Generally TaqMan probes are designed with lengths that exceed the length of primary amplification. The maximum length of a good TaqMan probe is 30 bases (16). The use of probes of length exceeding the primer is intended to obtain a higher probe Tm than the primary Tm (27). This is intended to prevent inefficient degradation of the probe. One of the reasons is that the upstream primer undergoes an extension process before the probe hybridizes with target (32). The longer the probe size, the time required for annealing will be longer (33). The specificity of PCR can also be affected by probe length. Probes that are too long are less specific to distinguish mismatch. This can lead to false-positive results, especially when the difference between 2 or more probes used lies solely on single nucleotide (27). Conversely, the shorter the probe, the annealing time is shorter, but the attachment of the probe to the undesirable area (mispriming) will increase (34).</span></p>
<p><span class="font6">Table 2. Results of Initial Analysis of Mutant Probe For Codon 315</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:middle;">
<p><span class="font3" style="font-style:italic;">Probe </span><span class="font0" style="font-style:italic;">∖</span><span class="font3" style="font-style:italic;">'amβ</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font3">Nucleotide sequence of JVafee 5—»3’</span></p></td><td colspan="7" style="vertical-align:top;">
<p><span class="font3">DNA Probe Design Criteria ⅛ </span><span class="font3" style="font-variant:small-caps;">.⅛⅛⅛j?</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">Ejg</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">Tm</span></p></td><td style="vertical-align:top;">
<p><span class="font3">%</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">Dimei-</span></p></td><td style="vertical-align:middle;">
<p><span class="font3" style="font-style:italic;">Jiabpin</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">raws</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">&gt; spear</span></p></td></tr>
<tr><td colspan="9" style="vertical-align:top;">
<p><span class="font3">Probe design of specific Kiidation. S315T</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MTI</span></p></td><td style="vertical-align:top;">
<p><span class="font3">CCACOGGCATCGAGGrCGTATG</span></p></td><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">22</span></p></td><td style="vertical-align:top;">
<p><span class="font3">70</span></p></td><td style="vertical-align:top;">
<p><span class="font3">63</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MT4</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Gatcaccaccggcatcgaggtc</span></p></td><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">ZZ</span></p></td><td style="vertical-align:top;">
<p><span class="font3"><sup>,</sup>iθ</span></p></td><td style="vertical-align:top;">
<p><span class="font3">63</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td colspan="9" style="vertical-align:top;">
<p><span class="font3">Frcbe design of specific mutation S315N</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN3</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Cgatcaccaacggcatcgaggt</span></p></td><td style="vertical-align:top;">
<p><span class="font3">S</span></p></td><td style="vertical-align:top;">
<p><span class="font3">.<sup>l</sup>0</span></p></td><td style="vertical-align:top;">
<p><span class="font3">59</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;">
<p><span class="font3">Z</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN5</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Atcaccaacggcatcgaggtcg</span></p></td><td style="vertical-align:top;">
<p><span class="font3">22</span></p></td><td style="vertical-align:top;">
<p><span class="font3"><sup>,</sup><sub>1</sub>'0</span></p></td><td style="vertical-align:top;">
<p><span class="font3">59</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;">
<p><span class="font3">Z</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN16</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Ccaacggcatcgaggtcgtatgg</span></p></td><td style="vertical-align:top;">
<p><span class="font3">23</span></p></td><td style="vertical-align:top;">
<p><span class="font3">70</span></p></td><td style="vertical-align:top;">
<p><span class="font3">60</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;">
<p><span class="font3">Z</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN17</span></p></td><td style="vertical-align:top;">
<p><span class="font3">CAACGGCATCGAGGrCGTATGGA</span></p></td><td style="vertical-align:top;">
<p><span class="font3">23</span></p></td><td style="vertical-align:top;">
<p><span class="font3">70</span></p></td><td style="vertical-align:top;">
<p><span class="font3">56</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;">
<p><span class="font3">Z</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN23</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Caccaacggcatcgaggtcgtat</span></p></td><td style="vertical-align:top;">
<p><span class="font3">23</span></p></td><td style="vertical-align:top;">
<p><span class="font3">70</span></p></td><td style="vertical-align:top;">
<p><span class="font3">56</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font3">Z</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN24</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Accaacggcatcgaggtcgtatg</span></p></td><td style="vertical-align:top;">
<p><span class="font3">23</span></p></td><td style="vertical-align:top;">
<p><span class="font3">70</span></p></td><td style="vertical-align:top;">
<p><span class="font3">56</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;">
<p><span class="font3">Z</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td colspan="9" style="vertical-align:top;">
<p><span class="font3">J⅛S⅛S J»®*® </span><span class="font3" style="font-variant:small-caps;">⅛h⅛i½</span><span class="font3"> mutasιS315 V</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MV1</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Ccgtcggcatcgaggtcgtatg</span></p></td><td style="vertical-align:top;">
<p><span class="font3">22</span></p></td><td style="vertical-align:top;">
<p><span class="font3">70</span></p></td><td style="vertical-align:top;">
<p><span class="font3">63</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;">
<p><span class="font3">-</span></p></td><td style="vertical-align:top;">
<p><span class="font3">2</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MV4</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Gatcaccgtcggcatcgaggtc</span></p></td><td style="vertical-align:top;">
<p><span class="font3">22</span></p></td><td style="vertical-align:top;">
<p><span class="font3">'.<sup>,</sup>0</span></p></td><td style="vertical-align:top;">
<p><span class="font3">63</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font3">Z</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MV8</span></p></td><td style="vertical-align:top;">
<p><span class="font3">ACCGrCGGCATCGAGGTCGTAT</span></p></td><td style="vertical-align:top;">
<p><span class="font3">22</span></p></td><td style="vertical-align:top;">
<p><span class="font3">.0</span></p></td><td style="vertical-align:top;">
<p><span class="font3">59</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
</table>
<p><span class="font6">Information: The probe name, for example K315MT1 shows: </span><span class="font6" style="font-style:italic;">M. tuberculosis katG genes</span><span class="font6">; 315 = target codon; M = type of probe is a mutant probe; T = amino acids formed by mutation of codon 315; number 1 = serial number of the probe design; Pjg = length; Tm = melting temperature (ºC); % GC = percentage of guanine and cytosine base content; underscore = mutation at codon 315</span></p>
<p><span class="font6">All mutant probe designs are then selected based on Tm values. Tm values can be used to predict the annealing temperature (Ta) of the primer or probe. In general, Ta can be determined as a temperature of 5ºC below the estimated Tm (35). A good Tm value for TaqMan probe is 5-10ºC higher than the primary Tm amplification value (31, 36). A higher probe probe will cause the probe to remain hybridized with the target DNA prior to the primary elongation process. The optimum Tm for the probe design to be effectively hybridized with the target DNA is ~ 8ºC over the primary Tm (16) or 70ºC (37,38). In this study, there were 85 mutant probes have a Tm of about 67-72 º C. The probe meets the range of Tm TaqMan probe criteria, which is Tm 5-10ºC above the primary. However, in order to obtain the best mutant probe, in this study re-selection based on the optimal Tm probe value in general (70ºC). The number of mutant probe designs meeting the optimal Tm criterion was 13 probes. After initial analysis, there were 11 mutant probes with optimal Tm that satisfied all of the probe's initial criteria (Table 2). The number of optimal mutant probe designs based on Tm for nucleotide changes at codon 315 varied. Changes in nucleotide composition occurred due to mutation variation at codon 315. The more GC content and the longer the probe, the Tm of the probe increased (39).</span></p>
<p><span class="font6">The third criterion assessed for obtaining good mutant probe design is the GC content. The requirement of a good GC content for the probe is 35-65% (15). The mutant probe obtained from the initial analysis (Table 2) has met the requirements of a good GC probe content. GC content outside the criterion range may cause instability of probe hybridization in the target area during the elongation process (26), and increase non-specific hybridization thereby decreasing the efficiency of PCR. The relationship between</span></p>
<p><span class="font6">Tm and GC content are seen in the design of the mutant probe for the S315G mutation. This type of probe has the highest Tm compared to other mutant probe designs though at the same length. Therefore, in this study no optimal mutant probe design for S315G mutation was obtained.</span></p>
<p><span class="font6">All subsequent designs were analyzed based on the general criteria of number of </span><span class="font6" style="font-style:italic;">runs</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">repeats</span><span class="font6">. The fewer the number of runs and repeats, the better the probe design is generated (40). The number of runs and repeats that can be accepted as a good probe is not more than 4 (16). Special base guanine, repetition sequentially as much as 3 or more must be avoided. 3 or more sequential guanine bases can fold the template into a tetraplex structure. This structure is very stable and cannot be recognized by the polymerase enzyme. Secondary structures are also formed when </span><span class="font6" style="font-style:italic;">repeats </span><span class="font6">are present. Repetition of the long dinucleotide can produce a complementary sequence to form a structure such as a </span><span class="font6" style="font-style:italic;">hairpin</span><span class="font6"> (41). In addition, the number of </span><span class="font6" style="font-style:italic;">runs</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">repeats </span><span class="font6">that exceed the criteria will also increase the cost for detection (42).</span></p>
<p><span class="font6">The mutant probe design that meets the number criteria of </span><span class="font6" style="font-style:italic;">runs</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">repeats</span><span class="font6"> is the mutant probes design for mutations S315T, S315N, and S315V. The maximum number of runs and repeats on each mutant probe design is 2. While the mutant probe design for the S315R mutation does not meet this criterion since it has 4 runs of the guanine base. Runs of the guanine base occurred in all mutant probe designs for the S315R mutation. A single base changed at the 315 codon from AGC to AGG and affected the base composition of this probe, resulting in 4 runs of the guanine base, as shown in Figure 4. In the figure, there were wildtype (WT) probe and K315MR8 mutant probe design (MT) with the position of mutation C → G (marked x) resulted in the addition of the number of G on the sequence of the mutant probe and formed the runs.</span></p><img src="https://jurnal.harianregional.com/media/36991-4.jpg" alt="" style="width:232pt;height:120pt;">
<p><span class="font6">Figure 4 </span><span class="font6" style="font-style:italic;">Runs</span><span class="font6"> on the K315MR8 mutant probe design</span></p>
<p><span class="font6">Caption: runs of 4 bases G on K315MR8 mutant probes are formed due to nucleotide changes C → G at codon 315. The formation of runs of 4 base Gs does not result in a nucleotide sequence of the K315MR probe that meets the initial stage analysis criteria for the TaqMan probe</span></p>
<p><span class="font6">Other criteria that also need to be considered to produce a good mutant probe are dimer and hairpin. The probe should not contain complementary bases so it can form a secondary structure, such as dimer and hairpin (17). Dimers are formed by intermolecular interactions between</span></p>
<p><span class="font6">two probes (43). The formation of dimers can occur in all mutant probes obtained. The type of dimer formed was a self dimer of 4 nucleotide bases. The dimer formation of one of the mutant probes, e.g. K315MN23 is shown in Figure 5. The dimer of the K315MN23 mutant probe is formed between the TCGA bases with its complementary base on the same probe (indicated by dashed lines in Fig. 5). Selected mutant probes based on early stage analysis (Table 2) have met the initial criteria of dimer count in Clone Manager Suite Software 6. Initial criteria for allowed dimers are less than 5 bases. However, it should be noted that dimer formation in the probe design can reduce hybridization efficiency resulting in decreased detection sensitivity (15).</span></p><img src="https://jurnal.harianregional.com/media/36991-5.jpg" alt="" style="width:232pt;height:138pt;">
<p><span class="font6">Figure 5 The result of dimer analysis on K315MN23 mutant probe design</span></p>
<p><span class="font6">Caption: All nucleotide sequences of the mutant probe form 4 dimers, including the K315MN23 probe and have met the criteria of good probe dimer quantities (&lt;5)</span></p>
<p><span class="font6">Hybridisation efficiency can also be reduced if the probe design forms another secondary structure, such as a hairpin. The curved hairpin structure can inhibit TaqMan hybridization probe process at target (17). The formation of the hairpin from the nucleotide sequence of the probe is shown by showing the free structure and energy (ΔG) image at the specific temperature set in Clone Manager Suite 6. Hairpin is affected by the annealing temperature used. The increasing annealing temperature can melt and reduce the formation of the hairpin structure. This occurs in the design of the resulting nucleotide probe sequence. At annealing temperatures from 1 to 36ºC, the nucleotide sequence of the mutant probe (Table 2) forms hairpin with different free energies at different temperatures. For example, a hairpin formed from the nucleotide sequence of the K315MT1 probe at 10ºC (Figure 6A) and 20ºC (Figure 6B) is shown with a red circle in Figure 6. The ΔG value at 10ºC is -0.6 kcals, whereas at 20ºC is - 0.1 kcals. Although hairpin with negative ΔG must be avoided, but the resulting mutant probe design meets the maximum hairpin criteria ΔG because the free energy of acceptably stable hairpin is -3 kcals (40, 44). In addition, all mutant probe designs do not form hairpins at annealing temperatures that are adjusted according to an experimental primer annealing temperature of 56ºC. Therefore, all nucleotide sequences of mutant probes have met the criteria of the secondary structure of hairpin.</span></p><img src="https://jurnal.harianregional.com/media/36991-6.jpg" alt="" style="width:180pt;height:130pt;">
<p><span class="font6">Figure 6. The hairpin structure depiction formed by the K315MT1 mutant probe with Clone Manager Suite 6</span></p>
<p><span class="font6">Caption: the red circle shows the hairpin structure and its free energy; (A) The structure of a hairpin K315MT1 at a temperature of 10ºC; (B) The structure of a hairpin K315MT1 at a temperature of 20ºC; (C) The hairpin structure is not formed at 56ºC, which is the primary optimization Ta based on Suryadi's experimental study (2013)</span></p>
<h1><a name="bookmark16"></a><span class="font6" style="font-weight:bold;"><a name="bookmark17"></a>The Analysis on Labeling of Mutant Probes</span></h1>
<p><span class="font6">Select mutant probe designs in the initial analysis (Table 2), then was analyzed the final stages. The final analysis criterion is based on the TaqMan probe labeling that there is no base G on the 2 bases at the 5 'end of the probe, and the amount of base C is more or equal to the base G on the nucleotide sequence of the probe. A final analysis is needed to find out the suitability of the probe design with TaqMan probe criteria and to produce a quality TaqMan probe. One of the factors that determine the TaqMan probe's quality is the fluorescence blackout efficiency of the probe label (26).</span></p>
<p><span class="font6">The efficiency of blackout of a good reporter label is influenced by several factors, such as the optimal distance between the two labels, as well as the number and position of the guanine base (16, 28, 33). Reporter blackouts can be performed optimally by quencher compounds located at a distance of 3-30 bases from the reporter. Quencher at a 2-base distance from the reporter will result in the Taq polymerase enzyme, which is not able to perform the exonuclease activity that plays a role in the degradation of the probe to produce a fluorescence signal (28). The basic structure G contains most atoms with electronegative properties that can contribute a free electron pair. This will result in the greatest electron-donor electron ability compared to other nucleotide bases. Therefore, the amount and position of base G are very necessary to be considered in order to obtain optimal signal detection efficiency (45).</span></p>
<p><span class="font6">The resulting mutant probe should not contain base G on the first and second nucleotides from the 5 'probe end (44). This criterion is important because the guanine base has a natural ability to extinguish fluorescence (quencher) from reporters such as FAM or other fluorescent compound derivatives. Based on the criteria of position G on the nucleotide sequence of the probe, 8 probes were found that met this criterion, of the 11 mutant probes in Table 2</span></p>
<p><span class="font6">analyzed. A probe that does not meet this criterion is 3 probes with position G as shown in Figure 7. The base G at the nucleotide sequence of the three probes is at the 1st position of the 5 'end for K315MT4 and K315MV4 probes, and at the 2nd position of the tip 5 'for K315MN3 probe.</span></p><img src="https://jurnal.harianregional.com/media/36991-7.jpg" alt="" style="width:235pt;height:89pt;">
<p><span class="font6">Figure 7. The base position of G on the K315MT4, K315MV4, and K315MN3 probes</span></p>
<p><span class="font6">Caption: These three mutant probes do not meet the final stages of analysis criteria, ie the minimum G position at the 3rd base from the 5 '</span></p>
<p><span class="font6">The second criterion for the final analysis of the probe is the amount of base C more or equal to the base G on the nucleotide sequence of the probe. In the 11 mutant probes analyzed in the final analysis, there were 5 mutant probes obtained such as K315MN16, K315MN17, K315MN24, K315MV1, and K315MV18 had fewer C than base G. Based on empirical observations, it was found that TaqMan probe with a higher number of G bases than base C is often resulting in decreasing normal fluorescence (40). This relates to the nature of a base G which can extinguish the reporter fluorescence, although the nucleotide sequence of the probe has been hydrolyzed by exonuclear activity (38). The extinction rate due to base G will increase as the amount of base G in the nucleotide sequence of the probe is increased (46).</span></p>
<p><span class="font6">After the initial and final analysis, it was obtained 3 best mutant probes as TaqMan probe, e.g. K315MT1 mutant probe, K315MN5 and K315MN23. The results of the analysis of the three probes with software can be seen in Table 3. The three TaqMan mutant probes can be used for detection of mutations at the codon of 315 KatG </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> genes. The criterion values of the three mutant probes (Table 3) have met the general criteria of the probe as described. Although the K315MN5 mutant probe (Table 3) contains more than 3 bases of G and C on 3 bases at the 3 'end, this does not affect the probe because the nucleotide sequence of the probe will not be elongated at the 3' end such as the primer. Therefore, this criterion is very important in the primary design so there is no non-specific elongation process (38). The three mutant probes (Table 3) have also met the end-stage criteria so that the label fluorescence efficiency is not reduced due to the nucleotide sequence of the probe.</span></p>
<p><span class="font6">Table 3. Final Analysis of </span><span class="font6" style="font-style:italic;">Probe</span><span class="font6"> Mutan</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:middle;">
<p><span class="font3" style="font-style:italic;">Probe</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font3">Nucleotide sequence </span><span class="font3" style="font-style:italic;">o?Probe</span><span class="font3"> 5<sup>,</sup>→3<sup>,</sup></span></p></td><td colspan="8" style="vertical-align:top;">
<p><span class="font3">DNA Probe Design Criteria Ai ,⅛⅛a</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">Eis</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">Tm</span></p></td><td style="vertical-align:top;">
<p><span class="font3">⅞</span></p>
<p><span class="font3">GC</span></p></td><td style="vertical-align:middle;">
<p><span class="font3" style="font-style:italic;">Dmer</span></p></td><td style="vertical-align:middle;">
<p><span class="font3" style="font-style:italic;">fiatrpir,</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">runs</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">repeat</span></p></td><td style="vertical-align:top;">
<p><span class="font3">Base</span></p>
<p><span class="font3">Amosnt</span></p>
<p><span class="font3">C:G</span></p></td></tr>
<tr><td colspan="10" style="vertical-align:top;">
<p><span class="font3">Probe design of specific mutation S315T</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MΓl</span></p></td><td style="vertical-align:top;">
<p><span class="font3">CCACCGGCATCGAGGrCGTATG</span></p></td><td colspan="2" style="vertical-align:top;"></td><td colspan="2" style="vertical-align:top;">
<p><span class="font3"><sub>U</sub> 4</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font3"><sup>2</sup></span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td colspan="10" style="vertical-align:top;">
<p><span class="font3">Probe design of specific mutation S315N</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN5</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Atcaccaacggcatcgaggtcg</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font3">59</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font2" style="font-style:italic;">1</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font3">73</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">K315MN23</span></p></td><td style="vertical-align:top;">
<p><span class="font5" style="font-variant:small-caps;">Caccaacggcatcgaggtcgtat</span></p></td><td style="vertical-align:top;">
<p><span class="font3">23</span></p></td><td style="vertical-align:top;">
<p><span class="font3">.<sup>,</sup>0</span></p></td><td style="vertical-align:top;">
<p><span class="font3">56</span></p></td><td style="vertical-align:top;">
<p><span class="font3">4</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font3">7:6</span></p></td></tr>
</table>
<p><span class="font6" style="font-weight:bold;">Information: </span><span class="font6">The probe name, for example K315MT1 shows: </span><span class="font6" style="font-style:italic;">M. tuberculosis katG genes</span><span class="font6">; 315 = target codon; M = type of probe is a mutant probe; T = amino acids formed by mutation of codon 315; number 1 = serial number of the</span></p>
<p><span class="font6">probe design; Pjg = length; Tm = melting temperature (ºC); % GC = percentage of guanine and cytosine base content; underscore = mutation at codon 315.</span></p>
<p><span class="font6">The probe probe design of 3 probes (Table 3) can be attached to labels. The label for TaqMan probe consists of reporter and quencher. The reporter label is a fluorescent label and placed on the 5 'probe. While the quencher can be fluorescence or non fluorescence compound that absorbs fluorescence from the reporter at the appropriate distance. Quencher can be placed in the center or 3 'probe end (38). The labels commonly used for TaqMan probes are FAM as reporter and TAMRA as quencher (15).</span></p>
<p><span class="font6">Installation of labels for the design of K315MT1 mutant probes with base changes G → C which marked with bold prints are shown in Figure 8. The reporter label can be embedded on base C at the 5 'end and the quencher label is affixed to a T base at the 3' probe end. Meanwhile, the labeling of the K315MN5 and K315MN23 mutant probe designs is shown in Figure 9. Both designs of this mutant probe have different nucleotide length and compositions, but the same detected nucleotide changes, ie G → A. The type of FAM reporter label can be used on K315MN5 and K315MN23 mutant probes as in Figure 9.</span></p>
<p><a href="#bookmark18"><span class="font1">∣</span><span class="font4" style="text-decoration:underline;">cc ⅞C3 ggc ate q⅛g gtc gta⅛</span></a></p>
<p><a href="#bookmark19"><span class="font4">Qj→g</span><span class="font1">⊂</span><span class="font4">g ate ace age ggc ate gag gee gta egg acg aac ace,,.</span></a></p>
<p><a href="#bookmark20"><span class="font4">Q→312 313 314 315 316 317 318 319 320 321 322 323 324...</span></a></p>
<p><span class="font4">Kcteraugaie</span></p>
<ul style="list-style:none;"><li>
<p><span class="font4">(7) &nbsp;&nbsp;&nbsp;: Kodon gen</span><span class="font4" style="font-style:italic;">PsrGM IMberciAiosfi</span><span class="font4"> Cl)' ^<sup>u</sup>Kl<sup>e</sup>°dda pro⅛a mutaa K-IleMTi</span></p></li></ul>
<p><span class="font4">Q: Nuldeodda </span><span class="font4" style="font-style:italic;">Hildtype</span><span class="font4"> penyusun kodon</span></p>
<p><span class="font6">Figure 8. Labeling of K315MT1 probe design</span></p>
<p><span class="font6">Caption:The K315MT1 probe is specific to G → C changes at codon 315, and can be paired with the FAM-TAMRA label</span></p>
<p><span class="font6">If the K315MN5 or K315MN23 mutant probe will be used in one reaction with the K315MT1 mutant probe, the reporter label of both mutant probe types should be differentiated. The use of reporter labels that provide different fluorescence colors for the S315N and S315T mutations will increase the detection specificity against base changes at codon 315. FAM reporter labels (green fluorescence) can generally be replaced by the VIC label</span></p>
<p><span class="font6">(fluorescent yellow fluorescence) due to fluorescence from both of these reporter labels typically can be extinguished by TAMRA as a quencher so that no quencher type replacement (30) is required. However, replacement of the label types used should also consider the availability of real-time PCR tools (15, 31).</span></p>
<p><span class="font6">The selected reporter and quencher labels were then affixed at opposite ends to the nucleotide sequence of the mutant probe (28). Figure 9 shows the label position for the K315MN5 probe (Figure A), with the FAM reporter affixed to the nucleotide A at the 5 'probe end. While the K315MN23 probe reporter (Figure B) is attached to the C nucleotide at the 5 'end of the probe. Both designs of this mutant probe use TAMRA as quencher. The position of TAMRA attachment to the K315MN5 probe is the C nucleotide at the 3 'probe end. Meanwhile, TAMRA for K315MN23 mutant probe is attached to the T nucleotide at the 3 'probe end.</span></p><img src="https://jurnal.harianregional.com/media/36991-8.jpg" alt="" style="width:230pt;height:121pt;">
<p><span class="font6">Figure 9. Labeling of mutant probe design for S315N mutation detection</span></p>
<p><span class="font6">Caption:both K315MN5 and K315MN23 best probes specific to G → A mutations; (A) K315MN5 mutant probe with FAM-TAMRA label, (B) K315MN23 mutant probe with FAM-TAMRA label</span></p>
<p><span class="font6">The mutant probes that can be labeled based on silico design research analysis with Clone Manager Suite 6 were K315MT1, K315MN5, and K315MN23 probe. The nucleotide sequence of these probes can recognize mutations in the codon of 315 KatG </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis </span><span class="font6">genes. Specific detection can be produced primarily specific serine mutations into asparagin at codon 315 with K315MN5 and K315MN23 mutant probes, and serine specific mutations into threonine at codon 315 with K315MT1 mutant probe. Therefore, the K315MT1 mutant probe can be used for detection of the S315T mutation. The use of DNA probe detection can provide faster results than conventional PCR and sequencing techniques, as well as cross-contamination and error due to numerous testing stages can be reduced as the amplification and detection process with DNA probe can be performed simultaneously with real-time PCR tool (16). If the test uses two types of mutant probes, such as K315MT1 and K315MN5 probes, the S315T mutation in the tested isolate will cause the K315MT1 probe to be hybridized with specific nucleotide sequence to the target. Then, TaqMan probe K315MT1 will be hydrolyzed and generated fluorescence signal as a marker of S315T mutation. The use of K315MN5, K315MN23 and</span></p>
<p><span class="font6">K315MT1 mutant probes for mutation detection at codon 315 can provide false-positive results in the presence of mutations in other codons and presence of wild-type isolates. However, further experimental testing is required in the laboratory to prove the performance of the K315MN5, K315MN23 and K315MT1 TaqMan designs.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">IV. &nbsp;&nbsp;&nbsp;CONCLUSION</span></p></li></ul>
<ul style="list-style:none;"><li>
<p><span class="font6">1. &nbsp;&nbsp;&nbsp;The best mutant probe that can be used for the detection of mutations at the codon of 315 </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> katG genes is a K315MT1 mutant probe with the nucleotide sequence 5'-FAM-CC ACC GGC ATC GAG GTC GTA TG-TAMRA-3 ' for detection of specific S315T mutations as well as probe mutant K315MN5 and K315MN23 with nucleotide sequences 5'-FAM-ATC ACC AAC GGC ATC GAG GTC G-TAMRA-3 'and 5'-FAM-C ACC AAC GGC ATC GAG GTC GTA T-TAMRA-3' for detection of specific mutations S315N .</span></p></li>
<li>
<p><span class="font6" style="font-weight:bold;">2.</span><span class="font6"> &nbsp;&nbsp;&nbsp;There were 3 DNA probe designs, such as K315MT1, K315MN5, and K315MN23 of the 11 probes that met the initial probe with TaqMan probe criteria for real-time PCR based on minimum base G positions on the 3rd base from tip 5 the probe and the amount of base G does not exceed the base C in the nucleotide sequence of the probe.</span></p></li></ul>
<p><span class="font6">REFERENCES</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[1] &nbsp;&nbsp;&nbsp;World Health Organization(WHO). 2014. </span><span class="font6" style="font-style:italic;">Global Tuberculosis Report 2014</span><span class="font6">. Perancis: World Health Organization.</span></p></li>
<li>
<p><span class="font6">[2] &nbsp;&nbsp;&nbsp;World Health Organization(WHO). 2015. </span><span class="font6" style="font-style:italic;">Global Tuberculosis Report 2015</span><span class="font6">. Perancis: World Health Organization.</span></p></li>
<li>
<p><span class="font6">[3] &nbsp;&nbsp;&nbsp;Lisdawati, V., N. Puspandari, L. Rif’ati, T. Soekarno, M, Melatiwati, K. Syamsidar, L. Ratnasari, N. Izzatun, dan I. Parwati. 2015. Molecular Epidemiology Study of </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> and Its Susceptibility to Anti-TuberculosisDrugs in Indonesia. </span><span class="font6" style="font-style:italic;">BMC Infectious Disease</span><span class="font6">. Volume 15: 366-373.</span></p></li>
<li>
<p><span class="font6">[4] &nbsp;&nbsp;&nbsp;Babamahmoodi, F., M. R. Mahdavi, H. Jalali, B. Talebi, P. Roshan, dan M. Mahdavi. 2014. Evaluation of Gene Mutations Involved in Drug Resistance in </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> Strains Derived from Tuberculosis Patients in Mazandaran, Iran, 2013. </span><span class="font6" style="font-style:italic;">Int J Mol Cell Med Summer</span><span class="font6">. Volume 3(3): 191-195.</span></p></li>
<li>
<p><span class="font6">[5] &nbsp;&nbsp;&nbsp;Silva, P. E. A. D., dan J. C. Palomino. 2011. Molecular Basis and Mechanisms of &nbsp;Drug Resistance in</span></p></li></ul>
<p><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6">: &nbsp;Classical and New</span></p>
<p><span class="font6">Drugs. </span><span class="font6" style="font-style:italic;">Journal of Antimicrobial Chemotherapy</span><span class="font6">. Volume 66(7): 1417-1430.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[6] &nbsp;&nbsp;&nbsp;Tseng, S. T., C. H. Tai, C. R. Li, C. F. Lin, dan Z. Y. Shi. 2015. The Mutations of </span><span class="font6" style="font-style:italic;">katG</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">inhA</span><span class="font6"> Genes of Isoniazid-Resistant </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis </span><span class="font6">Isolates in Taiwan. </span><span class="font6" style="font-style:italic;">Journal of &nbsp;Microbiology</span><span class="font6">,</span></p></li></ul>
<p><span class="font6" style="font-style:italic;">Imunology and Infection</span><span class="font6">. Volume 48: 249-255.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[7] &nbsp;&nbsp;&nbsp;Murray, J. L., P. Hu, dan D. A. Shafer. 2014. Seven Novel Probe System for Real-Time PCR Provide Absolute Single-Base Discrimination, Higher Signaling, and Generic Components. The Journal of Molecular Diagnostics. Vol. 16 (6): 627-638.</span></p></li>
<li>
<p><span class="font6">[8] &nbsp;&nbsp;&nbsp;Suryadi, T. 2013. Identifikasi Mutasi Gen </span><span class="font6" style="font-style:italic;">katG</span><span class="font6"> Pada Isolat P10 dan 151 </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis Multidrug Resistance</span><span class="font6"> di Bali Dengan Metode </span><span class="font6" style="font-style:italic;">Polymerase Chain Reaction</span><span class="font6">. </span><span class="font6" style="font-style:italic;">Skripsi</span><span class="font6">. Universitas Udayana, Bukit Jimbaran: 1-39.</span></p></li>
<li>
<p><span class="font6">[9] &nbsp;&nbsp;&nbsp;Deniariasih, N. W. 2013. Deteksi Mutasi Pada Gen </span><span class="font6" style="font-style:italic;">katG</span><span class="font6"> (Fragmen 0,7 kb) Isolat 86 dan P11 </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis Multidrug Resistance </span><span class="font6">Dengan Teknik </span><span class="font6" style="font-style:italic;">Polymerase Chain Reaction</span><span class="font6"> (PCR). </span><span class="font6" style="font-style:italic;">Skripsi</span><span class="font6">. Universitas Udayana, Bukit Jimbaran: 1-39.</span></p></li>
<li>
<p><span class="font6">[10] &nbsp;&nbsp;&nbsp;Hazbon, M. H., M. Brimacombe, M. B. D.Valle, M. Cavatore, M. I. Guerrero, M. V. Basil, H. B. Jacobe, C. Lavender, J. Fyfe, L. G. Garcia, C. I. Leon, M. Bose, F. Chaves, M. Murray, K. D. Eisenach, J. S. Osornio, M. D. Cave, A. P. D. Leon, dan D. Alland. 2006. Population Genetics Study of Isoniazid Resistance Mutations and Evolution of Multidrug-Resistant </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6">. </span><span class="font6" style="font-style:italic;">Antimicrobial Agents and Chemotherapy</span><span class="font6">. Volume 50(8): 2640-2649.</span></p></li>
<li>
<p><span class="font6">[11] &nbsp;&nbsp;&nbsp;Bostanabad, S. Z., L. P. Titov, A. Bahrmand, S. A. Nojoumi. 2008. Detection of Mutation in IsoniazidResistant </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> Isolates From Tuberculosis Patient in Belarus. </span><span class="font6" style="font-style:italic;">Indian Journal of Medical Microbiology</span><span class="font6">. Volume 26(2): 143-147.</span></p></li>
<li>
<p><span class="font6">[12] &nbsp;&nbsp;&nbsp;Jagielski, T., Z. Bakuta, K. Roeske, M. Kaminski, A. Napiorkowska, E. A. Kopec, Z. Zwolska, dan J. Bielecki. 2014. Detection of Mutations Associated With Isoniazid Resistance in Multidrug-Resistant </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> Clinical Isolates. </span><span class="font6" style="font-style:italic;">J. Antimicrob Chemother</span><span class="font6">. Volume 69(4): 2369-2375.</span></p></li>
<li>
<p><span class="font6">[13] &nbsp;&nbsp;&nbsp;Navarro, E., G. S. Heras, M. J. Castano, dan J. Solera. 2015. Real-Time PCR Detection Chemistry. </span><span class="font6" style="font-style:italic;">Clinica Chimica Acta</span><span class="font6">. Volume 439: 231-250.</span></p></li>
<li>
<p><span class="font6">[14] &nbsp;&nbsp;&nbsp;Espy, M. J., J. R. Uhl, L. M. Sloan, S. P. Buckwalter, M. F. Jones, E. A. Vetter, J. D. C. Yao, N. L. Wengenack, dan J. E. Rosenblatt. 2006. Real-Time PCR in Clinical Microbiology: Applications for Routine Laboratory Testing. </span><span class="font6" style="font-style:italic;">Journal of Clinical Microbiology Review</span><span class="font6">. Volume 19(1): 165-256.</span></p></li>
<li>
<p><span class="font6">[15] &nbsp;&nbsp;&nbsp;Dorak, M. T. 2006. </span><span class="font6" style="font-style:italic;">Real-Time PCR</span><span class="font6">. New York: Taylor and Francis Group.1-30.</span></p></li>
<li>
<p><span class="font6">[16] &nbsp;&nbsp;&nbsp;McPherson, M., dan S. Moller. 2006. </span><span class="font6" style="font-style:italic;">PCR</span><span class="font6"> Edisi 2. New York: Taylor &amp;&nbsp;Francis Group. 209-305.</span></p></li>
<li>
<p><span class="font6">[17] &nbsp;&nbsp;&nbsp;Walker, J.M. dan Rapley, R. 2008.</span><span class="font6" style="font-style:italic;">Medical Biomethods Handbook.</span><span class="font6">Totowa, New Jersey: Humana Press Inc. 13341.</span></p></li>
<li>
<p><span class="font6">[18] &nbsp;&nbsp;&nbsp;Wada, T., S. Maeda, A. Tamaru, S. Imai, A. Hase, dan K. Kobayashi.2004. Dual-Probe Assay For Rapid Detection of Drug-Resistant </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> by Real-Time PCR. </span><span class="font6" style="font-style:italic;">Journal of Clinical Microbiology</span><span class="font6">. Volume 42(11): 5277-5285.</span></p></li>
<li>
<p><span class="font6">[19] &nbsp;&nbsp;&nbsp;Kurreck, J. dan C. A. Stein. 2015. </span><span class="font6" style="font-style:italic;">Molecular Medicine</span><span class="font6">. Jerman: Wiley-VCH. 37-51.</span></p></li>
<li>
<p><span class="font6">[20] &nbsp;&nbsp;&nbsp;Chou, C.C., C. C. H. Chen, T. T. Lee, dan K. Peck. 2004. Optimization of Probe Length and The Number of Probes Per Gene for Optimal Microarray Analysis of Gene Expression. </span><span class="font6" style="font-style:italic;">Nucleic Acid Research</span><span class="font6">. Volume 32(12): 99-107.</span></p></li>
<li>
<p><span class="font6">[21] &nbsp;&nbsp;&nbsp;Kalendar, R., D. Lee, dan A. H. Schulman. 2011. Java Web Tools for PCR, In Silico PCR, and Oligonucleotide Assembly and Analysis. </span><span class="font6" style="font-style:italic;">Genomics</span><span class="font6">. Volume 98: 137-144.</span></p></li>
<li>
<p><span class="font6">[22] &nbsp;&nbsp;&nbsp;Yilmaz, L. S., S. Parnerkar, dan D. D. R. Noguera. 2011. mathFISH, A Web Tool That Uses Thermodynamics-Based Mathematical Models for </span><span class="font6" style="font-style:italic;">In Silico</span><span class="font6"> Evaluation of Oligonucleotide Probes for Fluorescence </span><span class="font6" style="font-style:italic;">In Situ</span><span class="font6"> Hybridization. </span><span class="font6" style="font-style:italic;">Applied And Environmental Microbiology</span><span class="font6">. Volume 77(3): 1118</span></p></li></ul>
<p><span class="font6">1122.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[23] &nbsp;&nbsp;&nbsp;Zhang, Y. dan W. W. Yew. 2009. Mechanisms of Drug Resistance in </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6">. </span><span class="font6" style="font-style:italic;">Int. J. Tuberc Lung Dis</span><span class="font6">. Volume 13(11): 1320-1330.</span></p></li>
<li>
<p><span class="font6">[24] &nbsp;&nbsp;&nbsp;Lina, M., B. Bela, dan A. Yasmon. 2009. Deteksi Mutasi Gen katG </span><span class="font6" style="font-style:italic;">Mycobacterium tuberculosis</span><span class="font6"> Dengan Metode PCR &nbsp;&nbsp;(</span><span class="font6" style="font-style:italic;">Polymerase &nbsp;&nbsp;Chain Reaction</span><span class="font6">)-</span></p></li></ul>
<p><span class="font6">Hibridisasi </span><span class="font6" style="font-style:italic;">Dot Blot</span><span class="font6"> Menggunakan Pelacak Oligonukleotida Bertanda <sup>32</sup>P. </span><span class="font6" style="font-style:italic;">Jurnal ilmiah Aplikasi Isotop dan Radiasi</span><span class="font6">. Volume 5 (1): 54-67.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[25] &nbsp;&nbsp;&nbsp;Smith, J. E. 2009. </span><span class="font6" style="font-style:italic;">Biotechnology</span><span class="font6"> 5<sup>th</sup> Edition. New York: Cambridge University Press. 41-44.</span></p></li>
<li>
<p><span class="font6">[26] &nbsp;&nbsp;&nbsp;Anonim a. 2004. Assay Formats for Use in Real-Time PCR. </span><span class="font6" style="font-style:italic;">Technical Note</span><span class="font6">. Jerman: Roche Applied Science. 1-13. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Available &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><a href="https://lifescience"><span class="font6">https://lifescience</span></a><span class="font6">.</span></p></li></ul>
<p><span class="font6">roche.com/wcsstore/RASCatalogAssetStore/Articles/A ssay%20Format%20for%20use%20in%20Real-Time%20PCR.pdf. (Cited March 17, 2016).</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[27] &nbsp;&nbsp;&nbsp;Alvandi, E. dan F Koohdani. 2014. Zip Nucleic Acid: A New Reliable Method To Increase The Melting Temperature of Real-Time PCR Probes. </span><span class="font6" style="font-style:italic;">Journal of Diabetes &amp;&nbsp;Metabolic Disorders</span><span class="font6">. Vol. 13 (26): 1-4.</span></p></li>
<li>
<p><span class="font6">[28] &nbsp;&nbsp;&nbsp;Livak, K. J., S. J. A. Flood, J. Marmaro, W. Giusti, dan K. Deetz. 1995. Oligonucleotides With Fluorescent Dyes At Opposite Ends Provide A Quenched Probe System Useful For Detecting PCR Product and Nucleic Acid Hybridization. </span><span class="font6" style="font-style:italic;">Genome Research</span><span class="font6">. Vol. 4: 357362.</span></p></li>
<li>
<p><span class="font6">[29] &nbsp;&nbsp;&nbsp;Anonim d. 2001. Allelic Discrimination Using The 5’ Nuclease Assay. USA: Applied Biosystems. 1-8. </span><a href="http://www.austincc.edu/mlt/mdfund/mdfund%20_Unit11AllelicDiscrimination.pdf"><span class="font6" style="text-decoration:underline;">http://www.austincc.edu/mlt/mdfund/mdfund</span></a><span class="font6" style="text-decoration:underline;"> </span><a href="http://www.austincc.edu/mlt/mdfund/mdfund%20_Unit11AllelicDiscrimination.pdf"><span class="font6" style="text-decoration:underline;">_Unit11AllelicDiscrimination.pdf</span><span class="font6">. </span></a><span class="font6">(Cited March 14, 2016).</span></p></li>
<li>
<p><span class="font6">[30] &nbsp;&nbsp;&nbsp;Logan, J., K. Edwards, dan N. Saunders. 2009. </span><span class="font6" style="font-style:italic;">RealTime PCR: Current Technology and Applications</span><span class="font6">. London: Caister Academic Press. 1-27. Available at:www.horizonpress.com/realtimepcr. </span><a href="http://citeseerx"><span class="font6">http://citeseerx</span></a><span class="font6">. ist.psu.edu/viewdoc/download?doi=10.1.1.261.223&amp;re p=rep1&amp;type=pdf. (Cited April 23, 2016).</span></p></li>
<li>
<p><span class="font6">[31] &nbsp;&nbsp;&nbsp;Anonim b. 2006. Real-Time PCR Applications Guide. </span><span class="font6" style="font-style:italic;">Bulletin 5279</span><span class="font6">. USA: Bio-Rad Laboratories Inc. 1-100. Available &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><a href="http://www.bio-rad.com/webroot/"><span class="font6">www.bio-rad.com/webroot/</span></a></p></li></ul>
<p><span class="font6">web/pdf/lsr/literature/Bulletin_5279.pdf. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(Cited</span></p>
<p><span class="font6">February 15, 2016).</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[32] &nbsp;&nbsp;&nbsp;Holland, P. M., R. D. Abramcon, R. Watson. Dan D. H. Gelfand. 1991. Detection of Spesific Polymerase Chain Reaction Product By Utilizing The 5’→3’ Exonuclease Activity of </span><span class="font6" style="font-style:italic;">Thermus aquaticus</span><span class="font6"> DNA Polymerase. </span><span class="font6" style="font-style:italic;">Proc Natl Acad Sci Biochemistry</span><span class="font6">. Vol. 88: 7276-7280.</span></p></li>
<li>
<p><span class="font6">[33] &nbsp;&nbsp;&nbsp;Liu, H., H. Wang, Z. Shi, H. Wang, C. Yang, S. Silke, W. Tan, dan Z. Lu. 2006. Taqman Probe Array For Quantitative Detection of DNA Targets. </span><span class="font6" style="font-style:italic;">Nucleic Acids Research</span><span class="font6">. Vol. 34(1): 1-8.</span></p></li>
<li>
<p><span class="font6">[34] &nbsp;&nbsp;&nbsp;Handoyo, D., dan A. Rudiretna. 2000. Prinsip Umum dan Pelaksanaan Polymerase Chain Reaction (PCR).Unitas. Vol. 9 (1): 17-29.</span></p></li>
<li>
<p><span class="font6">[35] &nbsp;&nbsp;&nbsp;Patel, N. K., dan N. Prakash. 2013. Principle and Tools For Primer Design. </span><span class="font6" style="font-style:italic;">Atmiya Spandan Biological Sciences</span><span class="font6">. Vol. 1 (1): 79-95.</span></p></li>
<li>
<p><span class="font6">[36] &nbsp;&nbsp;&nbsp;Bishop, J. L., S. A. Campbell, P. Farrell, M. Fitzgerald, M. Haugen, W. Kocmond, D.E. Madden, W. E. Murray, dan D. H. Persing. 2015. Designing Real-Time Assays on the SmartCycler® II System. United State: Cepheid Technical Support. 1-8.Available at: </span><a href="http://www.cepheid.com/us/%20component/phocadownload/category/2-support?download=8:smart-note-6-1"><span class="font6" style="text-decoration:underline;">http://www.cepheid.com/us/</span></a><span class="font6" style="text-decoration:underline;"> </span><a href="http://www.cepheid.com/us/%20component/phocadownload/category/2-support?download=8:smart-note-6-1"><span class="font6" style="text-decoration:underline;">component/phocadownload/category/2-</span></a><span class="font6" style="text-decoration:underline;"></span><a href="http://www.cepheid.com/us/%20component/phocadownload/category/2-support?download=8:smart-note-6-1"><span class="font6" style="text-decoration:underline;">support?download=8:smart-note-6-1</span><span class="font6">. </span></a><span class="font6">(Cited April 20, 2016).</span></p></li>
<li>
<p><span class="font6">[37] &nbsp;&nbsp;&nbsp;Proudnikov, D., V. Yuferov, Y. Zhou, K. S. Laforge, A. Ho, M. J. Kreek. 2003. Optimizing Primer-Probe Design for Fluorescent PCR. Journal of Neuroscience Methods. Vol 123: 31-45.</span></p></li>
<li>
<p><span class="font6">[38] &nbsp;&nbsp;&nbsp;Lazaro, R. D. dan M. Hernandez. 2013. </span><span class="font6" style="font-style:italic;">Real-Time PCR in Food Science Current Technology and Applications</span><span class="font6">. Norfolk, UK: Caister Academic Press. 512.</span></p></li>
<li>
<p><span class="font6">[39] &nbsp;&nbsp;&nbsp;Mulle, J. G., V. C. Patel, S. T. Warren, M. R. Hegde, D. J. Cutler, dan M. E. Zwick.2010. Empirical Evaluation of Oligonucleotide Probe Selection for DNA Microarrays. </span><span class="font6" style="font-style:italic;">Plos One</span><span class="font6">. Vol. 5 (3): 1-7.</span></p></li>
<li>
<p><span class="font6">[40] &nbsp;&nbsp;&nbsp;Anonim c. 2006. Beacon Designer 5.10 Manual. Corona Way: USA: Premier Biosoft International. 2289. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Available &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;at:</span></p></li></ul>
<p><a href="http://211.69.128.172/tlli/upfile/doc/7584c8_Beacon%25"><span class="font6">http://211.69.128.172/tlli/upfile/doc/7584c8_Beacon%</span></a><span class="font6"> 20Designer%20510Manual.pdf. (Cited February 25, 2016).</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6">[41] &nbsp;&nbsp;&nbsp;Kubista, M., J. M. Andrade, M. Bengtsson, A. Forootan, J. Jonak, K. Lind, R. Sindelka, R. Sjoback, B. Sjogreen, L. Strombom, A. Stahlberg, dan N. Zoric. 2006. The Real-Time Polymerase Chain Reaction. </span><span class="font6" style="font-style:italic;">Molecular Aspects of Medicine Review</span><span class="font6">. Vol. 27: 95125.</span></p></li>
<li>
<p><span class="font6">[42] &nbsp;&nbsp;&nbsp;Yuwono, T. 2008. </span><span class="font6" style="font-style:italic;">Biologi Molekuler</span><span class="font6">. Jakarta: Penerbit Erlangga. 49-74.</span></p></li>
<li>
<p><span class="font6">[43] &nbsp;&nbsp;&nbsp;Borah, P. 2011. Primer Designing For PCR.</span><span class="font6" style="font-style:italic;">Sci Vis</span><span class="font6">. Vol. 3: 134-136.</span></p></li>
<li>
<p><span class="font6">[44] &nbsp;&nbsp;&nbsp;Rychlik, W. 2008.</span><span class="font6" style="font-style:italic;">Oligo Primer Analysis Software Version 7</span><span class="font6">. USA: Molecular Biology Insights Inc. 90123.</span></p></li>
<li>
<p><span class="font6">[45] &nbsp;&nbsp;&nbsp;Nazarenko, I., R. Pires, B. Lowe, M. Obaidy, dan A. Rashtchian. 2002. Effect of Primary and Secondary Structure of Oligodeoxyribonucleotides on The Fluorescent Properties of Conjugated Dyes. </span><span class="font6" style="font-style:italic;">Nucleic Acids Research</span><span class="font6">. Volume 30 (9): 2089-2195.</span></p></li>
<li>
<p><span class="font6">[46] &nbsp;&nbsp;&nbsp;Mackay, I. M. 2007. </span><span class="font6" style="font-style:italic;">Real-Time PCR in Microbiology From Diagnosis to Characterization</span><span class="font6">. Norfolk, Uk: Caister Academic Press. 1-64</span></p></li></ul>