---
layout: full_article
title: "IN-PLANTA TRANSFORMATION METHOD MEDIATED WITH Agrobacterium tumefaciens FOR T-DNA TRANSFER IN TABLE GRAPE (Vitis vinifera L.)"
author: "Rindang Dwiyani, Hestin Yuswanti, Yuyun Fitriani, Bambang Sugiharto"
categories: jbb
canonical_url: https://jurnal.harianregional.com/jbb/full-52066 
citation_abstract_html_url: "https://jurnal.harianregional.com/jbb/id-52066"
citation_pdf_url: "https://jurnal.harianregional.com/jbb/full-52066"  
comments: true
---

<p><span class="font0">INTERNATIONAL JOURNAL OF BIOSCIENCES AND BIOTECHNOLOGY • Vol. 6 No. 2 • April 2019 eISSN: 2655-9994 pISSN: 2303-3371 </span><a href="https://doi.org/10.24843/IJBB.2019.v06.i02.p02"><span class="font0">https://doi.org/10.24843/IJBB.2019.v06.i02.p02</span></a></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font7" style="font-weight:bold;font-style:italic;"><a name="bookmark1"></a>IN-PLANTA</span><span class="font7" style="font-weight:bold;"> TRANSFORMATION METHOD MEDIATED WITH </span><span class="font7" style="font-weight:bold;font-style:italic;">Agrobacterium tumefaciens</span><span class="font7" style="font-weight:bold;"> FOR T-DNA TRANSFER IN TABLE GRAPE (</span><span class="font7" style="font-weight:bold;font-style:italic;">Vitis vinifera</span><span class="font7" style="font-weight:bold;"> L</span><span class="font7" style="font-weight:bold;font-style:italic;">.</span><span class="font7" style="font-weight:bold;">)</span></h1>
<p><span class="font7" style="font-weight:bold;">Rindang Dwiyani<sup>1*</sup>, Hestin Yuswanti<sup>1</sup>, Yuyun Fitriani<sup>1</sup>, and Bambang Sugiharto<sup>2 </sup></span><span class="font6"><sup>1</sup>Faculty of Agriculture, Udayana University, Jalan PB Sudirman Denpasar, Indonesia</span></p>
<ul style="list-style:none;"><li>
<p><span class="font6"><sup>2</sup>Centre Development of Advanced Sciences and Technology, Jember University, Jl. Kalimantan No.</span></p></li></ul>
<ul style="list-style:none;"><li>
<p><span class="font6">37, Kampus Tegalboto Jember – Jawa Timur, Indonesia <sup>*</sup>Corresponding author: </span><a href="mailto:rindangdwiyani@unud.ac.id"><span class="font6">rindangdwiyani@unud.ac.id</span></a></p></li></ul>
<h1><a name="bookmark2"></a><span class="font7" style="font-weight:bold;"><a name="bookmark3"></a>ABSTRACT</span></h1>
<p><span class="font7">The aim of the research is to investigate a simple method of in planta transformation method for T-DNA transfer in table grape. The T-DNA harbored the </span><span class="font7" style="font-style:italic;">S0SPS1</span><span class="font7"> gene under the control of promoter of the 35S </span><span class="font7" style="font-style:italic;">CaMV</span><span class="font7"> from the Cauliflower Mosaic Virus and contained the </span><span class="font7" style="font-style:italic;">NPTII </span><span class="font7">gene, a kanamycin-resistant gene as a selectable marker for transformant selection. Sixmonth plants originated from cuttings were used as target plants. We explored two methods of in planta transformation, namely ”dipping” and “sweeping”. For both methods, the leaves of the target plants were removed and those of shoots without leaves were used as the target of transformation. In the “dipping method”, those shoots were dipped with the agrobacterial suspension for 60 seconds. However, for the “sweeping method”, the scars (the spots where leaves were removed) were swept with agrobacterial suspension using a cotton bud. Those treated non-leafy-shoots (from both methods) then were grown to be leafy shoots. Those leafy shoots then were cut and transplanted into the soil and grown to be a whole plant. The leaves of those plants then were taken as samples for DNA extraction and PCR using primers of </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> gene (Forward: &nbsp;5’-GTCATCTCACCTTCCTCCTGCC-3’; Reverse: &nbsp;5’</span></p>
<p><span class="font7">GTCGCTTGGTCGGTCATTTCG-3’) with expected amplified band of 550 bp. We found that only the “sweeping method” plants amplified the 550 bp bands, while those of the “dipping method” did not. We suggest that the T-DNA was successfully integrated into the genome of plants treated with the “sweeping method” but not with the “dipping method”. Leaf sugar content (<sup>o</sup>Brix) of PCR-positive vines was higher than those of the wild-type vines, ensuring the integration of the T-DNA into the plant genome.</span></p>
<p><span class="font7" style="font-style:italic;">Keywords</span><span class="font7">: dipping, </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7">, sweeping, </span><span class="font7" style="font-style:italic;">S0SPS1</span><span class="font7">, T-DNA</span></p>
<h1><a name="bookmark4"></a><span class="font7" style="font-weight:bold;"><a name="bookmark5"></a>INTRODUCTION</span></h1>
<p><span class="font7">Genetic transformation is an efficient way to improve the characters of plants. The most common methods for the introduction of DNA into plant cells use </span><span class="font7" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font7"> or rapidly propelled tungsten microprojectiles that have been coated with DNA (Birch 1997; Hansen &amp;&nbsp;Wright 1999). However, we suggest that the cheapest and</span></p>
<p><span class="font7">simplest method is using </span><span class="font7" style="font-style:italic;">A. tumefaciens. </span><span class="font7">Genetic transformation method using </span><span class="font7" style="font-style:italic;">A. tumefaciens</span><span class="font7"> based on tissue culture (in vitro) methods has been reported elsewhere for some plant species such as a medicinal plant </span><span class="font7" style="font-style:italic;">Artemisia aucheri</span><span class="font7"> Boiss (Sharafi </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 2014), </span><span class="font7" style="font-style:italic;">Pisum sativu</span><span class="font7">m (Svabova </span><span class="font7" style="font-style:italic;">et a</span><span class="font7">l. 2005), tomato (El-Siddig </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 2009), Patchouli [</span><span class="font7" style="font-style:italic;">Pogostemon cablin</span><span class="font7"> (Blanco) Benth] ( Paul</span></p>
<p><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 2012), Orchid (Dwiyani </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 2010; Liau </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 2003; Semiarti </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 2007). However, those tissue culture based-methods are suggested to be time consuming and lead to soma clonal variation that affect both qualitative and quantitative characters of the plants (Labra </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 2004). The direct transformation method without any tissue culture steps is termed as </span><span class="font7" style="font-style:italic;">in-planta </span><span class="font7">transformation (Feldmann &amp;&nbsp;Marks 1987). The production of a large number of uniform plants in a short time, less labor efforts, and minimal reagents requirements are some of main the advantages of the </span><span class="font7" style="font-style:italic;">in-planta </span><span class="font7">transformation system (Bent 2000). In addition, tissue culture-based transformation methods require carefulness to maintain sterile condition, a common difficulty in the tissue culture work.</span></p>
<p><span class="font7">We explored </span><span class="font7" style="font-style:italic;">in-planta</span><span class="font7"> transformation of a table grape in order to provide a simple method of transformation that may be easier to be done compared to the </span><span class="font7" style="font-style:italic;">in-vitro</span><span class="font7"> method. </span><span class="font7" style="font-style:italic;">In-planta</span><span class="font7"> transformation method has been done for Arabidopsis by applying Agrobacterium to the Arabidopsis seeds as the target of transformation (Feldmann 1992; Feldmann &amp;&nbsp;Marks 1987) and a method of “clip ‘n squirt” using inflorescences as targeted cells (Chang </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 1994; Katavic </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 1994). </span><span class="font7" style="font-style:italic;">In-planta</span><span class="font7"> transformation method mediated with Agrobacterium was also reported for </span><span class="font7" style="font-style:italic;">Brassica napus</span><span class="font7"> L. (Li </span><span class="font7" style="font-style:italic;">et al.</span></p>
<p><span class="font7">2010), wheat (Razzaq </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 2011), pommelo (</span><span class="font7" style="font-style:italic;">Citrus maxima</span><span class="font7">) (Zhanga </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 2017) and Phalaenopsis orchid (Semiarti </span><span class="font7" style="font-style:italic;">et.al.</span><span class="font7"> 2014), but there is a lack of report for table grape. The current research explored two methods using </span><span class="font7" style="font-style:italic;">A. tumefaciens</span><span class="font7"> with non-leafy shoots (shoots whose leaves were removed) as targeted cells, namely ‘dipping method’ and ‘sweeping method’. The aim of the research was to investigate a simple method of </span><span class="font7" style="font-style:italic;">inplanta</span><span class="font7"> transformation for the table grape. The current method would be easier to be done and may be adopted for other species.</span></p>
<h1><a name="bookmark6"></a><span class="font7" style="font-weight:bold;"><a name="bookmark7"></a>MATERIALS AND METHODS</span></h1>
<p><span class="font7">We used a Sucrose Phospate Syntase (</span><span class="font7" style="font-style:italic;">S0SPS1</span><span class="font7">) gene that was obtained from Prof. Bambang Sugiharto, the Director of the Center Development of Advanced Sciences and Technology (CDAST), Jember University. The T-DNA harbored the </span><span class="font7" style="font-style:italic;">S0SPS1</span><span class="font7"> gene under the control of the 35S </span><span class="font7" style="font-style:italic;">CaMV</span><span class="font7"> promoter and the </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> gene, a kanamycin resistant gene as a selectable marker for transformant selection. This T-DNA was constructed in pKYS plasmid and it was cloned to </span><span class="font7" style="font-style:italic;">A. tumefaciens</span><span class="font7"> strain LBA4404. The experiment was conducted from January to October 2017 at the Laboratory of Plant Tissue Culture, the Faculty of Agriculture, Udayana University and the Laboratory of CDAST, Jember</span></p>
<p><span class="font7">University. The T-DNA construct is shown in Fig. 1.</span></p>
<div>
<p><span class="font3" style="font-weight:bold;">LB &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RB</span></p><img src="https://jurnal.harianregional.com/media/52066-1.jpg" alt="" style="width:389pt;height:47pt;">
<p><span class="font7" style="font-weight:bold;">Fig.1</span><span class="font7">. The structure of T-DNA in pKYS plasmid harbored </span><span class="font7" style="font-style:italic;">nptII</span><span class="font7"> gene and </span><span class="font7" style="font-style:italic;">S0SPS1</span><span class="font7"> gene. Right border, RB; Left border, LB; Promoter of the nopaline synthase gene, Pnos;</span></p>
<p><span class="font7">Polyadenylation site of the nopaline synthase gene, Tnos; Neomycin Phospotransferase gene, </span><span class="font7" style="font-style:italic;">nptII</span><span class="font7">; 35S promoter of Cauliflower Mosaic Virus, </span><span class="font7" style="font-style:italic;">P35SCaMV</span><span class="font7">; Sucrose Phosphate Synthase Gene, </span><span class="font7" style="font-style:italic;">S0SPS1 </span><span class="font7">(Source: Sugiharto </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 1997)</span></p>
</div><br clear="all">
<p><span class="font7">The preparation of Agrobacterium suspension was done using a method mentioned in Dwiyani (2012). A single colony of </span><span class="font7" style="font-style:italic;">A.tumefaciens</span><span class="font7"> that harbored the gene was cultured on 2 ml of liquid YEP Medium which contained 50 ppm kanamycin, 100 ppm gentamycin and 100 ppm ryfamphycin for 24 hours until the 0.6 Optical Density (OD) was reached. Then, 1 ml of the bacterial culture was added with the same medium until the volume of 20 ml for subsequent culture in the shaker until the OD of 0.6 was accomplished. The culture then was centrifuged with 1000 rpm for 5 minutes. The supernatant was removed and replaced by ½ MS liquid medium. The agrobacterial suspension was then added with 100 ppm acetosyringone and 2µL tween and was used for </span><span class="font7" style="font-style:italic;">in-planta</span><span class="font7"> transformation. The suspension was diluted ten times with ½ MS liquid medium before it was used.</span></p>
<p><span class="font7">Six-month of table grape (</span><span class="font7" style="font-style:italic;">Vitis vinifera</span><span class="font7"> L. var. Alphonse lavallee) plants originated from cuttings were used as target plants. We selected forty (40) healthy shoots from twenty (20) plants.. The leaves of the shoots were trimmed and removed. The purpose of removing the leaves was to allow buds in the leaf axil to grow to be new shoots. Besides that, the scars (spots where the leaves were removed) provided channels for Agrobacterium to enter the plant cells. The non-leafy shoots were then used as the target of the in-planta transformation (Fig. 2). We explored two methods in the current research, namely the “dipping method” and “sweeping method”. In the dipping method, the non-leafy shoots were dipped with the Agrobacterial suspension for 120 seconds (Fig. 3). However, for the “sweeping method”, the scars were swept with the Agrobacterial suspension using a cotton bud (Fig. 4). After 7 days of treatment, leaves</span></p>
<p><span class="font7">emerged and grew to be new shoots. At 14 &nbsp;grew to be a whole plant in the two (2)</span></p>
<p><span class="font7">days of treatment, the new shoots were cut &nbsp;months after the treatment (Fig. 5).</span></p>
<p><span class="font7">and transplanted in-to the soil. Those shoots</span></p><img src="https://jurnal.harianregional.com/media/52066-2.jpg" alt="" style="width:173pt;height:147pt;">
<p><span class="font7" style="font-weight:bold;">Fig. 2</span><span class="font7">. The non-leafy shoots (red arrows) as the target of </span><span class="font7" style="font-style:italic;">in planta</span><span class="font7"> transformation</span></p><img src="https://jurnal.harianregional.com/media/52066-3.jpg" alt="" style="width:334pt;height:121pt;">
<p><span class="font7" style="font-weight:bold;">Fig. 3</span><span class="font7">. Dipping of the non-leafy shoots on the “Dipping method”</span></p><img src="https://jurnal.harianregional.com/media/52066-4.jpg" alt="" style="width:400pt;height:118pt;">
<p><span class="font7" style="font-weight:bold;">Fig. 4</span><span class="font7">. Sweeping the scars (spots where leaves were removed) with Agrobacterial suspension using a cotton bud on the “sweeping method”</span></p>
<div><img src="https://jurnal.harianregional.com/media/52066-5.jpg" alt="" style="width:351pt;height:154pt;">
<p><span class="font7" style="font-weight:bold;">Fig. 5</span><span class="font7">. Young plants originated from the cutting of the new shoots after treated with agrobacterial suspension; A= young plant from the dipping method; B=young plant from the sweeping method; bar=3cm</span></p>
</div><br clear="all">
<h1><a name="bookmark8"></a><span class="font7" style="font-weight:bold;"><a name="bookmark9"></a>RESULTS AND DISCUSSION</span></h1>
<p><span class="font7">Before we did the transformation, we confirmed the gene of interest inside the Agrobaterium. Ten (10) single colonies of Agrobacterium were tested. The results can be seen in Fig. 6. In this image, the gene is still harbored by all of the single colonies</span></p>
<p><span class="font7">tested. They are indicated by 550bp length of band amplified using </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> primers (Forward: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’-</span></p>
<p><span class="font7">GTCATCTCACCTTCCTCCTGCC-3’;</span></p>
<p><span class="font7">Reverse: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5’</span></p>
<p><span class="font7">GTCGCTTGGTCGGTCATTTCG-3’).</span></p>
<div>
<p><span class="font3" style="font-weight:bold;">1 &nbsp;&nbsp;2 &nbsp;3 &nbsp;4 &nbsp;&nbsp;5 &nbsp;6 &nbsp;7 &nbsp;8 &nbsp;9 &nbsp;10 M</span></p>
<p><span class="font2" style="font-weight:bold;">550 bp</span></p><img src="https://jurnal.harianregional.com/media/52066-6.jpg" alt="" style="width:250pt;height:188pt;">
<p><span class="font2" style="font-weight:bold;">3000 bp</span></p>
<p><span class="font2" style="font-weight:bold;">1000 bp</span></p>
<p><span class="font2" style="font-weight:bold;">500 bp</span></p>
<p><span class="font7" style="font-weight:bold;">Fig. 6</span><span class="font7">. The gene confirmation in the </span><span class="font7" style="font-style:italic;">A. tumefaciens</span><span class="font7"> using </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> primers. P = plasmid; 1-10=single colonies of </span><span class="font7" style="font-style:italic;">A. tumefaciens</span><span class="font7">; M=DNA marker of 1 kb DNA ladder.</span></p>
</div><br clear="all">
<p><span class="font7">We used 40 shoots from 20 table grape plants for the transformation. Twenty (20) shoots were treated with the dipping method and the other twenty were treated with the sweeping method. However, after the new shoots were cut and transplanted; only nine were successfully grown to be young plants, five were from the dipping method and four were from the sweeping method. At the first analysis, we used only four plants as samples for the PCR analysis, </span><span class="font7" style="font-style:italic;">i.e</span><span class="font7">. 1 control plant, 2 plants originated from the shoots treated with the dipping method, and 1 plant originated from the shoots treated with the sweeping method. Without the selection process with kanamycin, the leaves of those plants were taken as samples for DNA extraction (Doyle &amp;&nbsp;Doyle 1990) and</span></p>
<p><span class="font7">then was PCR analyzed. In the current research, we verified the integration of T-DNA containing </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> gene in the table grape using primers for </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> gene (Forward: 5’-GTCATCTCACCTTCCTCCTGCC-3’;</span></p>
<p><span class="font7">Reverse: 5’ GTCGCTTGGTCGGTCATTTCG-3’) with expected amplified band of 550 bp. Fig. 7 shows the electrophoresis graph of the PCR results. We found that only the plant originated from the shoots of the sweeping method amplified the 550 bp band, indicating that the gene may integrated into the genome of the plant originated from the sweeping method, but did not occur with plants from the dipping method.</span></p>
<div>
<p><span class="font4" style="font-weight:bold;">P</span></p>
<p><span class="font4" style="font-weight:bold;">M</span></p>
<p><span class="font4" style="font-weight:bold;">3000 b</span></p>
<p><span class="font4" style="font-weight:bold;">1000</span></p><img src="https://jurnal.harianregional.com/media/52066-7.jpg" alt="" style="width:241pt;height:208pt;">
<p><span class="font4" style="font-weight:bold;">550</span></p>
<p><span class="font7" style="font-weight:bold;">Fig. 7</span><span class="font7">. The electrophoresis graph of PCR analysis using the </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> primers.</span></p>
<p><span class="font7">P=plasmid; M= DNA marker of 1 kb DNA ladder; C= control plant; 1-2 = plants of the dipping method; 3 = plant of the sweeping method</span></p>
</div><br clear="all">
<p><span class="font7">To ensure the results, then we did PCR for the other putative transformed plants. Two plants were from the sweeping method and one was from the dipping method. The PCR result is shown in Fig. 8.</span></p>
<p><span class="font7">The second PCR analysis again confirmed that the transgene was inserted into the plant genome of the sweeping method plant, but it did not occur with those of the dipping method.</span></p>
<div>
<p><span class="font1" style="font-weight:bold;">1 &nbsp;&nbsp;2 M P 3</span></p><img src="https://jurnal.harianregional.com/media/52066-8.jpg" alt="" style="width:105pt;height:168pt;">
<p><span class="font1" style="font-weight:bold;">3000 bp</span></p>
<p><span class="font1" style="font-weight:bold;">1000 bp</span></p>
<p><span class="font1" style="font-weight:bold;">550 bp</span></p>
</div><br clear="all">
<p><span class="font7" style="font-weight:bold;">Fig. 8</span><span class="font7">. The electrophoresis graph of the second PCR analysis for other transformed plants using primers of the </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> gene. 1-2 = plants of the sweeping method; 3= plant of the dipping method; P= plasmid; M= DNA marker of 1 kb DNA ladder.</span></p>
<p><span class="font7">We checked also the sucrose content of leaves of the PCR-positive vines and wildtype vines with hand-refractometer as the </span><span class="font7" style="font-style:italic;">S0SPS1</span><span class="font7"> gene was included in the T-DNA construct. The leaves were crushed with</span></p>
<p><span class="font7">mortar, and 1 µL of the extract was diluted 5 times by adding with 4 µL of distilled water and then measured with hand-refractometer. The result is in the Table 1.</span></p>
<div>
<p><span class="font7">Table 1. Leaf Sugar Content (<sup>o</sup>Brix) of PCR-PositiveVines and Wild Type vines (measured with Hand–refractometer)</span></p>
</div><br clear="all">
<div>
<p><span class="font5">Treatment</span></p>
<p><span class="font5">Wild Type Vines</span></p>
</div><br clear="all">
<div>
<p><span class="font5">Sugar Content (<sup>o</sup>Brix)</span></p>
<p><span class="font5">Vine 1 : 2.8</span></p>
<p><span class="font5">Vine 2 : 2.0</span></p>
<p><span class="font5">Vine 3 : 2.8</span></p>
<p><span class="font2">Vine 4 : 2.9</span></p>
</div><br clear="all">
<div>
<p><span class="font5">PCR-Positive Vines</span></p>
</div><br clear="all">
<div>
<p><span class="font5">Vine 1 : 5.8</span></p>
<p><span class="font5">Vine 2 : 5.4</span></p>
<p><span class="font5">Vine 3 : 5.2</span></p>
<p><span class="font5">Vine 4 : 5.8</span></p>
<p><span class="font5">Vine 5 : 5.1</span></p>
<p><span class="font2">Vine 6 : 5.3</span></p>
</div><br clear="all">
<p><span class="font7">The data in the Table 1 suggested that the </span><span class="font7" style="font-style:italic;">S0SPS1</span><span class="font7"> gene might be inserted in to the plant genome and was functionally worked on the putative vines. As we know that the </span><span class="font7" style="font-style:italic;">Sucrose Phosphate Synthase</span><span class="font7"> is a key enzyme for sucrose biosynthesis in plants (Bruneau </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">., 1991).</span></p>
<p><span class="font7">When scars (spots where leaves were removed) were swept with the Agrobacterium solution, the Agrobacterium might enter the plant cells, and the T-DNA was integrated into the plant genome. Thus the new shoot emerging from the node carried the gene of interest. The new shoot was cut and grown to be a whole plant, and we suggest it as a candidate of transformant. All transformants produced from this method (4 plants) confirmed the existence of the </span><span class="font7" style="font-style:italic;">NPTII</span><span class="font7"> gene. On the contrary, it did not occur in the dipping method, event when the scars were dipped for 120 seconds. The T-DNA failed to integrate into the plant genome, and the method did not produce any transformant. However, we need further research to increase immersion time to allow the chance of gene integration. We also need to confirm the cimeras and ensure that the gene was integrated in the whole plant.</span></p>
<p><span class="font7">Transferred DNA (T-DNA) enters the plant as a single stranded molecule (Stachel </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 1986; Tinland </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 1994; Yusibov </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 1994) that may eventually integrate into the nuclear genome. Although T-DNA</span></p>
<p><span class="font7">integration is random (Kim </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 2007), the mechanism of integration remains unclear (Park </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 2015). However, irradiated protoplasts show a higher DNA integration frequency than do non-irradiated protoplasts (Kohler </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 1989), suggesting that double stranded (ds)DNA damage sites could be targets of T-DNA integration (Park </span><span class="font7" style="font-style:italic;">et al. </span><span class="font7">2015). Indeed, T-DNA molecules preferentially integrate into dsDNA break sites (Chilton and Que, 2003; Salomon and Puchta, 1998; Tzfira </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 2003). In the current research, the touching of a cotton bud into the shoot scars in the sweeping method may produce dsDNA damage sites, providing a chance of T-DNA to integrate into the plant genome.</span></p>
<p><span class="font7">The transformation method for table grape had been regenerated in vitro (Dabauza &amp;&nbsp;Velasco 2012; Mezzetti </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. 2005), however, the </span><span class="font7" style="font-style:italic;">in planta</span><span class="font7"> method for table grape has been rarely studied. Fujita </span><span class="font7" style="font-style:italic;">et al. </span><span class="font7">(2009) did successful </span><span class="font7" style="font-style:italic;">in planta </span><span class="font7">transformation on table grape cultivars (Chardonnay and Cabernet Sauvignon). They used dormant buds on cuttings as the target of the transformation. The dormant buds were pricked four times with a needle and dipped into the </span><span class="font7" style="font-style:italic;">A. tumefaciens</span><span class="font7"> solution overnight. It was different from the current method. Although we had not done the southern blot yet for gene confirmation, however, this preliminary research suggested</span></p>
<p><span class="font7">that the current method of “sweeping method” was a novel simple method of </span><span class="font7" style="font-style:italic;">in planta</span><span class="font7"> transformation for T-DNA transfer in table grape, a similar protocol has not been reported elsewhere. The findings may guide future efforts to improve the transformation of other plant species.</span></p>
<h1><a name="bookmark10"></a><span class="font7" style="font-weight:bold;"><a name="bookmark11"></a>CONCLUSIONS</span></h1>
<p><span class="font7">The current preliminary research of </span><span class="font7" style="font-style:italic;">in planta</span><span class="font7"> transformation method for T-DNA transfer in table grape was suggested as a novel simple method in transferring the T-DNA into the plant genome. The method named “sweeping method”. It was done by sweeping the scars of shoots without leaves (the non-leafy shoots) with agrobacterial suspension. The production of transformed plant was done by cutting the new emerging shoots from the swept scars and those were then grown to be new plants.</span></p>
<h1><a name="bookmark12"></a><span class="font7" style="font-weight:bold;"><a name="bookmark13"></a>ACKNOWLEDGEMENTS</span></h1>
<p><span class="font7">The authors gratefully acknowledge the financial support provided by the Udayana University (Denpasar, Bali, Indonesia) through Competitive Grant Project (</span><span class="font7" style="font-style:italic;">Hibah Unggulan Udayana</span><span class="font7">, 2017). The authors also acknowledge Prof. Bambang Sugiharto from the Centre Development of Advanced Sciences and Technology (CDAST), Jember University, for facilitating the molecular research.</span></p>
<h1><a name="bookmark14"></a><span class="font7" style="font-weight:bold;"><a name="bookmark15"></a>REFERENCES</span></h1>
<p><span class="font7">Bent, A. F. (2000). Arabidopsis </span><span class="font7" style="font-style:italic;">in Planta </span><span class="font7">Transformation. Uses, Mechanisms, and Prospects for Transformation of Other Species. Plant Physiol. 124 : 1540-1547</span></p>
<p><span class="font7">Birch, R. G. (1997). Plant transformation: problems and strategies for practical application. Annu. Rev. Plant Physiol. Plant Mol. Biol. 48: 297–326</span></p>
<p><span class="font7">Bruneau, J. M., Worrell, A. C., Cambou, B., Lando, D., and Voelker T. A. (1991). Sucrose Phosphate Synthase, a Key Enzyme for Sucrose Biosynthesis in Plants. Plant Physiol. 96(2): 473–478</span></p>
<p><span class="font7">Chang, S. S., Park, S. K., Kim, B. C., Kang, B. J., Kim, D. U., and Nam, H. G. (1994). Stable genetic transformation of &nbsp;&nbsp;</span><span class="font7" style="font-style:italic;">Arabidopsis &nbsp;&nbsp;&nbsp;thaliana</span><span class="font7"> by</span></p>
<p><span class="font7">Agrobacterium inoculation </span><span class="font7" style="font-style:italic;">in planta</span><span class="font7">. Plant J. 5: 551–558</span></p>
<p><span class="font7">Chilton, M. D., and Que, &nbsp;Q. &nbsp;(2003).</span></p>
<p><span class="font7">Targeted integration of T-DNA into the tobacco genome &nbsp;at &nbsp;double</span></p>
<p><span class="font7">stranded breaks: new insights on the mechanism of T-DNA integration. Plant Physiol. 133:956–965.</span></p>
<p><span class="font7">Dabauza, M., and Velasco, L. (2012). Development &nbsp;of highly efficient</span></p>
<p><span class="font7">genetic transformation protocols for table grape Sugraone and Crimson Seedless. Methods Mol. Biol. 847: 227-235</span></p>
<p><span class="font7">Doyle, J. J., and Doyle, J. L. (1990). Isolation of plant DNA from fresh tissue. Focus 12: 13-15.</span></p>
<p><span class="font7">Dwiyani, R., Purwantoro, A., Indrianto, A., and Semiarti, E. (2010). Improvement of Genetic Transformation Efficiency in </span><span class="font7" style="font-style:italic;">Vanda tricolor</span><span class="font7"> Orchid Using Acetosyringone. Annales Bogoriense 14 (2): 27-32</span></p>
<p><span class="font7">Dwiyani, R. (2012). Mikropropagasi tanaman anggrek </span><span class="font7" style="font-style:italic;">Vanda tricolor</span><span class="font7"> Lindl. var </span><span class="font7" style="font-style:italic;">suavis</span><span class="font7"> yang membawa gen </span><span class="font7" style="font-style:italic;">Knotted 1-like Arabidopsis thaliana (KNAT1)</span><span class="font7">. Disertasi. Sekolah Pascasarjana</span></p>
<p><span class="font7">Universitas Gadjahmada, Indonesia (</span><span class="font7" style="font-style:italic;">In Indonesian</span><span class="font7">)</span></p>
<p><span class="font7">El-Siddig, M., El-Hussein, A., Siddig, M., Elballa, M., and Saker, M. (2009). Agrobacterium-Mediated Transformation and in Vitro Regeneration &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;of &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tomato</span></p>
<p><span class="font7">.(</span><span class="font7" style="font-style:italic;">Lycopersicon Esculentum</span><span class="font7"> Mill.) Plants Cv. Castlerock. The Journal of Genetic &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Engineering &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and</span></p>
<p><span class="font7">Biotechnology 7(1): 1-6</span></p>
<p><span class="font7">Feldmann, K. (1992). T-DNA insertion mutagenesis in Arabidopis: seed infection transformation. In Koncz, C., Chua, N-H., and J. Schell, eds. Methods &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;in &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Arabidopsis</span></p>
<p><span class="font7">Research.World &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Scientific,</span></p>
<p><span class="font7">Singapore. pp 274–289</span></p>
<p><span class="font7">Feldmann, K. A., and Marks, M. D. (1987). Agrobacterium-mediated transformation of germinating seeds of </span><span class="font7" style="font-style:italic;">Arabidopsis thaliana</span><span class="font7">: a non-tissue culture approach. Mol. Gen. Genet. 208: 1–9</span></p>
<p><span class="font7">Fujita, K., Matsuoka, T., Suzuki, S., and Takayanagi, T. (2009). In Planta Transformation Technique for Grapevines (</span><span class="font7" style="font-style:italic;">Vitis vinifera</span><span class="font7"> L) using Dormant Buds. J. Plant Biochem. and Biotech. 18 (2): 161-167</span></p>
<p><span class="font7">Hansen, G., and Wright, M. S. (1999). Recent advances in the transformation of plants. Trends Plant Sci. 4: 226– 231</span></p>
<p><span class="font7">Katavic, V., Haughn, G. W., Reed, D.,</span></p>
<p><span class="font7">Martin, M., and Kunst, L. (1994). </span><span class="font7" style="font-style:italic;">In planta</span><span class="font7"> transformation of </span><span class="font7" style="font-style:italic;">Arabidopsis thaliana</span><span class="font7">. Mol. Gen. Genet. 245: 363– 370</span></p>
<p><span class="font7">Kim, S. I., Veena, and Gelvin, S. B. (2007). Genome-wide &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;analysis &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;of</span></p>
<p><span class="font7">Agrobacterium T-DNA integration sites in the Arabidopsis genome generated under non-selective conditions. Plant J. 51: 779–791.</span></p>
<p><span class="font7">Kohler, F., Cardon, G., Pohlman, M., Gill, R., and Schieder, O. (1989). Enhancement of transformation rates in higher plants by low-dose</span></p>
<p><span class="font7">irradiation: are DNA repair systems involved in the incorporation of exogenous DNA into the plant genome? Plant Mol. Biol. 12: 189– 199.</span></p>
<p><span class="font7">Labra, M., Vannini, C., Grassi, F., Bracale, M., Balsemin, M., Basso, B., and Sala, F. (2004). Genomic stability in </span><span class="font7" style="font-style:italic;">Arabidopsis thaliana</span><span class="font7"> transgenic plants obtained by floral dip. Theor. Appl. Genet.109 (7):1512-1518</span></p>
<p><span class="font7">Li, J., Tan, X., Zhu, F., and Guo, J. (2010). A Rapid and Simple Method for Brassica &nbsp;&nbsp;&nbsp;&nbsp;Napus &nbsp;&nbsp;&nbsp;&nbsp;Floral-Dip</span></p>
<p><span class="font7">Transformation and Selection of Transgenic Plantlets. International J. of Biol. 2 (1) :127-131</span></p>
<p><span class="font7">Liau, C. H., You, S. J., Prasad, V., Hsiao, H. H., Lu, J. C., Yang. N. S., and Chan, M. S. (2003). </span><span class="font7" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font7">-mediated transformation of An </span><span class="font7" style="font-style:italic;">Oncidium</span><span class="font7"> Orchid. Plant Cell Reports 21 : 993-998</span></p>
<p><span class="font7">Mezzetti, B., Silvestroni, O., Costantini, E., Pandolfini, T., and Spena, A. (2005). Genetic transformation of table grape via organogenesis and field evaluation of </span><span class="font7" style="font-style:italic;">defH9-iaaM</span><span class="font7"> transgenic plants. Acta Horticulturae 689: 463468</span></p>
<p><span class="font7">Park, S. Y., Vaghchhipawala, Z., Vasudevan, B., Lee, L. Y., Shen, Y., Singer, K., Waterworth, W. M., Zhang, Z. J., West, C. E., Mysore, K. S., and Gelvin, B. (2015). Agrobacterium T-DNA integration into the plant genome can occur without the activity of key non-homologous end-joining proteins. The Plant J. 81: 934–946</span></p>
<p><span class="font7">Paul, A., Bakshi, S., Sahoo, D. P., Kalita, M.C., and Sahoo, L. &nbsp;&nbsp;(2012).</span></p>
<p><span class="font7">Agrobacterium-mediated &nbsp;&nbsp;&nbsp;&nbsp;genetic</span></p>
<p><span class="font7">transformation of </span><span class="font7" style="font-style:italic;">Pogostemon cablin </span><span class="font7">(Blanco) Benth. Using leaf explants: bactericidal effect of leaf extracts and counteracting strategies. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Appl.</span></p>
<p><span class="font7">Biochem. Biotechnol. 166 (8):1871-95.</span></p>
<p><span class="font7">Razzaq, A., Hafiz, I. A., Mahmood, I., and Hussain, A. (2011). Development of </span><span class="font7" style="font-style:italic;">in planta</span><span class="font7"> transformation protocol for wheat. African J. Biotech. 10 (5): 740-750</span></p>
<p><span class="font7">Salomon, S., and Puchta, H. 1998. Capture of genomic and T-DNA sequences during double-strand break repair in somatic plant cells. EMBO J. 17: 6086–6095.</span></p>
<p><span class="font7">Semiarti, E., Indrianto, A., Purwantoro, A., Isminingsih, S., Suseno, N., Ishikawa, N. T., Yoshioka, Y., Machida, Y., and Machida, C. (2007). Agrobacterium-mediated transformation of the wild orchid species </span><span class="font7" style="font-style:italic;">Phalaenopsis amabilis</span><span class="font7">. Plant Biotechnol..24 : 265-272.</span></p>
<p><span class="font7">Semiarti, E., Purwantoro, A., Mercuriani, I. S., Anggriasari, A. M., Jang, S., Suhandono, S., Machida, Y., and Machida, C. (2014). In planta transformation method for T-DNA transfer in orchids. &nbsp;&nbsp;&nbsp;&nbsp;In: AIP</span></p>
<p><span class="font7">Conference Proceedings 2014 Volume 1589 (Issue 1): 303-307</span></p>
<p><span class="font7">Sharafi, A., Sohi, H. H., Mirzaee, H., and Azadi, P. (2014). In vitro regeneration and Agrobacterium mediated genetic transformation of Artemisia aucheri Boiss. Physiol Mol Biol Plants 20(4):487–494</span></p>
<p><span class="font7">Stachel, S. E., Timmerman, B., and Zambryski, P. (1986). Generation of single-stranded T-DNA molecules during the initial stages of T-DNA transfer &nbsp;&nbsp;&nbsp;from </span><span class="font7" style="font-style:italic;">Agrobacterium</span></p>
<p><span class="font7" style="font-style:italic;">tumefaciens</span><span class="font7"> to plant cells. Nature 322: 706–712.</span></p>
<p><span class="font7">Sugiharto, B., Sakakibara, Sumadi, and Sugiyama. (1997). Differential Expression of Two Genes for Sucrose-Phosphate Synthase in Sugarcane: Molecular Cloning of the cDNAs and Comparative Analysis of Gene Expression. J. Plant Cell Physiol. 38: 961-965.</span></p>
<p><span class="font7">Svabova, L., Smykal, P., Griga, M., and Ondreji, V. (2005). Agrobacterium-mediated transformation of </span><span class="font7" style="font-style:italic;">Pisum</span></p>
<p><span class="font7" style="font-style:italic;">sativum</span><span class="font7">. Biologia Plantarum 49 (3): 361-370</span></p>
<p><span class="font7">Tinland, B., Hohn, B., and Puchta, H. (1994). </span><span class="font7" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font7"> transfers single-stranded transferred DNA (T-DNA) into the plant cell nucleus. Proc. Natl. Acad. Sci. USA 91: 8000– 8004.</span></p>
<p><span class="font7">Tzfira, T., Frankman, L. R., Vaidya, M., and Citovsky, V. (2003). Site-specific integration of </span><span class="font7" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font7"> T-DNA via doublestranded intermediates. Plant Physiol. 133: 1011–1023.</span></p>
<p><span class="font7">Yusibov, V. M., Steck, T. R., Gupta, V., and Gelvin, S. B. (1994). Association of single-stranded transferred DNA from </span><span class="font7" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font7"> with tobacco cells. Proc. Natl. Acad. Sci. USA, 91: 2994–2998.</span></p>
<p><span class="font7">Zhanga, Y., Zhanga, D., Zhong, Y., Changa, X., Hub M., and Chenga, C. (2017). A simple and efficient in planta transformation method for pommelo (</span><span class="font7" style="font-style:italic;">Citrus maxima</span><span class="font7">) using </span><span class="font7" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font7">. &nbsp;Scientia Horticulturae</span></p>
<p><span class="font7">214: 174–179</span></p>
<p><span class="font0">FACULTY OF AGRICULTURE, UDAYANA UNIVERSITY • 105</span></p>