---
layout: full_article
title: "THE LACTIC ACID BACTERIA OF BEBONTOT SPENT CHICKEN MEAT AND ANTIOXIDANT ACTIVITY OF THEIR ISOLATES"
author: "I. A Okarini, Heri Purnomo, Aulanni'am -, Liliek Eka Radiati, S. A Lindawati, I N.S Miwada"
categories: tropika
canonical_url: https://jurnal.harianregional.com/tropika/full-56840 
citation_abstract_html_url: "https://jurnal.harianregional.com/tropika/id-56840"
citation_pdf_url: "https://jurnal.harianregional.com/tropika/full-56840"  
comments: true
---

<div><img src="https://jurnal.harianregional.com/media/56840-1.jpg" alt="" style="width:65pt;height:63pt;">
<p><span class="font0" style="font-weight:bold;">e-journal FAPET UNUD</span></p>
</div><br clear="all">
<p><span class="font2" style="font-weight:bold;">e-Journal</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font3" style="font-weight:bold;"><a name="bookmark1"></a>Peternakan Tropika</span></h1>
<h2><a name="bookmark2"></a><span class="font9" style="font-weight:bold;"><a name="bookmark3"></a>Journal of Tropical Animal Science</span></h2>
<p><span class="font6" style="font-weight:bold;">email: </span><a href="mailto:peternakantropika@yahoo.com"><span class="font6" style="text-decoration:underline;">peternakantropika@yahoo.com</span></a></p>
<div><img src="https://jurnal.harianregional.com/media/56840-2.jpg" alt="" style="width:59pt;height:69pt;">
</div><br clear="all">
<p><span class="font1" style="font-style:italic;">Submitted Date: January 26, 2020</span></p>
<div>
<p><span class="font1" style="font-style:italic;">Accepted Date: January 28, 2020</span></p>
</div><br clear="all">
<p><span class="font1" style="font-style:italic;">Editor-Reviewer Article;: A.A.Pt. Putra Wibawa &amp;&nbsp;I Wyn. Wirawan</span></p>
<h2><a name="bookmark4"></a><span class="font8" style="font-weight:bold;"><a name="bookmark5"></a>THE LACTIC ACID BACTERIA OF </span><span class="font8" style="font-weight:bold;font-style:italic;">BEBONTOT</span><span class="font8" style="font-weight:bold;"> SPENT CHICKEN MEAT AND ANTIOXIDANT ACTIVITY OF THEIR ISOLATES</span></h2>
<p><span class="font7">Okarini I A<sup>1</sup>, Hari Purnomo<sup>2</sup>,Aulanni’am<sup>3</sup>, Liliek Eka Radiati<sup>2</sup>, S A Lindawati<sup>1</sup> and I N S Miwada<sup>1</sup></span></p>
<p><span class="font7"><sup>1</sup>Faculty of Animal Husbandry, Udayana University, Denpasar, Bali, Indonesia. </span><a href="mailto:i.aokarini@gmail.com"><span class="font7" style="text-decoration:underline;">i.aokarini@gmail.com</span><span class="font7">.</span></a><span class="font7"> +6281916541777</span></p>
<p><span class="font7"><sup>2</sup>Department of Animal Food Technology, Faculty of Animal Husbandry, Brawijaya University, Malang, Jawa Timur,Indonesia.</span><a href="mailto:purnomohari1120@gmail.com"><span class="font7"> </span><span class="font7" style="text-decoration:underline;">purnomohari1120@gmail.com</span><span class="font7">,</span></a><a href="mailto:lilieks_pdi@yahoo.com"><span class="font7"> </span><span class="font7" style="text-decoration:underline;">lilieks_pdi@yahoo.com</span></a><span class="font7" style="text-decoration:underline;"> </span><span class="font7"><sup>3</sup>Department of Biochemistry, Faculty of Mathematics and Natural Sciences, Brawijaya University, Malang</span></p>
<p><span class="font7">Jawa Timur,Indonesia. Email:</span><a href="mailto:aulani@ub.ac.id"><span class="font7">aulani@ub.ac.id</span></a></p>
<p><span class="font7" style="font-weight:bold;">Highlights</span><span class="font7" style="font-weight:bold;font-style:italic;">* </span><span class="font7" style="font-style:italic;">Bebontot or buntilan of spent chicken a novel Balinese traditional meat fermented product * Species-specific PCR assays were used to confirm the identity of the LAB strains, discriminated 63 profiles out of 72 LAB isolates </span><span class="font7" style="font-weight:bold;font-style:italic;">* </span><span class="font7" style="font-style:italic;">Pediococcus pentosaceus was dominant, followed by Lactobaccillus plantarum were the main species of bebontot.* Lactobaccillus plantarum had strong radical scavenging activities of 1.1-Diphenyl-2-picrylhydrazyl, DPPH than Pediococcus pentosaceus</span></p>
<p><span class="font7" style="font-weight:bold;">Abstract</span><span class="font7">* </span><span class="font7" style="font-style:italic;">The lactic acid bacteria (LAB) of bebontot or buntilan (Balinese traditional meat fermented product) of chicken breast meat of spent laying hen had been identified and these LAB were distributed as 58.33% Pediococcus acidilactici strain LMG 17680 and 41.67% Pediococcus acidilactici strain O-mls-1 at 0 day of the batters. The result observed at the end of fermentation (5 days) dried under the sun were isolate consist of 56.25% Pediococcus pentosaceus strain Ni 1386; 20.83% Lactobaccillus plantarum strain PA21 and 8.33% Lactobacillus plantarum strain Ni 1002. The average inhibition concentration (IC) of radical scavenging activity (RSA) of 1.1-diphenyl-2-picrylhydrazyl, DPPH after innoculation (0 day) was 66.83±3.55% and after 5days of fermentation was 83.37±4.06%. It is interesting to note that LAB isolates of bebontot spent-hen chicken breast meat of properness (nor isolated from human or infant feces) and could be benefit for antioxidant activity to process functional meat products for human consumption.</span></p>
<p><span class="font7" style="font-weight:bold;">Keywords: </span><span class="font7">Identification, antioxidant activity, LAB isolates, </span><span class="font7" style="font-style:italic;">bebontot</span></p>
<div><img src="https://jurnal.harianregional.com/media/56840-3.jpg" alt="" style="width:49pt;height:32pt;">
</div><br clear="all">
<p><span class="font7">technological means [12]. On the other hand [10] reported that the proteolyses system of LAB can contribute to release of health-enhancing bioactive peptides from food proteins, which generated to the fermentation and ripening process of meat by LAB, and they have bioactivities for functional foods. </span><span class="font7" style="font-style:italic;">Bebontot</span><span class="font7"> or </span><span class="font7" style="font-style:italic;">buntilan</span><span class="font7"> is a Balinese traditional meat fermented product, and this mostly prepared using air drying by traditional methods in small-scale enterprises. Generally it was made from mixed of lean meat and fat pork meat, chopped into 1.5-2.0 cm cubes, and mixed with fresh ground spices consist of turmeric, galangal, aromatic ginger, ginger, garlic, chilies, pepper and salt [13]; and in other region of Bali a difference mix spices used to produce </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7">, from pork with lard added, where it is also related to the environmental drying condition [14]. However there are no reports on identification of LAB isolate from </span><span class="font7" style="font-style:italic;">bebontot </span><span class="font7">spent chicken breast meat and hence the aims of this study were to find out the LAB strains involved in fermentation process from time of innoculation (0 day) and during at 5 days drying and to determine their antioxidant activity.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font7" style="font-weight:bold;">2. &nbsp;&nbsp;&nbsp;Materials and methods</span></p>
<ul style="list-style:none;">
<li>
<h3><a name="bookmark6"></a><span class="font7" style="font-weight:bold;"><a name="bookmark7"></a>2.1. &nbsp;&nbsp;&nbsp;Bebontot </span><span class="font7" style="font-weight:bold;font-style:italic;">chicken breast meat preparation.</span></h3></li></ul></li></ul>
<p><span class="font7">Skinless breast fillet (</span><span class="font7" style="font-style:italic;">Musculus Pectoralis Superficialis,</span><span class="font7"> left and right breast muscles of each bird ) of spent laying chicken (ISA-brown) after a period of laying of 76 weeks from an egg farm in Bali region were used in this study. The skinless breast fillet were packed in polyethylene pouches and stored at 4ºC before used for </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> production. Meat were chopped into 1.52.0cm cubes and mixed with fresh ground spices consist of coriander (12.5-15%), white pepper (2%) (dry seed), fresh alpinia galangal (35-37.5%) and garlic (25%), then salt (8%), sugar (10%) and coconut oil (5%) were also added. The bebontot ingredients mixture were wrapped in dried </span><span class="font7" style="font-style:italic;">Areca catechu palm</span><span class="font7"> sheaths and finally sun dried for 5 days.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark8"></a><span class="font7" style="font-weight:bold;"><a name="bookmark9"></a>2.2.</span><span class="font7" style="font-weight:bold;font-style:italic;"> &nbsp;&nbsp;&nbsp;Isolation and identification of LAB.</span></h3></li></ul>
<p><span class="font7">The wrappings </span><span class="font7" style="font-style:italic;">Areca catechu</span><span class="font7"> sheaths were aseptically removed, and 10 g of the </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> chicken meat were homogenized in 90 ml of sterile 0.85% sodium chloride (Merck, Darmstadt, Germany) diluents in a Stomacher (Lab-blender, Seward, London, UK) for 2 min. Serial decimal dilutions were prepared. The lactic acid bacteria isolation, count was determined after 48 h in an anaerobic-jar (</span><span class="font7" style="font-style:italic;">Gas-Pack, envelopes,</span><span class="font7"> BBL, Baltimore, Maryland, USA) at 37<sup>o</sup>C on MRS (De Man, Rogosa and Sharpe) agar (Pronadisa, cat.1043), after added with indicator of Brome Cresol Purple (BCP). Colonies were randomly selected from MRS plates containing less than 300 colonies and puried on MRS agar with added sterilizes 1% of calcium carbonate (Merck) and incubated in an anaerobic-jar at 37<sup>o</sup>C for 24 h. After overnight growth, yellowish clear zone around the colonies. Typically all isolates of LAB were selected for morphological examination using Gram-staining under microscope, and examined for production of catalase test and oxidase/gas. Only Gram-positive, catalase-negative, oxidase/gas-negative isolates were considered and stored at -20<sup>o</sup>C in MRS broth (Pronadisa, cat.1215) containing 30% glycerol (Panreac, Badalona, Spain) before being subjected to molecular identification. Species-specific PCR assays were used to confirm the identity of the strains belonging to the LAB group as previously reported [15], with slight modification. Briefly, DNA extraction was carried out from a single colony input in the PCR tubes were added 10 µl of sterile water distillate and then add 90 µl Insta Gene Matrix (Bio-Rad Laboratories, Hercules, CA) into the tube and incubated in a heat block/ thermal cycle with a temperature of 100<sup>o</sup> C for 15 min, then after incubated vortex for 2 min vortex and centrifugation for 3 min at 12.000 rpm. Sample is ready for use as a template DNA for amplification and 25 micro-liters used for PCR amplifications. DNA was prepared using the 16S r-RNA was amplified by PCR using a 9F forward primer, GAGTTTGATYMTGGCTCAG; and a 1541R reverse primer, AAGGAGGTGWTCCARCC.</span></p>
<p><span class="font7">The reaction mixture (50 μl total volume) contained 30 µl ddH2O, 5 µl of MgCl2, 5 µl of 10X Buffer 4 µl of de-oxy-nucleoside-tri-phosphates, 1 µl of each primer, 0.25 µl of Takara Taq Polymerase (Takara Bio Inc, Japan), and 5 µl of cell lyses as the template. PCR conditions were as follows: denaturation at 98°C for 20 s, annealing at 52°C for 45 s, and elongation at 72°C for 2 min. A total of 30 cycles were performed, followed by a final elongation for 4 min at 72°C. PCR products were purified with a Gene-aid PCR Fragments Extraction Kit according to the manufacturer's instructions (Gene-aid, Taiwan). Amplicon was sequenced with an automatic sequence analyzer (Applied Bio-systems 3130 DNA Analyzer; Applied Bio-systems, CA, USA) using the Big-Dye_Terminator v3.1 Cycle Sequencing Kit (Applied Bio-systems). Related sequences were identified by performing sequence database searches using the Basic Local Alignment Search Tool (BLAST), the address:</span><a href="mailto:blast@ncbi.nlm.nih.gov"><span class="font7"> </span><span class="font7" style="text-decoration:underline;">blast@ncbi.nlm.nih.gov</span><span class="font7">.</span></a><span class="font7"> Sequence data for related species were retrieved from GenBank </span><a href="mailto:retrieve@ncbi.nlm.nih.gov"><span class="font7">(</span><span class="font7" style="text-decoration:underline;">retrieve@ncbi.nlm.nih.gov</span><span class="font7">)</span></a><span class="font7"> ([16]; [17]).</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark10"></a><span class="font7" style="font-weight:bold;"><a name="bookmark11"></a>2.3.</span><span class="font7" style="font-weight:bold;font-style:italic;"> &nbsp;&nbsp;&nbsp;DPPH radical scavenging activity of LAB isolate</span></h3></li></ul>
<p><span class="font7">The radical-scavenging activity (RSA) was estimated with the aqueous supernatant obtained from raw </span><span class="font6">breast meat according to the method of Blois as described by [18]; [3], with slight modifications. 0.1 ml quantity of aqueous supernatant of isolate diluted to 1 ml of water and vortex, then filtered using Millipore 0.45 </span><span class="font7" style="font-style:italic;">μ</span><span class="font6">m filter membrane, then take into test tube 0.1 ml and was added 1.0 ml fresh of methanolic DPPH solution (0.2 mM). The mixture was vortexed and left to stand at room temperature for 30 min. A tube containing 1.0 mL of methanol and 1.0 mL of methanolic DPPH solution (0.2 mM) served as the control. The absorbance of the solution was measured at 517 nm (GENESYS 10S UV-VIS Spectrophotometer, Thermo Scientific). Each strains of isolated at 0 day and 5 days were prepared in three replicates and absorbance measured in two replicate readings. The percentage of DPPH radical scavenging activity was obtained from the following equation: Radical-scavenging activity (%) =</span></p>
<p><span class="font4" style="font-weight:bold;">absorbance value of testin g solution A <sup>-</sup> τ—z----;—3---;—;—∙—J χlθθ</span></p>
<p><span class="font4" style="font-weight:bold;">v absorbance value of control solution <sup>?</sup></span></p>
<ul style="list-style:none;"><li>
<p><span class="font7" style="font-weight:bold;">3. &nbsp;&nbsp;&nbsp;Results</span></p>
<ul style="list-style:none;">
<li>
<h3><a name="bookmark12"></a><span class="font7" style="font-weight:bold;"><a name="bookmark13"></a>3.1.</span><span class="font7" style="font-weight:bold;font-style:italic;"> &nbsp;&nbsp;&nbsp;Identification of LAB isolate from bebontot spent chicken meat.</span></h3></li></ul></li></ul>
<p><span class="font7">Results of the 16S rDNA sequence analysis in these research (Table 1) on microbial diversity in spontaneously fermented of LAB </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> chicken meat products were distributed two strains of </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici</span><span class="font7"> at 0 day of the batters (before wrapped) as 58.33% </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici strain LMG 17680</span><span class="font7"> (530 base pair, 100% maximum identity, accession number AJ249891.1; GenBank ID number 7672203) and 41.67% of </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici strain O-mls-1</span><span class="font7"> (649 base pair, 99% maximum identity, accession number JN836485.1; GenBank ID number 374722731) were found in this study</span><span class="font7" style="font-style:italic;">. Pediococcus acidilactici</span><span class="font7"> strain LMG 17680, it was similar with the reference strains, and it was</span></p>
<p><span class="font7">corroborated with other researchers as well as [19], and </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici</span><span class="font7"> strain –O mls-1 similarity with isolate from yak milk cheese [20].</span></p>
<p><span class="font7">One strain of the isolate was 56.25% of </span><span class="font7" style="font-style:italic;">Pediococcus pentosaceus</span><span class="font7"> strain Ni 1386 (1538 base pair, 100% maximum identity, accession number AB598987.1; GenBank ID number</span></p>
<table border="1">
<tr><td colspan="7" style="vertical-align:bottom;">
<p><span class="font5">Table 1. Distribution of lactic acid bacteria of </span><span class="font5" style="font-style:italic;">bebontot</span><span class="font5"> from spent chicken meat at day 0 to the end (at day 5) fermentation.</span></p></td></tr>
<tr><td rowspan="2" style="vertical-align:middle;">
<p><span class="font5">Accession number</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font5">Description spesies</span></p></td><td style="vertical-align:bottom;">
<p><span class="font5">Length of sequences</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">GenBank</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Maximum</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Total of</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">DPPH</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">(basepair)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">ID number</span></p></td><td style="vertical-align:bottom;">
<p><span class="font5">Identifications (%)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Isolate (%)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">RSA (%)</span></p></td></tr>
<tr><td colspan="3" style="vertical-align:bottom;">
<p><span class="font5" style="font-style:italic;">before wrapping/ the batters</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AJ249891.1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Pediococcus acidilactici strain LMG 17680</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">530</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">7672203</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">100</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">7 (58.33)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">68.76%</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">JN836485.1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Pediococcus acidilactici strain O-mls-1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">649</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">374722731</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">99</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">5 (41.67)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">66.34%</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td colspan="3" style="vertical-align:bottom;">
<p><span class="font5" style="font-style:italic;">the end (at day 5) ripening/fermentation</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AB598987.1</span></p></td><td style="vertical-align:top;">
<p><span class="font5">Pediococcus pentosaceus strain Ni 1386</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1538</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">385541184</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">100</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">27(56.25)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">85.58%</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">JX244277.1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Lactobaccillus plantarum strain PA21</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1561</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">396576520</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">100</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">10(20.83)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">84.69%</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AB598950.1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Lactobaccillus plantarum strain Ni 1002</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1529</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">385541147</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">99</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">4 (8.33)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">82.93%</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td colspan="3" style="vertical-align:bottom;">
<p><span class="font5" style="font-style:italic;">at day 4 ripening/fermentation</span></p></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AB362734.1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Lactobaccillus plantarum strain NRIC 1725</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1554</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">157907466</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">99</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">2 (4.17)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">NE</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AB261004.1</span></p></td><td style="vertical-align:top;">
<p><span class="font5">Lactobaccillus plantarum strain G51104J1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">503</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">106365513</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">99</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">2(4.17)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">NE</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AB598983.1</span></p></td><td style="vertical-align:top;">
<p><span class="font5">Pediococcus pentosaceus strain Ni 1382</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1538</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">385541180</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">100</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1 (2.08)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">NE</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AB598980.1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Pediococcus pentosaceus strain Ni 1379</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1537</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">385541177</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">100</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1 (2.08)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">NE</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font5">AB481102.1</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">Pediococcus pentosaceus strain KT3CE27</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1539</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">245002877</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">99</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">1 (2.08)</span></p></td><td style="vertical-align:middle;">
<p><span class="font5">NE</span></p></td></tr>
</table>
<p><span class="font7">385541184), two strains lactobacilli as 8.33% of </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum</span><span class="font7"> strain Ni 1002 (1529 base pair, 99% maximum identity, accession number AB598950.1; GenBank ID number 385541147) and 20.83% of </span><span class="font7" style="font-style:italic;">Lactobaccillus plantarum</span><span class="font7"> strain PA21 (1561 base pair, 100% maximum identity, accession number JX244277.1; GenBank ID number 396576520) were found during of fermentation (5 days dried under the sun) of </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7">. In a comparison of ribosomal proteins </span><span class="font7" style="font-style:italic;">Pediococcus pentosaceus</span><span class="font7"> are related to </span><span class="font7" style="font-style:italic;">Lactobacillus brevis</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum,</span><span class="font7"> these were similarity with the reference strains, and it was corroborated with other researchers as well as [21] isolate from mixed pasture of timothy and orchardgrass silage, this bacteria is also similar with </span><span class="font7" style="font-style:italic;">Pediococcus pentosaceus</span><span class="font7"> strain Ni1386 and </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum </span><span class="font7">strain Ni1002. Then for </span><span class="font7" style="font-style:italic;">Lactobaccillus plantarum</span><span class="font7"> strain PA21 was similarity with isolate from </span><span class="font7" style="font-style:italic;">Pandanus amaryllifolius</span><span class="font7"> [22].</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark14"></a><span class="font7" style="font-weight:bold;"><a name="bookmark15"></a>3.2.</span><span class="font7" style="font-weight:bold;font-style:italic;"> &nbsp;&nbsp;&nbsp;Antioxidant activity of LAB isolate from bebontot spent chicken meat.</span></h3></li></ul>
<p><span class="font7">The radical scavenging activity (RSA) of DPPH as inhibition concentration (IC) of </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici</span><span class="font7"> strain LMG 17680 (68.76%) and </span><span class="font7" style="font-style:italic;">Pediococcus. acidilactici</span><span class="font7"> strain O-mls-1 (range between 52.00%-73.05%) or the average IC of RSA was 66.83±3.55% at 0 day of the batters.</span></p>
<p><span class="font7">The result observed at the end of ripening/fermentation (5 days) dried under the sun were isolate consist of </span><span class="font7" style="font-style:italic;">Pediococcus pentosaceus</span><span class="font7"> strain Ni 1386 (85.58% IC of RSA); </span><span class="font7" style="font-style:italic;">Lactobaccillus plantarum</span><span class="font7"> strain PA21 (the mean IC of RSA 79.89%) and </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum</span><span class="font7"> strain Ni 1002 (the mean IC of RSA 82.93%). The average IC of RCA after 5days of fermentation was 83.37±4.06%.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font7" style="font-weight:bold;">4. &nbsp;&nbsp;&nbsp;Discussion</span></p>
<ul style="list-style:none;">
<li>
<h3><a name="bookmark16"></a><span class="font7" style="font-weight:bold;"><a name="bookmark17"></a>4.1.</span><span class="font7" style="font-weight:bold;font-style:italic;"> &nbsp;&nbsp;&nbsp;Identification of LAB isolate</span></h3></li></ul></li></ul>
<ul style="list-style:none;"><li>
<p><span class="font7">[23] &nbsp;&nbsp;&nbsp;also reported that </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici</span><span class="font7"> had been described as one of the predominant LAB isolated from Iberian dry fermented sausages (10 </span><span class="font7" style="font-style:italic;">chorizos</span><span class="font7"> and 12 </span><span class="font7" style="font-style:italic;">salchichones</span><span class="font7">), traditional meat products produced in the central-west of Spain (Extremadura), and this species according to [24] was commonly used for commercial starter culture. While [25] noted as commercial probiotic cultures for human use, and [26] stated that it was used for fermented meat product in pilot processing of Iberian sausages.</span></p></li></ul>
<p><span class="font7">It is interesting to note that in this study </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum</span><span class="font7"> were similar to the one reported by [27] by using SDS-PAGE-sequencing of 16S-rRNA gene; and for </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici, Pediococcus pentosaceus</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum</span><span class="font7"> were similar with the one who founded by [28], where they use phenotypic identification approach. </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum</span><span class="font7"> found in </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> chicken breast meat was similar to the one reported by [29], and it was also found dominate the LAB flora in a Greek sausage [30]. Furthermore [31] reported that </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum</span><span class="font7">, </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici </span><span class="font7">and </span><span class="font7" style="font-style:italic;">Pediococcus pentosaceus</span><span class="font7"> BT520 have been used for the production of Som-fug, a Thai fermented fish at 30<sup>o</sup>C.</span></p>
<p><span class="font7">The dominant species identified in this study are different than the one reported by [15] where in their study on identification of dominant species in other indigenous Balinese traditional meat fermented poduct known as </span><span class="font7" style="font-style:italic;">urutan</span><span class="font7"> pork meat they found </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum, Pediococcus acidilactici, Lactobacillus. farciminis, Lactobacillus fermentum</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">Lactobacillus hilgardii. Urutan</span><span class="font7"> pork meat was prepared using pork meat with 30% fat added with spices/herb and packed in natural casing then sun dried for 5 days. While [14] prepared bebontot from lean meat and pork fat and wrapped in </span><span class="font7" style="font-style:italic;">Areca cathecu palm</span><span class="font7"> sheaths (</span><span class="font7" style="font-style:italic;">upih</span><span class="font7">, Balinese word) and found LAB isolate were dominated by genus </span><span class="font7" style="font-style:italic;">Lactobacillus sp</span><span class="font7">., and </span><span class="font7" style="font-style:italic;">Streptococcus sp.</span><span class="font7"> However </span><span class="font7" style="font-style:italic;">Leuconostoc</span><span class="font7"> sp was found in </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> from Tabanan region besides both of LAB, and it is possibly due to different mixed spices, type of meat and fat used and difference in temperature and relative humidity at one region to other regions in Bali island. Furthermore [32] reported that the several lactobacilli dominate the endogenous LAB had been isolated from infant feces, and identify by partial 16S rRNA sequencing, they confirming as potential starter cultures for fermented sausage. [33] noted that difference in identified microorganisms involved during traditional meat fermentation (depend on the type of meat products) were related to the diversity in formulation (the ingredients and raw materials used), and to the technology applied (fermentation and ripening time) such as different temperature, duration and relative humidity.</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark18"></a><span class="font7" style="font-weight:bold;"><a name="bookmark19"></a>4.2.</span><span class="font7" style="font-weight:bold;font-style:italic;"> &nbsp;&nbsp;&nbsp;Antioxidant activity of LAB isolate</span></h3></li></ul>
<p><span class="font7">The results in this study, it was higher than reported by [3], these probably due to mix between meat (spent laying hen meat, it abundant heme) and of spices/herbs contained of micronutrients on the batters as substrates for grow of LAB </span><span class="font7" style="font-style:italic;">bebontot,</span><span class="font7"> may active and harbors systems for protection against reactive oxygen species in fermentation process. [34] reported that presence of micronutrients, especially manganese of spices and herbs could increase of the antioxidant activity.</span></p>
<p><span class="font7">The results of the radical scavenging ability of DPPH (0.2mM) on the intact cells and intracellular extract of intestinal bacteria </span><span class="font7" style="font-style:italic;">Bifidobacterium longum</span><span class="font7"> ATCC 15708 (41.6% -52.1%) and </span><span class="font7" style="font-style:italic;">Lactobacillus acidophilus</span><span class="font7"> ATCC 4356 (20.8% - 43.2%) contributes to the antioxidant effect [3]. It was probably due to caused by using substrates for grew both (</span><span class="font7" style="font-style:italic;">Bifidobacterium longum</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">Lactobacillus acidophilus</span><span class="font7">) isolated from infant and human intestinal. It could also probably due to the influence on metabolism of nutrients for growing LAB, and therefore it was not properness for used as cultured meat products to human foods.</span></p>
<p><span class="font7">Furthermore [8] reported that for the DPPH (0.2 mM) radical of </span><span class="font7" style="font-style:italic;">Lactobacillus. fermentum </span><span class="font7">isolated from gastrointestinal mucosa of healthy weaning piglets showed 64.26% scavenging activity at 10<sup>6</sup> </span><span class="font7" style="font-style:italic;">cfu</span><span class="font7">/ml and 87.89% scavenging activity at 10<sup>9</sup> </span><span class="font7" style="font-style:italic;">cfu</span><span class="font7">/ml. It was used as feed supplement on basal diet of crossbred pigs. These result showed that antioxidant capacity of LAB was analyses as a function of LAB cell concentration as described by [4]. This result was later contradicted with the results of LAB isolate of </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> chicken meat in this study, because it appears antioxidant activity of LAB isolate not only oriented towards cell concentration but also towards related with material nutrition original source for grow bacteria autochthonous or where of LAB isolated from meat product or based to other materials. Although all of LAB can grow in optimal medium but first growing influence physiologies condition of LAB when isolated. It seems that the RSA of LAB isolated from </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> samples were possibly related to the length of the sequence of each native bacteria species which was identified from enzymatic activity of raw material at high drying temperature. This tight regulation of the intracellular Mn concentration would explain why antioxidant activity in LAB isolates of </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> could be increased by the addition of spices /herbs to the growth medium during drying or fermentation .</span></p>
<p><span class="font7">Although LAB have long been considered as catalase-negative microorganisms, two groups of LAB with of catalase activity (heme-dependent catalase and nonheme Mn-containing catalase) have been reported in the last decade in genera </span><span class="font7" style="font-style:italic;">Lactobacillus</span><span class="font7">, </span><span class="font7" style="font-style:italic;">Pediococcus</span><span class="font7">, and </span><span class="font7" style="font-style:italic;">Leuconostoc</span><span class="font7"> [35]. It is probably due to that in this study also contained of two group of catalase activity of LAB isolate </span><span class="font7" style="font-style:italic;">bebontot</span><span class="font7"> chicken meat, can eliminate oxygen in a reaction that produces H</span><span class="font4">2</span><span class="font7">O</span><span class="font4">2</span><span class="font7">, there by preventing the formation of extremely damaging ROS like O</span><span class="font4">2</span><span class="font7"><sup>-</sup> (superoxide) and OH</span><span class="font7" style="font-weight:bold;"><sup>.</sup> </span><span class="font7">(hydroxyl radical). Furthermore [36] reported that </span><span class="font7" style="font-style:italic;">MnKat</span><span class="font7"> is the only manganese-dependent catalase isolated from LAB where higher Mn concentrations intracellular (varies greatly among LAB) have been implicated ini oxidative stress resistance, acting as an O</span><span class="font4">2</span><span class="font7"><sup>-</sup> scavenger that could replace superoxide dismutase.</span></p>
<p><span class="font7">The several author ([10]; [1]) reported that microorganisms can producing antioxidant factors have been considered to play an important role in ameliorating the aging process, cardiovascular disease and diabetes, because meat proteins are hydrolyzed during the fermentation (ripening and drying) of meat products by muscle and microbial protease exerts a combination action, where they can contribute to release health-enhancing bioactive peptides. Our results should open the way to improve these products by introducing antioxidative LAB strains.</span></p>
<p><span class="font7" style="font-weight:bold;">References</span></p>
<ul style="list-style:none;"><li>
<p><span class="font7">[1] &nbsp;&nbsp;&nbsp;Wójciak, K M, and Dolatowski, Z J 2012 Oxidative stability of femented meat products </span><span class="font7" style="font-style:italic;">Acta Sci. Pol., Technol. Aliment.</span><span class="font7" style="font-weight:bold;"> 11</span><span class="font7">(2): 99-109</span></p></li>
<li>
<p><span class="font7">[2] &nbsp;&nbsp;&nbsp;Lin, MY, and Yen, C L 1999 Antioxidative ability of lactic acid bacteria </span><span class="font7" style="font-style:italic;">J. Agric. Food Chem</span><span class="font7">, </span><span class="font7" style="font-weight:bold;">47</span><span class="font7">:1460-1466</span></p></li>
<li>
<p><span class="font7">[3] &nbsp;&nbsp;&nbsp;Lin, M Y, and Chang, F J 2000 Antioxidative effect of intestinal bacteria </span><span class="font7" style="font-style:italic;">Bifidobacterium longum</span><span class="font7"> ATCC 5708 and </span><span class="font7" style="font-style:italic;">Lactobacillus acidophilus</span><span class="font7"> ATCC 4356 </span><span class="font7" style="font-style:italic;">Dig. Dis. Sci. </span><span class="font7" style="font-weight:bold;">45</span><span class="font7">:1617-1622</span></p></li>
<li>
<p><span class="font7">[4] &nbsp;&nbsp;&nbsp;Stecchini, M L, Torre, M D, and Munari, M 2001 Determination of peroxy radical scavenging of lactic acid bacteria </span><span class="font7" style="font-style:italic;">Int. J. Food Microbiol.</span><span class="font7" style="font-weight:bold;"> 64</span><span class="font7">:183-188</span></p></li>
<li>
<p><span class="font7">[5] &nbsp;&nbsp;&nbsp;Kullisaar, T, Zilmer, M, Mikelsaar, M, Vihalemm, T, Annuk, H, Kairane, C, and Kilk, A 2002 Two antioxidative lactobacilli strains as promising probiotics </span><span class="font7" style="font-style:italic;">Int. J. Food Microbiol.</span><span class="font7" style="font-weight:bold;"> 72</span><span class="font7">:215-224</span></p></li>
<li>
<p><span class="font7">[6] &nbsp;&nbsp;&nbsp;Kim, H S, Chae, H S, Jeong, S G, Ham, J S, Im, S K, Ahn, C N, and Lee, J M 2005 Antioxidantactivity of some yogurt starter cultures </span><span class="font7" style="font-style:italic;">Asian-Aust. J. Anim. Sci. </span><span class="font7" style="font-weight:bold;">18</span><span class="font7">:255-258</span></p></li>
<li>
<p><span class="font7">[7] &nbsp;&nbsp;&nbsp;Kim, H S, Chae, H S, Jeong, S G, Ham, J S, Im, S K, Ahn, C N, and Lee, J M 2006 </span><span class="font7" style="font-style:italic;">In vitro</span><span class="font7"> antioxidative properties of lactobacilli </span><span class="font7" style="font-style:italic;">Asian-Aust. J. Anim. Sci.</span><span class="font7" style="font-weight:bold;"> 19</span><span class="font7">:262-265</span></p></li>
<li>
<p><span class="font7">[8] &nbsp;&nbsp;&nbsp;Wang, A N, Yi, X W, Yu, H F, Dong, B, and Qiao, S Y 2009 Free radical scavenging activity of &nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font7" style="font-style:italic;">Lactobacillus fermentum in vitro</span><span class="font7"> and its antioxidative effect on</span></p></li></ul>
<p><span class="font7">growing-finishing pigs </span><span class="font7" style="font-style:italic;">J. Appl. Microbiol.</span><span class="font7" style="font-weight:bold;"> 107</span><span class="font7">:1140-1148</span></p>
<ul style="list-style:none;"><li>
<p><span class="font7">[9] &nbsp;&nbsp;&nbsp;Nedelcheva, P, Denkova, Z, Denev, P, and Krastanov, A 2010 Probiotic strain </span><span class="font7" style="font-style:italic;">Lactobacillus plantarum</span><span class="font7"> NBIMCC 2415 with antioxidant activity as a starter culture in the production of dried fermented meat products </span><span class="font7" style="font-style:italic;">Biotechnol. &amp;&nbsp;Biotechnol</span><span class="font7">. EQ. 24/2010/1 DOI: 10 2478/V10133-010-0016-4</span></p></li>
<li>
<p><span class="font7">[10] &nbsp;&nbsp;&nbsp;Arihara, K 2006 Strategies for designing novel functional meat products </span><span class="font7" style="font-style:italic;">Meat Sci</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">74</span><span class="font7">:219-229</span></p></li>
<li>
<p><span class="font7">[11] &nbsp;&nbsp;&nbsp;De Macedo, F, R E, Miyague, L, Costa, L B, and Luciano, F B 2013 Control of Listeria monocytogenes growth by bacteriocin-producing starter cultures in the manufacturing of dry fermented sausage </span><span class="font7" style="font-style:italic;">African J of Microbiol Res</span><span class="font7" style="font-weight:bold;"> 7 </span><span class="font7">(9): 710-718</span></p></li>
<li>
<p><span class="font7">[12] &nbsp;&nbsp;&nbsp;Talon R, Leroy, S, and Lebert, I 2007 Microbial ecosystems of traditional fermented meat products: the importance of indigenous starters </span><span class="font7" style="font-style:italic;">Meat Sci</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">77</span><span class="font7">:55-62</span></p></li>
<li>
<p><span class="font7">[13] &nbsp;&nbsp;&nbsp;Arihantana, M B 1981 </span><span class="font7" style="font-style:italic;">Production and stability of bebontot</span><span class="font7"> M App Sc Thesis Department of Food Science and Technology, Univ of New South Wales, Sydney</span></p></li>
<li>
<p><span class="font7">[14] &nbsp;&nbsp;&nbsp;Hartawan, M 2002 </span><span class="font7" style="font-style:italic;">Identifikasi Bakteri Asam Laktat, Perubahan Mikrobiologis dan Biokimiawi Selama Fermentasi Bebontot</span><span class="font7"> Thesis Program Pascasarjana Program Study Bioteknologi Pertanian Univ Udayana Denpasar</span></p></li>
<li>
<p><span class="font7">[15] &nbsp;&nbsp;&nbsp;Antara, N S, Sujaya, I N, Yokota, A, Asano, K, Aryanta, W R, and Tomita, F 2002 Identification and succession of lactic acid bacteria during fermentation of “urutan”, a Balinese indigenous fermented sausage </span><span class="font7" style="font-style:italic;">World J. Microbiol. &amp;&nbsp;Biotech</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">18</span><span class="font7">: 255262</span></p></li>
<li>
<p><span class="font7">[16] &nbsp;&nbsp;&nbsp;Altschul, S F, Madden, T L, Schäffer, A A, Zhang, J, Zhang, Z, Miller, W, and Lipman, D J 1997 Gapped blast and psi-blast: a new generation of protein database search programs. </span><span class="font7" style="font-style:italic;">Nucleic Acids Res</span><span class="font7" style="font-weight:bold;"> 25 </span><span class="font7">(17): 3389-3402</span></p></li>
<li>
<p><span class="font7">[17] &nbsp;&nbsp;&nbsp;Benson, D A, Boguski, M S, Lipman D J, and Ostell, J 1997 GenBank </span><span class="font7" style="font-style:italic;">Nucleic Acids Res </span><span class="font7" style="font-weight:bold;">25 </span><span class="font7">(1): 1-6</span></p></li>
<li>
<p><span class="font7">[18] &nbsp;&nbsp;&nbsp;Jang, A, Liu X D, Shin, M H, Lee, B D, Lee, S K, Lee, J H, and Jo, C 2008 Antioxidative potential of raw breast meat from broiler chicks fed a dietary medicinal herb extract mix. </span><span class="font7" style="font-style:italic;">Poult. Sci.</span><span class="font7" style="font-weight:bold;"> 87</span><span class="font7">:2382–2389</span></p></li>
<li>
<p><span class="font7">[19] &nbsp;&nbsp;&nbsp;Mora, D, Fortina, M. G, Parini, C, Daffonchio, D, and Manachini, P L 2000 Genomic subpopulations within the species </span><span class="font7" style="font-style:italic;">Pediococcus acidilactici</span><span class="font7"> detected by multilocus typing analysis: relationships between pediocin AcH/PA-1 producing and nonproducing strains. Microbiology (Reading, Engl.) 146 (PT 8), 2027-2038</span></p></li>
<li>
<p><span class="font7">[20] &nbsp;&nbsp;&nbsp;Yang, J, Chen, Z and Mustapha, A 2011 Phenotypic, genotypic and physiological characterization of latic acid bacteria from Chinese yak milk cheeses Unpublished Submitted (11-SEP-2011) Food Sciences Program, University of Missouri, Columbia, MO 65211, USA</span></p></li>
<li>
<p><span class="font7">[21] &nbsp;&nbsp;&nbsp;Tohno, M, Kobayashi, H, Nomura, M, Uegaki, R and Cai, Y 2012 Identification and characterization of lactic acid bacteria isolated from mixed pasture of timothy and orchardgrass, and its badly preserved silage </span><span class="font7" style="font-style:italic;">Anim. Sci. J.</span><span class="font7" style="font-weight:bold;"> 83 </span><span class="font7">(4), 318-330</span></p></li>
<li>
<p><span class="font7">[22] &nbsp;&nbsp;&nbsp;Jalilsood, T, Foo, H L, Mustafa, S, Yusoff, K, and Abdul Rahim, R 16S rDNA gene fragments 2012 </span><span class="font7" style="font-style:italic;">Unpublished</span><span class="font7"> Submitted (25-JUN-2012) Cell and Molecular Biology, Universiti Putra Malaysia, Serdang, Selangor 43400, Malaysia</span></p></li>
<li>
<p><span class="font7">[23] &nbsp;&nbsp;&nbsp;Ruiz-Moyano, S, Martín, A, Benito, M J, Pérez-Nevado, F, and Córdoba, M G 2008 Screening of lactic acid bacteria and bifidobacteria for potential probiotic use in Iberian dry fermented sausages </span><span class="font7" style="font-style:italic;">Meat Sci</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">80</span><span class="font7">:715-721</span></p></li>
<li>
<p><span class="font7">[24] &nbsp;&nbsp;&nbsp;Tyӧppӧnen, S, Petӓjӓ, E, and, Mattila-Sandholm, T 2003 Bioprotectives and probiotics for sausages </span><span class="font7" style="font-style:italic;">Int. J. of Food Microbiol</span><span class="font7">, </span><span class="font7" style="font-weight:bold;">83</span><span class="font7">, 233-244</span></p></li>
<li>
<p><span class="font7">[25] &nbsp;&nbsp;&nbsp;Erkkilӓ, S, and Petӓja, E 2000 Screening of commercial meat starter cultures at low pH and in the presence of bile salts for potential use </span><span class="font7" style="font-style:italic;">Meat Sci</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">55</span><span class="font7">, 297-300</span></p></li>
<li>
<p><span class="font7">[26] &nbsp;&nbsp;&nbsp;Benito, M J, Martín, A, Aranda, E Pérez-Nevado, F, Ruiz-Moyano, S, and Córdoba, M G 2007 Characterization and selection of autochthonous lactic acid bacteria isolated from traditional lberian dry-fermented Salchichon and Chorizo sausages </span><span class="font7" style="font-style:italic;">J. of Food Sci</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">72</span><span class="font7">, 193-201</span></p></li>
<li>
<p><span class="font7">[27] &nbsp;&nbsp;&nbsp;Benito, M J, Serradilla, M J, Ruiz-Moyano, S, Alberto, A, Pérez-Nevado, F, and Córdoba, M G 2008 Rapid differentiation of lactic acid bacteria from autochthonous fermentation of Iberian dry-fermented sausages </span><span class="font7" style="font-style:italic;">Meat Sci</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">80</span><span class="font7">:656661</span></p></li>
<li>
<p><span class="font7">[28] &nbsp;&nbsp;&nbsp;Kozacinski, L, Drosinos, E H, Caklovica, F, Cocolin, L, Gasparik-Reichardt, J, and Veskovic 2008 Investigation of microbial association of traditionally fermented sausages </span><span class="font7" style="font-style:italic;">Food Tech. and Biotech.</span><span class="font7" style="font-weight:bold;"> 46</span><span class="font7">:93-106</span></p></li>
<li>
<p><span class="font7">[29] &nbsp;&nbsp;&nbsp;Rantsiou, K, E H Drosinos, M Gialitaki, I Metaxopoulos, G Comi, and L Cocolin 2006 Use of molecular tools to characterize </span><span class="font7" style="font-style:italic;">Lactobacillus</span><span class="font7"> spp Isolated from Greek traditional fermented sausages </span><span class="font7" style="font-style:italic;">Int. J. of Food Microbiol</span><span class="font7" style="font-weight:bold;"> 112</span><span class="font7">:215-222</span></p></li>
<li>
<p><span class="font7">[30] &nbsp;&nbsp;&nbsp;Drosinos, E H, Mataragas, M, Xiraphi, N, Moschonas, G, Gaitis, F, and Metaxopoulus, J 2005 Characterization of the microbial flora from a traditional Greek fermented sausage </span><span class="font7" style="font-style:italic;">Meat Sci</span><span class="font7">. </span><span class="font7" style="font-weight:bold;">69</span><span class="font7">:307-317</span></p></li>
<li>
<p><span class="font7">[31] &nbsp;&nbsp;&nbsp;Riebroy, S, Benjakul, S, and Visessanguan, W 2008 Proporties and acceptability of Som-fug, a Thai fermented fish mince, inoculated with lactic acid bacteria starters </span><span class="font7" style="font-style:italic;">Lwt-Food Sci. and Tech</span><span class="font7">., </span><span class="font7" style="font-weight:bold;">41</span><span class="font7">(4):569-580</span></p></li>
<li>
<p><span class="font7">[32] &nbsp;&nbsp;&nbsp;Rubio, R, Jofré, A, Martín, B, Aymerich, T, and Garriga, M 2014 Characterization of lactic acid</span></p></li></ul>
<p><span class="font7">bacteria isolated from infant faeces as potential probiotic starter cultures for fermented sausages </span><span class="font7" style="font-style:italic;">Food Microbiol</span><span class="font7">, </span><span class="font7" style="font-weight:bold;">38</span><span class="font7">(4):303-311 Abstract</span></p>
<ul style="list-style:none;"><li>
<p><span class="font7">[33] &nbsp;&nbsp;&nbsp;Lebert, l, Leroy, S Giammarinaro, P, Lebert, A, Chacornac, J P, Bover-Cid, S, Vidal-Carou, M C, and Talon, R 2007 Diversity of microorganisms in the environment and dry fermented sausage of small traditional French processing units </span><span class="font7" style="font-style:italic;">Meat Sci.</span><span class="font7" style="font-weight:bold;">76</span><span class="font7">:112-122</span></p></li>
<li>
<p><span class="font7">[34] &nbsp;&nbsp;&nbsp;Leroy, F, and De Vuyst, L 2005 Simulation of the effect of sausage ingredients and technology on the functionality of the bacteriocin-producing Lactobacillus sakei CTC494 strain </span><span class="font7" style="font-style:italic;">Int. J. Food Microbiol</span><span class="font7" style="font-weight:bold;"> 100</span><span class="font7">:141-152</span></p></li>
<li>
<p><span class="font7">[35] &nbsp;&nbsp;&nbsp;Igarashi, T, Kono, Y, and Tanaka, K 1996 Molecular cloning of manganese catalase from </span><span class="font7" style="font-style:italic;">Lactobaccillus plantarum J. of Biol. Chemis.</span><span class="font7" style="font-weight:bold;"> 271</span><span class="font7">, 29521–29524</span></p></li>
<li>
<p><span class="font7">[36] &nbsp;&nbsp;&nbsp;Rochat, T, Gratadoux, J J, Gruss, A, Corthier, G, Maguin, E, Langella, P, van de Guchte, M (2006) Production of a heterologous nonheme catalase by </span><span class="font7" style="font-style:italic;">Lactobacillus casei</span><span class="font7">: an efficient tool for removal of H2O2 and protection of </span><span class="font7" style="font-style:italic;">Lactobacillus bulgaricus </span><span class="font7">from oxidative stress in milk Applied and Environmental Microbiology </span><span class="font7" style="font-weight:bold;">72</span><span class="font7">, 5143 – 5149</span></p></li></ul>
<p><span class="font6" style="font-style:italic;">Okarini, I. A., et al, Peternakan Tropika Vol. 8 No. 1 Th. 2020 : 37 - 45</span></p>
<p><span class="font6">Page 45</span></p>