---
layout: full_article
title: "PRIMER DESIGN OF CVPDr DNA FRAGMENT SEQUENCES THAT AMPLIFY SPECIFIC FRAGMENTS TO DISTINCT THE RESISTANT FRAGMENT FROM Triphasia trifolia (Burm. F.) P. Wils. AND THE SUSCEPTIBLE FRAGMENT FROM Citrus nobilis Lour."
author: "Ni Made Ayuratih Utami, I Gede Putu Wirawan, I Ketut Suada"
categories: jbb
canonical_url: https://jurnal.harianregional.com/jbb/full-60847 
citation_abstract_html_url: "https://jurnal.harianregional.com/jbb/id-60847"
citation_pdf_url: "https://jurnal.harianregional.com/jbb/full-60847"  
comments: true
---

<p><span class="font2">INTERNATIONAL JOURNAL OF BIOSCIENCES AND BIOTECHNOLOGY • Vol. 7 No. 2 • April 2020 eISSN: 2655-9994 pISSN: 2303-3371 </span><a href="https://doi.org/10.24843/IJBB.2020.v07.i02.p05"><span class="font2">https://doi.org/10.24843/IJBB.2020.v07.i02.p05</span></a></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font6" style="font-weight:bold;"><a name="bookmark1"></a>PRIMER DESIGN OF CVPD</span><span class="font1" style="font-weight:bold;"><sup>r</sup> </span><span class="font6" style="font-weight:bold;">DNA FRAGMENT SEQUENCES THAT AMPLIFY SPECIFIC FRAGMENTS TO DISTINCT THE RESISTANT FRAGMENT FROM</span></h1>
<h1><a name="bookmark2"></a><span class="font6" style="font-weight:bold;font-style:italic;"><a name="bookmark3"></a>Triphasia trifolia</span><span class="font6" style="font-weight:bold;"> (Burm. F.) P. Wils. AND THE SUSCEPTIBLE FRAGMENT FROM </span><span class="font6" style="font-weight:bold;font-style:italic;">Citrus nobilis</span><span class="font6" style="font-weight:bold;"> Lour</span><span class="font6" style="font-weight:bold;font-style:italic;">.</span></h1>
<p><span class="font6" style="font-weight:bold;">Ni Made Ayuratih Utami</span><span class="font1" style="font-weight:bold;"><sup>*</sup></span><span class="font6" style="font-weight:bold;">, I Gede Putu Wirawan, and I Ketut Suada </span><span class="font5">Department of Agroecotechnology, Faculty of Agriculture, Udayana University <sup>*</sup>Corresponding author: </span><a href="mailto:ayuratihutami@gmail.com"><span class="font5">ayuratihutami@gmail.com</span></a></p>
<h1><a name="bookmark4"></a><span class="font6" style="font-weight:bold;"><a name="bookmark5"></a>ABSTRACT</span></h1>
<p><span class="font6">CVPD<sup>r</sup> is a DNA fragment that indicates that plants are resistant to CVPD. Previous research using primers that amplified 841 bp CVPD<sup>r</sup> fragment was able to amplify the fragment from </span><span class="font6" style="font-style:italic;">Triphasia trifolia</span><span class="font6"> that considers being a resistant plant, </span><span class="font6" style="font-style:italic;">Citrus aurantifolia</span><span class="font6"> var. seedless which considers being a tolerant plant, and some susceptible citrus plants to CVPD disease. In this study, we designed some primers that amplified only CVPD<sup>r</sup> DNA fragment from </span><span class="font6" style="font-style:italic;">T. trifolia </span><span class="font6">which consider as the resistant plant and a primer that amplified only DNA fragmen from </span><span class="font6" style="font-style:italic;">Citrus nobilis</span><span class="font6"> which consider as the susceptible citrus plants. The primers for CVPD<sup>r</sup> on </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> &nbsp;&nbsp;&nbsp;are &nbsp;&nbsp;&nbsp;TCATCTGCATGGGATACC &nbsp;&nbsp;&nbsp;for &nbsp;&nbsp;&nbsp;forward &nbsp;&nbsp;&nbsp;primer &nbsp;&nbsp;&nbsp;and</span></p>
<p><span class="font6">GCCTTGAGCTTGTAAGTG for reverse primer which turned out to amplify the DNA of </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> and also the </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> cultivar Denpasar and only succeeded in not amplifying the </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> cultivar Gianyar. The primers for CVPD<sup>r</sup> &nbsp;&nbsp;on </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> are</span></p>
<p><span class="font6">GAATGGCTTAGCAGAAAGG for forward primer and GGTTGTAGATGGACATAGG for reverse primer turned out can not only amplify the DNA </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> but also amplify </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">.</span></p>
<p><span class="font6" style="font-style:italic;">Keywords:</span><span class="font6"> CVPD<sup>r</sup> fragment, Primer Design, </span><span class="font6" style="font-style:italic;">Citrus nobilis, Triphasia trifolia</span></p>
<h1><a name="bookmark6"></a><span class="font6" style="font-weight:bold;"><a name="bookmark7"></a>INTRODUCTION</span></h1>
<p><span class="font6">Citrus fruit production in Bali has fluctuated over the past five years. In 2016, citrus production in Bali from the previous year decreased by 45,133 tons based on various factors decreasing citrus production (Badan Pusat Staristik, 2018), one of which was an attack of Citrus Vein Phloem Degeneration (CVPD). Some popular plants, some non-commercialized citrus plants and some citrus relatives, are proven to be resistant to CVPD.</span></p>
<p><span class="font6">CVPD is caused by </span><span class="font6" style="font-style:italic;">Liberibacter asiaticus</span><span class="font6"> which develops in shoots and is transmitted by vector insect </span><span class="font6" style="font-style:italic;">Diaphorina citri </span><span class="font6">(Tirtawijaya, 1983; Jaqoueix et al., 1994; 1996; Wirawan, et al., 2004). In addition to oranges, CVPD can be transmitted to several members of the oranges (</span><span class="font6" style="font-style:italic;">Rutaceae</span><span class="font6">) tribe such as </span><span class="font6" style="font-style:italic;">Poncirus trifoliata</span><span class="font6"> (L) Raf, Kemuning </span><span class="font6" style="font-style:italic;">/ Murraya paniculata</span><span class="font6"> (L) Jack, </span><span class="font6" style="font-style:italic;">Swinglea glutinosa</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">Clausena indica</span><span class="font6">.</span></p>
<p><span class="font6">Martasari et al. (2004), stated that many citrus farmers complained about the condition of their oranges which were</span></p>
<p><span class="font6">attacked by various diseases both by bacteria and viruses. All citrus plants cultivated are susceptible to CVPD disease attacks (Mahayani, 2013). In several cases several types of citrus plants were reported, notably commercialized citrus plants and some of their relative plants, known to be resistant to CVPD. Among them are seedless lime, Tahiti lime, </span><span class="font6" style="font-style:italic;">Triphachia trifolia</span><span class="font6"> (kinkit orange), and </span><span class="font6" style="font-style:italic;">Poncirus trifoliata</span><span class="font6"> (karatachi). Citrus plants that are resistant to CVPD are thought to contain genes that produce a trait that is able to break the pathogenic CVPD infection or is able to resist transmission of pathogens carried by vector insects (Wirawan et al., 2004).</span></p>
<p><span class="font6">Previous studies was done using DNA fragments isolated using RAPD and mutations from </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">, these fragments were indicated to be resistant factors to CVPD disease and then referred to as CVPD<sup>r </sup>DNA fragments. Yuniti et al. (2018) found polymorphisms in CVPD<sup>r</sup> DNA fragments in several citrus species (</span><span class="font6" style="font-style:italic;">Citrus</span><span class="font6">, spp) and 12 types of citrus plants that were sampled were plants that had CVPD<sup>r</sup> DNA fragments. Mutations that occur in some CVPD<sup>r</sup> DNA fragments cause changes in bases in DNA fragment sequences so that repairing sequences of some of these plants can be distinguished according to their level of sensitivity to CVPD disease.</span></p>
<p><span class="font6">The purpose of this study is to develop primers that only amplify DNA fragments from resistant plants (</span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">) and susceptible plants (</span><span class="font6" style="font-style:italic;">Citrus nobilis</span><span class="font6">). This primer is expected to be a specific primer that can be used to detect CVPD<sup>r</sup> in resistant plants (</span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">) and susceptible plants (</span><span class="font6" style="font-style:italic;">Citrus nobilis</span><span class="font6">).</span></p>
<h1><a name="bookmark8"></a><span class="font6" style="font-weight:bold;"><a name="bookmark9"></a>MATERIALS AND METHODS</span></h1>
<p><span class="font6">The study was conducted in two stages, namely in </span><span class="font6" style="font-style:italic;">silico</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">in vitro</span><span class="font6">. Research conducted in silico or using a computer is a bionformatics research to design the desired primer.</span></p>
<p><span class="font6">The results of </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> DNA sequencing from seven regions in Bali and </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> DNA (Yuniti et al., 2018) were converted to FASTA format. The sequences in the FASTA format are then aligned using the Ugene (Okonechnikov, </span><span class="font6" style="font-style:italic;">et al.</span><span class="font6">, 2012) application with the MUSCLE algorithm. Sequences are sought for conservative regions which are then used as templates for primer designs.</span></p>
<p><span class="font6">The primer design is done with </span><span class="font6" style="font-style:italic;">in silico</span><span class="font6"> method using the Clone Manager program through several stages. First install the Clone Manager Pro 9 Program. Prepare the sequential data in FASTA format. Open the Clone Manager Pro 9 program, select the primer menu on the menu bar, then choose design. In the dialog box, enter the desired</span></p>
<p><span class="font6">primer and reverse primer lengths (18-24 bases); Click the criteria menu then enter the desired primer parameters. Then select next. Enter the data sequence that has been previously saved and the target region is equipped with the desired base position as the start and end of the target region, select finish. In the dialog box primer results will appear based on the primer criteria and sorted by rank.</span></p>
<p><span class="font6">The Clone Manager program will display several primer candidates according to the required criteria. The best primers candidates then tested for their annealing place using the Ugene application.</span></p>
<p><span class="font6" style="font-style:italic;">In vitro</span><span class="font6"> research was carried out after the primer design was completed, which is</span></p>
<p><span class="font6">the primer test using the PCR method in the laboratory.</span></p>
<p><span class="font6" style="font-weight:bold;">RESULTS AND DISSCUSSION</span></p>
<p><span class="font6" style="font-weight:bold;font-style:italic;">Triphasia trifolia</span><span class="font6" style="font-weight:bold;"> Primer</span></p>
<p><span class="font6">The CVPD<sup>r</sup> fragment sequence from kinkit orange is input into the Clone Manager application along with the desired primer parameters. The Clone Manager application then display several suitable primer candidates and sorted by rank. There are five candidate pairs of </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> primers who fit the proposed criteria. Information about the five primer pairs can be seen in table. 1.</span></p>
<div>
<p><span class="font6">Table. 1 </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> primers candidate from the Clone Manager Program</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:middle;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Jenis</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font4">Primer sequence</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">GC</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Tm</span></p></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font4">Length</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">3’ End</span></p></td><td rowspan="2" style="vertical-align:bottom;">
<p><span class="font4">Any</span></p>
<p><span class="font4">Dimer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Product</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">%</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">(<sup>o</sup>C)</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Dimer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Length</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">F</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">TCATCTGCAT-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">50</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">57</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">18</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">4</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">302</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Kinkit 1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">R</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">GGGATACC GCCTTGAGCT-</span></p>
<p><span class="font4">TGTAAGTG</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">57</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">18</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">F</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">GTCATCTGCA-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">52</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">59</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">19</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">4</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">303</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Kinkit 2</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">R</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">TGGGATACC GCCTTGAGCT-TGTAAGTG</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">57</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">18</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">F</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">GGATTGGTCA-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">50</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">60</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">20</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">3</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">401</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Kinkit 3</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">R</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">GCCTACAAAC GCCTTGAGCT-TGTAAGTG</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">57</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">18</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">F</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">GGTCAGCCTA-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">60</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">20</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">2</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">396</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Kinkit 4</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">R</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">CAAACTTTGG GCCTTGAGCTT-GTAAGTG</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">57</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">18</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">F</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">GGTCATCTGCA-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">55</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">62</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">20</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">4</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">304</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Kinkit 5</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">R</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">TGGGATACC GCCTTGAGCTT-GTAAGTG</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">50</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">57</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">18</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;"></td></tr>
</table>
</div><br clear="all">
<p><span class="font6">From the five pairs of primer candidates produced by Clone manager suite 9, primers were selected that best suit their needs. The selected forward kinkit primer then written as F-Kinkit and the reverse primer is written as R-Kinkit.</span></p>
<p><span class="font6">F-Kinkit: TCATCTGCATGGGATACC R-Kinkit: GCCTTGAGCTTGTAAGTG</span></p>
<p><span class="font6">The results of the analysis and primer information can be seen in Fig. 1 and Fig. 2. Based on these pictures it is known that the primer pair has fulfilled good primer criteria.</span></p>
<div><img src="https://jurnal.harianregional.com/media/60847-1.jpg" alt="" style="width:298pt;height:198pt;">
<p><span class="font6" style="font-weight:bold;">Fig. 1. </span><span class="font6">F-Kinkit and R-Kinkit primer information</span></p>
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/60847-2.jpg" alt="" style="width:300pt;height:200pt;">
<p><span class="font6" style="font-weight:bold;">Fig. 2. </span><span class="font6">The results of the analysis of the primer design of F-Kinkit and R-Kinkit</span></p>
</div><br clear="all">
<p><span class="font6">The length of the F-Kinkit and R-Kinkit is 18 bases. Each has a GC percentage of 50% and Tm 57̊C. The general rule that is followed by most primer design programs is to use a base percentage of G and C between 40% to 60% (Dinda Eling, 2014).</span></p>
<p><span class="font6">Primers may not contain complementary base sequences (palindromes) on the same strand. This can result in the formation of hairpins. The primers will naturally fold and produce unproductive priming. The best primers chosen are primers that do not or have little hairpin structure, dimers, and cross dimers (Breslauer et al., 1986).</span></p>
<p><span class="font6">Based on the analysis results in the program (Figure 4.2), the two primers do not form a hairpins structure. The selected primer pair has a dimer at the end of 3 'ie two on the primer forward and one on the reverse</span></p>
<p><span class="font6">primer. The other dimers formed each bind four base strands. Cross dimers are also formed between the two primers, which are at the end of 3 'by two bases and the other dimer by five bases.</span></p>
<p><span class="font6">Primer attachment position was again tested using the Eugene program. Primers that have been made stick to the sequences of Kinkit Oranges (</span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">) and Denpasar Siem orange (</span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6">). The position of the primer attachment in each sequence is as follows:</span></p>
<div><img src="https://jurnal.harianregional.com/media/60847-3.jpg" alt="" style="width:183pt;height:84pt;">
<p><span class="font6" style="font-weight:bold;">Fig. 3. </span><span class="font6">Position of primer attachment to (A) Kinkit Oranges (</span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">) and (B) Denpasar Siem Oranges (</span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6">)</span></p>
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/60847-4.jpg" alt="" style="width:185pt;height:84pt;">
</div><br clear="all">
<p><span class="font6">The primers attach to the </span><span class="font6" style="font-style:italic;">Triphasia trifolia</span><span class="font6"> sequences and </span><span class="font6" style="font-style:italic;">Citrus nobilis</span><span class="font6"> cul. Denpasar because the two oranges have a similar sequence of bases.</span></p>
<p><span class="font6" style="font-weight:bold;font-style:italic;">C. nobilis</span><span class="font6" style="font-weight:bold;"> Primer</span></p>
<p><span class="font6">Primer for </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> was obtained after aligning seven samples of </span><span class="font6" style="font-style:italic;">C. nobilis</span></p>
<p><span class="font6">cultivar in several regions in Bali so that the same base sequence can be seen. The sequence is then used as a template for primer design. Data sequence of the seven samples of siem orange has different lengths, so the extra bases on the front and back of the sequence are removed so that the sequences of the seven samples are the same length. There were 733 bases after the</span></p>
<p><span class="font6">sequence length of the seven sampled samples were then analyzed with CM suite 9 to obtain the desired primer pair.</span></p>
<p><span class="font6">There are 59 primer pairs that match the proposed criteria. Four of the 59 primer pairs were ranked first based on ratings in the Clone Manager application. Primer information with the first rank can be seen in table. 2.</span></p>
<p><span class="font6">Table. 2 </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> primers candidate from the Clone Manager Program</span></p>
<table border="1">
<tr><td style="vertical-align:middle;">
<p><span class="font4">Primer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Jenis</span></p>
<p><span class="font4">Primer</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Primer sequence</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">GC %</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Tm (<sup>o</sup>C)</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Length</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">3’ End Dimer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Any</span></p>
<p><span class="font4">Dimer</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Product Length</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Primer</span></p>
<p><span class="font4">Siem 1</span></p></td><td style="vertical-align:top;">
<p><span class="font4">F</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">AGAATGGCTTAG-CAGAAAGG</span></p></td><td style="vertical-align:top;">
<p><span class="font4">45</span></p></td><td style="vertical-align:top;">
<p><span class="font4">59</span></p></td><td style="vertical-align:top;">
<p><span class="font4">20</span></p></td><td style="vertical-align:top;">
<p><span class="font4">1</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">144</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font4">R</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">TGGTTGTAGATG-GACATAGG</span></p></td><td style="vertical-align:top;">
<p><span class="font4">45</span></p></td><td style="vertical-align:top;">
<p><span class="font4">58</span></p></td><td style="vertical-align:top;">
<p><span class="font4">20</span></p></td><td style="vertical-align:top;">
<p><span class="font4">1</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Primer</span></p>
<p><span class="font4">Siem 2</span></p></td><td style="vertical-align:top;">
<p><span class="font4">F</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">GAATGGCTTAGC-AGAAAGG</span></p></td><td style="vertical-align:top;">
<p><span class="font4">47</span></p></td><td style="vertical-align:top;">
<p><span class="font4">57</span></p></td><td style="vertical-align:top;">
<p><span class="font4">19</span></p></td><td style="vertical-align:top;">
<p><span class="font4">1</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">142</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font4">R</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">GGTTGTAGATGG-ACATAGG</span></p></td><td style="vertical-align:top;">
<p><span class="font4">47</span></p></td><td style="vertical-align:top;">
<p><span class="font4">56</span></p></td><td style="vertical-align:top;">
<p><span class="font4">19</span></p></td><td style="vertical-align:top;">
<p><span class="font4">1</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Primer</span></p>
<p><span class="font4">Siem 3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">F</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">GAATGGCTTAGC-AGAAAGG</span></p></td><td style="vertical-align:top;">
<p><span class="font4">47</span></p></td><td style="vertical-align:top;">
<p><span class="font4">57</span></p></td><td style="vertical-align:top;">
<p><span class="font4">19</span></p></td><td style="vertical-align:top;">
<p><span class="font4">1</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">189</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font4">R</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">GTTTGTAGGCTG-ACCAATC</span></p></td><td style="vertical-align:top;">
<p><span class="font4">47</span></p></td><td style="vertical-align:top;">
<p><span class="font4">57</span></p></td><td style="vertical-align:top;">
<p><span class="font4">19</span></p></td><td style="vertical-align:top;">
<p><span class="font4">2</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Primer</span></p>
<p><span class="font4">Siem 4</span></p></td><td style="vertical-align:top;">
<p><span class="font4">F</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">AGAATGGCTTAG-CAGAAAGG</span></p></td><td style="vertical-align:top;">
<p><span class="font4">45</span></p></td><td style="vertical-align:top;">
<p><span class="font4">59</span></p></td><td style="vertical-align:top;">
<p><span class="font4">20</span></p></td><td style="vertical-align:top;">
<p><span class="font4">1</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">191</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font4">R</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">AGTTTGTAGGCT-GACCAATC</span></p></td><td style="vertical-align:top;">
<p><span class="font4">45</span></p></td><td style="vertical-align:top;">
<p><span class="font4">59</span></p></td><td style="vertical-align:top;">
<p><span class="font4">20</span></p></td><td style="vertical-align:top;">
<p><span class="font4">2</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;"></td></tr>
</table>
<p><span class="font6">F-Siem: GAATGGCTTAGCAGAAAGG R-Siem: GGTTGTAGATGGACATAGG</span></p>
<p><span class="font6">Compared to the first and fourth primer pairs, the selected primer pairs have a higher GC percentage of 47%. Tm F-Siem is 57̊C, while R-Siem is 56̊C. The primer pair has a Tm difference of 1̊C which is tolerable.</span></p>
<p><span class="font6">The primer pair does not form a hairpin structure. Each primer has a dimer at the end of 3 'of one base and the other dimer of three bases. Cross dimers are also expected to be formed, which is one end 3 'and three bases complementary to each other.</span></p>
<table border="1">
<tr><td style="vertical-align:top;"></td><td colspan="2" style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">0 [Search 5] Primer Pair &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font3" style="font-weight:bold;text-decoration:underline;">∣</span><span class="font1" style="font-weight:bold;text-decoration:underline;">∙o-</span><span class="font3" style="font-weight:bold;text-decoration:underline;">∣∣</span><span class="font1" style="font-weight:bold;text-decoration:underline;">⅛</span><span class="font3" style="font-weight:bold;text-decoration:underline;">∣∣∣</span><span class="font1" style="font-weight:bold;text-decoration:underline;">⅛</span><span class="font3" style="font-weight:bold;text-decoration:underline;">∣</span></p></td></tr>
<tr><td rowspan="3" style="vertical-align:top;"></td><td colspan="2" style="vertical-align:top;">
<p><span class="font6">^ © I S 1<sup>1</sup>⅞ I ¾ I 0</span></p></td></tr>
<tr><td colspan="2" style="vertical-align:top;">
<p><span class="font0" style="font-weight:bold;">Selected Pair:</span></p>
<p><span class="font0" style="font-weight:bold;">Rank: 1= PCR Primer Pair Search</span></p>
<p><span class="font0" style="font-weight:bold;">Primer A: &nbsp;&nbsp;&nbsp;&nbsp;Molecule: daerah Pos: 193</span></p>
<p><span class="font0" style="font-weight:bold;">PrimerB: &nbsp;&nbsp;&nbsp;&nbsp;Molecule: daerah Pos: 334Compi</span></p></td></tr>
<tr><td style="vertical-align:top;"></td><td style="vertical-align:middle;">
<p><span class="font1">A Z Length: 19 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GAATGGCTTAGCAGAAAGG</span></p>
<p><span class="font1">B Z Length: 19 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GGTTGTAGATGGACATAGG</span></p>
<p><span class="font1">Amplifiedproduct: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Size: 142 ⅛≡ Product GC 35%</span></p>
<p><span class="font1">Calctemperatures: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Melting T 69°C, Annealing T 43°C</span></p>
<p><span class="font1">Linkedmolecule: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;daeιaħ &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Po≡: 193, 334 C</span></p>
<p><span class="font1">Infb I Analyze</span></p></td></tr>
</table>
<p><span class="font6" style="font-weight:bold;">Fig. 4. </span><span class="font6">Primer information of F-Siem dan R-Siem</span></p><img src="https://jurnal.harianregional.com/media/60847-5.jpg" alt="" style="width:313pt;height:207pt;">
<p><span class="font6" style="font-weight:bold;">Fig. 5. </span><span class="font6">The results of the analysis of the primer design of F-Siem and R-Siem</span></p>
<p><span class="font6">The primer attachment sites for each </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> sample in the Ugene application are as follows:</span></p>
<p><span class="font6">Table</span><span class="font6" style="text-decoration:underline;">. 3 Primer attachment sites F-Siem and R-Siem on each sample </span><span class="font6" style="font-style:italic;text-decoration:underline;">C. nobilis</span></p>
<p><span class="font6">Orange cultivar &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Attachment sites &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Product length</span></p>
<table border="1">
<tr><td style="vertical-align:middle;">
<p><span class="font6">Siem</span></p></td><td style="vertical-align:middle;">
<p><span class="font6">193-334</span></p></td><td style="vertical-align:middle;">
<p><span class="font6">142</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font6">Siem Denpasar</span></p></td><td style="vertical-align:middle;">
<p><span class="font6">244-385</span></p></td><td style="vertical-align:middle;">
<p><span class="font6">142</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">Siem Buleleng</span></p></td><td style="vertical-align:top;">
<p><span class="font6">235-376</span></p></td><td style="vertical-align:top;">
<p><span class="font6">142</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">Siem Gianyar</span></p></td><td style="vertical-align:top;">
<p><span class="font6">240-381</span></p></td><td style="vertical-align:top;">
<p><span class="font6">142</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">Siem Payangan</span></p></td><td style="vertical-align:top;">
<p><span class="font6">202-343</span></p></td><td style="vertical-align:top;">
<p><span class="font6">142</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">Siem Pecatu</span></p></td><td style="vertical-align:top;">
<p><span class="font6">236-377</span></p></td><td style="vertical-align:top;">
<p><span class="font6">142</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font6">Siem Tabanan</span></p></td><td style="vertical-align:middle;">
<p><span class="font6">249-390</span></p></td><td style="vertical-align:middle;">
<p><span class="font6">142</span></p></td></tr>
</table>
<h1><a name="bookmark10"></a><span class="font6" style="font-weight:bold;"><a name="bookmark11"></a>Primer Design Test Results by PCR (Polymerase Chain Reaction) Method</span></h1>
<p><span class="font6">Pair of kinkit primer and siem orange primer design results were tested to find out whether the primer succeeded in amplifying the DNA of oranges that had been isolated. Testing is done by PCR method. Each primer was tested on the three DNA samples namely DNA </span><span class="font6" style="font-style:italic;">Triphacia trifolia, Citrus nobilis</span><span class="font6"> cul. Denpasar, and </span><span class="font6" style="font-style:italic;">Citrus nobilis</span><span class="font6"> cul. Gianyar.</span></p>
<p><span class="font6">DNA isolation from the leaves of siem orange and kinkit orange was carried out to test the primer that had been made.</span></p>
<p><span class="font6">Total DNA isolation was carried out using the Mini Genomic DNA Kit from Genaid. The success of the DNA isolation stage can be seen from the visualization of agarose gel electrophoresis on the UV transilluminator (Fig. 6). The success of the total DNA isolation process is evidenced by the presence or absence of DNA bands in agarose gels during electrophoresis.</span></p>
<div><img src="https://jurnal.harianregional.com/media/60847-6.jpg" alt="" style="width:185pt;height:226pt;">
<p><span class="font6" style="font-weight:bold;">Fig. 6. </span><span class="font6">Visualization of the total DNA isolation results of </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">, </span><span class="font6" style="font-style:italic;">C. nobilis cul. </span><span class="font6">Gianyar</span><span class="font6" style="font-style:italic;">,</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">C. nobilis cul.</span><span class="font6"> Denpasar</span></p>
</div><br clear="all">
<div>
<p><span class="font5">Total DNA</span></p>
</div><br clear="all">
<p><span class="font6">The PCR results were then electrophoresed on 1% agarose gel using a 1% TAE buffer solution. Warming is done for 20 minutes (100 volts). The agarose gel is then transferred to a UV transilluminator for</span></p>
<p><span class="font6">PCR DNA bands. The presence or absence of DNA bands indicates the primer success in amplifying the target fragment needed. Visualization of PCR results using F-Kinkit and R-Kinkit primers can be seen in Fig. 7.</span></p>
<div><img src="https://jurnal.harianregional.com/media/60847-7.jpg" alt="" style="width:100pt;height:187pt;">
</div><br clear="all">
<p><span class="font6" style="font-weight:bold;">Fig. 7. </span><span class="font6">The results of DNA amplification using F-Kinkit and R-Kinkit primers (1) </span><span class="font6" style="font-style:italic;">C. nobilis </span><span class="font6">cul. Gianyar, (2) </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> cul. Denpasar, and (3) </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">.</span></p>
<p><span class="font6">The primer pair F-Kinkit and R-Kinkit succeeded in amplifying the DNA of </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> cul. Denpasar, and </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">. DNA bands are seen in lines two and three which prove that DNA was successfully amplified by the primer pairs that had been</span></p>
<p><span class="font6">designed. Lane one filled with </span><span class="font6" style="font-style:italic;">C. nobilis </span><span class="font6">DNA DNA solution. Gianyar did not show the DNA band (Fig. 7). DNA is not amplified according to </span><span class="font6" style="font-style:italic;">in silico</span><span class="font6"> tests using the Ugene application.</span></p>
<div><img src="https://jurnal.harianregional.com/media/60847-8.jpg" alt="" style="width:103pt;height:188pt;">
</div><br clear="all">
<p><span class="font6" style="font-weight:bold;">Fig. 8. </span><span class="font6">The results of DNA amplification using F-Siem and R-Siem primers (1) </span><span class="font6" style="font-style:italic;">C. nobilis cul.</span></p>
<p><span class="font6">Gianyar, (2) </span><span class="font6" style="font-style:italic;">C. nobilis cul.</span><span class="font6"> Denpasar, and (3) </span><span class="font6" style="font-style:italic;">T. trifolia.</span></p>
<p><span class="font6">Primer F-Siem and R-Siem amplified all three DNA samples. The primer is expected not to apply </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> DNA. The</span></p>
<p><span class="font6">amplification of </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> DNA is probably due to the high level of homology of </span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6"> and </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> DNA. The high level</span></p>
<p><span class="font6">of homology between genes allows the primer to amplify several genes, although there are differences in several nucleotides (Kamel, 2003).</span></p>
<h1><a name="bookmark12"></a><span class="font6" style="font-weight:bold;"><a name="bookmark13"></a>CONCLUSION</span></h1>
<p><span class="font6">In this study it can be concluded that Primer design results for amplifying CVPDr DNA fragments in the plant of kinkit (</span><span class="font6" style="font-style:italic;">T. trifolia</span><span class="font6">) namely TCATCTGCATGGGATACC primer primer &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;reverse &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;primer</span></p>
<p><span class="font6">GCCTTGAGCTTGTAAGTG turned out to amplify the DNA of kinkit orange and also the siem orange cultivar Denpasar. The primer only succeeded in not amplifying the Gianyar cultivar siem orange. Primer design results to amplify CVPDr DNA fragments on the plant of siam (</span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6">), namely GAATGGCTTAGCAGAAAGG forward primer &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;reverse &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;primer</span></p>
<p><span class="font6">GGTTGTAGATGGACATAGG are not able to only amplify </span><span class="font6" style="font-style:italic;">C. nobilis</span><span class="font6"> but also amplify </span><span class="font6" style="font-style:italic;">T. trifolia.</span></p>
<h1><a name="bookmark14"></a><span class="font6" style="font-weight:bold;"><a name="bookmark15"></a>ACKNOWLEDGEMENTS</span></h1>
<p><span class="font6">This study was supported by Udayana University Research Grant No. 383-5/UN</span></p>
<p><span class="font6">14.4.A/LT//2018</span></p>
<h1><a name="bookmark16"></a><span class="font6" style="font-weight:bold;"><a name="bookmark17"></a>REFERENCES</span></h1>
<p><span class="font6">Badan Pusat Statistik Provinsi Bali. 2018. Produksi buah jeruk dirinci menurut</span></p>
<p><span class="font6">Kabupaten/Kota di Bali, 2000-2018. </span><a href="https://bali.bps.go.id/dynamictable/&#x9;2017/05/18/128/produksi-ton-buah-jeruk-dirinci-menurut-kabupaten-kota-di-bali-2011-2015.html"><span class="font6">https://bali.bps.go.id/dynamictable/</span></a><span class="font6"> </span><a href="https://bali.bps.go.id/dynamictable/&#x9;2017/05/18/128/produksi-ton-buah-jeruk-dirinci-menurut-kabupaten-kota-di-bali-2011-2015.html"><span class="font6">2017/05/18/128/produksi-ton-buah-</span></a><span class="font6"></span><a href="https://bali.bps.go.id/dynamictable/&#x9;2017/05/18/128/produksi-ton-buah-jeruk-dirinci-menurut-kabupaten-kota-di-bali-2011-2015.html"><span class="font6">jeruk-dirinci-menurut-kabupaten-</span></a><span class="font6"></span><a href="https://bali.bps.go.id/dynamictable/&#x9;2017/05/18/128/produksi-ton-buah-jeruk-dirinci-menurut-kabupaten-kota-di-bali-2011-2015.html"><span class="font6">kota-di-bali-2011-2015.html</span></a></p>
<p><span class="font6">Breslauer, K.J., R. Frank, H. Blocker, L. A. Markey. 1986. Predicting DNA duplex stability from the base sequence. Proc. Natl. Acad. Sci. USA, 83:3746-3750.</span></p>
<p><span class="font6">Jaqoueix, S., J.M Bove, and M. Garnier. 1994. The phloem limited bacterium of greening disease of citrus is as member of the alpha subdivision of proteobacteria. International J. Systemic Bacteriol. 44:397-86.</span></p>
<p><span class="font6">Kamel, A. 2003. Bioinformatic tools and guideline for PCR primer design. African Journal of Biotechnology 2 (5):91-95.</span></p>
<p><span class="font6">Mahayani, S. 2013. Analisis Ekspresi Klon Gen CVPD<sup>r</sup> Dalam Sel </span><span class="font6" style="font-style:italic;">Escherichia coli.</span><span class="font6"> Jurnal Agroknow 1(1): 33-38.</span></p>
<p><span class="font6">Martasari, C., A. Supriyanto, Hardiyanto, D. Agisimanto, dan H. Mulyanto. 2004. Keragaman Jeruk Siam di Indonesia. Proseding Seminar Jeruk Siam Nasional. 57-69.</span></p>
<p><span class="font6">Okonechnikov K, Golosova O, Fursov M, the UGENE team. 2012. Unipro UGENE: &nbsp;a unified bioinformatics</span></p>
<p><span class="font6">toolkit. Bioinformatics 28: &nbsp;1166</span></p>
<p><span class="font6">1167.</span></p>
<p><span class="font6">Wirawan, I. G. P., S. Liliek dan N. Wijaya. 2004. Penyakit CVPD pada tanaman jeruk. Denpasar. Udayana Press.</span></p>
<p><span class="font6">Wirawan, I. G. P. 2016. Distribution of CVPD<sup>r</sup> gene among some citrus plants in bali. International Journal of Bioscience and Biotechnology, 3(2).</span></p>
<p><span class="font6">Yuniti, I. G. A. D. 2018. Polimorfisme fragmen DNA CVPD<sup>r</sup> pada beberapa jenis tanaman jeruk di Bali. (</span><span class="font6" style="font-style:italic;">Disertasi</span><span class="font6">). &nbsp;&nbsp;Universitas Udayana.</span></p>
<p><span class="font6">Denpasar.</span></p>
<p><span class="font2">100 • FACULTY OF AGRICULTURE, UDAYANA UNIVERSITY</span></p>