---
layout: full_article
title: "Total Genomic DNA Extraction Studies from Seaweeds"
author: "Wildan Mujahidul Basyar, Made Pharmawati, Ida Ayu Astarini"
categories: atbes
canonical_url: https://jurnal.harianregional.com/atbes/full-58096 
citation_abstract_html_url: "https://jurnal.harianregional.com/atbes/id-58096"
citation_pdf_url: "https://jurnal.harianregional.com/atbes/full-58096"  
comments: true
---

<p><span class="font3">Advances in Tropical Biodiversity and Environmental Sciences 4(1): 10-14, February, 2020 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;e-ISSN:2622-0628</span></p>
<p><span class="font3">DOI: 10.24843/ATBES.2020.v04.i01.p03 Available online at:</span><a href="https://ojs.unud.ac.id/index.php/ATBES/article/view/49860"><span class="font3"> https://ojs.unud.ac.id/index.php/ATBES/article/view/5</span></a><span class="font3">8096</span></p>
<p><span class="font3">10</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font5"><a name="bookmark1"></a>Total Genomic DNA Extraction Studies from Seaweeds</span></h1>
<h3><a name="bookmark2"></a><span class="font4"><a name="bookmark3"></a>Wildan Mujahidul Basyar, Made Pharmawati*, Ida Ayu Astarini</span></h3>
<p><span class="font3">Biology Study Program, Faculty of Mathematics and Natural Sciences, Udayana University Jl. Kampus Unud Bukit Jimbaran, Badung-80361, Bali, Indonesia *Corresponding author: </span><a href="mailto:made_pharmawati@unud.ac.id"><span class="font3">made_pharmawati@unud.ac.id</span></a></p>
<p><span class="font3" style="font-weight:bold;">Abstract</span><span class="font3">. One of marine resources that has high value is seaweed. Seaweed is a carrageenan producer used in the food industry. Seaweed contains many minerals, vitamins and proteins that are useful for health. Carotenoids are pigments found in seaweed that function as antioxidants. The genes involved in carotenoid biosynthesis have been studied and provide opportunities for genetic improvement of seaweed. DNA is a basic requirement in molecular analysis. Therefore, a suitable method of DNA extraction from seaweed is needed. The aim of this research was to investigate DNA extraction method from several seaweed species and test the DNA quality through PCR-RAPD. Seaweed samples were collected from Pantai Bumi Perkemahan Taman Nasional Bali Barat and DNA was extracted using Doyle and Doyle’s method with modifications. PCR-RAPD was conducted using primer UBC127 and OPD 11 to test the quality of DNA. Results showed that 3 hours incubation in 60ºC had the best result of DNA extraction. However, the quality of DNA was low, as indicated by inconsistent PCR-RAPD products. Further optimization in DNA extraction is needed to obtain high quality DNA for genetic analysis.</span></p>
<p><span class="font3" style="font-weight:bold;font-style:italic;">Keywords:</span><span class="font3"> DNA; seaweed; quality; PCR-RAPD</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">I. &nbsp;&nbsp;&nbsp;INTRODUCTION</span></p></li></ul>
<p><span class="font3">Bali has a variety of coastal and marine ecosystems such as mangroves, corals and seaweed that have important values such as for food and medicine. Development of marine resources into a source of medicines, is important for both the development of science and economic.</span></p>
<p><span class="font3">Market demand for natural ingredients as a source of food and medicine is increasing rapidly [1]. One of the ingredients that is currently very popular in the market is carotenoids. Carotenoids are fat-soluble pigments, which are a group of yellow, orange, red and brown pigments found in various organisms such as: phytoplankton, seaweed, bacteria and certain plants. Humans and animals consume carotenoids to meet the needs in their bodies as antioxidants and are very dependent on external supplies [2].</span></p>
<p><span class="font3">Currently, carotenoids are produced on an industrial scale synthetically using chemicals. This synthetic carotenoid has the disadvantage of being a stereoisomer not found in natural carotenoids [3]. In addition, contamination can occur with reaction intermediates and contain less nutrients [3]. Therefore, there is an opportunity for carotenoid production from natural sources. One natural resource that has not been much explored is seaweed.</span></p>
<p><span class="font3">All seaweed produces carotenoids. The genes responsible for carotenoid biosynthesis (Crt gene groups) have been characterized [4, 5]. The discovery of these genes provides an opportunity to engineer the biosynthesis pathway of astaxanthin and other carotenoids in order to produce various carotenoids from various types of</span></p>
<p><span class="font3">seaweed that have the potential to produce higher carotenoid production [5].</span></p>
<p><span class="font3">Genetic engineering and other genetic analyses require DNA as material in research. Therefore, suitable DNA extraction technique from seaweed that can produce DNA with good quality and quantity is needed.</span></p>
<p><span class="font3">Methods of DNA extraction should be quick, reproducible and free from contamination. The quality of DNA determines the success of DNA analysis through amplification or digestion. The presence of contaminant can reduce amplification efficiency [6]. Several methods can be used to test the quality of DNA. Polymerase chain reaction (PCR) is one method that is used to confirm the DNA quality.</span></p>
<p><span class="font3">The aim of this research was to investigate DNA extraction method from several seaweed species and test DNA quality through PCR-RAPD.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">II. &nbsp;&nbsp;&nbsp;RESEARCH METHODS</span></p></li></ul>
<p><span class="font3" style="font-style:italic;">Sample Collection</span></p>
<p><span class="font3">Sampling was carried out at Pantai Bumi Perkemahan Taman Nasional Bali Barat. Seaweed samples collected were </span><span class="font3" style="font-style:italic;">Ulva reticulata</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Ulva lactuca</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Turbinaria sp</span><span class="font3">., </span><span class="font3" style="font-style:italic;">Gracilaria</span><span class="font3"> sp., </span><span class="font3" style="font-style:italic;">Sargassum</span><span class="font3"> sp., </span><span class="font3" style="font-style:italic;">Laurencia</span><span class="font3"> sp</span><span class="font3" style="font-style:italic;">.</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Padina</span><span class="font3"> sp., </span><span class="font3" style="font-style:italic;">Halimeda</span><span class="font3"> sp 1, </span><span class="font3" style="font-style:italic;">Halimeda opuntia</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Amphiroa</span><span class="font3"> sp. (Fig. 1). Samples were taken by hand in intertidal areas during low tide. Identification was done based on morphology using several references [7, 8].</span></p>
<p><span class="font3">Seaweeds were dried on newspaper and air dried without sunlight. After that the dried seaweeds were cut</span></p>
<p><span class="font3">with scissors until it was small then blended to form fine seaweed powder.</span></p><img src="https://jurnal.harianregional.com/media/58096-1.jpg" alt="" style="width:188pt;height:353pt;">
<p><span class="font3">Fig. 1. Seaweed species collected from Pantai Bumi Perkemahan Taman Nasional Bali Barat. a. </span><span class="font3" style="font-style:italic;">Gracilaria</span><span class="font3"> sp., b. </span><span class="font3" style="font-style:italic;">Turbinaria</span><span class="font3"> sp., c. </span><span class="font3" style="font-style:italic;">Padina</span><span class="font3"> sp,, d. </span><span class="font3" style="font-style:italic;">Sargassum</span><span class="font3"> sp., e. </span><span class="font3" style="font-style:italic;">Ulva reticulata</span><span class="font3">, f. </span><span class="font3" style="font-style:italic;">U. lactuca</span><span class="font3">, g. </span><span class="font3" style="font-style:italic;">H. macroloba</span><span class="font3">, h. </span><span class="font3" style="font-style:italic;">H. opuntia</span><span class="font3">, i. </span><span class="font3" style="font-style:italic;">Laurencia</span><span class="font3"> sp., j. </span><span class="font3" style="font-style:italic;">Amphiroa</span><span class="font3"> sp.</span></p>
<p><span class="font3" style="font-style:italic;">Seaweed DNA Extraction</span></p>
<p><span class="font3">DNA extraction was carried out based on the Doyle and Doyle [9] method by modifying the incubation time in the extraction buffer. As much as 0.1 gr of fine blended seaweed powder was incubated at 65ºC in 1 ml of DNA extraction buffer (2% CTAB, 1.4 M NaCl, 20 mM EDTA, 100 mM Tris pH 8 added 1% β mercaptoethanol and 1 % PVP) for 1 hour (the mixture was mixed every five minutes). Centrifugation was done at a rotating speed of 12,000 rpm for 10 minutes. The resulting supernatant was transferred to a new Eppendorf tube and then a mixture of phenol: chloroform: isoamylalcohol (25: 24: 1) with the same volume was added. Centrifugation at a rotational speed of 12,000 rpm for 5 minutes was conducted. The supernatant was transferred to a new Eppendorf tube then a chloroform: isoamylalcohol (24: 1) mixture was added in the same volume as the supernatant. The mixture was then</span></p>
<p><span class="font3">centrifuged at a rotational speed of 12,000 rpm for 5 minutes.</span></p>
<p><span class="font3">The resulting supernatant was transferred to a new Eppendorf tube then cold isopropanol was added 2 times the supernatant volume and incubated at cold temperature (-20ºC) for 30 minutes. Subsequently centrifuged 10,000 rpm for 5 minutes and the supernatant removed. The resulting pellet was washed with 70% ethanol then centrifuged at a rotational speed of 10,000 rpm for 4 minutes. Washing liquid was then discarded and the pellets were dried and then added by 100 µl of sterile water.</span></p>
<p><span class="font3">In the second modification, the sample used was 0.1 gr of fine powder of seaweed and incubated at 65ºC in 1.5 ml of DNA extraction buffer for 2 hours. The third modification method was carried out using a sample of 0.1 g of fine powder of seaweed and incubated at 65ºC in 2 ml of DNA extraction buffer for 3 hours.</span></p>
<p><span class="font3" style="font-style:italic;">PCR-RAPD</span></p>
<p><span class="font3">In this study, 2 RAPD primers (Random Amplified Polymorphic DNA) were used. The primers were UBC127 (5’-ATCTGGCAGC-3’) and OPD11 (5’-ACGGCCATTG-3’).</span></p>
<p><span class="font3">PCR reactions with UBC127 and OPD11 primers were carried out at a total volume of 20 µl containing a mixture: 1X polymerase buffer, 4.5mM MgCl</span><span class="font2">2</span><span class="font3">, 0.25mM dNTP, 1x Taq DNA polymerase, 3M primers and 15 ng DNA, 5% glycerol and the remaining sterile water up to a volume of 20 µl. The PCR programs used were: initial denaturation at 94ºC for 5 minutes with one cycle, denaturation at 94ºC for 1 minute, annealing at 36ºC for 1 minutes, elongation at 72ºC for 2 minutes with 40 times cycles and the final elongation was done at 72ºC for 7 minutes for one cycle.</span></p>
<p><span class="font3" style="font-style:italic;">Electrophoresis of DNA and PCR Product</span></p>
<p><span class="font3">In electrophoresis of total seaweed DNA extraction, 0.8% agarose was used for visualization, while for PCR product, 1% agarose in 1X TAE (40mM Tris-acetate pH 7.9 and 2mM Na</span><span class="font2">2</span><span class="font3">EDTA) was used. DNA samples were mixed with loading blue with a ratio of 5 µl DNA samples were added 1 µl loading blue, then put in a gel well and electrified 100 volts for 30 minutes. Staining was done by soaking the gel in ethidium bromide for 30 minutes. DNA observation was carried out using a UV transilluminator (Bio-Rad Mini) and a photo shoot was taken.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">III. &nbsp;&nbsp;&nbsp;RESULTS AND DISCUSSION</span></p></li></ul>
<p><span class="font3">Extraction with an incubation time of 1 hour showed DNA bands on </span><span class="font3" style="font-style:italic;">U. reticulata</span><span class="font3"> and </span><span class="font3" style="font-style:italic;">H. opuntia</span><span class="font3">. DNA bands with smears were observed on </span><span class="font3" style="font-style:italic;">Sargassum sp</span><span class="font3">. and </span><span class="font3" style="font-style:italic;">Padina sp</span><span class="font3">, while in </span><span class="font3" style="font-style:italic;">Turbinaria sp</span><span class="font3">., </span><span class="font3" style="font-style:italic;">Gracilaria sp</span><span class="font3">., </span><span class="font3" style="font-style:italic;">H. opuntia</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Amphiroa sp</span><span class="font3">. and </span><span class="font3" style="font-style:italic;">H. macroloba</span><span class="font3">, no DNA band was observed (Figure 2).</span></p>
<div><img src="https://jurnal.harianregional.com/media/58096-2.jpg" alt="" style="width:236pt;height:81pt;">
<p><span class="font3">Figure 2. DNA extraction results from several seaweed species with 1 hour incubation in CTAB buffer. 1. </span><span class="font3" style="font-style:italic;">Sargassum sp</span><span class="font3">., </span><span class="font3" style="font-style:italic;">2. Padina sp</span><span class="font3">., 3-5. </span><span class="font3" style="font-style:italic;">Turbinaria sp</span><span class="font3">., 6. </span><span class="font3" style="font-style:italic;">Gracilaria sp</span><span class="font3">., 7. </span><span class="font3" style="font-style:italic;">U. reticulate</span><span class="font3">, 8. </span><span class="font3" style="font-style:italic;">H. opuntia</span><span class="font3">, 9. </span><span class="font3" style="font-style:italic;">Amphiroa </span><span class="font3">sp., 10. </span><span class="font3" style="font-style:italic;">H. macroloba</span><span class="font3">. Arrows indicate DNA bands.</span></p>
</div><br clear="all">
<p><span class="font3">and 8). Therefore, electrophoresis was repeated and 10 µl DNA from each species was loaded. Result from the electrophoresis was shown in Figure 4.</span></p>
<div><img src="https://jurnal.harianregional.com/media/58096-3.jpg" alt="" style="width:46pt;height:100pt;">
</div><br clear="all">
<p><span class="font3">In DNA extraction with 1 hour incubation, DNA was not extracted successfully in several samples such as </span><span class="font3" style="font-style:italic;">Turbinaria</span><span class="font3"> sp., </span><span class="font3" style="font-style:italic;">Gracilaria</span><span class="font3"> sp., </span><span class="font3" style="font-style:italic;">Amphiroa</span><span class="font3"> sp. And </span><span class="font3" style="font-style:italic;">H. macroloba</span><span class="font3">. This can be due to the lack of extraction buffer volume and lack of incubation time in this sample so that the buffer is less optimal in lyse the cell walls and membranes. According to Jain [10] a longer incubation period in lysis buffer increases the exposure of the cells to the lysis buffer which lead to a more complete cell lysis. Besides that, according to Rogers and Bendich [11], the use of a buffer in a small volume (1 ml) is unable to separate proteins and polysaccharides so that it contaminates the results of isolation.</span></p>
<p><span class="font3">Extraction by modifying the incubation period for 2 hours produced DNA that is clearly visible in several species of seaweed such as </span><span class="font3" style="font-style:italic;">Sargassum</span><span class="font3"> sp., </span><span class="font3" style="font-style:italic;">H. opuntia</span><span class="font3">, and </span><span class="font3" style="font-style:italic;">H. macroloba</span><span class="font3"> (Fig. 3). Extraction by incubation for 3 hours produced DNA in </span><span class="font3" style="font-style:italic;">Sargassum</span><span class="font3"> sp., </span><span class="font3" style="font-style:italic;">Ulva lactuca</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Ulva reticulata</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Gracilaria</span><span class="font3"> sp. and </span><span class="font3" style="font-style:italic;">Amphiroa</span><span class="font3"> sp. as shown in Figure 3.</span></p><img src="https://jurnal.harianregional.com/media/58096-4.jpg" alt="" style="width:236pt;height:86pt;">
<p><span class="font3">Figure 3. The results of DNA extraction from several species of seaweed with incubation of 2 hours (1-4) and 3 hours (5-11). 1. </span><span class="font3" style="font-style:italic;">Sargassum sp</span><span class="font3">., 2. </span><span class="font3" style="font-style:italic;">H. opuntia</span><span class="font3">, 3-4. </span><span class="font3" style="font-style:italic;">H. macroloba</span><span class="font3">, 5. </span><span class="font3" style="font-style:italic;">Sargassum</span><span class="font3"> sp., 6. </span><span class="font3" style="font-style:italic;">U. lactuc</span><span class="font3">a. 7. </span><span class="font3" style="font-style:italic;">Laurencia sp</span><span class="font3">., 8. </span><span class="font3" style="font-style:italic;">Turbinaria sp</span><span class="font3">., 9. </span><span class="font3" style="font-style:italic;">U. reticulate</span><span class="font3">, 10. </span><span class="font3" style="font-style:italic;">Gracilaria sp</span><span class="font3">., 11. </span><span class="font3" style="font-style:italic;">Amphiroa sp</span><span class="font3">. Arrows indicate DNA bands.</span></p>
<p><span class="font3">By using an incubation time of 3 hours, DNA was obtained from </span><span class="font3" style="font-style:italic;">Halimeda macroloba</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Amphiroa</span><span class="font3"> sp. and </span><span class="font3" style="font-style:italic;">Gracilaria</span><span class="font3"> sp. which did not work on incubation for 1 hour. DNA that appeared with smear in some seaweed samples indicated that DNA had degraded [12]. DNA breaks into small fragments with small molecular size due to the activity of the nuclease enzymes that cut DNA [13].</span></p>
<p><span class="font3">Very faint DNA bands were observed for </span><span class="font3" style="font-style:italic;">Laurencia</span><span class="font3"> sp and </span><span class="font3" style="font-style:italic;">Turbinaria sp</span><span class="font3"> as shown in Fig. 3 (sample number 7</span></p>
<p><span class="font3">Figure 4. Electrophoresis of DNA extracted from Turbinaria sp. (1) and Laurencia sp. (2) where 10 µl DNA was loaded for each species. Arrows indicate DNA bands</span></p>
<p><span class="font3">To test DNA quality, RAPD PCR was carried out using primers UBC 127 and OPD 11. PCR-RAPD technique has widely been used for analyzing the genetic variability in plants. This technique is rapid, simple and does not require prior genetic information of the samples [15]. Result showed that only UBC127 primer produced PCR products and the PCR products were only from 3 samples namely </span><span class="font3" style="font-style:italic;">U. reticulata</span><span class="font3">, </span><span class="font3" style="font-style:italic;">Amphiroa sp</span><span class="font3">., </span><span class="font3" style="font-style:italic;">Sargassum sp</span><span class="font3"> (Figure 5). This showed that the quality of DNA was low.</span></p>
<p><span class="font3">PCR products using UBC 127 primer exhibit thin DNA bands. This may because the conditions in PCR reaction are not yet optimal despite the possibility that there are contaminant compounds contained in seaweed DNA such as carrageenan, alginate, fucoidan and other polysaccharides. These contaminants become inhibitors in the PCR reaction. The presence of polyphenols, and other secondary metabolites also influence PCR reaction [16].</span></p>
<p><span class="font3">The success of DNA extraction depends on four important steps: effective cell disruption; nucleoprotein complexes denaturation; nucleases inactivation and free from contamination [14].</span></p>
<h2><a name="bookmark4"></a><span class="font0"><a name="bookmark5"></a>1 &nbsp;&nbsp;2 &nbsp;&nbsp;3 4 5 &nbsp;&nbsp;6 &nbsp;&nbsp;7 1kb</span></h2>
<p><span class="font0">3000 bp</span></p>
<p><span class="font0">1000 bp</span></p>
<p><span class="font3">Figure 5. PCR-RAPD of seaweed using primer UBC127. 1. </span><span class="font3" style="font-style:italic;">H. opuntia</span><span class="font3">, 2. </span><span class="font3" style="font-style:italic;">H. macroloba</span><span class="font3">, 3. </span><span class="font3" style="font-style:italic;">Laurencia sp</span><span class="font3">., 4. </span><span class="font3" style="font-style:italic;">Turbinaria sp</span><span class="font3">., 5</span><span class="font1">. </span><span class="font3" style="font-style:italic;">U. reticulata</span><span class="font3">, 6. </span><span class="font3" style="font-style:italic;">Amphiroa sp</span><span class="font3">., 7. </span><span class="font3" style="font-style:italic;">Sargassum sp</span><span class="font3">. Arrows indicate PCR products.</span></p>
<p><span class="font3">PCR with RAPD primers provides a great chance of success for every sample, because RAPD primers are short DNA sequences (10-12 base pairs) making it easier to amplify DNA. Primers with short sequences are highly</span></p>
<p><span class="font3">likely to be attached to the DNA site of the sample used. According to Bardakci [17] in amplification using RAPD primer, DNA from all sources is capable to amplification.</span></p>
<p><span class="font3">However, several factors affected the success of RAPD amplification such as DNA quality and PCR-RAPD protocol. Concentrations of primers, MgCl</span><span class="font2">2</span><span class="font3">, template DNA should be optimized [16].</span></p>
<p><span class="font3">In this study, the colour of extracted DNA was brown indicated contamination with phenolic compounds and the DNA solution was viscous for most samples. The viscosity of DNA shows the co-precipitation of polysaccharide with DNA [18].</span></p>
<p><span class="font3">DNA extraction from seaweed was reported successful using fresh materials [19]. Furthermore, the treatment with enzymes and grinding with liquid nitrogen have resulted in good quality of DNA [20, 21]. However, enzymes are expensive and liquid nitrogen is not accessible in every area. Commercial DNA extraction kit was also successful in extracting DNA from seaweed [22, 23]. The high price of DNA extraction kit is a limitation in several laboratories. Further modifications of DNA protocol using CTAB lysis buffer need to be studied to obtain high quality and quantity of DNA from seaweed.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">IV. &nbsp;&nbsp;&nbsp;CONCLUSION</span></p></li></ul>
<p><span class="font3">DNA extraction with an incubation time of 3 hours in 2 ml of extraction buffer showed the best results. The quality of seaweed DNA obtained was low as indicated by faint DNA band as well as smeared DNA in several species. The low DNA quality was also indicated by the failure of amplification of seaweed DNA using PCR-RAPD. Only three DNA samples produced PCR product with primer UBC127, while no product for all seaweed DNA tested when primer OPD 11 used.</span></p>
<p><span class="font3">REFERENCES</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">[1] &nbsp;&nbsp;&nbsp;Jamshidi-Kia F, Z. Lorigooini, H. Amini-Khoei. 2018. Medicinal plants: Past history and future perspective. </span><span class="font3" style="font-style:italic;">J. Herbmed. Pharmacol</span><span class="font3">. 7(1): 1-7.</span></p></li>
<li>
<p><span class="font3">[2] &nbsp;&nbsp;&nbsp;Fiedor J, K. Burda. 2014. Potential role of carotenoids as antioxidants in human health and disease. </span><span class="font3" style="font-style:italic;">Nutrients</span><span class="font3">. 6(2):466–488.</span></p></li>
<li>
<p><span class="font3">[3] &nbsp;&nbsp;&nbsp;Ralley L, E.M.A. Enfissi, N. Misawa, W. Schuch, P.M. Bramley, P.D. Fraser. 2004. Metabolic engineering of ketocarotenoid formation in higher plants. </span><span class="font3" style="font-style:italic;">Plant J.</span><span class="font3"> 39:477–486.</span></p></li>
<li>
<p><span class="font3">[4] &nbsp;&nbsp;&nbsp;Misawa N, S. Tomy, K. Kondo, A. Yokohama, S. Kajiwara, T. Saito, T. Ohtani, W. Miki. 1995. Structure and functional analysis of a marine bacterial carotenoid biosynthesis. </span><span class="font3" style="font-style:italic;">J. Bacteriol</span><span class="font3">. 177(22): 65756584.</span></p></li>
<li>
<p><span class="font3">[5] &nbsp;&nbsp;&nbsp;Nishida Y, K. Adeshi, H. Kasai, Y. Shizuri, K. Shindo, A. Sawabe, S. Komemushi, M. Wataru, N.</span></p></li></ul>
<p><span class="font3">Misawa. &nbsp;2005. Elucidation of a carotenoid</span></p>
<p><span class="font3">biosynthesis gene cluster encoding a novel enzyme, 2,2’-β-hydroxylase, from </span><span class="font3" style="font-style:italic;">Brevundimonas</span><span class="font3"> sp. Strain SD212 and combinational biosynthesis of new and rare xanthophylls. </span><span class="font3" style="font-style:italic;">Appl. Environ. Microbiology</span><span class="font3">, 71: 4286-4296</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">[6] &nbsp;&nbsp;&nbsp;Pereira JC, R. Chaves, E. Bastos, A. Leitão, H. Guedes-Pinto. 2011. An efficient method for genomic DNA extraction from different Molluscs species. </span><span class="font3" style="font-style:italic;">Int. J. Mol. Sci.</span><span class="font3"> 12: 8086-8095.</span></p></li>
<li>
<p><span class="font3">[7] &nbsp;&nbsp;&nbsp;N’Yeurt, ADR. 1999. </span><span class="font3" style="font-style:italic;">A preliminary illustrated field guide to the common marine algae of the Cook Islands</span><span class="font3">. The University of the South Pacific, Suva, Fiji.</span></p></li>
<li>
<p><span class="font3">[8] &nbsp;&nbsp;&nbsp;Al-Yamani FY., I. Polikarpov, A. Al-Ghunaim, T. Mikhaylova. 2014. </span><span class="font3" style="font-style:italic;">Field guide of marine macroalgae (Chlorophyta, Rhodophyta, Phaeophyceae) &nbsp;&nbsp;of</span></p></li></ul>
<p><span class="font3" style="font-style:italic;">Kuwait</span><span class="font3">. Kuwait Institute for Scientific Research, Safat, Kuwait.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">[9] &nbsp;&nbsp;&nbsp;Doyle JJ, J.L. Doyle. 1990. Isolation of plant DNA from fresh tissue. </span><span class="font3" style="font-style:italic;">Focus</span><span class="font3">, 12: 13-15.</span></p></li>
<li>
<p><span class="font3">[10] &nbsp;&nbsp;&nbsp;Jain SA., F.T. de Jesus, G.M. Marchioro, E.D. de Araújo. 2013. Extraction of DNA from honey and its amplification by PCR for botanical identification. </span><span class="font3" style="font-style:italic;">Food Sci. Technol</span><span class="font3">. 33(4): 753-756.</span></p></li>
<li>
<p><span class="font3">[11] &nbsp;&nbsp;&nbsp;Rogers, S.O., A.J. Benedich. 1985. Extraction of DNA from milligram amounts of fresh herbarium and Mummified plant tissues. </span><span class="font3" style="font-style:italic;">Plant Mol. Biol</span><span class="font3">. 5: 69-76</span></p></li>
<li>
<p><span class="font3">[12] &nbsp;&nbsp;&nbsp;Das SS., S. Das (Sur), P. Ghosh. 2013. Optimization of DNA isolation and RAPD-PCR protocol of </span><span class="font3" style="font-style:italic;">Acanthus volubilis</span><span class="font3"> wall a rare mangrove plant from Indian Sundarban, for conservation concern. </span><span class="font3" style="font-style:italic;">European J. Exp. Biol</span><span class="font3">. 3(6):33-38</span></p></li>
<li>
<p><span class="font3">[13] &nbsp;&nbsp;&nbsp;Sahu SK., M. Thangaraj, K. Kathiresan. 2012. DNA extraction protocol for plants with high levels of secondary metabolites and polysaccharides without using liquid nitrogen and phenol. </span><span class="font3" style="font-style:italic;">ISRN Mol. Biol</span><span class="font3">. 2012, Article ID 205049, doi:10.5402/2012/205049</span></p></li>
<li>
<p><span class="font3">[14] &nbsp;&nbsp;&nbsp;Tan SC, B. C. Yiap. 2009. DNA, RNA, and protein extraction: the past and the present. </span><span class="font3" style="font-style:italic;">J. Biomed. Biotechnol.</span><span class="font3"> &nbsp;&nbsp;&nbsp;2009, Article ID 574398,</span></p></li></ul>
<p><span class="font3">doi:10.1155/2009/574398</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">[15] &nbsp;&nbsp;&nbsp;Shinde VM, K. Dhalwal, K.R. Mahadik, K.S. Joshi, B.K. Patwardhan. 2007. RAPD analysis for determination of components in herbal medicine. </span><span class="font3" style="font-style:italic;">Evid Based Complement Alternat Med</span><span class="font3">. 4(Suppl 1): 21–23. doi: 10.1093/ecam/nem109.</span></p></li>
<li>
<p><span class="font3">[16] &nbsp;&nbsp;&nbsp;Skoric M, B. Šiler, T. Banjanac, J.N. Živkov, S. Dmitrovic, D. Misic, D. Grubisic. 2012. &nbsp;&nbsp;&nbsp;The</span></p></li></ul>
<p><span class="font3">reproducibility of RAPD profiles: Еffects of PCR components on RAPD analysis of four </span><span class="font3" style="font-style:italic;">Centaurium </span><span class="font3">species. </span><span class="font3" style="font-style:italic;">Arch. Biol. Sci., Belgrade</span><span class="font3">, 64 (1), 191-199. DOI:10.2298/ABS1201191S191</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">[17] &nbsp;&nbsp;&nbsp;Bardakci F. 2001. Random amplified polymorphic</span></p></li></ul>
<p><span class="font3">DNA (RAPD) markers. Turk. J. Biol. 25: 185-196.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">[18] &nbsp;&nbsp;&nbsp;Souza HAV, L.A.C. Muller, R.L. Brandão, M.B. Lovato. 2012. Isolation of high quality and polysaccharide-free DNA from leaves of </span><span class="font3" style="font-style:italic;">Dimorphandra mollis</span><span class="font3"> (Leguminosae), a tree from the Brazilian Cerrado. </span><span class="font3" style="font-style:italic;">Genet. Mol. Res</span><span class="font3">. 11 (1): 756-764</span></p></li>
<li>
<p><span class="font3">[19] &nbsp;&nbsp;&nbsp;Ramakrishnan GS, A.A. Fathima, M. Ramya. 2017. A rapid and efficient DNA extraction method suitable for marine macroalgae. </span><span class="font3" style="font-style:italic;">Biotech.</span><span class="font3"> 7(6):364. doi:10.1007/s13205-017-0992-2.</span></p></li>
<li>
<p><span class="font3">[20] &nbsp;&nbsp;&nbsp;Wang G, Y. Li, P. Xia, D. Duan. 2005. A simple method for DNA extraction from sporophyte in the brown alga </span><span class="font3" style="font-style:italic;">Laminaria japonica</span><span class="font3">. </span><span class="font3" style="font-style:italic;">J. Appl. Phycol. </span><span class="font3">17(1):75–79. doi: 10.1007/s10811-005-5557-9.</span></p></li>
<li>
<p><span class="font3">[21] &nbsp;&nbsp;&nbsp;Joubert Y, J. Fleurence. DNA isolation protocol for seaweeds. </span><span class="font3" style="font-style:italic;">Plant Mol. Biol. Rep</span><span class="font3">. 2005. 23:197. doi: 10.1007/BF02772712.</span></p></li>
<li>
<p><span class="font3">[22] &nbsp;&nbsp;&nbsp;Irvan R. Hengkengbala, G.S. Gerung, S. Wullur. 2018. DNA extraction and amplification of the rbcL (ribulose-1,5-bisphosphate carboxylase/oxygenase large subunit) gene of red seaweed </span><span class="font3" style="font-style:italic;">Gracilaria</span><span class="font3"> sp. from Bahoi Waters, North Minahasa Regency. &nbsp;</span><span class="font3" style="font-style:italic;">J.</span></p></li></ul>
<p><span class="font3" style="font-style:italic;">Aquatic Sci. Management.</span><span class="font3"> 6(2): 33-38.</span></p>
<ul style="list-style:none;"><li>
<p><span class="font3">[23] &nbsp;&nbsp;&nbsp;Mohammed ZAA., N.A.H. Al–Bdairi, R.K. Abed. 2018. RAPD marker variability among algae species in Iraq. Biochem. Cell. Arch. 18, Suppl. 1: 953-958.</span></p></li></ul>