---
layout: full_article
title: "DNA MICROARRAY BASED EXPRESSIONAL PROFILING OF HRPXO DEPENDENT UP-REGULONS IN XANTHOMONAS ORYZAE PV. ORYZAE"
author: "Byoung Ho So, Jongun Kim, Hee Wan Kang"
categories: jbb
canonical_url: https://jurnal.harianregional.com/jbb/full-13592 
citation_abstract_html_url: "https://jurnal.harianregional.com/jbb/id-13592"
citation_pdf_url: "https://jurnal.harianregional.com/jbb/full-13592"  
comments: true
---

<p><span class="font0">ISSN: 9 772303 337008</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font4"><a name="bookmark1"></a>ARTICLE</span></h1>
<h2><a name="bookmark2"></a><span class="font8" style="font-weight:bold;"><a name="bookmark3"></a>DNA MICROARRAY BASED EXPRESSIONAL PROFILING OF </span><span class="font8" style="font-weight:bold;font-style:italic;">HRPXO </span><span class="font8" style="font-weight:bold;">DEPENDENTUP-REGULONS IN </span><span class="font8" style="font-weight:bold;font-style:italic;">XANTHOMONAS ORYZAE</span><span class="font8" style="font-weight:bold;"> PV</span><span class="font8" style="font-weight:bold;font-style:italic;">. ORYZAE</span></h2>
<p><span class="font7" style="font-weight:bold;">Byoung Ho So<sup>1</sup>, Jongun Kim<sup>1</sup>, Hee Wan Kang<sup>1,2*</sup></span></p>
<p><span class="font7" style="font-style:italic;"><sup>1</sup>Graduate School of Biotechnology and Information Technology, Hankyong National University, Ansung 456-749, Korea</span></p>
<p><span class="font7" style="font-style:italic;"><sup>2</sup>Institute of Genetic Engineering, Hankyong National University, Ansung 456-749, Korea</span></p>
<p><span class="font7">*Corresponding author: Hee-Wan Kang</span></p>
<p><span class="font7">Graduate School of Biotechnology and Information Technology, Hankyong National University, Ansung 456-749, Korea</span></p>
<p><span class="font7">Tel: +82-31-670-5420; Fax: +82-31-676-2602; E-mail: </span><a href="mailto:kanghw2@hknu.ac.kr"><span class="font7">kanghw2@hknu.ac.kr</span></a></p>
<p><span class="font7" style="font-weight:bold;">ABSTRACT</span></p>
<p><span class="font7">Thirty-mer oligonucleotides for microarray analysis were designed from the annotated open reading frames (ORFs) in the whole genome sequence of </span><span class="font7" style="font-style:italic;">X. oryzae</span><span class="font7"> pv. </span><span class="font7" style="font-style:italic;">oryzae</span><span class="font7"> KACC10331. Mutant </span><span class="font7" style="font-style:italic;">hrpX</span><span class="font7">o</span><span class="font7" style="font-style:italic;">::TN</span><span class="font7"> and wild-type strain KACC10859 were cultured in </span><span class="font7" style="font-style:italic;">hrp</span><span class="font7"> inducing medium (XOM2), and cDNAs, which were synthesized from the total RNA samples from both strains, were hybridized on a DNA microarray. The microarray data showed that 210 genes were down-regulated more than 2-fold in </span><span class="font7" style="font-style:italic;">hrpXo</span><span class="font7">::TN, while 115 genes were up-regulated more than 2-fold. The HrpXo regulons differently included 54 hypothetical genes: type III secretory genes (11 </span><span class="font7" style="font-style:italic;">hrp</span><span class="font7"> genes); genes encoding type III secretory effectors (</span><span class="font7" style="font-style:italic;">xopP</span><span class="font7"> genes and </span><span class="font7" style="font-style:italic;">avr/pthA</span><span class="font7">); genes encoding type II secretory effectors (6 proteases, a lipase, a polygalacturonase, and 4 cellulases); 7 iron-uptake genes; 6 </span><span class="font7" style="font-style:italic;">pil</span><span class="font7"> genes encoding fimbrial assembly membrane proteins; and 14 transposon genes. Significant plant-inducible promoter (PIP) sequences were newly identified on HrpXo regulons. The validity of the expressional profiles was further confirmed by reverse transcription (RT)-PCR.</span></p>
<h2><a name="bookmark4"></a><span class="font8" style="font-weight:bold;"><a name="bookmark5"></a>INTRODUCTION</span></h2>
<p><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> (</span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8">), a causal agent of bacterial blight (BB), is economically the most important bacterial pathogen in rice plants (</span><span class="font8" style="font-style:italic;">Oryza sativa</span><span class="font8"> L.) worldwide. The hypersensitive response and pathogenicity (mediated by the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> gene) and the production of extracellular polysaccharides (EPS), extracellular enzymes, and lipopolysaccharides (LPS) are known to be critical factors in pathogenesis (Chan and Goodwin 1999; Shen and Ronald 2002). Additionally, various virulence genes have been newly identified from the </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> genome by transposon mutagenesis (Park </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2007; Wang </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2008).</span></p>
<p><span class="font8">The </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes encode proteins involved in the type III secretion system (T3SS), which is involved in the secretion of effector proteins from bacteria to plant (Büttner and Bonas 2002; Grulebeck </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Keyu </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005). The </span><span class="font8" style="font-style:italic;">hrp </span><span class="font8">gene cluster in </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> is composed of 27 genes, from </span><span class="font8" style="font-style:italic;">hrpa</span><span class="font8">2 to </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">F (Lee </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005), and the expression of these genes is regulated by 2 regulators</span><span class="font8" style="font-style:italic;">, hrpG </span><span class="font8">and </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8">, which are separate from the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> gene</span></p>
<p><span class="font8">cluster (Oku </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1995; Tsuge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005). Hrpg belongs to the OmpR family and activates the expression of </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">X, an AraC-like transcription activator that controls </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes along with some effector proteins (Tsuge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006; Wengelnik and Bonus 1996; Wengelnik </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1999). The HrpX regulons of </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> species have been found to include a consensus sequence motif termed the plant-inducible promoter region (PIP) box (TTCGC-N15-TTCGC) around the promoter regions (Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006; Koebnik </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006; Tsuge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">.2005). In </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8">, an additional conserved </span><span class="font8" style="font-style:italic;">cis</span><span class="font8">-regulatory element, a -10 boxlike sequence that resembles the binding motif of RNA polymerase, was found downstream of the PIP box (Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006). By using PIP box sequences, numerous candidate genes belonging to Hrp regulons, other than the genes in the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> cluster, have been identified from the </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> genome database (da Silva </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2002; Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006; Koebnik </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006). It has been demonstrated that HrpXo regulates the transcriptional expression of genes associated with type II secretory proteins such as cysteine proteases (Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2004). Recently, it was</span></p>
<p><span class="font8">reported that polygalacturonase (PG) Wang<sup>a)</sup> </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">.2008; and extracellular proteases (Wei </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2007) in </span><span class="font8" style="font-style:italic;">X. campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8"> are regulated by HrpX. These reports suggest that HrpX can potentially function in regulating the expression of other genes in addition to the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes. A useful </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">-inducing medium, XOM2, was established for cultivation of Xoo and has greatly contributed to elucidating the </span><span class="font8" style="font-style:italic;">in vitro</span><span class="font8"> functional roles of </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> (Tsuge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2002).</span></p>
<p><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> genomes, including the genomes of Xoo, </span><span class="font8" style="font-style:italic;">X. axonopodis</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">citri</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">X. campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8">, have been completely sequenced (da Silva </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2002; Lee </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Ochiai </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005), and our knowledge of the sequences provides a powerful insight into the general and specific features of pathogenicity-related genes in </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> genomes</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> On the basis of the sequence information</span><span class="font8" style="font-style:italic;">,</span><span class="font8"> DNA microarrays of the </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pathovars </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">oryzicola</span><span class="font8"> have been constructed as biosensors, which can be used for high-throughput genome-wide gene expression analysis to study the regulation of </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes (Seo </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2008). In addition, DNA microarrays of the phytopathogenic bacteria </span><span class="font8" style="font-style:italic;">Pseudomonas syringae</span><span class="font8">, </span><span class="font8" style="font-style:italic;">Ralstonia solanacearum</span><span class="font8">, </span><span class="font8" style="font-style:italic;">Xanthomonas axonopodis</span><span class="font8">, </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">Xylella fastidiosa</span><span class="font8"> have been used to probe genome-wide expression related to pathogenesis and to assess the genomic diversity of meaningful bacterial isolates by (Astua-Monge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Guidot </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2009; He </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2007; He </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2006; Koide </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006; Lu </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Wang </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006). DNA microarray chip technology promises to allow monitoring of the entire genome on a single chip, thereby providing researchers with a better picture of the simultaneous interactions among thousands of genes (de Morgan </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2010).</span></p>
<p><span class="font8">In this study, our aim was to identify the genome-wide </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> regulons by using an Xoo DNA microarray</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> The </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> microarray was constructed using 30-mers derived from annotated open reading frames (ORFs) in the genome sequence of Xoo KACC10331. The transcriptional profiles of the </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> mutant</span></p>
<p><span class="font8">were compared with those of the wild-type strain KACC10859. The resultant </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> regulons included different genes, including T3SS genes, T3SS effector genes, type II secretory system (T2SS) effector genes such as genes encoding proteases and cellulases, as well as genes involved in the uptake of iron, fimbrial-assembly genes, and metabolism-related genes.</span></p>
<p><span class="font8" style="font-weight:bold;">MATERIALS AND METHODS</span></p>
<h2><a name="bookmark6"></a><span class="font8" style="font-weight:bold;"><a name="bookmark7"></a>Bacterial strains and culture</span></h2>
<p><span class="font8">Wild type strain, </span><span class="font8" style="font-style:italic;">X</span><span class="font8">. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae </span><span class="font8">KACC10859 was obtained from the Korean Agricultural Culture Collection (KACC) at the National Academy of Agricultural Science (NAAS), Suwon, Korea. Transposon mutant, </span><span class="font8" style="font-style:italic;">hrpXo::TN</span><span class="font8"> were a gift from Dr. Cho of National Academy of Agricultural Science, Suwon. </span><span class="font8" style="font-style:italic;">Xoo </span><span class="font8">strains were cultured on pepton sucrose agar (PSA) medium (Tsuchiya </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1982) at 28°C for 3 days. Antibiotic, kanamycin were added to final concentration (20 mg/liter) 20 for mutant, </span><span class="font8" style="font-style:italic;">hrpXo:: TN</span><span class="font8">.</span></p>
<h2><a name="bookmark8"></a><span class="font8" style="font-weight:bold;"><a name="bookmark9"></a>Induction of </span><span class="font8" style="font-weight:bold;font-style:italic;">hrpXo</span><span class="font8" style="font-weight:bold;"> gene expression</span></h2>
<p><span class="font8">Method of Tsuge </span><span class="font8" style="font-style:italic;">et a</span><span class="font8">l. (2002) was applied to modification for inducing hrpXo gene expression. Wild type and mutant, </span><span class="font8" style="font-style:italic;">hrpXo::TN</span><span class="font8"> strains were grown on PSA for 3 days and the bacterial cells were washed twice distilled water (DW) and resuspended in DW at an optical density of 2.0 /ml at 600 nm. The bacterial cells were harvested by centrifugation at 6,000 rpm and suspended in 5 ml of XOM2 medium and incubated with shaking for 18 h at 25 C. The cells were used for total RNA extraction.</span></p>
<h2><a name="bookmark10"></a><span class="font8" style="font-weight:bold;"><a name="bookmark11"></a>Fabrication of DNA microarray</span></h2>
<p><span class="font8">Fifty-mer oligonucleotides were designed from the respective ORFs in the genomic sequence data of </span><span class="font8" style="font-style:italic;">X. oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> KACC10331 by employing the following criteria: melting temperatures were normalized within 2°C; the G+C content of the designed oligonucleotides was restricted to 50 ± 5%, matching the 63.7%</span></p>
<p><span class="font8">G+C content of </span><span class="font8" style="font-style:italic;">X. oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> KACC10331; “no sequence homology” to other regions of the genome was restricted to a maximum of 35 bp, with no exact sequence matches of more than 15 bp. Finally, 3,057 oligonucleotides were directly synthesized on CombiMatirx CustomArray™ 12K Microarray (Ghindilis </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2007)according to the manufacturer’s protocol (Macrogen, Korea).</span></p>
<h2><a name="bookmark12"></a><span class="font8" style="font-weight:bold;"><a name="bookmark13"></a>cDNA labeling and hybridization</span></h2>
<p><span class="font8">The experiments were performed according to the standard CombiMatrix protocol described in detail at). Total RNA from mutant </span><span class="font8" style="font-style:italic;">hrpXo::TN </span><span class="font8">and the wild-type strain KACC10859, which were grown on XOM2 media, was purified by using the RNeasy mini kit in accordance with the manufacturer’s instructions (Qiagen, Valencia, CA, U.S.A.). Labeled cDNA was synthesized by random primed reverse transcription reactions in the presence of Cy3/Alexa 546 or Cy5/Alexa 647 by using SuperScript II (Invitrogen, USA). One microgram of RNA was mixed with 2 μg of random primers, heated to 80°C for 10 min, and cooled on ice for 2 min. Unlabeled deoxynucleoside triphosphates (10 mM each of dATP, dTTP, dGTP, and dCTP), 200 units of reverse transcriptase, 5× reverse transcription buffer, and 1 μl of SUPERase-In RNase inhibitor (Ambion, U.S.A.) were added to a final volume of 20 μl. The samples were incubated at 42°C for 2 h. The reaction was stopped by the addition of 3.5 μl of stop-solution (0.5 M NaOH, 50 mM EDTA), and the RNA was degraded by incubation at 65°C for 15 min. The mixture was neutralized by adding 5 μl of 1 M Tris-HCl (pH 7.5), and 21.5 μl of 1× TE buffer was added to a final volume of 50 μl. After purification of the cDNA sample by using a MinElute PCR purification kit (Qiagen, Germany), 16 μl of the sample was heated to 80°C for 10 min and cooled on ice for 2 min. Four microliters of 10 mM dTTP, either 2.5 μl of terminal deoxynucleotidyl transferase, and 2.5 μl of 10× tailing buffer were added to a final volume of 25 μl. The samples were incubated at 37°C for 30 min, heated to 95°C for 10 min, and then immediately cooled on ice for 2 min.</span></p>
<p><span class="font8">Five microliters of 6× ligation mix (Cy3/Alexa 546 or Cy5/Alexa 647) and 2 μl of T4 DNA ligase (Genishpere, U.S.A.) were added to a final volume of 32 μl. The sample was incubated at room temperature for 30 min, and 14.5 μl of 1× TE buffer was added to a final volume of 50 μl.</span></p>
<p><span class="font8">For hybridization, 2× hybridization buffer (6× SSPE, 0.05% Tween-20, 20 mM EDTA, 5× Denhardt’s solution, 1 mg Salmon sperm DNA, and 0.05% SDS) was added to the CombiMatirx CustomArray™ 12K Microarray assembly and prehybridized at 65°C for 10 min, and 10 μl of the labeled cDNA was added to a prewarmed microarray and then incubated at 65°C overnight in a dark humidified chamber. After hybridization, the microarrays were washed with pre-warmed washing buffer A (6× SSPE), B (3× SSPE), and C (0.5× SSPE) at 42°C for 15 min each and finally washed with PBS buffer.</span></p>
<h2><a name="bookmark14"></a><span class="font8" style="font-weight:bold;"><a name="bookmark15"></a>Image scan</span></h2>
<p><span class="font8">ElectraSenseTM Reader (</span><span class="font8" style="text-decoration:underline;">http://www. combimatrix.com/</span><span class="font8"> products electrasense.htm) and ElectraSenseTM 12K microarrays were used for all electrochemical assays according to the standard CombiMatrix protocol. The microarrays were assembled with special ElectraSenseTM caps to create hybridization chambers. After hybridization, the microarrays were washed and incubated with HRP-Avidin solution in BSA Peroxidase Stabilizer (1:1000) for 45 min. Finally, the hybridization chambers were filled with TMB solution, and the microarrays were inserted into the ElectraSenseTM Reader for detection.</span></p>
<h2><a name="bookmark16"></a><span class="font8" style="font-weight:bold;"><a name="bookmark17"></a>Reverse transcriptase-PCR analysis</span></h2>
<p><span class="font8">The mutant strain </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">::</span><span class="font8" style="font-style:italic;">TN</span><span class="font8"> and wild-type strain KACC10859 were cultured in 5 ml of the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">-inducing medium XOM2 (Tsuge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">., 2002) and were extracted with an RNeasy kit (Qiagen, Valencia, CA, U.S.A.), which was followed by DNase I (Promega, U.S.A.) treatment. The RNA concentration was determined using a NanoDrop photometer (NanoDrop, Wilmington, DE, U.S.A.). Primer sets designed from the 67 genes were used for reverse transcriptase</span></p>
<p><span class="font8">(RT)-PCR analysis to verify the </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8">-regulated genes appearing on the DNA microarray. After the quality of the RNA was confirmed on a 1% agarose gel, it was added to a 20-μl reaction mixture containing 20 pmol of downstream primer, 4 μl of 5× RT buffer, 2 μl of dNTPs (10 mM), 10 units of RNase inhibitor, and 10 units of reverse transcriptase, and incubated in a PTC-225<sup>TM</sup> thermocycler (MJ Research, USA) under the following conditions: 42°C for 30 min. After the cDNA synthesis reaction, 10 μl of each cDNA was PCR-amplified by HQ Taq Polymerase (JK Biotech. Korea) using the gene-specific primer sets in Table 1 under the following conditions: 35 cycles of 94°C for 30 sec, 60°C for 30 sec, and 72°C for 1.5 min. The PCR amplicons were visualized by ethidium bromide staining after 1% agarose gel electrophoresis. As a positive control, 16S rRNA was used. The primer set was as follows: 16S_F, 5′-GGAGGAACATCAGTGGCGAAGG-3′ and 16S_R, 5′-CCGAAGGCACCAATCCATCTCTG-3′.</span></p>
<p><span class="font8" style="font-weight:bold;">RESULTS AND DISCUSSION</span></p>
<h2><a name="bookmark18"></a><span class="font8" style="font-weight:bold;"><a name="bookmark19"></a>Microarray data analysis</span></h2>
<p><span class="font8">In this study, we aimed to perform genomewide identification of </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> regulons in </span><span class="font8" style="font-style:italic;">Xoo </span><span class="font8">by using DNA microarray analysis. The XOM2 media was formulated to induce the expression of </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes, including </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8">, and has been used to study </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8">-dependent expression in the </span><span class="font8" style="font-style:italic;">Xoo </span><span class="font8">genome (Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2004; Tsuge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2002). Therefore, we employed this medium to induce expression of the </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> gene. Labeled cDNAs were synthesized from the total RNA from </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo::TN and the wild type strain KACC10859, which were cultured in XOM2, and the synthesized cDNAs were hybridized with the Xoo DNA microarray plate with 3 duplications. Oligonucleotides consisting of 30 bases were designed from the annotated ORFs in the whole genome sequence data of Xoo KACC10331, but ORFs with length less than 50 bp and those with repetitive sequences, such as transposable elements, were excluded from this study. Finally,</span></p>
<p><span class="font8">individual oligonucleotides designed from the 3,017 ORFs were synthesized on the microarray. The expression profile of the wild-type strain KACC10859 differed from that of the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo:: TN mutant strain (Fig. 1 A), and the differences were observed in the 3 duplicated samples. Fig. 1B shows a scatter plot of the up- and down-regulated genes in </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo::TN, when compared with the expression in the wild-type strain. A large number of genes were down-regulated. In total, 210 genes were down-regulated more than 2-fold in the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo::TN mutant, while 115 genes were up-regulated, suggesting that numerous </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo-regulated genes are distributed in the Xoo genome. The oligonucleotides are directly synthesized on individually addressable microelectrodes in a semiconductor matrix, and therefore, a relatively fixed quantity can be provided for microarray analysis.</span></p>
<p><span class="font8">In a previous study, the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">-inducing XVM2 medium, which provides an environment similar to the host intercellular components, was extensively used for inducing virulence genes in different </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> species (Wengelnik </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1996). A DNA microarray consisting of 279 potential pathogenicity-related virulence genes from </span><span class="font8" style="font-style:italic;">X</span><span class="font8">. </span><span class="font8" style="font-style:italic;">axonopodis</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">citri</span><span class="font8"> was applied for comparison between the transcriptomes from </span><span class="font8" style="font-style:italic;">X</span><span class="font8">. </span><span class="font8" style="font-style:italic;">axonopodis</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">citri</span><span class="font8"> colonies cultured in XVM2 and nutrient broth media (Gustavo </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005). When </span><span class="font8" style="font-style:italic;">X</span><span class="font8">. </span><span class="font8" style="font-style:italic;">axonopodis</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">citri</span><span class="font8"> was grown in XVM2 medium, 31 genes, including those encoding cell wall degradation enzymes, Hrps, and avirulence proteins were profiled to be up-regulated, indicating that the artificial media can induce the expression of pathogenicity-related genes. The XOM2 medium was modified for Xoo with the same objective and mainly used for studying </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8">-regulated genes (Tsuge </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2002). In addition, microarray analysis to compare the global gene expression in representative strains of Xoo and </span><span class="font8" style="font-style:italic;">X. oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzicola</span><span class="font8"> (</span><span class="font8" style="font-style:italic;">Xoc</span><span class="font8">) grown in PSB and XOM2 has been performed (Seo </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2008). It was found that 247 Xoo genes and 39 Xoc genes</span><span class="font8" style="font-style:italic;">,</span><span class="font8"> including bacterial-motility and signal-transduction related gene</span><span class="font8" style="font-style:italic;">s</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrp,</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">rax</span></p>
<div><img src="https://jurnal.harianregional.com/media/13592-1.jpg" alt="" style="width:481pt;height:262pt;">
<p><span class="font7">Fig 1. Expression profiling (A) of the </span><span class="font7" style="font-style:italic;">hrpX</span><span class="font7">::TN mutant and </span><span class="font7" style="font-style:italic;">X. oryzae</span><span class="font7"> pv. </span><span class="font7" style="font-style:italic;">oryzae</span><span class="font7"> KACC10859 on DNA microarray, and scatter plot (B) of </span><span class="font7" style="font-style:italic;">hrpX</span><span class="font7"> regulatory genes in the </span><span class="font7" style="font-style:italic;">hrpX</span><span class="font7">::TN mutant compared to wild-type strain. M: mutant strains; W: wild-type strains. The scale indicates the relative level of expression of each gene, where red indicates expression and green indicates repression.</span></p>
</div><br clear="all">
<p><span class="font8">genes, are up-regulated in XOM2. Nevertheless, it was reasonably considered that the artificial XOM2 medium can provide environmental signals for inducing different genes, such as metabolism-related genes as well as </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes, under different media conditions. However, we aimed to determine how </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> induces target genes by comparing the expressional profiles of the </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> mutant and wild-type strains under the same induction medium, XOM2. Thus, we believe that the down-regulated genes on the mutant microarray are dependent on </span><span class="font8" style="font-style:italic;">hrpX </span><span class="font8">expression, although their expression intensities are different.</span></p>
<p><span class="font8">To date, different </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8">-induced genes have been identified by the presence of a PIP box, a </span><span class="font8" style="font-style:italic;">cis</span><span class="font8">-regulatory element with a motif [TTCGC-N<sub>15</sub>-TTCGC] that exists in promoter regions of HrpX regulons. The PIP box has been used as effective marker for screening HrpX regulons (Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006; Koebnik </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2006). Genome-wide screening for PIP boxes was performed on the whole genomic sequence of Xoo (Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">.</span></p>
<p><span class="font8">2006; Lee </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005). On the basis of the PIP box sequences, 17 HrpXo regulons were identified in the genome sequence of Xoo KACC10331. Of these, the cysteine proteinase gene, iron receptor gene, and </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes were coincident to those observed in the present study. Furthermore, in this study, we investigated the existence of PIP box sequences in the </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">-regulated genes on a microarray. As shown in Fig. 2, perfect PIP boxes, containing the motif TTCGT- N<sub>15</sub>-TTCGT, were identified in the promoter regions of 4 </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes (XOO0095, XOO0096, XOO0094, and XOO0076). Genes encoding cysteine protease contained imperfect PIP boxes (TTCGn-N12-TTCGn in XOO0387</span><span class="font8" style="font-style:italic;">,</span><span class="font8"> TTCGC-N12-TTCGC in XOO1480, and TTCGT-N<sub>13</sub>-TTCGT in XOO1487) in their promoter regions. Significant PIP boxes were further identified in other </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">-regulated genes, including the </span><span class="font8" style="font-style:italic;">xrvA</span><span class="font8"> gene of virulence regulator (XOO2744), a cellulase gene (XOO4036), and 2 </span><span class="font8" style="font-style:italic;">pili</span><span class="font8"> genes (XOO1157, XOO1158), but not in the other </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> regulons in this study. In </span><span class="font8" style="font-style:italic;">Xanthomonas </span><span class="font8">species, genes with an imperfect PIP box and</span></p>
<div>
<table border="1">
<tr><td style="vertical-align:top;">
<p><span class="font6">XOOO387</span></p>
<p><span class="font6">XOO1480</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p>
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">tgcctcagcgct------</span></p>
<p><span class="font6">tgcctcagcgct------</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p>
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">Cystein protease</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">XOO1487</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGT</span></p></td><td style="vertical-align:top;">
<p><span class="font6">aagcggcatcgcc-----</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGT</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">XOO2744</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">gcagcgtgtccactctta</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCCG</span></p></td><td style="vertical-align:top;">
<p><span class="font3">Xtv A; viru Ience regulator</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">XOO4036</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTGTC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">tccacccgcacaggccac</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font3">cellulase</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">X001157</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTGCC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">tagcgcgtgaagggc---</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTGTC</span></p></td><td style="vertical-align:top;">
<p><span class="font3">pilM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">XOO1158</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">ggtcgaaaacgcc-----</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font3">P</span><span class="font9">∣</span><span class="font3">IN</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">X000095</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">gcgtacaagcgcaat---</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">hrp1</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">X000096</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">ttgcccgttaagtgt---</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGT</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">hrp2</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">XOO0094</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">Gagcgacttccgata---</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">hrcC</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">XOO0076</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">cacaccagctatcgc---</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;"></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font6">PIP</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font6">-N15-</span></p></td><td style="vertical-align:top;">
<p><span class="font6">TTCGC</span></p></td><td style="vertical-align:top;">
<p><span class="font3">hrpE</span></p></td></tr>
</table>
<p><span class="font7">Fig. 2. PIP sequences in the promoter regions of </span><span class="font7" style="font-style:italic;">hrp</span><span class="font7">Xo-regulated genes on the </span><span class="font7" style="font-style:italic;">X. oryzae</span><span class="font7"> pv. </span><span class="font7" style="font-style:italic;">oryzae</span><span class="font7"> microarray. The window boxes indicate consensus sequences in PIP (standard sequence).</span></p>
</div><br clear="all">
<p><span class="font8">genes without a PIP box have been reported to be expressed in an HrpX-dependent manner (Noël </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2002), suggesting that the PIP box is not an absolute criterion for identifying HrpXo regulons.</span></p>
<h2><a name="bookmark20"></a><span class="font8" style="font-weight:bold;font-style:italic;"><a name="bookmark21"></a>hrp</span><span class="font8" style="font-weight:bold;">Xo dependent up-regulated genes </span><span class="font8" style="font-weight:bold;font-style:italic;">Pathogenesis-related genes</span></h2>
<p><span class="font8">Virulence and regulatory genes required for bacterial pathogenicity are commonly found in pathogenicity islands (PAIs) that encode for a type III protein secretion system assembled from </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> gene products. During infection, phytopathogenic bacteria use T3SS to deliver effector proteins into the host plant, thereby causing disease symptoms in susceptible hosts and defense responses in resistant hosts. An </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> gene cluster containing 27 genes, including </span><span class="font8" style="font-style:italic;">hpa2</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">hrpF</span><span class="font8">, was identified in the </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> genome. The </span><span class="font8" style="font-style:italic;">Xoo hrp</span><span class="font8"> PAI (31.3 kb) was larger than its counterparts in </span><span class="font8" style="font-style:italic;">Xac</span><span class="font8"> (25.6 kb) and </span><span class="font8" style="font-style:italic;">Xcc</span><span class="font8"> (23.1 kb), owing to the presence of 4 transposed genes (about 6 kb) located between the </span><span class="font8" style="font-style:italic;">hpaB</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">hrpF </span><span class="font8">genes (Lee, </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2005). In this result, it was revealed that </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> regulates the transcriptional expression of </span><span class="font8" style="font-style:italic;">hrpF</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrpE</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrpD6</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hpaA, hrcV</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrpB3, hrpB8, hrcC, hpa1</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">hpa</span><span class="font8">2 (Table 1). Previously,</span></p>
<p><span class="font8">microarray experiments were performed to compare global gene expression between </span><span class="font8" style="font-style:italic;">Xoo </span><span class="font8">and </span><span class="font8" style="font-style:italic;">Xoc</span><span class="font8"> grown in nutrient medium (PSB) and those grown in </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">-inducing XOM2 media (Seo </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">., 2008). The </span><span class="font8" style="font-style:italic;">hpa1</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">hrpE</span><span class="font8"> genes in </span><span class="font8" style="font-style:italic;">X. oryzae </span><span class="font8">pv. </span><span class="font8" style="font-style:italic;">oryzicola</span><span class="font8"> are specifically induced in </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">-inducing XOM2 media, but not in nutrient media, indicating that </span><span class="font8" style="font-style:italic;">hpa1</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">hrpE</span><span class="font8"> are under different or additional regulatory controls from the other </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes, which is unlikely to be the case in </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8">. In contrast, expression of </span><span class="font8" style="font-style:italic;">Xoo hpa1</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrpG</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrcC</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">hrpE</span><span class="font8"> were up-regulated in XOM2. However, it is not clear whether the induced expression of the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes in XOM2 is due to environmental signals in the media or </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">. On the basis of our data, we considered that </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> expression is induced by environmental factors in plants or XOM2 media and that the activated HrpX then stimulates the expression of other </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes such as </span><span class="font8" style="font-style:italic;">hpa1</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrpG</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrcC</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">hrpE.</span><span class="font8"> HrpX is known to be a regulator of </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes and several effector genes (Wengelnik and Bonas 1996).</span></p>
<p><span class="font8">Furthermore, we identified 3 T3SS effector genes XOO4466 (</span><span class="font8" style="font-style:italic;">XopP</span><span class="font8">), XOO2127 (</span><span class="font8" style="font-style:italic;">Avr/PthA</span><span class="font8">), and XOO4033 (</span><span class="font8" style="font-style:italic;">XopP</span><span class="font8">) in this microarray analysis. </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> harbors multiple genes for T3S effectors of the transcription activator-like (TAL) effector family,</span></p>
<p><span class="font8">called the </span><span class="font8" style="font-style:italic;">avrBs3/pthA</span><span class="font8"> family (Lee </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Ochiai </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005). AvrXa7, AvrXa10, and AvrXa27 are representative TAL effectors that elicit disease resistance in hosts with corresponding resistance genes and thereby function as determinants of race–cultivar specificity (Gu </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Yang </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2000). Hrp outer proteins (Hops) from </span><span class="font8" style="font-style:italic;">P. syringae </span><span class="font8">strains and </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> outer proteins (Xops) have been shown to be T3S effectors. Recently, on the basis of the natural features of N-terminal amino acids of Hop, Xop proteins, and T3S substrates, T3S system-dependent translocation of 60 effector candidates was identified from the genome data of </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> MAFF311018 and screened using </span><span class="font8" style="font-style:italic;">Bordetella pertussis</span><span class="font8"> calmodulin-dependent adenylate cyclase (Cya) reporter for translocation into plant cells (Frutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2008). Finally, 16 T3SS effector genes that are also regulated by hrpX were identified. Of these, XOO4466 and XOO4033, from this study, were included among the effectors.</span></p>
<p><span class="font8">The </span><span class="font8" style="font-style:italic;">rtx</span><span class="font8"> genes (XOO4535 and XOO1389) have been shown to be up-regulated by HrpXo in </span><span class="font8" style="font-style:italic;">Bradyrhizobium elkanii</span><span class="font8">, a symbiotic nitrogenfixing bacterium that produces the enol-ether amino acid rhizobitoxine; the </span><span class="font8" style="font-style:italic;">rtxC</span><span class="font8"> gene is required for the production of rhizobitoxine, which is regarded as a phytotoxin responsible for progressing chlorosis symptoms in soybean (Okazaki </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2004). RTX toxins are important virulence factors for a variety of human and animal pathogens (Welch </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1992) and have been found in several plant pathogenic bacteria, including </span><span class="font8" style="font-style:italic;">X. fastidiosa</span><span class="font8">, </span><span class="font8" style="font-style:italic;">Rhizobium leguminosarum</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">Erwinia carotovora</span><span class="font8"> (Kuhnert </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1997; Oresnik </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1999; Simpson </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2000). The genes for 2 apparent RTX toxins, </span><span class="font8" style="font-style:italic;">rtxA</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">rtxC</span><span class="font8">, were identified in the Xoo genome but were not detected in the Xcc or Xac genomes.</span></p>
<p><span class="font8">The virulence gene </span><span class="font8" style="font-style:italic;">xrvA,</span><span class="font8"> which encodes a histone-like nucleoid-structuring protein, was also revealed to be an </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> regulon. Recently, it was reported that </span><span class="font8" style="font-style:italic;">xrvA</span><span class="font8"> demonstrates enhanced expression in XOM2 medium and further regulates pathogenicity-related genes such as </span><span class="font8" style="font-style:italic;">hrpG</span><span class="font8">, </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8">, </span><span class="font8" style="font-style:italic;">rpfC</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">rpfG</span><span class="font8">, which play an</span></p>
<p><span class="font8">important role in the hypersensitive response and EPS and LPS biosynthesis (Feng </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2009). The </span><span class="font8" style="font-style:italic;">xrvA</span><span class="font8"> function found in the study by Feng </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. seems to be contrary to the result of our microarray analysis, but this is not conclusive because we do not know how virulence genes are interracially associated with other unknown pathogenesis-related genes.</span></p>
<p><span class="font8">In the present study, it was revealed that 7 iron-receptor proteins, which encode the genes XOO4431, XOO0407, XOO0008, XOO0009, XOO0029, and XOO1368, are regulated by </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">. In </span><span class="font8" style="font-style:italic;">X. campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8">, TonB, ExbB, and ExbD have been reported to play a critical role in Fe<sup>3+</sup> uptake and induction of a hypersensitive response in pepper (Wiggerich </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 1997; Wiggerich and Puhler 2000). Forty-two putative iron uptake-related genes, including TonB-dependent receptor proteins, were annotated in the genomic sequence of Xoo KACC10331. In a recent report, mutation in a TonB-dependent receptor led to the loss of pathogenicity and also caused a reduction in mobility and extracellular enzymes (Wu </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2010). A mutation in XOO1368, homologous to bfeA, resulted in reduced pathogenicity (Wang </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2008), suggesting that sequestration of iron is important for </span><span class="font8" style="font-style:italic;">Xoo </span><span class="font8">virulence. A 3-gene cluster (</span><span class="font8" style="font-style:italic;">bfeABR</span><span class="font8">) is also present in the </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> genome. The </span><span class="font8" style="font-style:italic;">bfeA</span><span class="font8"> gene was isolated from </span><span class="font8" style="font-style:italic;">Bordetella pertussis</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">Bordetella bronchiseptica,</span><span class="font8"> obligate respiratory pathogens of mammals that encode a predicted outermembrane siderophore receptor exhibiting a high degree of similarity to the ferric enterobactin receptors. The extremely limited availability of free iron requires pathogenic bacteria to utilize specialized uptake mechanisms to successfully compete with the host and other microorganisms for iron.</span></p>
<p><span class="font8">Table 1. Downregulated genes in hrpX: TN mutant strain</span></p>
<table border="1">
<tr><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">Gene ID</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">Description</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">Expression folds</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">Gene ID</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">Description</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">Expression folds</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO0832</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">holC;DNA polymerase III subunit chi</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.02</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO1623</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">rpsT;30S ribosomal protein S20</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.58</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font1">XOO4019</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">type VI secretion system Vgr family protein</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.03</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">XOO0079</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">hpaA;HpaA</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.59</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0844</span></p></td><td style="vertical-align:top;">
<p><span class="font1">protease</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.03</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO1822</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.59</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO1259</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">peptidase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.03</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO4535</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">RTS beta protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.60</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO0094</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hrp 1</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.05</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO3277</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.62</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO2522</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.05</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO0094</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hrcC;HrcC</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.62</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO4431</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">cirA;TonB-dependent receptor</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.07</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO1157</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">pilM;fimbrial assembly membrane protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.62</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font1">XOO1943</span></p></td><td style="vertical-align:top;">
<p><span class="font1">nodQ;ATP sulfurylase/adenylylsulfate kinase</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.07</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">XOO2231</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">proP;Prop transport protein</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.63</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO3773</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">nrtB;permease</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.08</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO4418</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">salR;sal operon transcriptional repressor</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.66</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font1">XOO2294</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">phosphatase</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.10</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">XOO3464</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">Type I restriction enzyme StySPI specificity protein</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.67</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font1">XOO3594</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">rplA;50S ribosomal protein L1</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.13</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">XOO1831</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">histidine kinase-response regulator hybrid protein</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.68</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO0495</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">radC;DNA repair protein RadC</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.15</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO3372</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">thiG;thiazole synthase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.68</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO2348</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.16</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO0019</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">;hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.69</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO3127_</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.17</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO0093</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hrpB8;HrpB8</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.69</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO3184</span></p></td><td style="vertical-align:top;">
<p><span class="font1">rplT;50S ribosomal protein L20</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.17</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO0704</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.72</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0348</span></p></td><td style="vertical-align:top;">
<p><span class="font1">;pseudouridylate synthase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.18</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO3238</span></p></td><td style="vertical-align:top;">
<p><span class="font1">tpiA;triosephosphate isomerase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.73</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0408</span></p></td><td style="vertical-align:top;">
<p><span class="font1">csd2 family</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.18</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO2744</span></p></td><td style="vertical-align:top;">
<p><span class="font1">xrvA;virulence regulator</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.75</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO4263</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">icfG;IcfG protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.19</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO1077</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">Cellulase S</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.75</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO2561</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">;hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.20</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO3557</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">rplQ;50S ribosomal protein L17</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.76</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO3449</span></p></td><td style="vertical-align:top;">
<p><span class="font1">uidR;transcriptional regulator uid family</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.20</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO3269</span></p></td><td style="vertical-align:top;">
<p><span class="font1">integral membrane protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.78</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0869</span></p></td><td style="vertical-align:top;">
<p><span class="font1">CRISPR-associated proteins</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.21</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO2084</span></p></td><td style="vertical-align:top;">
<p><span class="font1">tRNA/rRNA methyltransferase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.78</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO4265</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">SpolI AA</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.22</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO3000</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">cytochrome C</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.83</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO1497</span></p></td><td style="vertical-align:top;">
<p><span class="font1">cysteine protease</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.23</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO3255</span></p></td><td style="vertical-align:top;">
<p><span class="font1">truA;tRNA pseudouridine synthase A</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.85</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0705</span></p></td><td style="vertical-align:top;">
<p><span class="font1">oxidoreductase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.23</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO1456</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.85</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO2530</span></p></td><td style="vertical-align:top;">
<p><span class="font1">Dehydrogenases</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.26</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO1590</span></p></td><td style="vertical-align:top;">
<p><span class="font1">fic;Cell filamentation protein Fic</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.85</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0409</span></p></td><td style="vertical-align:top;">
<p><span class="font1">TolB protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.26</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO4465</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.87</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO1612</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.27</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO4426</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.87</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO1908</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.29</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO0174</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.89</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO1262</span></p></td><td style="vertical-align:top;">
<p><span class="font1">;hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.30</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO0411</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.90</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0407</span></p></td><td style="vertical-align:top;">
<p><span class="font1">piuB;iron-uptake factor</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.31</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO4490</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.91</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO1201</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">omp21;outer membrane protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.31</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO2127</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">avirulence protein, Avr/PthA</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.92</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font1">XOO0028</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">kdgK;2-keto-3-deoxygluconate kinase</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.31</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">XOO0703</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">rsuA;ribosomal small subunit pseudouridylate synthase</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.92</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0747</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.33</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO3561</span></p></td><td style="vertical-align:top;">
<p><span class="font1">rpsM;30S ribosomal protein S13</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.96</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO1606</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.34</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO2746</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.96</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO2499</span></p></td><td style="vertical-align:top;">
<p><span class="font1">phy;putative phytase precursor</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.35</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO0288</span></p></td><td style="vertical-align:top;">
<p><span class="font1">transglycolase; epimerase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.99</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO2989</span></p></td><td style="vertical-align:top;">
<p><span class="font1">methyltransferase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.35</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO1531</span></p></td><td style="vertical-align:top;">
<p><span class="font1">phuR;outer membrane hemin receptor</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.00</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO2346</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.35</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO1160</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">pilP;fimbrial assembly protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.05</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font1">XOO2147</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">cycH;C-type cytochrome biogenesis protein</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-2.38</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">XOO2986</span></p></td><td style="vertical-align:middle;">
<p><span class="font1">glycosyltransferase</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;">-3.08</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO2047</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.38</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO1401</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">acetyltransferase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.09</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO3205</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.38</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO1055</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.10</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO1528</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">short chain dehydrogenase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.38</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO2845</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">tsr;chemotaxis protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.10</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO4482</span></p></td><td style="vertical-align:top;">
<p><span class="font1">yojM;superoxide dismutase like protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.39</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO3559</span></p></td><td style="vertical-align:top;">
<p><span class="font1">rpsD;30S ribosomal protein S4</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.13</span></p></td></tr>
</table>
<table border="1">
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO2263</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.01</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO4527</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">oxidoreductase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.16</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO0947</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">thiC;thiamine biosynthesis protein ThiC</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.01</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO0420</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.16</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO3584</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">rplC;50S ribosomal protein L3</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.02</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO4466</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XopP, Nucleoside hydrolases</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.20</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO1994</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">brf;bacterioferritin</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.40</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO2022</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">serine peptidase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.21</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO03911</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">prnA;tryptophan halogenase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.42</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO2667</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.22</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO1069</span></p></td><td style="vertical-align:top;">
<p><span class="font1">Transglutaminase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.44</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO2246</span></p></td><td style="vertical-align:top;">
<p><span class="font1">;hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.25</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO3281</span></p></td><td style="vertical-align:top;">
<p><span class="font1">metallopeptidase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.44</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO0077</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hrpD6;HrpD6</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.31</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO0084</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hrcV;HrcV</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.44</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO3735</span></p></td><td style="vertical-align:top;">
<p><span class="font1">vanR;transcriptional regulator gntR family</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.33</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO3570</span></p></td><td style="vertical-align:top;">
<p><span class="font1">rplE;50S ribosomal protein L5</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.46</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO4355</span></p></td><td style="vertical-align:top;">
<p><span class="font1">Dipeptidyl aminopeptidases</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.36</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO2688</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.52</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO1541</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.43</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO2042</span></p></td><td style="vertical-align:top;">
<p><span class="font1">ldp;dihydrolipoamide dehydrogenase</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.53</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO3479</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hcp;hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.47</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font1">XOO2705</span></p></td><td style="vertical-align:top;">
<p><span class="font1">efP;elongation factor P</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-2.54</span></p></td><td style="vertical-align:top;">
<p><span class="font1">XOO4532</span></p></td><td style="vertical-align:top;">
<p><span class="font1">hypothetical protein</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;">-3.59</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO3566</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">rpsE;30S ribosomal protein S5</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.54</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO0008</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">tonB;TonB protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.63</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font1">XOO0699</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">;hypothetical protein</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-2.55</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">XOO3410</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1">methyltransferase</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;">-3.64</span></p></td></tr>
</table>
<p><span class="font8" style="font-weight:bold;font-style:italic;">Type II secretory system related genes</span></p>
<p><span class="font8">We found that </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo regulates T2SS effector genes, including 4 cellulase genes (XOO4036, XOO1077, XOO2699, and XOO4019), 6 protease genes (XOO0007, XOO1497, XOO0387, XOO1480, XOO1487, and XOO0844), a polygalacturonase gene (XOO2699), and a lipase gene (XOO0526). Cell-wall degrading enzymes, such as cellulases, pectinases, xylanases, and proteases, are secreted by plant pathogen cells to break down the components of host cell walls and may play a crucial role in virulence and bacterial nutrition (Barras </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 1994; Shen and Ronald 2002). Currently, genes encoding celldegrading enzymes (CDEs) such as cellulase and xylanase are considered to play a role in the virulence of Xoo (Hu </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2007; Rajeshwari </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Ray </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2000; Sun </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005; Wang </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2008). Moreover, this study showed that </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo regulates the transcriptional expression of cellulase genes in </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8">. It has been reported that purified cellulase and lipase proteins induce defense responses in rice that are suppressible by </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> in a T3S-dependent manner (Jha </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2007). Therefore, cellulase genes are considered to participate in diverse virulence functions associated directly or indirectly with the expression of the main pathogenicity-related genes, such as </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes. In the case of other </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> species, 2 pathogenicity-related genes (</span><span class="font8" style="font-style:italic;">pghAxc</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">pghBxc</span><span class="font8">) encoding functional polygalacturonase (PG) from </span><span class="font8" style="font-style:italic;">X. campestris</span><span class="font8"> pv.</span></p>
<p><span class="font8" style="font-style:italic;">campestris</span><span class="font8"> 8004 are regulated by HrpX (Wang<sup>a)</sup>, </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2008). It was also reported that </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> negatively regulates the α-amylase isozymes in </span><span class="font8" style="font-style:italic;">X. axonopodis </span><span class="font8">pv. </span><span class="font8" style="font-style:italic;">citri</span><span class="font8"> (Yamazaki </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2008) and extracellular proteases in </span><span class="font8" style="font-style:italic;">X. campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8"> (Wei </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2007). In a previous study conducted using SDS-PAGE and RT-PCR analysis, </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> was shown to regulate the expression of genes associated with T2S proteins, i.e., 2 cysteine proteases (Cysp1 and Cysp2) (Furutani </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2004), which shows good agreement with our results. In other phytopathogenic bacteria, it is known that the TTSS effectors encoding cysteine proteases, such as XopD, AvrXv4, AvrPphB, and AvrRpt2, proteolyze specific host targets and are indirectly recognized by R proteins as a consequence of their action in plant cells (Andrew and Mary 2004). Thus, it can be reasonably assumed that </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8">Xo-dependent cysteine proteases may function as T3SS effectors or indirectly control the recognition of the host plant.</span></p>
<p><span class="font8" style="font-weight:bold;font-style:italic;">Bacterial mobility</span></p>
<p><span class="font8">On the basis of the microarray data obtained in this study, it was revealed that </span><span class="font8" style="font-style:italic;">hrpXo </span><span class="font8">regulates </span><span class="font8" style="font-style:italic;">pilMNEGIP</span><span class="font8">, which encodes fimbrial as asembly proteins that are known as type IV pili. Type IV pili are flexible, filamentous structures protruding from the cell surface of gram-negative bacteria (Shi and Sun 2002; Strom andLory 1993). It has been revealed that type IV pili promote the attachment of bacterial pathogens</span></p>
<p><span class="font8">to the specific receptors of host cells during colonization (Bieber </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 1998). Such attachment is an essential event for the initiation of infection. Limited to colonizing the water-conducting xylem vessels of plants, phytopathogenic bacteria develop biofilms that contribute to the blockage of sap flow and thereby cause plant stress and disease (Maxwell </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2003; Meng </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005). How bacteria are disseminated in the xylem vessels from feeding sites is an important question. Generally, it is known that type IV pili in bacteria play an important role in twitching and social gliding motility for cell movement (Mattick 2002). On the basis of colony morphologies, there is also strong evidence for a functional role of </span><span class="font8" style="font-style:italic;">pil</span><span class="font8"> genes in twitching motility in the phytopathogenic bacteria </span><span class="font8" style="font-style:italic;">Ralstonia solanacearum</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">X. fastidiosa</span><span class="font8"> (Liu </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">.2002; Meng </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">.2005)</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> The distribution of pili genes in the </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> genome was analyzed according to the </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> sequence information released on the NCBI website. Interestingly, 26 pili-related genes were present in different regions of the </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> genome, including </span><span class="font8" style="font-style:italic;">pilA, pilZ, pilH, pilMNOPQ, pilTU, pilDCE, pilBRS, pilGHIJL, pilF, pilE1Y1</span><span class="font8">, </span><span class="font8" style="font-style:italic;">pilX</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">pilV</span><span class="font8"> (Lee </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2005). </span><span class="font8" style="font-style:italic;">Xoo</span><span class="font8"> genome analysis suggests that a number of pili gene orthologs may encode proteins involved in the biogenesis and function of type IV pili. However, the functional roles of other pili genes have not been proved in the Xoo genome. It was recently reported that Xoo </span><span class="font8" style="font-style:italic;">pilQ </span><span class="font8">plays a critical role in pathogenicity, twitching motility, and biofilm production (Lim, </span><span class="font8" style="font-style:italic;">et al</span><span class="font8">. 2008). It was previously reported that 14 pili genes (</span><span class="font8" style="font-style:italic;">pilIJKLMNOPQRSTUV</span><span class="font8">) are involved in type IV pilus biogenesis in </span><span class="font8" style="font-style:italic;">E. coli</span><span class="font8"> (Sakai andKomano 2002). The genes </span><span class="font8" style="font-style:italic;">pilS, pilV</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">pilU</span><span class="font8"> encode the major pilin, minor pilin, and prepilin peptidase, respectively. The C-terminal segments of </span><span class="font8" style="font-style:italic;">pilV </span><span class="font8">genes are under the control of multiple DNA inversions of shufflon and determine recipient specificity in liquid mating. The </span><span class="font8" style="font-style:italic;">pilL</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">pilN</span><span class="font8"> gene products are outer-membrane lipoproteins, and the </span><span class="font8" style="font-style:italic;">pilQ</span><span class="font8"> gene product is a cytoplasmic ATPase and is essential for thin pilus formation (Saiki </span><span class="font8" style="font-style:italic;">et al.</span><span class="font8"> 2001). The remaining </span><span class="font8" style="font-style:italic;">pil</span><span class="font8"> genes are likely to</span></p>
<p><span class="font8">encode structural proteins that function in the establishment of the pilin transport apparatus and the basal body of the thin pilus. Most of these </span><span class="font8" style="font-style:italic;">pil</span><span class="font8"> gene products contain signal sequences or trans-membrane domains, suggesting that they are transported to the peri-plasmic space, inner membrane, and outer membrane.</span></p>
<h2><a name="bookmark22"></a><span class="font8" style="font-weight:bold;"><a name="bookmark23"></a>Verification of microarray results</span></h2>
<p><span class="font8">The effect of the </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> mutation on the expression of different genes was validated by using RT-PCR. The RT-PCR 20-mer primers were generated from 12 T3SS-related </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes, 4 T2SS-related cysteine protease-encoding genes, 3 cellulase genes, and 6 type-IV pili genes. As shown in Fig. 2, no, or very weak, amplification was obtained from the genes in the </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">::Tn5 mutant, although the magnitude of expression was somewhat different to those observed in the microarray analysis. As expected, all </span><span class="font8" style="font-style:italic;">hrp </span><span class="font8">gene primers tested amplified RT-PCR bands with reduced intensity in the </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">::Tn5 mutant, compared with the intensities for the wild-type strain. In addition, 4 genes encoding cysteine proteases and cellulases (XOO4036) did not amplify an RT-PCR band in the </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">::Tn5 mutant, indicating that their transcriptional expression is absolutely dependent on </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8">. Furthermore, RT-PCR analysis also verified that when compared with the wild-type strain, the </span><span class="font8" style="font-style:italic;">hrpXo</span><span class="font8"> mutant visually represents low transcriptional expression of </span><span class="font8" style="font-style:italic;">pilE</span><span class="font8">, </span><span class="font8" style="font-style:italic;">pilG</span><span class="font8">, </span><span class="font8" style="font-style:italic;">pilI</span><span class="font8">, and </span><span class="font8" style="font-style:italic;">pilP</span><span class="font8">, but not </span><span class="font8" style="font-style:italic;">pilM</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">pilN</span><span class="font8">. The result confirmed that the microarray result is largely available.</span></p>
<p><span class="font8">In conclusion, DNA microarray analysis of </span><span class="font8" style="font-style:italic;">X. oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> revealed that in addition to regulating the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes, the </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> regulon regulates numerous new genes possessing different functions. Furthermore, the potential interactive roles between the </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> regulons and the host plant should be elucidated for pathogenesis mechanisms.</span></p>
<div><img src="https://jurnal.harianregional.com/media/13592-2.jpg" alt="" style="width:478pt;height:218pt;">
<p><span class="font2">XOO0387</span></p>
<p><span class="font2">XOO0066 XOO0076 XOO0077</span></p>
<p><span class="font2">XOO1480 &nbsp;</span><span class="font2" style="font-weight:bold;">XOO1487 </span><span class="font2">XOO1497</span></p>
<p><span class="font7">Fig.3. Verification of microarray data by RT-PCR. Total RNAs from wild-type strain KACC10859 and the </span><span class="font7" style="font-style:italic;">hrpX</span><span class="font7">:: TN mutant were used as PCR templates with primer pairs in Table 2. Amplicons were visualized after agarose gel electrophoresis and ethidium bromide staining.</span></p>
</div><br clear="all">
<h2><a name="bookmark24"></a><span class="font8" style="font-weight:bold;"><a name="bookmark25"></a>REFERENCES</span></h2>
<p><span class="font8">Andrew H, Mary BM. 2004. Cysteine proteases in phytopathogenic bacteria: identification of plant targets and activation of innate immunity. Current Opinion in Plant Biology 7: 1-7</span></p>
<p><span class="font8">Astua-Monge G, Freitas-Astua J, Bacocina G , Roncoletta J, Carvalho SA, Machado MA. 2005.Expression profiling of virulence and pathogenicity genes of </span><span class="font8" style="font-style:italic;">Xanthomonas axonopodis pv. citri.</span><span class="font8"> J. Bacteriol. 187: 12011205.</span></p>
<p><span class="font8">Barras F. 1994. Exreacellular enzymes and pathogenesis of soft-rot </span><span class="font8" style="font-style:italic;">Erwinia</span><span class="font8">. Ann Rev Phytopathol. 32: 201-234</span></p>
<p><span class="font8">Bieber D, Ramer SW, Wu CY, Murray WJ, Tobe T, Fernandez R, Schoolnik GK. 1998. Type IV pili, transient bacterial aggregates, and virulence of enteropathogenic </span><span class="font8" style="font-style:italic;">Escherichia coli</span><span class="font8">. Science 280: 2114-2118.</span></p>
<p><span class="font8">Bu¨ttner D, Bonas U. 2002. Getting across— bacterial type III effector proteins on their way to the plant cell. EMBO J. 21: 5313– 5322.</span></p>
<p><span class="font8">Chan JW, Goodwin PH. 1999. The molecular genetics of virulence of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8">. Biotechnol. advance 17: 489-580</span></p>
<p><span class="font8">da Silva ACR and Ferro JA. 2002. Comparison of genomes of two </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> pathogens with differing host specificities. Nature 417: 459-463.</span></p>
<p><span class="font8">de Morgan A, Brodsky LY, Ronin Y, Nevo E, Korol A, Kashi, Y. 2010. Genome-wide analysis of DNA turnover and gene expression in stationary-phase </span><span class="font8" style="font-style:italic;">Saccharomyces cerevisiae. </span><span class="font8">Microbiol 156: 1758–1771</span></p>
<p><span class="font8">Feng JX, Song ZZ, Duan CJ, Zhao S, Wu YQ, Wang C, J. Dow M, Tang JL. 2009. The </span><span class="font8" style="font-style:italic;">xrvA</span><span class="font8"> gene of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8">, encoding an H-NS-like protein, regulates virulence in rice. Microbiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 155: 3033-3044</span></p>
<p><span class="font8">Furutani A, Nakayama, T, Ochiai, H, Kaku H, Kubo Y, Tsuge S. 2006. Identification of novel HrpXo regulalons preceded by two cis-acting elements, a plant-inducible promoter box and a -10 box-like sequence, from the genome database of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8">. FEMS Microbiol Letter 259: 133-141</span></p>
<p><span class="font8">Furutani A, Takaoka M, Sanada H, Noguchi Y, Oku T, Tsuno K, Ochiai H, Tsuge S. 2009. Identification of Novel Type III Secretion Effectors in </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae. </span><span class="font8">Mol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> Plant- Microbe Interact</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 22: 96–106</span></p>
<p><span class="font8">Furutani A, Tsuge S, Ohnishi K, Hikichi Y, Oku T, Tsuno, K, Inoue Y, Ochiai H, Kaku H, Kubo Y. 2004. Evidence for HrpXo-dependent expression of type II secretory proteins in </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8">. J. Bacteriol</span><span class="font8" style="font-style:italic;">. </span><span class="font8">186: 1374–1380.</span></p>
<p><span class="font8">Ghindilis AL, Smith MW, Schwarzkopf KR, Roth KM, Peyvan,K, Munro SB, Lodes MJ, St¨over AG, Bernards K, Dill K, McShea A. 2007. CombiMatrix oligonucleotide arrays: Genotyping and gene expression assays employing electrochemical detection. Biosens</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> Bioelectro. 22: 1853–1860</span></p>
<p><span class="font8">Grulebeck D, Thieme F, Bonas U. 2005. Type III effector proteins from the plant pathogen </span><span class="font8" style="font-style:italic;">Xanthomonas</span><span class="font8"> and their role in the interaction with the host plant. J. Plant Physiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8">163: 233-255.</span></p>
<p><span class="font8">Gu K, Yang B, Tian D, Wu L, Wang D, Sreekala C, Yang F, Chu Z, Wang GL, White FF, Yin Z. 2005. R gene expression induced by a type-III effector triggers disease resistance in rice. Nature 435: 1122-1125.</span></p>
<p><span class="font8">Gustavo AM, Juliana FA, Gisele B, Juliana R, Se´rgio AC, Marcos AM. 2005. Expression Profiling of Virulence and Pathogenicity Genes of </span><span class="font8" style="font-style:italic;">Xanthomonas axonopodis</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">citri. </span><span class="font8">J. Bacteriol. 187: 1201-1205</span></p>
<p><span class="font8">Guidot A, Coupat B, Fall S, Prior P, Bertolla F. 2009. Horizontal gene transfer between </span><span class="font8" style="font-style:italic;">Ralstonia solanacearum</span><span class="font8"> strains detected by comparative genomic hybridization on microarrays. The ISME Journal</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 3: 549-562 He YQ, Zhang L, Jiang BL, Zhang ZC, Xu RQ, Tang DJ, Qin J, Jiang W, Zhang X, Liao J, Cao JR, Zhang SS, Liang XX, Wei ML, Lu GT, Feng JX, Chen B, Cheng J, Tang JL. 2007. Comparative and functional genomics reveals genetic diversity and determinants of host specificity among reference strains and a large collection of Chinese isolates of</span></p>
<p><span class="font8">the phytopathogen </span><span class="font8" style="font-style:italic;">Xanthomonas campestris </span><span class="font8">pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8">. Genome Biol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 8: R218.</span></p>
<p><span class="font8">He YW, Xu M, Lin K, Ng YJ, Wen CM, Wang LH, Liu ZD, Zhang HB, Dong YH, Dow JM, Zhang LH. 2006. Genome scale analysis of diffusible signal factor regulon in </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8">: identification of novel cellcell communicationdependent genes and functions. Mol. microbial</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 59: 610-622.</span></p>
<p><span class="font8">Hu J, Qian W, He CZ. 2007. The </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae eglXoB</span><span class="font8"> endoglucanase gene is required for virulence to rice. </span><span class="font8" style="font-style:italic;">FEMS </span><span class="font8">Microbiol letter 269: 273-279.</span></p>
<p><span class="font8">Jha G, Rajeshwari R, Sonti RV. 2007. Functional interplay between two </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae </span><span class="font8">pv. </span><span class="font8" style="font-style:italic;">oryazae</span><span class="font8"> secretion systems in modulating virulence on rice. Mol. Plant- Microbe Interact. 20: 31-40.</span></p>
<p><span class="font8">Koebnik R, Kruger A, Thieme F, Urban A, Bonas U. 2006. Specific binding of the </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">vesicatoria</span><span class="font8"> AraC-type transcriptional activator HrpX to plantinducible promoter boxes. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 188: 7652-7660.</span></p>
<p><span class="font8">Koide T, Vencio RZ, Gomes SL. 2006. Global gene expression analysis of the heat shock response in the phytopathogen </span><span class="font8" style="font-style:italic;">Xylella fastidiosa</span><span class="font8">. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 188: 5821-5830.</span></p>
<p><span class="font8">Kuhnert P, Heyberger-Meyer B, Burnens AP, Nicolet J, Frey J. 1997. Detection ofRTXtoxin genes in gram-negative bacteria with a set of specific probes. Appl. Environ. Microbiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 63: 2258–2265.</span></p>
<p><span class="font8">Lee BM, Park YJ, Park DS, Kang HW et al. 2005. The genome sequence of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pathovar </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> KACC10331, the bacterial blight pathogen of rice. Nuc. Acids Res. 33: 577-586.</span></p>
<p><span class="font8">Lim SH, So BH, Park YJ, Lee BM, Kang HW. 2008. Functional analysis of </span><span class="font8" style="font-style:italic;">pilQ</span><span class="font8"> gene in </span><span class="font8" style="font-style:italic;">Xanthomanas oryzae</span><span class="font8"> pv.</span><span class="font8" style="font-style:italic;">oryzae.</span><span class="font8"> J. Microbiol</span><span class="font8" style="font-style:italic;">. </span><span class="font8">46: 214-220.</span></p>
<p><span class="font8">Liu HL, Kang YW, Genin S, Schell MA, Denny TP. 2001. Twitching motility of </span><span class="font8" style="font-style:italic;">Ralstonia solanacearum</span><span class="font8"> requires a type IV pilus</span></p>
<p><span class="font8">system. Microbiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 147: 3215-3229.</span></p>
<p><span class="font8">Lu SE, Wang N, Wang J, Chen ZJ, Gross DC. 2005. Oligonucleotide microarray analysis of the salA regulon controlling phytotoxin production by </span><span class="font8" style="font-style:italic;">Pseudomonas syringae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">syringae</span><span class="font8">. Mol. Plant Microbe Interact</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 18: 324-333.</span></p>
<p><span class="font8">Mattick, JS. 2002. Type pili and twitching motility. </span><span class="font8" style="font-style:italic;">Annu.</span><span class="font8"> Rev. Microbiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 56: 289-314.</span></p>
<p><span class="font8">Maxwell JL, Crossman K, Findlay YQ, He J, Feng X, Tang JL. 2003. Biofilm dispersal in </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> is controlled by cell-cell signaling and is required for full virulece to plants. Proc. Natl. Acad. Sci</span><span class="font8" style="font-style:italic;">. </span><span class="font8">100: 10995-11000.</span></p>
<p><span class="font8">Meng YY, Li Y, Galvani CD, Hao G, Turner JN, Burr TJ, Hoch HC. 2005. Upstream migration of </span><span class="font8" style="font-style:italic;">Xylella fastidiosa</span><span class="font8"> via pilus-driven twitching motility. J Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 187: 5560-5567.</span></p>
<p><span class="font8">Noe¨l L, Thieme F, Ninnstiel D and Bonas U. 2001. cDNA-AFLP analysis unravels a genome-wide </span><span class="font8" style="font-style:italic;">hrpG</span><span class="font8">-regulon in the plant pathogen </span><span class="font8" style="font-style:italic;">Xanthomonas.</span><span class="font8"> Mol Microbiol 41: 1271–1281</span></p>
<p><span class="font8">Ochiai H, Inous Y, Takeya M, Sasaki A and Kaku H. 2005. Genome sequence of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv</span><span class="font8" style="font-style:italic;">. oryzae</span><span class="font8"> suggests contribution of large numbers of effecter genes and insertion sequences to its race diversity. Jap.Agri. Res</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 39: 275-287.</span></p>
<p><span class="font8">Okazaki S, Sugawara M, Minamisawa K. 2004. </span><span class="font8" style="font-style:italic;">Bradyrhizobium elkanii rtxC</span><span class="font8"> Gene Is Required for Expression of Symbiotic Phenotypes in the FinalStepofRhizobitoxineBiosynthesis. Appl Environ. Microbiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 70: 535-541</span></p>
<p><span class="font8">Oku T, Alvarez AM, Kado CI. 1995. Conservation of the hypersensitivity-pathogenicity regulatory gene </span><span class="font8" style="font-style:italic;">hrpX</span><span class="font8"> of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">X. oryzae</span><span class="font8">. DNA Seq. 5: 245– 249.</span></p>
<p><span class="font8">Oresnik IJ, Twelker S, Hynes MF. 1999. Cloning and characterization of a </span><span class="font8" style="font-style:italic;">Rhizobium leguminosarum</span><span class="font8"> gene encoding a bacteriocin with similarities to RTX toxins. Appl. Environ. Microbiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 65: 2833–2840.</span></p>
<p><span class="font8">Park YJ, Song ES, Kim YT, Noh TH, Kang HW, Lee BM. 2007. Analysis of virulence and growth ofa purine auxotrophic mutant of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pathovar </span><span class="font8" style="font-style:italic;">oryzae.</span><span class="font8"> FEMS Microbiol. Letter 276: 55-59</span></p>
<p><span class="font8">Rajeshwari R, Jha G, Sonti V. 2005. Role of an In Planta-expressed xylanase of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv.</span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> in promoting virulence on rice. Mol Plant Microbe Interact. 18: 830837</span></p>
<p><span class="font8">Ray S, Rajeshwari R, Sonti RV. 2000. Mutants of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv.</span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> deficient in general secretory pathway are virulence deficient and unable to secret xylanase. Mol Plant Microbe Interact</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 13: 394-401</span></p>
<p><span class="font8">Sakai D, Komano T. 2002. Genes required for plasmid R64 thin-pilus biogenesis: identification and localization of products of the </span><span class="font8" style="font-style:italic;">pilK, pilM, pilO, pilP, pilR,</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">pilT </span><span class="font8">genes. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 184: 444-452</span></p>
<p><span class="font8">Sakai D, Horiuchi T, Komano T. 2001. ATPase activity and multimer formation of PilQ protein are thin pilus biogenesis in plasmid R64. J. Bio.Chem</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 276: 17968-17975</span></p>
<p><span class="font8">Seo YS, Sriariyanun M, Wang L, Pfeiff J, Phetsom J, Lin Y, Jung KH, Chou HH, Bogdanove A, Ronald P. 2008. A two-genome microarray for the rice pathogens </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae </span><span class="font8">pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> and </span><span class="font8" style="font-style:italic;">X. oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzicola</span><span class="font8"> and its use in the discovery of a difference in their regulation of </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes. BMC microbial</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 8: 86-99</span></p>
<p><span class="font8">Simpson AJ, Reinach FC, Arruda P, Abreu FA, Acencio M, Alvarenga R, Alves LM, Araya JE, Baia GS, Baptista CS, Barros MH. 2000. The genome sequence of the plant pathogen </span><span class="font8" style="font-style:italic;">Xylella fastidiosa</span><span class="font8">. Nature 406: 151–157.</span></p>
<p><span class="font8">Shen Y, Ronald P. 2002. Molecular determinants of disease and resistance in interactions of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> and rice. Microbes Infect</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 4: 1361-1367</span></p>
<p><span class="font8">Shi W, Sun H. 2002. Type IV Pilus-Dependent Motility and Its Possible Role in Bacterial Pathogenesis. Infection immunol 70: 1–4.</span></p>
<p><span class="font8">Strom MS, Lory S. 1993. Structure function and biogenesis of the type IV pili. Annu. rev</span><span class="font8" style="font-style:italic;">.</span></p>
<p><span class="font8">Microbiol 47: 565-597.</span></p>
<p><span class="font8">Sun QH, Hu J, Huang GX, Ge C, Fang RX, He CZ. 2005. Type-II secretion pathway structural gene </span><span class="font8" style="font-style:italic;">xpsE</span><span class="font8">, xylanase- and celllulase secretion and virulence in </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae.</span><span class="font8"> Plant Pathol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 54: 15-21</span></p>
<p><span class="font8">Tsuge S, Furutani A, Fukunaka R, Oku T, Tsuno K, Ochiai H, Inoue Y, Kaku H, Kubo Y. 2002. Expression of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae </span><span class="font8">pv. </span><span class="font8" style="font-style:italic;">oryzae hrp</span><span class="font8"> genes in a novel synthetic medium, XOM2. J. Gen. Plant Pathol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 68: 363–371.</span></p>
<p><span class="font8">Tsuge S, Nakayama T, Terashima S, Ochiai H, Furutani A, Oku T, Tsuno K, Kubo Y, Kaku H. 2006. Gene Involved in Transcriptional Activation of the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> Regulatory Gene </span><span class="font8" style="font-style:italic;">hrpG</span><span class="font8"> in </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. oryzae. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 188: 4158-4162</span></p>
<p><span class="font8">Tsuge S, Terashima S, Furutani A, Ochiai H, Oku T, Tsuno K, Kaku H, Kubo Y. 2005. Effects on Promoter Activity of Base Substitutions in the </span><span class="font8" style="font-style:italic;">cis-</span><span class="font8">Acting Regulatory Element of HrpXo Regulons in </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv</span><span class="font8" style="font-style:italic;">. oryzae.</span><span class="font8"> J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 187: 2308-2314</span></p>
<p><span class="font8">Wang L, Wei R, He C. 2008. Two </span><span class="font8" style="font-style:italic;">Xanthomonas </span><span class="font8">extracellular polygalacturonases, PghAxc and PghBxc, are regulated by type III secretion regulators HrpX and HrpG and are required for virulence. Mol Plant Microbe Interact. 21: 555–563.</span></p>
<p><span class="font8">Wang JC, So BH, Lee DS, Park YJ, Lee BM, Kang HW. 2008. Genome Wide- Identification of virulence Genes in </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> by Transposon Mutagenesis. Plant Pathol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 57: 1136-1145</span></p>
<p><span class="font8">Wang N, Lu SE, Wang J, Chen ZJ, Gross DC. 2006. The expression of genes encoding lipodepsipeptide phytotoxins by </span><span class="font8" style="font-style:italic;">Pseudomonas syringae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">syringae</span><span class="font8"> is coordinated in response to plant signal molecules. Mol Plant Microbe Interact</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 19: 257-269.</span></p>
<p><span class="font8">Welch R, Forestier A, Lobo C, Pellett A, Thomas SW, Rowe G. 1992. The synthesis and function of the </span><span class="font8" style="font-style:italic;">Escherichia coli</span><span class="font8"> hemolysins and related RTX exotoxins. FEMS Microbiol</span></p>
<p><span class="font8">Immunol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 105: 29–36.</span></p>
<p><span class="font8">Wei K, Tang DJ, He YQ, Feng JX, Jiang BL, Lu GT, Chen B, Tang JL. 2007. </span><span class="font8" style="font-style:italic;">hpaR</span><span class="font8">, a Putative </span><span class="font8" style="font-style:italic;">marR</span><span class="font8"> Family Transcriptional Regulator, Is Positively Controlled by HrpG and HrpX and Involved in the Pathogenesis, Hypersensitive Response,andExtracellular Protease Production of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pathovar </span><span class="font8" style="font-style:italic;">campestris.</span><span class="font8"> J. Bacteriol</span><span class="font8" style="font-style:italic;">. </span><span class="font8">189: 2055-2062</span></p>
<p><span class="font8">Wengelnik K, Bonas U. 1996. HrpXv, an AraC-type regulator, activates expression of five of six loci in the </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> cluster of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pv. vesicatoria essential for pathogenicity and induction of the hypersensitive reaction. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 178: 3462–3469.</span></p>
<p><span class="font8">Wengelnik K, Marie C, Russel M, Bonas U. 1996. Expression and localization of HrpA1, a protein of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">vesicatoria</span><span class="font8"> essential for pathogenicity and induction of the hypersensitive reaction. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 178: 1061–1069.</span></p>
<p><span class="font8">Wengelnik K, Rossier O, Bonas U. 1999. Mutations in the regulatory gene </span><span class="font8" style="font-style:italic;">hrpG</span><span class="font8"> of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">vesicatoria</span><span class="font8"> result in constitutive expression of all </span><span class="font8" style="font-style:italic;">hrp</span><span class="font8"> genes. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 181: 6828–6831.</span></p>
<p><span class="font8">Wiggerich HG, Klauke B, Koplin R, Priefer UB, Puhler A. 1997. Unusual structure of the tonB-exb DNA region of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8">: tonB, exbB, and exbD1 are essential for ferric iron uptake, but exbD2 is not. J. Bacteriol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 179: 71037110</span></p>
<p><span class="font8">Wiggerich HG, Puhler A. 2000. The </span><span class="font8" style="font-style:italic;">exbD2</span><span class="font8"> gene as well as the iron-uptake genes </span><span class="font8" style="font-style:italic;">tonB</span><span class="font8">, </span><span class="font8" style="font-style:italic;">exbB </span><span class="font8">and </span><span class="font8" style="font-style:italic;">exbD1</span><span class="font8"> of </span><span class="font8" style="font-style:italic;">Xanthomonas campestris</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">campestris</span><span class="font8"> are essential for the induction of a hypersensitive response on pepper (</span><span class="font8" style="font-style:italic;">Capsicum annuum</span><span class="font8">). Microbiol</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 146</span><span class="font8" style="font-weight:bold;">: </span><span class="font8">1053– 1060</span></p>
<p><span class="font8">Wu J, Wu M, He C. 2010. Identification and functional analysis of Tdrxoo, the member of TonB-dependent-receptor family proteins in </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. oryzae.</span></p>
<p><span class="font8">Wei Sheng Wue Bao 50: 155-161</span></p>
<p><span class="font8">Yamazaki A, Hirata H, Tsuyumu S. 2008. Type III regulators HrpG and HrpXct control synthesis of α-amylase, which is involved in planta multiplication of </span><span class="font8" style="font-style:italic;">Xanthomonas axonopodis</span><span class="font8"> pv.</span><span class="font8" style="font-style:italic;">citri.</span><span class="font8"> J. Gen. Plant Pathol. 74: 254–257.</span></p>
<p><span class="font8">Yang B, Zhu W, Johnson LB, White FF. 2000. The virulence factor AvrXa7 of </span><span class="font8" style="font-style:italic;">Xanthomonas oryzae</span><span class="font8"> pv. </span><span class="font8" style="font-style:italic;">oryzae</span><span class="font8"> is a type III secretion pathway-dependent nuclear-localized double-stranded DNA-binding protein. Proc Natl Acad Sci. U.S.A</span><span class="font8" style="font-style:italic;">.</span><span class="font8"> 97: 9807-9812.</span></p>
<p><span class="font4">20 </span><span class="font0">• </span><span class="font3" style="font-variant:small-caps;">udayana university biosciences and biotechnology forum</span></p>