---
layout: full_article
title: "ANALYSIS OF GENE TARGETING EVENTS IN BUCKWHEAT (Fagopyrum esculentum) TRANSFORMED b THE MUTANT STRAINS OF Agrobacterium tumefaciens THAT HAVE DEFICIENCY IN T-­??DNA INTEGRATION STEP INTO PLANT GENOME"
author: "Mineo Kojima, Putu Suparthana, Tsutomu Shimizu, Masahiro Nogawa"
categories: jbb
canonical_url: https://jurnal.harianregional.com/jbb/full-13307 
citation_abstract_html_url: "https://jurnal.harianregional.com/jbb/id-13307"
citation_pdf_url: "https://jurnal.harianregional.com/jbb/full-13307"  
comments: true
---

<a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font7"><a name="bookmark1"></a>ARTICLE</span></h1>
<p><span class="font14" style="font-weight:bold;">ANALYSIS OF GENE TARGETING EVENTS IN BUCKWHEAT (</span><span class="font14" style="font-weight:bold;font-style:italic;">Fagopyrum esculentum</span><span class="font14" style="font-weight:bold;">) TRANSFORMED b THE MUTANT STRAINS OF </span><span class="font14" style="font-weight:bold;font-style:italic;">Agrobacterium tumefaciens</span><span class="font14" style="font-weight:bold;"> THAT HAVE DEFICIENCY IN T--‐‑DNA INTEGRATION STEP INTO PLANT GENOME</span></p>
<p><span class="font13" style="font-weight:bold;">Mineo Kojima<sup>1*</sup>, Putu Suparthana<sup>2</sup> , Tsutomu Shimizu<sup>3</sup> and Masahiro Nogawa<sup>4</sup></span></p>
<ul style="list-style:none;"><li>
<p><span class="font13"><sup>1</sup>Professor emeritus, Shinshu University, </span><span class="font13" style="font-style:italic;">Department of Applied Biology, Faculty of Textile Science and Technology, Shinshu University, 3--‐‑15--‐‑1 Tokida, Ueda, Nagano 386--‐‑8567, Japan. Email:</span></p></li></ul>
<p><a href="mailto:kojima@silk.plala.or.jp"><span class="font13" style="font-style:italic;">kojima@silk.plala.or.jp</span></a><span class="font13"> <sup>2</sup></span><span class="font13" style="font-style:italic;">Department of Food Science and Technology, Faculty of Agricultural Technology, Udayana University, Gedung GA, Kampus Bukit Jimbaran, Bali, Indonesia</span></p>
<ul style="list-style:none;"><li>
<p><span class="font13"><sup>3</sup></span><span class="font13" style="font-style:italic;">Life Science Research Institute, Kumiai Chemical Industry Co., Ltd., 276 Tamari, Kakegawa, Shizuoka 463--‐‑0011, Japan.</span></p></li>
<li>
<p><span class="font13"><sup>4</sup></span><span class="font13" style="font-style:italic;">Bioscience and Biotechnology Course,</span><span class="font13"> Division of </span><span class="font13" style="font-style:italic;">Applied Biology, Faculty of Textile Science and Technology, Shinshu University, 3--‐‑15--‐‑1 Tokida, Ueda, Nagano 386--‐‑8567, Japan.</span></p></li></ul>
<ul style="list-style:none;"><li>
<p><span class="font13"><sup>*</sup>Corresponding author</span></p></li></ul>
<p><span class="font13" style="font-weight:bold;">ABSTRACT</span></p>
<p><span class="font13">Gene targeting events were analyzed in buckwheat </span><span class="font13" style="font-style:italic;">Fagopyrum esculentum</span><span class="font13">, var. Shinano No. 1) that were </span><span class="font13" style="font-style:italic;">in planta</span><span class="font13"> transformed T</span><span class="font10">1</span><span class="font13">) by the three strains of </span><span class="font13" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font13"> harboring a gene targeting vector for an endogenous gene acc.no.AB327276); M--‐‑1 mutant </span><span class="font13" style="font-style:italic;">abvA</span><span class="font13">::</span><span class="font13" style="font-style:italic;">Tn5</span><span class="font13">) and M--‐‑31 mutant </span><span class="font13" style="font-style:italic;">virA</span><span class="font13">::</span><span class="font13" style="font-style:italic;">Tn5</span><span class="font13">) strains, both of which are capable of transferring its T--‐‑DNA into nucleus of host plant cell, but have a deficiency in T--‐‑DNA integration step into chromosome and LBA4404 strain, being a commonly used strain. The results of both phenotype examination and Southern blot analysis implied that a targeting construct s) was integrated into a different locus loci) each depending on the strain of </span><span class="font13" style="font-style:italic;">A. tumefaciens</span><span class="font13"> used for transformation. Both 5’--‐‑end and 3’--‐‑end flanking DNA segments that were expected from precise insertion of targeting construct into the endogenous U--‐‑gene locus were obtained by PCR with 3 23%) transformants out of the randomly selected 13 transformants by M--‐‑1 mutant strain, while only 3’--‐‑end flanking DNA segment was obtained with 3 21%) transformants out of 14 transformants by M--‐‑31 strain. Taken together, the results suggested the potential of usage of a mutant strain of </span><span class="font13" style="font-style:italic;">A. tumefaciens</span><span class="font13"> for gene targeting which has a deficiency in T--‐‑DNA integration step into plant genome.</span></p>
<p><span class="font13" style="font-weight:bold;font-style:italic;">Keywords: </span><span class="font13" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font13">; gene targeting; buckwheat; </span><span class="font13" style="font-style:italic;">Fagopyrum esculentum</span><span class="font13">; mutant strains of </span><span class="font13" style="font-style:italic;">A. tumefaciens</span></p>
<p><span class="font14" style="font-weight:bold;">INTRODUCTION</span></p>
<p><span class="font15">D</span><span class="font3" style="font-weight:bold;">espite many attempts, gene targeting has been</span></p>
<p><span class="font3" style="font-weight:bold;">extremely difficult in higher plants Hanin and Paszkowski, 2003; Hanin </span><span class="font3" style="font-style:italic;">et al</span><span class="font3" style="font-weight:bold;">.,</span></p>
<p><span class="font14">2001; Terada </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2002; Endo </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2006; Offringa et al., 1993). Gene targeting could achieve precise targeted gene replacement without additional disruption of the targeted genomes. Thus, gene targeting through homologous recombination has long been a major goal in plant genetic transformation field Hanin and Paszkowski, 2003; Hanin </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2001; Terada </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2002; Endo </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2006; Offringa et al., 1993) .</span></p>
<p><span class="font14">In the organisms ranging from bacteria to plants to animals, double--‐‑strand DNA breaks are repaired by either homologous recombination, which requires sequence homology, or non--‐‑ homologous end joining, which requires no or very limited sequence homology Hanin and Paszkowski, 2003) . The usage of these alternative repair pathways depends on the organisms. Homologous recombination is prevalent in prokaryotes and yeast, while non--‐‑homologous end joining is predominant in multicellular eukaryotes. This difference is considered to affects greatly homologous recombination competence, or gene targeting competence, of the organisms Hanin and Paszkowski, 2003).</span></p>
<p><span class="font14">Exogenous DNA is believed to be integrated into plant genome by the machinery of plant. Also, integration mechanism of exogenous DNA, namely either homologous recombination or non--‐‑ homologous end joining, is believed to be determined principally by host plant Hanin and Paszkowski, 2003). However, we argue based on evidences presented below that there should be possibility that </span><span class="font14" style="font-style:italic;">Agrobacterium tumefaciens </span><span class="font14">contribute to integration of transforming DNA into plant genome to some extent. First, among numerous microorganisms in nature, only </span><span class="font14" style="font-style:italic;">Agrobacteria</span><span class="font14"> are capable of transforming plants. It seems to be reasonable to assume that they are so, since they have evolved some ability that is important for the integration of transforming DNA into plant genome. Second, some gene products proteins) of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14">, for example, VirD2, VirE2, VirF, and possibly other as yet unidentified products, are transferred, independently or in association with T--‐‑DNA, from </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> cell to plant cell during transformation Tzfira and Citovsky, 2006). These gene products might play some role in integration of transforming DNA into plant genome. Finally, we have isolated two mutants of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14">, M--‐‑1 mutant </span><span class="font14" style="font-style:italic;">abvA</span><span class="font14">::</span><span class="font14" style="font-style:italic;">Tn5</span><span class="font14">) and M--‐‑31 mutant </span><span class="font14" style="font-style:italic;">virA</span><span class="font14">::</span><span class="font14" style="font-style:italic;">Tn5</span><span class="font14">), both of which are capable of transferring its T--‐‑DNA into nucleus of host plant cell, but have a deficiency in its integration step into chromosome Majumder </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2000; Majumder </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2001).</span></p>
<p><span class="font14">Taking the consideration above into account, we speculated as follows; when plants are transformed by a wild strain of </span><span class="font14" style="font-style:italic;">A. tumefaciens </span><span class="font14">having an intact ability, the transforming DNA would be immediately integrated into random loci via illegitimate recombination after being transferred inside nucleus of host plant cell. By contrast, when the transforming &nbsp;DNA is</span></p>
<p><span class="font14">transferred into plant cell by using M--‐‑1 or M--‐‑31 mutant strain having some deficiency in T--‐‑DNA integration machinery, the transforming DNA could stay in nucleus for a longer period before its integration into chromosome and therefore it would be allowed to have sufficient time to search</span></p>
<p><span class="font14">for homologous locus on host chromosome. The present study was carried out to examine this speculation. As far as we know, mutants of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> have never been used in the studies of gene targeting.</span></p>
<p><span class="font14">In the present study, buckwheat plant </span><span class="font14" style="font-style:italic;">Fagopyrum esculentum</span><span class="font14">, var. Shinano No.1) was used as an experimental plant and its endogenous genomic DNA segment of 3.8 kb acc.no.AB327276), being tentatively referred to as a U--‐‑gene, was chosen as a model gene to be targeted for the following reasons; we have developed a simple and efficient </span><span class="font14" style="font-style:italic;">in planta </span><span class="font14">transformation method for buckwheat Kojima </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 2000a; Kojima </span><span class="font14" style="font-style:italic;">et al.</span><span class="font14">, 2000b Kojima et al. 2006). Buckwheat plants grow rapidly and produce many seeds within three months. The U--‐‑gene was suitable for Southern blot analysis since it gave strong bands and few interfering band. There are, however, some disadvantages in this system. First, genome sequencing project of buckwheat has not yet started and information concerning its genes and genome structure are lacking. Moreover, buckwheat is a self--‐‑incompatible pollination plant, and therefore only pollination between different types of flowers can result in fertilization. As a result, most of plants at T</span><span class="font12">1 </span><span class="font14">generation should be heterozygous for the integrated transgene, otherwise being gene targeted in most of transformants at T</span><span class="font12">0 </span><span class="font14">generation.</span></p>
<p><span class="font14">In most of the preceding studies of gene targeting, plants have been transformed with the conventional methods using the </span><span class="font14" style="font-style:italic;">in vitro </span><span class="font14">cultured cells. It, however, has been revealed that plant genome become unstable, resulting in somaclonal variations, during </span><span class="font14" style="font-style:italic;">in vitro</span><span class="font14"> culture Risseuw </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 1997). In contrast, </span><span class="font14" style="font-style:italic;">in planta </span><span class="font14">transformation includes no </span><span class="font14" style="font-style:italic;">in vitro</span><span class="font14"> culture of plant cells, and hence would not cause plant genome to be unstable during transformation. In present paper, the occurrence of gene targeting was examined at T</span><span class="font12">1 </span><span class="font14">generation by phenotypes of transformants and Southern blot and PCR analyses.</span></p>
<p><span class="font14" style="font-weight:bold;">MATERIALS AND METHODS</span></p>
<h3><a name="bookmark2"></a><span class="font14" style="font-weight:bold;"><a name="bookmark3"></a>Construction of targeting vector</span></h3>
<p><span class="font14">Genomic DNA was extracted from 5 days old seedlings of buckwheat </span><span class="font14" style="font-style:italic;">Fagopyrum esculentum </span><span class="font14">var. Shinano No.1) and digested with </span><span class="font14" style="font-style:italic;">EcoR</span><span class="font14">I and </span><span class="font14" style="font-style:italic;">Xho</span><span class="font14">I. The digested DNA was ligated with a pBluescript cloning vector Stratagen, San Diego, USA) and then transformed to </span><span class="font14" style="font-style:italic;">Escherichia coli </span><span class="font14">JM109). From among the genomic DNA segments of buckwheat cloned in pBluescript, a DNA segment was selected as a gene to be targeted because it was appropriate size 3.8 kb) and had a </span><span class="font14" style="font-style:italic;">Bgl</span><span class="font14">II site in its middle, which seemed to be suitable for construction of gene targeting construct. This genomic DNA segment 3.8 kb) of buckwheat was tentatively referred as a U--‐‑gene, and its sequence data have been submitted to the DDBJ database under accession no. AB327276 Fig. 1). Any genes with significant homology to the nucleotide sequence of the U--‐‑gene were not found by NCBI BLAST http://www. ncbi.lm.nih.gov/BLAST). A cloning vector, pBluescript, was digested by </span><span class="font14" style="font-style:italic;">Pvu</span><span class="font14">I, and a DNA</span></p>
<p><span class="font14">segment 445 bp) containing a multi--‐‑cloning site was recovered, which was then inserted into a </span><span class="font14" style="font-style:italic;">Bgl</span><span class="font14">II site in the middle of U--‐‑gene. Using U--‐‑gene with a insertion of the DNA segment 445 bp) of pBluescript plasmid as a template, PCR was carried out to obtain DNA segment 3.2 kb) that contained the DNA segment 445 bp) from pBluescript plasmid in the middle of a central part 2.8 kb) of U--‐‑gene, in which L--‐‑124 primer, 5’ GCAACCAGGAAATCGAGCTT 3’ and L--‐‑67 primer, 5’ CATAAGAAGCGGCTCCTACT 3’ were used. The PCR products 3.2 kb) was blunt--‐‑ended and subsequently inserted into the site of β--‐‑glucuronidase gene segment of a pIG121--‐‑Hm binary vector Ohta </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 1990), resulting in a targeting vector Fig. 1B).</span></p>
<ul style="list-style:none;"><li>
<h3><a name="bookmark4"></a><span class="font14" style="font-weight:bold;font-style:italic;"><a name="bookmark5"></a>A. tumefaciens</span><span class="font14" style="font-weight:bold;"> strains and transformation of buckwheat</span></h3></li></ul>
<p><span class="font14">The targeting vector constructed above was introduced into three strains of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> by transformation; M--‐‑1 mutant </span><span class="font14" style="font-style:italic;">abvA</span><span class="font14">::</span><span class="font14" style="font-style:italic;">Tn5</span><span class="font14">) strain, M--‐‑ 31 mutant </span><span class="font14" style="font-style:italic;">virA</span><span class="font14">::</span><span class="font14" style="font-style:italic;">Tn5</span><span class="font14">) strain and LBA4404 strain. The M--‐‑1 and M--‐‑31 mutant strains were</span></p>
<div><img src="https://jurnal.harianregional.com/media/13307-1.jpg" alt="" style="width:482pt;height:297pt;">
<p><span class="font4" style="font-weight:bold;">ZJJSil</span></p>
<p><span class="font6" style="font-weight:bold;">U-gene</span></p>
<p><span class="font4" style="font-weight:bold;">Psfl</span></p>
<p><span class="font3" style="font-weight:bold;">1.0 kb</span></p>
<p><span class="font3" style="font-weight:bold;">2.3 kb 2.5 kb</span></p>
<p><span class="font3" style="font-weight:bold;">3' end</span></p>
<p><span class="font3" style="font-weight:bold;">3.8 kb</span></p>
<p><span class="font4" style="font-weight:bold;">EcoRI</span></p>
<p><span class="font2" style="font-weight:bold;">(acc. no. AB327276)</span></p>
<p><span class="font4" style="font-weight:bold;">Dig-U (5'-half) probe</span></p>
<p><span class="font4" style="font-weight:bold;">Dig-U (3-half) probe</span></p>
<p><span class="font6" style="font-weight:bold;">(B)</span></p>
<p><span class="font6" style="font-weight:bold;">U-gene targeting construct</span></p>
<p><span class="font3" style="font-weight:bold;">5' end</span></p>
<p><span class="font4" style="font-weight:bold;">Sg/Ii</span></p>
<p><span class="font2" style="font-weight:bold;font-style:italic;">Pvu</span><span class="font2" style="font-weight:bold;"> Il segment (445 bp) of pBluescript plasmid </span><span class="font3" style="font-weight:bold;">3- <sub>enc</sub></span><span class="font16" style="font-weight:bold;">∣</span></p>
<p><span class="font4" style="font-weight:bold;">Sacl Sg/Il</span></p>
<p><span class="font2" style="font-weight:bold;font-style:italic;">Pvu</span><span class="font2" style="font-weight:bold;"> Il segment (445 bp) of pBluescript plasmid</span></p>
<p><span class="font3" style="font-weight:bold;">5' end</span></p>
<p><span class="font3" style="font-weight:bold;">190 bp</span></p>
<p><span class="font3" style="font-weight:bold;">5’ end</span></p>
<p><span class="font3" style="font-weight:bold;">2.2 kb</span></p>
<p><span class="font13" style="font-weight:bold;">Fig. 1. </span><span class="font13">Schematic presentation of a U--‐‑gene A) and a U--‐‑gene targeting construct B), and locations of Dig--‐‑labeled probes used for Southern blot analysis C). Kojima et al.)</span></p>
</div><br clear="all">
<div>
<p><span class="font3" style="font-weight:bold;">3' end</span></p>
<table border="1">
<tr><td style="vertical-align:top;"></td><td rowspan="2" style="vertical-align:middle;">
<p><span class="font3" style="font-weight:bold;">◄“ 0.7 kb “►</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3" style="font-weight:bold;">-------------------- </span><span class="font16" style="font-weight:bold;">∠</span><span class="font3" style="font-weight:bold;">.ι kb ---------------</span></p></td></tr>
</table>
</div><br clear="all">
<p><span class="font14">isolated previously in our laboratory Majumder </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">. 2000; Majumder </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">. 2001). Both of mutant strains are capable of transferring its T--‐‑DNA into the nucleus of host plant cell, but have a deficiency in integrating it into the chromosome. The LBA4404 strain is widely used and has been shown to integrate transgenes randomly to plant genomes Miao and Lam, 1995; Gelvin 2003).</span></p>
<p><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> was cultured on LB agar medium containing kanamycin 50 µμg/ml), rifampicin 10 µμg/ml) and hygromycin B 50 µμg/ml) at 28ºC for 2 days. Bacterial cells were harvested with a loop, suspended at a density of 1.0 x 10<sup>9</sup> cells/ml in water and used as an inoculum.</span></p>
<p><span class="font14">The seedlings of 4 to 5 days old buckwheat var. Shinano No.1) were </span><span class="font14" style="font-style:italic;">in planta</span><span class="font14"> transformed by our reported method Kojima </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">. 2000a; Kojima </span><span class="font14" style="font-style:italic;">et al.</span><span class="font14"> 2000b; Kojima et al. 2006). Ten seedlings were transformed by each strain of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> or treated with water nontransformant) and grown individually to maturation to set seeds in a pot ø 23 cm) under ambient conditions. The random cross--‐‑pollination was aided by using plumage between plants in each pot. Seeds T</span><span class="font11">1</span><span class="font14">) were harvested separately from each pot and mixed well, from which seeds were randomly selected for all experiments in this paper. In our </span><span class="font14" style="font-style:italic;">in planta </span><span class="font14">transformation method, any step of selection for transformants was not included. Still very high transformation efficiency was obtained as being described below. In our previous paper, PCR analysis indicated that 70% of T</span><span class="font11">1 </span><span class="font14">plants were transformed Kojima </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">. 2000a).</span></p>
<h3><a name="bookmark6"></a><span class="font14" style="font-weight:bold;"><a name="bookmark7"></a>Isolation of genomic DNA from buckwheat</span></h3>
<p><span class="font14">Genomic DNA was extracted from leaves of young plants by the method of Chen </span><span class="font14" style="font-style:italic;">et al.</span><span class="font14"> 2003) with some modification. Young leaves up to 0.8 g) were pulverized in liquid nitrogen in a mortar. The resultant powder was transferred to a tube 2.0 ml), to which 700 µμl of extraction buffer pre--‐‑ warmed at 65ºC was added. The extraction buffer was composed of tris 100 mM, pH 8.0), NaCl 1.4 M), ethylenediamine--‐‑N,N,N’,N’--‐‑tetraacetic acid EDTA, 20 mM), cetyltrimethylammonium</span></p>
<p><span class="font14">bromide CTAB, 3%), 2--‐‑mercapthoethanol 25 mM) and RNase 10 µμg, Nippon Gene, Tokyo, Japan). The powder was well suspended in the extraction buffer and then incubated at 65ºC, for 5 min. Thereafter, a mixture 570 µμl) of chloroform and isoamylalcohol 24:1 v/v) was added to the tube, followed by vigorous shaking 20 times. Subsequently, the tube was centrifuged at 1.3x10<sup>4 </sup>g for 5 min. Then, the supernatant was discarded and the precipitate crude DNA) was washed with 70% ethanol, followed by drying </span><span class="font14" style="font-style:italic;">in vacuo</span><span class="font14">. The dried precipitate was dissolved in 400 µμl of 10 mM tris--‐‑HCl, pH 8.0, buffer with 1 mM EDTA TE buffer) containing RNase 10 µμg) and incubated at 37˚C for 30 min. Thereafter, 80 µμl of 60 mM tris--‐‑ HCl, pH 7.8, containing EDTA 60 mM), sodium dodecylsulfate 3%) and proteinase K 40 µμg, Wako Pure Chemical Industries, Osaka, Japan) was added to the tube and incubated at 55˚C for 30 min, followed by extraction, each once, with phenol, a mixture of phenol and chloroform 1:1, v/v) and chloroform. Finally, DNA was ethanol--‐‑ precipitated, washed with 70% ethanol, dried </span><span class="font14" style="font-style:italic;">in vacuo</span><span class="font14"> and finally dissolved in TE buffer 100 µμl). The purity of preparation was checked by agarose gel electrophoresis and the DNA contents were determined by optical density at 260 n</span></p>
<h3><a name="bookmark8"></a><span class="font14" style="font-weight:bold;"><a name="bookmark9"></a>Southern blot analysis</span></h3>
<p><span class="font14">Genomic DNA 20 µμg) from buckwheat was digested with restriction endonucleases, fractionated on 1% agarose gel and transferred to a nylon membrane Hybond N<sup>+</sup>; GE Healthcare, UK). The two probes, Dig--‐‑U 5’--‐‑half) probe and Dig--‐‑U 3’--‐‑half) probe, were prepared by a PCR DIG labeling method Roche Diagnostics, Tokyo, Japan) Fig. 1C). The following primers were used: for a Dig--‐‑U 5’--‐‑half) probe; L--‐‑124 primer, 5’ GCAACCAGGAAATCGAGCTT 3’ and L--‐‑ 239primer,5’CAGGCTTTACACTTTATGCT3’; for a Dig--‐‑U 3’--‐‑half) probe, T3 promoter sequence primer, 5’ AATTAACCCTCACTAAAGGG 3’ and T7 promoter sequence primer, 5’ GTAATACGACTCACTATAGGGC 3’. These two primer sequences were included in the</span></p>
<p><span class="font14">vector pBlurscript) used for subcloning of the template DNA segment. The DNA transferred membrane was probed with the probes prepared above and DNA bands were then detected using a Dig DNA Detection Kit Roche Diagnostics, Tokyo, Japan).</span></p>
<h3><a name="bookmark10"></a><span class="font14" style="font-weight:bold;"><a name="bookmark11"></a>PCR analysis</span></h3>
<p><span class="font14">PCR was conducted to amplify 5’--‐‑ and 3’--‐‑ end flanking DNA segments of U--‐‑gene targeting construct that was integrated in the genomes of buckwheat transformants T</span><span class="font11">1</span><span class="font14">). Instead of conventional PCR consisting of one round of reactions, nested PCR was used to confer an adequate level of specificity to the process to permit reproducible result. The reaction mixture 25 µμl) was composed of a template genomic DNA 500 ng), 50 mM KCl, 10 mM tris--‐‑HCl pH 8.3), 1.5 mM MgCl</span><span class="font11">2</span><span class="font14">, 200 µμM each of dNTPs, 0.2 µμM of each primer and 0.63 units of Ex Taq polymerase Takara Shuzo, Kusatsu, Japan). The locations and directions of primers used are shown in Fig. 2. In the nested PCR for 5’--‐‑end flanking DNA segment, the first PCR was performed at 94˚C for 1 min for initial step, followed by 94˚C for 1 min, 55˚C for 30 s, 72˚C for 2.5 min for 40 cycles and then a final extension at 72˚C for 5 min, using a primer set of T3 and L--‐‑133 primers. In the second PCR, reaction was done using 0.5 µμl of</span></p>
<p><span class="font14">the first PCR reaction mixture as a template and a primer set of L--‐‑261 and L--‐‑263 primers. The other conditions were the same as for the first PCR except for extention period at 72˚C, being 2 min and 20 s. In the nested PCR for 3’--‐‑end flanking DNA segment, a primer set of T7 and L--‐‑265 primers was used in the first PCR and a primer set of L--‐‑140 and L--‐‑271 primers in the second PCR. The other PCR conditions were the same as for 5’--‐‑ end flanking DNA segment except for extention period at 72˚C, being 1.5 min, in both the first and second PCR. The nucleotide sequences of primers used are as follows: L--‐‑133 primer, 5’ GCATCAATAACCAAACTTGC 3’; L--‐‑263 primer, 5’ GCAAGCCAATAACATATGCA 3’; L--‐‑261 primer, 5’ TAATCGCCTTGCAGCACATC 3’; L--‐‑ 265 primer, 5’ TACAATTATGCTCCCTCTAC 3’; L--‐‑140 primer, 5’ GAGGTCGACGGTATCGATAA 3’; L--‐‑271 primer, 5’ TACATTGCTGGTTCTAAGC 3’. The reaction mixture of the second PCR 10 µμl) was subjected to agarose gel 1%) electrophoresis and the gel was stained with ethidium bromide to visualize the product. To identify the PCR product, the products were purified by a DNA and GFX PCR--‐‑DNA and Gel Band Purification Kit GE Healthcare, UK) and sequenced using a Big</span></p>
<p><span class="font14">Dye Terminator V 3.1 Cycle Sequencing Kit and 3737 DNA sequencer of Applied Biosystems Japan</span></p>
<p><span class="font14">Tokyo, Japan).</span></p>
<div><img src="https://jurnal.harianregional.com/media/13307-2.jpg" alt="" style="width:482pt;height:214pt;">
<p><span class="font13" style="font-weight:bold;">Fig. 2. </span><span class="font13">Locations and directions of PCR primer sets used for amplification of 5’--‐‑end and 3’--‐‑end flanking DNA segments of a U--‐‑gene targeting construct being integrated into the genomes of transformants. Kojima et al.)</span></p>
</div><br clear="all">
<p><span class="font14" style="font-weight:bold;">RESULTS AND DISCUSSION</span></p>
<h3><a name="bookmark12"></a><span class="font14" style="font-weight:bold;"><a name="bookmark13"></a>Phenotype of transformant at T</span><span class="font12" style="font-weight:bold;">1 </span><span class="font14" style="font-weight:bold;">generation</span></h3>
<p><span class="font14">The randomly selected 15 seeds T</span><span class="font11">1</span><span class="font14">) each of transformants by the three different strains of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> harboring a U--‐‑gene targeting vector and nontransformants were sowed on soil in each pot and allowed to grow to maturation Fig. 3). The phenotype or appearance that transformants took on was remarkable for three respects. First, almost all plants in three pots which were transformed T</span><span class="font11">1</span><span class="font14">) by the three strains differed in phenotype from that of nontransformants, suggesting &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;that almost all</span></p>
<p><span class="font14">plantsweretransformants.Second,thephenotype of all individual transformant plant in each pot, which was transformed by the respective strain of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14">, was very similar each other, that is, the phenotype of transformants by the respective strain was very homogenous. Finally, a clear difference was observed in phenotype between transformants by different strain of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14">; transformants by LBA4404 strain were the shortest, had the smallest leaves and</span></p>
<p><span class="font14">aged earlier. Transformants by M--‐‑1 mutant strain were taller and had the biggest leaves. Transformants by M--‐‑31 mutant strain were the tallest, and had bigger leaves and flowers with insufficiently opened petals.</span></p>
<p><span class="font14">The U--‐‑gene in a target vector should be of no function since the U--‐‑gene was destructed by inserting a </span><span class="font14" style="font-style:italic;">Pvu</span><span class="font14">II DNA segment of pBluescript plasmid in its middle Fig. 1B). Therefore, the altered phenotypes of transformants was reasonably assumed to be caused not by the direct effect of gene function of transgene integrated into their genomes but by the disarrangement of genome due to integration of transgene. Consequently, it seemed likely that a targeting construct s) was integrated into a different locus loci) each in respective transformant that varied with the strain of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> used for transformation.</span></p>
<h3><a name="bookmark14"></a><span class="font14" style="font-weight:bold;"><a name="bookmark15"></a>Southern blot analysis</span></h3>
<p><span class="font14">Genomic DNA from the randomly selected 10 transformants T</span><span class="font12">1</span><span class="font14">) by the respective strain and 10 nontransformants were digested</span></p>
<div><img src="https://jurnal.harianregional.com/media/13307-3.jpg" alt="" style="width:282pt;height:253pt;">
<p><span class="font9">βo cm</span></p>
<p><span class="font2" style="font-weight:bold;">Nontransformed (water-treated)</span></p>
<p><span class="font2" style="font-weight:bold;">Transformed by M-1 mutant strain of </span><span class="font2" style="font-weight:bold;font-style:italic;">A. tumefaciens</span></p>
<p><span class="font2" style="font-weight:bold;">Transformed by LBA4404 strain of </span><span class="font2" style="font-weight:bold;font-style:italic;">A. tumefaciens</span></p>
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/13307-4.jpg" alt="" style="width:84pt;height:253pt;">
<p><span class="font2" style="font-weight:bold;">Transformed by M-31 mutant strain of </span><span class="font2" style="font-weight:bold;font-style:italic;">A. tumefaciens</span></p>
</div><br clear="all">
<div>
<h2><a name="bookmark16"></a><span class="font5" style="font-weight:bold;"><a name="bookmark17"></a>20 days-old</span></h2>
</div><br clear="all">
<div>
<h2><a name="bookmark18"></a><span class="font5" style="font-weight:bold;"><a name="bookmark19"></a>34 days-old</span></h2>
</div><br clear="all">
<div>
<p><span class="font13" style="font-weight:bold;">Fig.3. </span><span class="font13">Phenotypes of buckwheat plants transformed T</span><span class="font10">1</span><span class="font13">) by various strains of </span><span class="font13" style="font-style:italic;">A. tumefaciens</span><span class="font13"> harboring a U--‐‑gene targeting vector. The randomly selected 15 plants were grown in each pot. Kojima et al.)</span></p>
</div><br clear="all">
<p><span class="font14">with </span><span class="font14" style="font-style:italic;">Pst</span><span class="font14">I and </span><span class="font14" style="font-style:italic;">Sac</span><span class="font14">I and then applied to agarose gel electrophoresis. Subsequently, the two blotted membranes were prepared. One blotted membrane was probed with a Dig--‐‑U 5’--‐‑half) probe, while another blotted membrane was probed with a DigU 3’--‐‑half) probe Fig. 1C and Fig. 4). On the bases of the structure of the U--‐‑gene targeting construct and the restriction sites of </span><span class="font14" style="font-style:italic;">Pst</span><span class="font14">I and </span><span class="font14" style="font-style:italic;">Sac</span><span class="font14">I within U--‐‑gene Fig. 1A and Fig. 4C), the band pattern of Southern blot analysis was expected to be as follows. If a targeting construct is integrated into genome by a true gene targeting through a precise homologous recombination, the pattern of band on the blot probed with a Dig--‐‑U 5’--‐‑half) probe will not change, while one on the blot probed with a Dig--‐‑U 3’--‐‑half) probe will change. On the other hand, if targeting construct is integrated randomly into genome, additional</span></p>
<p><span class="font14">bands, besides bands with nontransformants, will appear on both blots probed with a Dig--‐‑U 5’--‐‑half) and a Dig--‐‑U 3’--‐‑half) probes.</span></p>
<p><span class="font14">With transformants by LBA4404 strain, additional bands were observed with two out of seven transformants on the blot probed with a Dig--‐‑U 5’--‐‑half) probe, while additional bands were observed with five out of seven transformants on the blot probed with a Dig--‐‑U 3’--‐‑ half) probe Fig. 4). With transformants by M--‐‑1 mutant, additional bands appeared with five out of seven transformants on both blots probed with a Dig--‐‑U 5’--‐‑half) and a Dig--‐‑U 3’--‐‑half) probes. With transformants by M--‐‑31 mutant, additional bands were observed with two out of seven transformants on the blot probed with a Dig--‐‑U 5’--‐‑ half) probe, while a significant change was not observed in band pattern on the blot</span></p>
<div><img src="https://jurnal.harianregional.com/media/13307-5.jpg" alt="" style="width:237pt;height:83pt;">
<p><span class="font2" style="font-weight:bold;">Non-transformants</span></p>
<p><span class="font2" style="font-weight:bold;">Transformants by LBA4404 strain</span></p>
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/13307-6.jpg" alt="" style="width:118pt;height:65pt;">
<p><span class="font2" style="font-weight:bold;">Transformants by M-31 mutant strain</span></p>
</div><br clear="all">
<div>
<p><span class="font2" style="font-weight:bold;">Transformants by M-1 mutant strain</span></p>
</div><br clear="all">
<div>
<p><span class="font6" style="font-weight:bold;">(B) </span><span class="font3" style="font-weight:bold;">Probed by Dig-U (3'-half) probe</span></p><img src="https://jurnal.harianregional.com/media/13307-7.jpg" alt="" style="width:431pt;height:86pt;">
</div><br clear="all">
<div>
<p><span class="font2" style="font-style:italic;">Pvu</span><span class="font2"> Il fraqment of pBluescript (445 bp)</span></p><img src="https://jurnal.harianregional.com/media/13307-8.jpg" alt="" style="width:429pt;height:45pt;">
<p><span class="font2" style="font-weight:bold;">3' end segment of U-gene accompanying flanking DNA</span></p>
<p><span class="font2" style="font-weight:bold;">5' end segment of U-gene (1.0 kb) accompanying flanking DNA</span></p>
<p><span class="font13" style="font-weight:bold;">Fig. 4. </span><span class="font13">Southern blot analysis of genomic DNA from buckwheat plants transformed T</span><span class="font10">1</span><span class="font13">) by various strains of </span><span class="font13" style="font-style:italic;">A. tumefaciens</span><span class="font13"> harboring a U--‐‑gene targeting vector. Genomic DNA was digested with </span><span class="font13" style="font-style:italic;">Pst</span><span class="font13"> I and </span><span class="font13" style="font-style:italic;">Sac</span><span class="font13"> I, and the blot was probed with either a Dig--‐‑U 5’ half) probe Panels A) or a Dig--‐‑U 3’ half) probe Panels B) Fig. 1C). Left end lane, the digested pBluescript plasmid containing a U--‐‑gene with a insertion of the 445 bp DNA segment of pBluescript plasmid; the second lane from left end, the digested pBluescript plasmid containing a U--‐‑gene; the remaining lanes, different plant. The bands with an arrowhead are the bands that appeared in positions different from ones of nontransformants. The genomic DNA of the same plant was loaded in the corresponding lane of Panels A and B. Panel C, the expected </span><span class="font13" style="font-style:italic;">Pst</span><span class="font13"> I / </span><span class="font13" style="font-style:italic;">Sac</span><span class="font13"> I restriction products of the U--‐‑gene targeting construct being integrated into transformant genome. Kojima et al.)</span></p>
</div><br clear="all">
<p><span class="font14">probed with a Dig--‐‑U 3’--‐‑half). Thus, analysis of the results of Southern blot experiment did not reveal the occurrence of a transformant s) that filled a criterion for gene targeted transformant.</span></p>
<p><span class="font14">However, the result of Southern blot experiment in Fig. 4 demonstrated clearly that pattern of bands was different between blots of transformants by different strain of </span><span class="font14" style="font-style:italic;">A.tumefaciens. </span><span class="font14">This result indicated that a targeting construct s) was integrated into a different locus loci) each depending on the strain of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> used for transformation, being consistent with the above--‐‑ mentioned notion deduced from phenotype of transformants Fig. 3).</span></p>
<h3><a name="bookmark20"></a><span class="font14" style="font-weight:bold;"><a name="bookmark21"></a>PCR analysis</span></h3>
<p><span class="font14">Nested PCR was carried out to amplify the 5’--‐‑end and 3’--‐‑end flanking DNA segments of the integrated targeting construct using primer sets shown in Fig. 2. If a targeting construct is integrated into genome by a true gene targeting through a precise homologous recombination, both5’--‐‑endand3’--‐‑endflankingDNAsegmentscan be amplified by the PCR. With nontransformants, neither 5’--‐‑end nor 3’--‐‑end flanking DNA segment was amplified, as expected Fig. 5). With the transformants by LBA4404 strain, neither 5’--‐‑end</span></p>
<p><span class="font14">nor 3’--‐‑end flanking DNA segment was amplified , implying that a targeting construct was integrated into the random loci of the genome. With the transformants by M--‐‑1 mutant strain, an expected size of 5’--‐‑end flanking DNA segment, 2.3 kb, was obtained with 5 38%) transformants out of the randomly selected 13 transformants, while an expected size of 3’--‐‑end flanking DNA segment, 1.5 kb, was obtained with 7 54%) transformants out of 13 transformants. Among those 13 transformants by M--‐‑1 mutant strain, both 5’--‐‑end and 3’--‐‑end flanking DNA segments was obtained with 3 23%) transformants, transformants in lanes 1, 3 and 8, whereas either or neither 5’--‐‑end or 3’--‐‑ end flanking DNA was obtained with the remaining transformants. With the transformants by M--‐‑31 mutant strain, an expected size of 5’--‐‑end flanking DNA segment was not obtained with any transformants out of the randomly selected 14 transformants, while 3’--‐‑end flanking DNA segment was obtained with 3 21%) out of 14 transformants. The amplified 5’--‐‑end and 3’--‐‑end flanking DNA segments were sequenced, indicating that both sequences were congruent with the sequence of the expected locus, respectively data not shown).</span></p>
<div>
<p><span class="font6" style="font-weight:bold;">(A) </span><span class="font3" style="font-weight:bold;">5'-end flanking DNA</span></p><img src="https://jurnal.harianregional.com/media/13307-9.jpg" alt="" style="width:333pt;height:51pt;">
<p><span class="font0" style="font-weight:bold;">MKDI 2345 DlB O 101112 13 14 M K O 1 2 3 4 5 D 7 B O 101112 13 14 M K D 1 2 3 4 5 O 7 B O 1011 12 13</span></p>
<p><span class="font2" style="font-weight:bold;">Non-transformants &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transformants by &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transformants by</span></p>
<p><span class="font2" style="font-weight:bold;">LBA4404 strain &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font3" style="font-variant:small-caps;">m-1</span><span class="font2" style="font-weight:bold;"> mutant strain</span></p>
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/13307-10.jpg" alt="" style="width:107pt;height:50pt;">
<p><span class="font0" style="font-weight:bold;">M K D 1 2 3 4 5 O 7 B Q 101112 13 14</span></p>
<p><span class="font2" style="font-weight:bold;">Transformants by M-31 mutant strain</span></p>
</div><br clear="all">
<div>
<p><span class="font6" style="font-weight:bold;">(B) </span><span class="font3" style="font-weight:bold;">3'-end flanking DNA</span></p><img src="https://jurnal.harianregional.com/media/13307-11.jpg" alt="" style="width:472pt;height:57pt;">
<p><span class="font2" style="font-weight:bold;">Non-transformants &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transformants by &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transformants by &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Transformants by</span></p>
<p><span class="font2" style="font-weight:bold;">LBA4404 strain &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;M-1 mutant strain &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;M-31 mutant strain</span></p>
<p><span class="font13" style="font-weight:bold;">Fig. 5. </span><span class="font13">PCR amplification of 5’--‐‑and 3’--‐‑end flanking DNA segments of U--‐‑gene targeting constructs that were integrated into genomes of buckwheat plants transformed T</span><span class="font10">1</span><span class="font13">) by various strains of </span><span class="font13" style="font-style:italic;">A. tumefaciens</span><span class="font13"> harboring a U--‐‑gene targeting vector. PCR was done as described in the text. Panels A, 5’--‐‑end flanking DNA product; Panels B, 3’--‐‑end flanking DNA product; M1 and M2, DNA size markers; lane 0, no--‐‑template PCR reaction product; lanes 1 to 14, different plant. The genomic DNA of the same plant was used as a template in the corresponding lane of Panels A and B. Kojima et al.)</span></p>
</div><br clear="all">
<p><span class="font14">That both 5’--‐‑end and 3’--‐‑end flanking DNA segments of targeting construct being integrated into genome were amplified by PCR suggested strongly that a true gene targeting occurred in those transformants T</span><span class="font11">1</span><span class="font14">) by M--‐‑1 mutant strain. However, caution should be exercised in interpretation of the result since the experimental system under present study has some disadvantages as mentioned above. Buckwheat is a self--‐‑incompatible pollination plant, resulting in that most of plants at T</span><span class="font11">1 </span><span class="font14">generation should be heterozygous for the integrated transgene. Moreover, we have not yet proved that the gene targeting construct is integrated in only one locus of the genomes of these transformants. Under such a circumstance, it was appropriate to reason that amplification of both 5’--‐‑end and 3’--‐‑end flanking DNA segments of the gene targeting construct was just an indication of presence of at least one copy of gene targeting construct which was precisely integrated in a locus out of two or more U--‐‑gene loci in the heterozygous diploid genomes of transformants T</span><span class="font11">1</span><span class="font14">). Only one side end flanking DNA segment of either 5’--‐‑end or 3’--‐‑end flanking DNA segment of the gene targeting construct was obtained with a majority of transformants by M--‐‑1 mutant strain and 21% transformants by M--‐‑31 mutant strain Fig. 5). Regarding targeting mechanisms of those transformants, the two possibilities could be pointed out: the first possibility is that the gene targeting construct might be integrated into genomes of transformants by an ectopic gene targeting mechanism. However, we assumed that an ectopic gene targeting model could not be the case with gene targeting events reported here. In an ectopic gene targeting model, genomic DNA is copied from target locus to T--‐‑DNA. Subsequently, the synthesized T--‐‑DNA is integrated elsewhere in the genome without changing the target locus Hanin and Paszkowski 2003; Endo </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">. 2006). The results reported in this paper cannot be consistent with this model because Southern blot analyses demonstrated that band position of endogenous U--‐‑gene was changed with many transformants data not</span></p>
<p><span class="font14">shown). Consequently, the targeting construct should have been integrated in either the exact locus or proximate locus of an endogenous U--‐‑ gene, not in other locus elsewhere in the genome. The second and plausible possibility suggested from the above result Fig. 5) was that deletion or rearrangement of DNA sequence might take place in either the 5’--‐‑end or 3’--‐‑end flanking DNA region of the integrated gene targeting construct. It was reported that gene targeted transformants being accompanied by deletions or rearrangements of surrounding genomic region were detected frequently in tobacco </span><span class="font14" style="font-style:italic;">Nicotiana tabacum</span><span class="font14">) and </span><span class="font14" style="font-style:italic;">Chlamydomonas reinhardtii</span><span class="font14"> Risseuw </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">., 1997; Risseuw et al., 1995; Puchta et al., 1996; Nelson and Lefebvre, 1995). Hooykaas </span><span class="font14" style="font-style:italic;">et al</span><span class="font14">. claimed that non--‐‑precise recombination, a recombination producing such transformants as mentioned above, presumably occurs frequently in plants and they, however, have been missed in most of previous studies because they used a selection method only for a precise recombination Risseuw et al., 1997). We now speculate that a non--‐‑precise recombination mechanism works as well as in the system under present study.</span></p>
<p><span class="font14">Taken together the results above, it appears to be promising to use the mutant strain of </span><span class="font14" style="font-style:italic;">A. tumefaciens</span><span class="font14"> for gene targeting that is capable of transferring its T--‐‑DNA into nucleus of host plant cell, but has some deficiency in its integration step into chromosome, although we were unable to obtain the genuine gene targeted transformants in the present study.</span></p>
<h3><a name="bookmark22"></a><span class="font14" style="font-weight:bold;"><a name="bookmark23"></a>REFERENCES</span></h3>
<p><span class="font14">Chen S, Jin W, Wang M, Zhang F, Zhou J, Jia O, Wu Y, Liu F, Wu P. 2003. Distribution and characterization of over 1000 T--‐‑DNA tags in rice genome. Plant J. 36: 105--‐‑113.</span></p>
<p><span class="font14">Endo M, Osakabe K, Ichikawa H, Toki, S. 2006. Molecular characterization of true and ectopic gene targeting events at the acetolactate synthase gene in </span><span class="font14" style="font-style:italic;">Arabidopsis</span><span class="font14">. Plant Cell Physiol. 47: 372--‐‑379.</span></p>
<p><span class="font14">Gelvin SB. 2003. </span><span class="font14" style="font-style:italic;">Agrobacterium</span><span class="font14">--‐‑mediated plant transformation: the biology behind the “gene--‐‑jockeying” tool. Microbiol. Mol. Biol. Rev. 67: 16--‐‑37.</span></p>
<p><span class="font14">Hanin M, Volrath S, Bogucki A, Briker M, Ward E, Paszkowski J. 2001. Gene targeting in </span><span class="font14" style="font-style:italic;">Arabidopsis</span><span class="font14">. Plant J. 28: 671--‐‑677.</span></p>
<p><span class="font14">Hanin M, Paszkowski J. 2003. Plant genome modification by homologous recombination. Curr. Opin. Plant Biol. 6: 157--‐‑162.</span></p>
<p><span class="font14">Kojima M, Arai Y, Iwase N, Shiratori K, Shioiri H, Nozue M. 2000a. Development of a simple and efficient method for transformation of buckwheat plants </span><span class="font14" style="font-style:italic;">Fagopyrum esculentum</span><span class="font14">) using </span><span class="font14" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font14">. Biosci. Biotechnol. Biochem. 64: 845--‐‑847.</span></p>
<p><span class="font14">Kojima M, Hihara M, Koyama S, Shioiri H, Nozue M, Yamamoto K, Sasaki T. 2000b. Buckwheat transformed with cDNA of a Rice MADS--‐‑box gene is stimulated in branching. Plant Biotechnol. 17: 35--‐‑42.</span></p>
<p><span class="font14">Kojima M, Suparthana P, Teixeira da Silva JA, Nogawa M. 2006. Development of </span><span class="font14" style="font-style:italic;">in planta</span><span class="font14"> transformation methods using </span><span class="font14" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font14">. In: Teixeira da Silva JA. ed). Floriculture, Ornamental and Plant Biotechnology, Advances and Topical Issues, vol.2. Global Science Books, London, p. 41--‐‑48.</span></p>
<p><span class="font14">Majumder P, Yoshida H, Shioiri H, Nozue M, Kojima M. 2000. M--‐‑31 mutant </span><span class="font14" style="font-style:italic;">virA</span><span class="font14">::</span><span class="font14" style="font-style:italic;">Tn5</span><span class="font14">) of </span><span class="font14" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font14"> is capable of transferring its T--‐‑DNA into the nucleus of host cell, but incapable of integrating it into the chromosome. J. Biosci. Bioeng. 90: 328--‐‑331.</span></p>
<p><span class="font14">Majumder P, Shioiri H, Nozue M, Kojima M. 2001. Isolation and characterization of a new virulence gene </span><span class="font14" style="font-style:italic;">abv A</span><span class="font14">) of </span><span class="font14" style="font-style:italic;">Agrobacterium tumefaciens</span><span class="font14">. J. Gen. Plant. Pathol. 67: 124--‐‑133.</span></p>
<p><span class="font14">Miao ZH, Lam E., 1995. Targeted disruption of the TGA 3 locus in </span><span class="font14" style="font-style:italic;">Arabidopsis thaliana</span><span class="font14">. Plant J. 7: 359--‐‑365.</span></p>
<p><span class="font14">Nelson JAE, Lefebvre PA. 1995. Targeted disruption of the NIT8 gene in </span><span class="font14" style="font-style:italic;">Chlamydomonas reinhardtii</span><span class="font14">. Mol. Cell. Biol. 15: 5762--‐‑5769.</span></p>
<p><span class="font14">Offringa R, Franke--‐‑van Dĳk MEI, deGroot MJA, van den Elzen PJM, Hooykaas PJJ. 1993. Nonreciprocal homologous recombination between </span><span class="font14" style="font-style:italic;">Agrobacterium </span><span class="font14">transferred DNA and a plant chromosomal locus. Proc. Natl. Acad. Sci. U.S.A. 90: 7346--‐‑7350.</span></p>
<p><span class="font14">Ohta S, Mita S, Hattori T, Nakamura K. 1990. Construction and expression in tobacco of β--‐‑glucuronidase GUS) reporter gene containing an intron within coding sequence. Plant Cell Physiol. 31: 805--‐‑813.</span></p>
<p><span class="font14">Puchta H, Dujon B, Hohn B. 1996. Two different but related mechanisms are used in plants for the repair of genomic double--‐‑strand breaks by homologous recombination. Proc. Natl. Acad. Sci. U.S.A. 93: 5055--‐‑5060.</span></p>
<p><span class="font14">Risseuw E, Offringa R, Franke--‐‑van Dĳk MEI, HooykaasPJJ.1995.Targetedrecombination in plants using </span><span class="font14" style="font-style:italic;">Agrobacterium</span><span class="font14"> coincides with additional rearrangements at target locus. Plant J. 7: 109--‐‑119.</span></p>
<p><span class="font14">Risseuw E, Franke--‐‑van Dĳk MEI, Hooykaas PJJ. 1997. Gene targeting and instability of </span><span class="font14" style="font-style:italic;">Agrobacterium</span><span class="font14"> T--‐‑DNA loci in the plant genome. Plant J. 11: 717--‐‑728.</span></p>
<p><span class="font14">Terada R, Urawa H, Inagaki Y, Tsugane K, Iida S. 2002. Efficient gene targeting by homologous recombination in rice. Nature biotechnol. 20: 1030--‐‑1033.</span></p>
<p><span class="font14">Tzfira T, Citovsky V. 2006. </span><span class="font14" style="font-style:italic;">Agrobacterium</span><span class="font14">--‐‑ mediated genetic transformation of plants: biology and biotechnology. Curr. Opin. Biotechnol. 17: 1--‐‑8.</span></p>
<p><span class="font3">10 </span><span class="font1">• </span><span class="font4" style="font-variant:small-caps;">asia oceania biosciences and biotechnology consortium</span></p>