---
layout: full_article
title: "CYTOLOGICAL VARIATION OF (AAG)7 REPEAT ON LETTUCE CHROMOSOMES BY FLUORESCENCE IN SITU HYBRIDIZATION (FISH)"
author: "Winny Dewi Widarmi, Shinji Kikuchi, Hidenori Sassa, Takato Koba"
categories: jbb
canonical_url: https://jurnal.harianregional.com/jbb/full-73082 
citation_abstract_html_url: "https://jurnal.harianregional.com/jbb/id-73082"
citation_pdf_url: "https://jurnal.harianregional.com/jbb/full-73082"  
comments: true
---

<p><span class="font1">INTERNATIONAL JOURNAL OF BIOSCIENCES AND BIOTECHNOLOGY • Vol. 8 No. 2 • April 2021</span></p>
<p><span class="font1">eISSN: 2655-9994</span></p>
<p><span class="font1">pISSN: 2303-3371</span></p>
<p><a href="https://doi.org/10.24843/IJBB.2021.v08.i02.p06"><span class="font1">https://doi.org/10.24843/IJBB.2021.v08.i02.p06</span></a></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font7" style="font-weight:bold;"><a name="bookmark1"></a>CYTOLOGICAL VARIATION OF (AAG)</span><span class="font5" style="font-weight:bold;">7 </span><span class="font7" style="font-weight:bold;">REPEAT ON LETTUCE CHROMOSOMES BY FLUORESCENCE </span><span class="font7" style="font-weight:bold;font-style:italic;">IN SITU</span><span class="font7" style="font-weight:bold;"> HYBRIDIZATION (FISH)</span></h1>
<p><span class="font7" style="font-weight:bold;">Winny Dewi Widarmi <sup>1,2</sup>*, Shinji Kikuchi <sup>1</sup>, Hidenori Sassa <sup>1</sup>, and Takato Koba <sup>1 1</sup></span><span class="font7">Laboratory of Genetics and Plant Breeding, Graduate School of Horticulture, Chiba</span></p>
<p><span class="font7">University, Matsudo 648, Matsudo, Chiba 271-8510, Japan</span></p>
<p><span class="font7" style="font-weight:bold;"><sup>2</sup></span><span class="font7">Corteva Agriscience (PT Dupont Agricultural Product Indonesia), Cibis Nine 10<sup>th</sup> floor, Jl. TB Simatupang No. 2, East Cilandak, South Jakarta 12560, Indonesia *Corresponding author: </span><a href="mailto:winnydw@yahoo.com"><span class="font7">winnydw@yahoo.com</span></a></p>
<h1><a name="bookmark2"></a><span class="font7" style="font-weight:bold;"><a name="bookmark3"></a>ABSTRACT</span></h1>
<p><span class="font7">Cytological studies using fluorescence </span><span class="font7" style="font-style:italic;">in situ</span><span class="font7"> hybridization (FISH) technique provides phylogenetical information in closely related taxa and have been widely applied for karyotyping and studying chromosomal organization and evolution in plant species. In the present study, FISH using a microsatellite sequence of (AAG)</span><span class="font5">7 </span><span class="font7">as the probe was performed in order to discriminate the chromosomes in four </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species, </span><span class="font7" style="font-style:italic;">i.e.</span><span class="font7">, </span><span class="font7" style="font-style:italic;">L. sativa, L. serriola</span><span class="font7">, </span><span class="font7" style="font-style:italic;">L. salign</span><span class="font7">a, and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7">. The experiment was carried out in April to September 2018 at Laboratory of Genetic and Plant Breeding of Breeding of Graduate School of Horticulture, Chiba University, Japan. Different distribution patterns of (AAG)</span><span class="font5">n </span><span class="font7">signals were shown on the chromosomes in the four </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species studied, In </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7">, FISH with (AAG)</span><span class="font5">7 </span><span class="font7">sequences revealed dispersed distribution patterns with one pair of bright signals, respectively. While in </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">L. virosa,</span><span class="font7"> distinct signals with different intensities were observed in two pairs of chromosomes of </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> and five pairs of chromosomes of </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7">. In conclusion, the AAG repeat signals could be used as cytogenetic landmarks for chromosome identification in </span><span class="font7" style="font-style:italic;">Lactuca </span><span class="font7">species.</span></p>
<p><span class="font7" style="font-style:italic;">Keywords:</span><span class="font7"> (AAG)</span><span class="font5">7 </span><span class="font7">repeat, FISH, microsatellite, lettuce</span></p>
<h1><a name="bookmark4"></a><span class="font7" style="font-weight:bold;"><a name="bookmark5"></a>INTRODUCTION</span></h1>
<p><span class="font7">Cultivated lettuce (</span><span class="font7" style="font-style:italic;">Lactuca sativa</span><span class="font7"> L.) is an economically important member of the genus </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7">, which includes closely related wild species, </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7">, </span><span class="font7" style="font-style:italic;">L. salign</span><span class="font7">a and </span><span class="font7" style="font-style:italic;">L. virosa.</span><span class="font7"> All of them have 2</span><span class="font7" style="font-style:italic;">n</span><span class="font7"> = 2</span><span class="font7" style="font-style:italic;">x</span><span class="font7"> =18 chromosomes, and the estimated genome size of </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> is 2.7 Gb, with repetitive sequences accounting for 74.2% of the genome (Koopman </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 1998; Michelmore </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 1994; Reyes-Chin-Wo </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 2017). So far, karyotype analyses were carried out to</span></p>
<p><span class="font7">demonstrate the relationships among the cultivated lettuce and its closely related species (Haque and Godward 1985; Koopman and De Jong 1996; Lindqvist 1960; Matoba </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">., 2007). Koopman </span><span class="font7" style="font-style:italic;">et al</span><span class="font7">. (1993) applied chromosome banding techniques to discriminate the chromosomes of </span><span class="font7" style="font-style:italic;">Lactuca </span><span class="font7">species. Widarmi </span><span class="font7" style="font-style:italic;">et al. (2019</span><span class="font7">) and Matoba </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> (2007) applied FISH technique using 45S and 5SrDNAs to identify the chromosomes of </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species. From these studies, it was clarified that chromosomal characteristics of</span></p>
<p><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7"> were almost similar to each other, while cytological features of </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> are slightly different from </span><span class="font7" style="font-style:italic;">L. sativa/L. serriola</span><span class="font7">, and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7"> is significantly different from the other species.</span></p>
<p><span class="font7">Fluorescence </span><span class="font7" style="font-style:italic;">in situ</span><span class="font7"> hybridization (FISH) using repetitive DNA sequences as the probes results in chromosome-specific hybridization patterns that allows identification of all chromosomes within a species, making this technique a powerful tool for the karyotyping and studying chromosomal organization and evolution in many plant species (Mukai </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> 1993; Pedersen and Langridge 1997; Jiang and Gill 2006).</span></p>
<p><span class="font7">The distribution of microsatellite (AAG)</span><span class="font5">n </span><span class="font7">on the chromosomes of wheat, barley, and related species in Triticeae were well studied. The sequence (AAG)</span><span class="font5">5 </span><span class="font7">was used as the probe for </span><span class="font7" style="font-style:italic;">in situ</span><span class="font7"> hybridization to identify different chromatin classes in wheat (Cuadrado </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 2000; 2008). The oligonucleotide sequence (AAG)</span><span class="font5">7 </span><span class="font7">was used for genome and chromosome identification via </span><span class="font7" style="font-style:italic;">in situ</span><span class="font7"> hybridization in cultivated barley and related species of the Triticeae (Poaceae) (Pedersen </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 1996) The sequence (AAG)</span><span class="font5">9 </span><span class="font7">was used for the complementary identification of all A-genome chromosomes in diploid and polyploid wheat (Badaeva </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 2015, 2016). Also the sequence (GAA)</span><span class="font5">6 </span><span class="font7">was used to perform a more accurate karyotype analysis of</span></p>
<p><span class="font7" style="font-style:italic;">Zanthoxylum armatum</span><span class="font7"> (Luo </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 2018). Thus, probing AAG repeats for </span><span class="font7" style="font-style:italic;">in situ </span><span class="font7">hybridization seems to be a useful diagnostic tool in cytogenetics in many plant species.</span></p>
<p><span class="font7">In the present investigation, we aim to perform FISH using (AAG)</span><span class="font5">7 </span><span class="font7">as the probe in order to clarify the possibility of using the microsatellite repetitive sequences for discrimination and/or identification of the chromosomes in </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species.</span></p>
<h1><a name="bookmark6"></a><span class="font7" style="font-weight:bold;"><a name="bookmark7"></a>MATERIALS AND METHODS</span></h1>
<p><span class="font7">Chromosomal study of lettuce chromosomes by FISH using (AAG)</span><span class="font5">7 </span><span class="font7">as the probe was conducted in April to September 2018 at Laboratory of Genetic and Plant Breeding of Graduate School of Horticulture, Chiba University, Japan.</span></p>
<p><span class="font7" style="font-weight:bold;">Plant materials and chromosome sample preparation</span></p>
<p><span class="font7">Seeds of </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> ‘Shinano Hope’, </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7">, </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7">, and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7"> were kindly provided by Nagano Vegetable and Ornamental Crops Experimental Station, Japan. They were germinated and planted at the Graduate School of Horticulture, Chiba University, Matsudo, Japan. Root tips of 1-2 cm long were collected from the seedlings and young plants after a few weeks growth in 500mL pots. The root tips were fixed in 3 ethanol: 1 acetic acid (v/v) at 4°C for least 1 day before preparation. Fixed root tips were washed three times with distilled water, and the samples</span></p>
<p><span class="font7">were stored in 70% ethanol at 4°C. Chromosomes preparations were made through enzymatic maceration, </span><span class="font7" style="font-style:italic;">i.e.</span><span class="font7">, root tips were digested with 10 µl enzymatic mixture (4% Cellulase Onozuka RS, Yakult; 2% Pectolyase Y-23, Kyowa; and 1% Pectinase from </span><span class="font7" style="font-style:italic;">Aspergillus</span><span class="font7">, Sigma) at 37°C for 45-90 minutes. Squashing was made in 45% acetic acid and the slides were stored at -80°C for more than12 h. The glass covers were removed by forceps for the following FISH experiment.</span></p>
<p><span class="font7" style="font-weight:bold;">DNA probe preparation</span></p>
<p><span class="font7">The (AAG)</span><span class="font5">7 </span><span class="font7">probe was labelled with digoxigenin-11-dUTP (Roche) by terminal deoxynucleotidyl transferase (TdT) (Thermo Fisher Scientific) following the manufacturer’s instructions.</span></p>
<p><span class="font7" style="font-weight:bold;">FISH analysis</span></p>
<p><span class="font7">Chromosome slides were re-fixed in 1% paraformaldehyde/1× PBS for 10 min at room temperature. Procedures for hybridization and detection were followed as those described by Kikuchi </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7">(2008). The slides were counter-stained with 4,6-diamidino-2-phenylindole (DAPI) in a VectaShield antifade solution (Vector Laboratories). All FISH images were captured with an Olympus BX53 fluorescence microscope with a CCD camera (CoolSNAP MYO; Photometrics). The FISH images were processed by Metamorph, Metavue imaging series version 7.8 and edited with Adobe</span></p>
<p><span class="font7">Photoshop CC 2017. CHIAS IV </span><a href="http://www2.kobeu.ac.jp/~ohmido/index03.htm"><span class="font7">(</span><span class="font7" style="text-decoration:underline;">http://www2.kobeu.ac.jp/~ohmido/index03.</span></a><span class="font7" style="text-decoration:underline;"> </span><a href="http://www2.kobeu.ac.jp/~ohmido/index03.htm"><span class="font7" style="text-decoration:underline;">htm</span><span class="font7">)</span></a><span class="font7"> was used for the construction of the ideogram.</span></p>
<h1><a name="bookmark8"></a><span class="font7" style="font-weight:bold;"><a name="bookmark9"></a>RESULTS AND DISCUSSION</span><br><br><span class="font7" style="font-weight:bold;"><a name="bookmark10"></a>RESULT</span></h1>
<p><span class="font7">Images of mitotic metaphase of four </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species visualized after FISH using (AAG)</span><span class="font5">7 </span><span class="font7">sequence as the probe are shown in Figure. 1.</span></p>
<p><span class="font7">Different distribution patterns of (AAG)</span><span class="font5">n </span><span class="font7">signals were shown in all four </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species studied. In </span><span class="font7" style="font-style:italic;">L. sativ</span><span class="font7">a and </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7">, FISH with (AAG)</span><span class="font5">7 </span><span class="font7">sequences revealed dispersed distribution patterns with one pair of bright signals, respectively. While in </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7">, distinct signals with different intensities were observed in two pairs of chromosomes of </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> and five pairs of chromosomes of </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7">.</span></p>
<div><img src="https://jurnal.harianregional.com/media/73082-1.jpg" alt="" style="width:408pt;height:473pt;">
<p><span class="font7" style="font-weight:bold;">Fig. 1. </span><span class="font7">FISH of (AAG)</span><span class="font5">7 </span><span class="font7">(red) probe in four </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species. (a-c) </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> cv. ‘Shinano Hope’ showing dispersed signals distribution in all chromosomes and two bright signal sites of (AAG)</span><span class="font5">7</span><span class="font7">, (d-f) </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7"> showing dispersed signals distribution in all chromosomes with one pair of weak signals, (g-i) </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> showing bright signals in two pairs of chromosomes, (j-l) </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7"> showing strong signals. Scale bar=10 µm</span></p>
</div><br clear="all">
<p><span class="font7">Figure 2 shows FISH karyotypes of four </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species, which were arranged according to the order of the lengths of the chromosomes based on the FISH images shown in Figure 1. Quantitative ideograms of</span></p>
<p><span class="font7">the prometaphase chromosomes of the four lettuce species generated by CHIAS IV based on the characteristics of condensation patterns, chromosome lengths, and FISH signals are shown in Fig. 3.</span></p>
<p><span class="font2" style="font-weight:bold;">1 &nbsp;2 &nbsp;3 456789</span></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Lactuca sattva</span></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Lactuca somola</span></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Lactuca sat</span><span class="font3" style="font-weight:bold;font-style:italic;">∣</span><span class="font0" style="font-weight:bold;font-style:italic;">gna</span></p>
<p><span class="font0" style="font-weight:bold;font-style:italic;">Lactuca virosa</span></p>
<p><span class="font7" style="font-weight:bold;">Fig. 2. </span><span class="font7">FISH karyotypes of four </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species with (AAG)</span><span class="font5">n </span><span class="font7">signals. Homologous pairs of chromosomes were arranged and numbered according to their lengths. Horizontal lines show the positions of the centromeres. Scale Bar=10 µm</span></p>
<p><span class="font7">Dispersed signal distribution was observed in prometaphase chromosomes of </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7">, with one pair of strong pericentromeric signals on Chromosome 1. </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7"> also showed similar dispersed distribution as those of </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> chromosomes, with one pair of small telomeric signals on Chromosome 2. By contrast, distinct signals were observed on </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7"> chromosomes. In </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7">, Chromosomes 3 and 8 showed clear signals at the interstitial regions of the long arm and subtelomeric region of the short arm, respectively. While, in </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7">,</span></p>
<p><span class="font7">Chromosome 1 had four major signals, two on the proximal region of the short and long arms, and two on the interstitial region of the long arm. Chromosome 3 showed two pairs of signals, one on the interstitial region and one on the subtelomeric region of the long arm. Chromosomes 2 and 5 showed one pair of clear signals on the long arms of the chromosomes and Chromosome 4 had one pair of signals on the proximal region of the chromosome. With their distribution patterns, Chromosomes 1 to 5 could be clearly discriminated by using (AAG)</span><span class="font5">7 </span><span class="font7">probe.</span></p>
<div><img src="https://jurnal.harianregional.com/media/73082-2.jpg" alt="" style="width:413pt;height:289pt;">
<p><span class="font7" style="font-weight:bold;">Fig. 3. </span><span class="font7">FISH ideogram of prometaphase chromosomes of </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> (</span><span class="font7" style="font-weight:bold;">a</span><span class="font7">), </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7"> (</span><span class="font7" style="font-weight:bold;">b</span><span class="font7">), </span><span class="font7" style="font-style:italic;">L. saligna </span><span class="font7">(</span><span class="font7" style="font-weight:bold;">c</span><span class="font7">) and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7"> (</span><span class="font7" style="font-weight:bold;">d</span><span class="font7">) with (AAG)</span><span class="font5">n </span><span class="font7">signals (red) constructed by CHIAS IV. Numerals denote the chromosome numbers according to their chromosome lengths. Black, gray, and white areas represent chromosome regions with heavy, intermediate and slight condensation, respectively. The gaps in the middle of the chromosomes indicate the centromere regions. Homologous pairs of chromosomes were arranged and numbered according to their lengths. Horizontal lines show the positions of the centromeres</span></p>
</div><br clear="all">
<h1><a name="bookmark11"></a><span class="font7" style="font-weight:bold;"><a name="bookmark12"></a>DISCUSSION</span></h1>
<p><span class="font7">Schmidt and Heslop-harrison (1996) demonstrated that microsatellites, representing a substantial fraction of the genome, showed chromosome-specific amplification in plants. The repeat AAG presented different </span><span class="font7" style="font-style:italic;">in situ</span><span class="font7"> hybridization patterns that provided cytogenetic landmarks for chromosome identification in barley, </span><span class="font7" style="font-style:italic;">Hordeum vulgare</span><span class="font7"> ssp. vulgare (Carmona </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> 2013). Also, Sonah </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> (2011)</span></p>
<p><span class="font7">described that AAG repeat units are major contributors to the genomes of dicots.</span></p>
<p><span class="font7">As shown in the previous reports (Lindqvist 1960; Koopman and De Jong 1996), the karyotype analyses indicated the closest relationship between </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7">. Our results showed that karyotype of </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7"> was similar to that of </span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7">, since the dispersed signal distribution patterns of (AAG)</span><span class="font5">7 </span><span class="font7">repeats were similarly observed in both species studied. However, </span><span class="font7" style="font-style:italic;">L. sativa and L. serriola</span><span class="font7"> could be distinguished easily by</span></p>
<p><span class="font7">the presence of the AAG pericentromeric and telomeric signals on the Chromosomes 1 and 2, respectively. On the other hand, </span><span class="font7" style="font-style:italic;">L. saligna </span><span class="font7">and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7"> showed their specific karyotype features with AAG signals compare to the other two species (Figure 2). Lindqvist (1960), Haque and Godward (1985), Koopman </span><span class="font7" style="font-style:italic;">et al.,</span><span class="font7"> (1993), and Koopman and De Jong (1996) described that cytological features of </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7"> and </span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7"> are different from those of </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7">/</span><span class="font7" style="font-style:italic;">L. serriola</span><span class="font7">. Based on the sequence comparisons of chloroplast DNAs in </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> species, Wei </span><span class="font7" style="font-style:italic;">et al.</span><span class="font7"> (2017) revealed that </span><span class="font7" style="font-style:italic;">L. sativa</span><span class="font7">/</span><span class="font7" style="font-style:italic;">L. serriola </span><span class="font7">group was different from that of </span><span class="font7" style="font-style:italic;">L. saligna</span><span class="font7">/</span><span class="font7" style="font-style:italic;">L. virosa</span><span class="font7">. This fact indicates that the distribution of microsatellite sequence on the chromosomes reflects the phylogenetic differentiation of the species in the genus </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7">. Cytological analyses on the distribution of microsatellite sequences would provide insight into the phylogenetic differentiation of the species in the genus </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7">.</span></p>
<h1><a name="bookmark13"></a><span class="font7" style="font-weight:bold;"><a name="bookmark14"></a>CONCLUSIONS</span></h1>
<p><span class="font7">In conclusion, the AAG repeat presented different </span><span class="font7" style="font-style:italic;">in situ</span><span class="font7"> hybridization patterns that provided cytogenetic landmarks for chromosome identification in </span><span class="font7" style="font-style:italic;">Lactuca </span><span class="font7">species. These such variation results provide insight into the distribution pattern and evolution of SSRs in the genome of </span><span class="font7" style="font-style:italic;">Lactuca.</span></p>
<h1><a name="bookmark15"></a><span class="font7" style="font-weight:bold;"><a name="bookmark16"></a>REFERENCES</span></h1>
<p><span class="font7">Badaeva ED, Amosova A V, Goncharov NP, Macas J, Ruban AS, Grechishnikova I V, Zoshchuk SA, Houben A, Schmid M. (2015). E-Mail A Set of Cytogenetic Markers Allows the Precise Identification of All A-Genome Chromosomes in Diploid and Polyploid Wheat. Cytogenetic and Genome Research. 146 &nbsp;&nbsp;(2015) &nbsp;&nbsp;71–79.</span></p>
<p><span class="font7">doi:10.1159/000433458.</span></p>
<p><span class="font7">Badaeva ED, Ruban AS, Zoshchuk SA, Surzhikov SA, Knüpffer H, Kilian B. (2016). Molecular cytogenetic characterization &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;of &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Triticum</span></p>
<p><span class="font7">timopheevii chromosomes provides new insight on genome evolution of T. zhukovskyi. Plant Systematics and Evolution. 302 (8) (2016) 943–956. doi:10.1007/s00606-016-1309-3.</span></p>
<p><span class="font7">Carmona A, Friero E, de Bustos A, Jouve N, Cuadrado A. (2013). Cytogenetic diversity of SSR motifs within and between Hordeum species carrying the H genome: H. vulgare L. and H. bulbosum L. Theoretical and Applied Genetics. 126(4) &nbsp;(2013) &nbsp;949–961.</span></p>
<p><span class="font7">doi:10.1007/s00122-012-2028-y.</span></p>
<p><span class="font7">Cuadrado A, Cardoso M, Jouve N. (2008). Physical organisation of simple sequence repeats (SSRs) in Triticeae: structural, functional and evolutionary implications. Cytogenetic and Genome Research. 120(3–4) (2008) 210–219. doi:10.1159/000121069.</span></p>
<p><span class="font7">Cuadrado A, Schwarzacher T, Jouve N. (2000). Identification of different chromatin classes in wheat using in situ hybridization with simple sequence repeat oligonucleotides. Theoretical and Applied Genetics. 101(5–6) (2000) 711–717. doi:10.1007/s001220051535.</span></p>
<p><span class="font7">Haque MZ, Godward MBE. (1985). Comparison between Two Genera, Species and Cultivars in Lactuceae I.</span></p>
<p><span class="font7">Karyotype analysis. Cytologia 50 (1985) 725-738</span></p>
<p><span class="font7">Jiang J, Gill BS. (2006). Current status and the future of fluorescence in situ hybridization (FISH) in plant genome research. Genome. 49(9) (2006) 1057– 1068. doi:10.1139/G06-076.</span></p>
<p><span class="font7">Kikuchi S, Matsui K, Tanaka H, Ohnishi O, Tsujimoto H. (2008). Chromosome evolution among seven Fagopyrum species revealed by fluorescence in situ hybridization (FISH) probed with rDNAs. Chromosom Science. 11(1) (2008) 37–43.</span></p>
<p><span class="font7">Koopman WJM, Guetta E, van de Wiel CCM, Vosman B, van den Berg RG. (1998). Phylogenetic relationships among </span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> (Asteraceae) species and related genera based on ITS-1 DNA sequences. American Journal of Botany. 85(11) &nbsp;(1998) &nbsp;1517–1530.</span></p>
<p><span class="font7">doi:10.2307/2446479.</span></p>
<p><span class="font7">Koopman WJM, De Jong JH. (1996). A numerical analysis of karyotypes and DNA amounts in lettuce cultivars and species ( Lactuca subsect. Lactuca , Compositae). &nbsp;&nbsp;&nbsp;&nbsp;Acta &nbsp;&nbsp;&nbsp;&nbsp;Botanica</span></p>
<p><span class="font7">Neerlandica. 45(2) (1996) 211–222.</span></p>
<p><span class="font7">doi:10.1111/j.1438-8677.1996.tb00510.x.</span></p>
<p><span class="font7">Koopman WJM, De Jong JH, De Vries IM. (1993). Chromosome banding patterns in Lettuce species (Lactuca sect. Lactuca, &nbsp;&nbsp;&nbsp;&nbsp;Compositae). &nbsp;&nbsp;&nbsp;&nbsp;Plant</span></p>
<p><span class="font7">Systematics and Evolution. 185(3–4) (1993) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;249–257.</span></p>
<p><span class="font7">doi:10.1007/BF00937661.</span></p>
<p><span class="font7">Lindqvist K. (1960). Cytogenetic Studies In The Serriola Group of Lactuca. Hereditas. 46(1–2) &nbsp;(1960) &nbsp;75–151.</span></p>
<p><span class="font7">doi:10.1111/j.1601-5223.1960.tb03080.x.</span></p>
<p><span class="font7">Luo X, Liu J, Wang J, Gong W, Chen L, Wan W. (2018). FISH analysis of </span><span class="font7" style="font-style:italic;">Zanthoxylum armatum</span><span class="font7"> based on oligonucleotides for 5S rDNA and</span></p>
<p><span class="font7">(GAA) </span><span class="font5">6</span><span class="font7">. Schwarzacher T, editor. Genome. 61(9) &nbsp;(2018) &nbsp;699–702.</span></p>
<p><span class="font7">doi:10.1139/gen-2018-0009.</span></p>
<p><span class="font7">Matoba H, Mizutani T, Nagano K, Hoshi Y, Uchiyama H. (2007). Chromosomal study of lettuce and its allied species (</span><span class="font7" style="font-style:italic;">Lactuca</span><span class="font7"> spp., Asteraceae) by means of karyotype analysis and fluorescence </span><span class="font7" style="font-style:italic;">in situ</span><span class="font7"> hybridization. Hereditas. 144(6) (2007) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;235–243.</span></p>
<p><span class="font7">doi:10.1111/j.2007.0018-0661.02012x.</span></p>
<p><span class="font7">Michelmore RW, Kesseli R V., Ryder EJ. (1994). Genetic mapping in lettuce. Phillips, R. L. and Vasil, I. K. (eds.). DNA-Based Markers in Plants. Kluwer Academic Publishers. p. 223–239.</span></p>
<p><span class="font7">Mukai Y, Nakahara Y, Yamamoto M. (1993). Simultaneous discrimination of the three genomes in hexaploid wheat by multicolor fluorescence </span><span class="font7" style="font-style:italic;">in situ </span><span class="font7">hybridization using total genomic and highly repeated DNA probes. Genome. 36(3) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1993) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;489–494.</span></p>
<p><span class="font7">doi:10.1139/g93-067.</span></p>
<p><span class="font7">Pedersen C, Langridge P. (1997). Identification of the entire chromosome complement of bread wheat by two-colour FISH. Genome. 40(5) (1997) 589–593. doi:10.1139/g97-077.</span></p>
<p><span class="font7">Pedersen C, Rasmussen SK, Linde-Laursen I. (1996). Genome and chromosome identification in cultivated barley and related species of the Triticeae (Poaceae) by in situ hybridization with the GAA-satellite sequence. Genome. 39(1):93–104. doi:10.1139/g96-013.</span></p>
<p><span class="font7">Reyes-Chin-Wo S, Wang Z, Yang X, Kozik A, Arikit S, Song C, Xia L, Froenicke L, Lavelle DO, Truco M-J, et al. (2017). Genome assembly with in vitro proximity ligation data and wholegenome triplication in lettuce. Nat Commun. doi:10.1038/ncomms14953.</span></p>
<p><span class="font7">Schmidt T, Heslop-harrison JS. (1996). The physical and genomic organization of</span></p>
<p><span class="font7">microsatellites in sugar beet.</span></p>
<p><span class="font7">Proceedings of the National Academy of Sciences of U S A. 93(16) (1996) 8761–8765.</span></p>
<p><span class="font7">Sonah H, Deshmukh RK, Sharma A, Singh</span></p>
<p><span class="font7">VP, Gupta DK. (2011). Genome-Wide</span></p>
<p><span class="font7">Distribution and Organization of Microsatellites in Plants: An Insight into Marker Development in Brachypodium. PLoS One. 6(6) (2011) 21298.</span></p>
<p><span class="font7">doi:10.1371/journal.pone.0021298.</span></p>
<p><span class="font7">Wei Shi-Xin Zhu G Van den Berg Freek T Bakker M Eric Schranz ZR. (2017). Phylogenetic relationships within Lactuca L. (Asteraceae), including African species, based on chloroplast DNA sequence comparisons. Genetic Resources and Crop Evolution. 64 (2017) &nbsp;55–71. &nbsp;doi:10.1007/s10722-</span></p>
<p><span class="font7">015-0332-5.</span></p>
<p><span class="font7">Widarmi, WD., Kikuchi, S., Sassa, H., and</span></p>
<p><span class="font7">Koba T. (2019). Characterization of lettuce chromosomes based on condensation patterns and physical mapping of 45S and 5S rDNAs using fluorescence in situ hybridization.</span></p>
<p><span class="font7">Cytologia (Tokyo). 85 (1) (2019) 49</span></p>
<p><span class="font7">55. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;doi</span></p>
<p><a href="https://doi.org/10.1508/cytologia.85.49"><span class="font7">https://doi.org/10.1508/cytologia.85.49</span></a></p>
<p><span class="font1">FACULTY OF AGRICULTURE, UDAYANA UNIVERSITY • 53</span></p>