---
layout: full_article
title: "MOLECULAR DETECTION OF PORK CONTAMINATION IN BEEF SOLD IN TRADITIONAL MARKETS, YOGYAKARTA, INDONESIA"
author: "Sri Wijayanti Wulandari, Diah Asta Putri, Nurul Suwartiningsih, Violita Syukroni"
categories: jbb
canonical_url: https://jurnal.harianregional.com/jbb/full-87975 
citation_abstract_html_url: "https://jurnal.harianregional.com/jbb/id-87975"
citation_pdf_url: "https://jurnal.harianregional.com/jbb/full-87975"  
comments: true
---

<p><span class="font2">Molecular Detection of Pork Contamination in Beef Sold in Traditional Markets, Yogyakarta, Indonesia</span></p>
<p><span class="font2">Wulandari, S.W., Putri, D.A., Suwartiningsih, N. &amp;&nbsp;Syukroni, V.</span></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font5" style="font-weight:bold;"><a name="bookmark1"></a>MOLECULAR DETECTION OF PORK CONTAMINATION IN BEEF SOLD IN TRADITIONAL MARKETS, YOGYAKARTA, INDONESIA</span></h1>
<h2><a name="bookmark2"></a><span class="font3" style="font-weight:bold;"><a name="bookmark3"></a>Sri Wijayanti Wulandari<sup>1*</sup>, Diah Asta Putri<sup>2</sup>, Nurul Suwartiningsih<sup>3</sup>, Violita Syukroni<sup>4</sup></span></h2>
<p><span class="font3"><sup>1</sup>Laboratory of Biochemistry and Biotechnology, Department of Biology, Faculty of Applied Science and Technology, Universitas Ahmad Dahlan, Jl. Ringroad Selatan, Tamanan, Banguntapan, Bantul 55191</span></p>
<p><span class="font3"><sup>2</sup>Laboratory of Botany, Department of Biology, Faculty of Applied Science and Technology, Universitas Ahmad Dahlan, Jl. Ringroad Selatan, Tamanan, Banguntapan, Bantul 55191 <sup>3</sup>Biology Teaching Laboratory, Faculty of Applied Science and Technology, Universitas Ahmad Dahlan, Jl. Ringroad Selatan, Tamanan, Banguntapan, Bantul 55191</span></p>
<p><span class="font3"><sup>4</sup>Faculty of Biology, Universitas Gadjah Mada, Jl. Teknika Selatan, Sinduadi, Mlati, Sleman 55281</span></p>
<p><span class="font3" style="font-style:italic;">*Corresponding author, e-mail: </span><a href="mailto:sri.wijayanti@bio.uad.ac.id"><span class="font3" style="font-style:italic;">sri.wijayanti@bio.uad.ac.id</span></a></p>
<h2><a name="bookmark4"></a><span class="font3" style="font-weight:bold;"><a name="bookmark5"></a>ABSTRACT</span></h2>
<p><span class="font3">Beef is one of the animal protein sources needed by the body to meet nutritional requirements. Consequently, fresh beef (whole or milled) is one of the highest-demanded comestibles on the market. Some meat kiosks in Yogyakarta traditional markets are also in service of milling whole meat. However, reports are stating that some kiosks are neglecting aspects concerning halal. One of those aspects is separating tools used to process beef and pork. Reports said that kiosk owners fail to meet that essential requirement causing Muslims to be at risk of consuming contaminated beef without their knowledge. Hence, this study aims to examine whether fresh and ground beef obtained from Pasar Pathuk and Pasar Kranggan, Kota Yogyakarta, is free from pork contamination. The examination was carried out using Polymerase Chain Reaction (PCR) on 14 samples of seven fresh and seven ground beef obtained from both markets. DNA isolation from all samples was done using FavorGen® FavorPrepTM Tissue Genomic DNA Extraction Mini Kit. Isolated DNA was further examined by PCR analysis using P14 and MTCB primers. Results showed that P14 primers could amplify the PRE-1 gene (481 bp) designed as a pork molecular marker only on positive control (fresh pork); meanwhile, MTCB primers could amplify the cytochrome b gene (1141 bp) designed as a mammal molecular marker on all samples involved in this research. Based on the results, we concluded that both fresh and ground beef sold in Pasar Pathuk and Pasar Kranggan, Yogyakarta are not contaminated by pork DNA.</span></p>
<p><span class="font3" style="font-style:italic;">Keywords</span><span class="font3">: beef, MTCB primer, P14 primer, pork contamination</span></p>
<h2><a name="bookmark6"></a><span class="font3" style="font-weight:bold;"><a name="bookmark7"></a>INTRODUCTION</span></h2>
<p><span class="font3">The vast majority of the Indonesian population is Muslim. As a Muslim-majority country, the Indonesian</span></p>
<p><span class="font3">government must ensure that food circulating in the community is safe, of good quality, nutritious and halal-certified (Triasih et al., 2016). One of the halal</span></p>
<p><span class="font3">certification criteria requires that food be free from any pork elements as stated in the Quran (An-Nahl:115), “He has only forbidden you to eat carrion, blood, swine, and what is slaughtered in the name of any other than Allah. But if someone is compelled by necessity—neither driven by desire nor exceeding immediate need—then surely Allah is All-Forgiving, Most Merciful.”. Meat is the source of animal protein necessary for the body to meet nutritional needs. Meat is often milled as the main ingredient for many meat-processed food products (e. g. sausages, meatballs, and corned beef). Many people who do not have a milling machine at home would go to meat kiosks in the markets that offer milling services. Quoted by Redaksikibar (2013), several meat kiosks offer milling services in Pasar Pathuk and Pasar Kranggan that violate the basic rule of providing halal comestible; separating tools used to mill pork and beef. This becomes a substantial problem because it is a transgression of someone’s belief and an</span></p>
<h2><a name="bookmark8"></a><span class="font3" style="font-weight:bold;"><a name="bookmark9"></a>MATERIALS AND METHODS</span></h2>
<p><span class="font3">This research was conducted from January – to April 2021 in the Laboratory of Biochemistry and Biotechnology, Universitas Ahmad Dahlan Yogyakarta, Indonesia. All samples were collected from</span></p>
<p><span class="font2">eISSN: 2655-9994 pISSN: 2303-3371 </span><a href="https://doi.org/10.24843/IJBB.2022.v09.i02.p03"><span class="font2">https://doi.org/10.24843/IJBB.2022.v09.i02.p03</span></a></p>
<p><span class="font3">offence to Law Number (No) 8 of 1999 on Consumer Protection.</span></p>
<p><span class="font3">Technological advances in analyzing pork contamination have increased. One of those advances is the use of Polymerase Chain Reaction (PCR) to identify the presence of pork elements in meat-processed food. Fibriana et al. (2012) conducted a study using PCR to identify pork contamination on meatballs sold on the street of Salatiga, Central Java. One of thirteen samples of this study was found contaminated by pork meat. A similar study has been conducted in Surabaya using RT-PCR. In that study, five samples out of 30 tested positive for pork contamination on milled beef (Susilowati, 2019). However, there is limited literature on molecular identification of pork meat contamination in Yogyakarta; hence this research was conducted. Fourteen samples were used in this study. Those samples were taken from Pasar Kranggan and Pasar Pathuk, Kota Yogyakarta.</span></p>
<p><span class="font3">Kranggan traditional market and Pathuk traditional market, Yogyakarta City, Indonesia. Fresh pork, fresh beef, and ddH2O were used as controls in this research. Figure 1 shows research location.</span></p>
<div><img src="https://jurnal.harianregional.com/media/87975-1.jpg" alt="" style="width:436pt;height:413pt;">
<p><span class="font3">Figure 1. Locations where the samples were taken</span></p>
</div><br clear="all">
<h2><a name="bookmark10"></a><span class="font3" style="font-weight:bold;"><a name="bookmark11"></a>Sample Preparation and DNA Extraction</span></h2>
<p><span class="font3">Samples taken from traditional markets were transferred to the laboratory using a cooler box. From each sample, as much as 30 mg were milled and transferred into a 1.5 ml microtube. 200 µl FATG1 buffer, proteinase K was added into the microtube. Samples were then vortexed. Samples were incubated at 60</span><span class="font4">°</span><span class="font3">C for 1-3 hours. After the first incubation, 200 µl of FATG2 buffer was added and samples were incubated for the second time for 10 minutes. After the second incubation, 200 µl of 96% ethanol was added and samples</span></p>
<p><span class="font3">were vortexed. FATG mini columns were transferred into a collection tube and centrifuged for 1 minute (18,000 rpm). The collection tube was discarded and replaced by a new collection tube. 400 µl of W1 buffer was added to the column. Samples were then centrifuged full-speed for another 1 minute. The collection tube was discarded and replaced by a new collection tube. 750 µl wash buffer was added into the column and was centrifuged full-speed for 1 minute. The collection tube was discarded and replaced by a new 1,5 ml microtube. 100 µl elution buffer was added to the column and the column was incubated at room</span></p>
<p><span class="font3">temperature for 3 minutes. Samples were then centrifuged for 2 minutes. Pellet formed after centrifugation was stored at 4</span><span class="font4">°</span><span class="font3">C.</span></p>
<h2><a name="bookmark12"></a><span class="font3" style="font-weight:bold;"><a name="bookmark13"></a>Qualitative and Quantitative Analysis of DNA</span></h2>
<p><span class="font3">Qualitative analysis was carried out using 1.5% agarose gel immersed in 0.5x TAE buffer solution. 3 µl of DNA sample, 2 µl loading dye, and 5 µl GelRed were used for every well in agarose gel. The electrophoresis tank was set to 100 volts and 15 minutes. Observation of agarose gel was carried out on a UV transilluminator to detect DNA bands. Quantitative analysis</span></p>
<p><span class="font3">was carried out using Nanodrop.</span></p>
<h2><a name="bookmark14"></a><span class="font3" style="font-weight:bold;"><a name="bookmark15"></a>PCR Amplification</span></h2>
<p><span class="font3">PCR was used to amplify the target gene using two pairs of primers as shown in Table 1. P14 primers design was adapted from Fibriana et al. (2012) and the MTCB design was adapted from Naidu et al. (2012). As for PCR condition is shown in Table 2. As much as 1.2 µl DNA sample, 12.5 µl </span><span class="font3" style="font-style:italic;">MasterMix</span><span class="font3">, 1 µl primer forward and 1 µl primer reversed were mixed into each of the PCR tubes. 9.3 µl ddH</span><span class="font1">2</span><span class="font3">O was added to the tubes. The tubes were then placed into a PCR machine. PCR products were then stored at -20</span><span class="font4">°</span><span class="font3">C for further use.</span></p>
<p><span class="font3">Table 1. Primers Sequence</span></p>
<p><span class="font3">Primer &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sequence</span></p>
<p><span class="font3">P14F &nbsp;&nbsp;&nbsp;5’-CCCCGTCTCCTTCCTCCGGTGGTTGATG-3’</span></p>
<p><span class="font3">P14R &nbsp;&nbsp;&nbsp;5’-CTGCGACACATGATGCCTTTATGTCCCAGC-3’</span></p>
<p><span class="font3">MTCBF &nbsp;5’-CCHCCATAAATAGGNGAAGG-3’</span></p>
<p><span class="font3">MTCBR &nbsp;5’- WAGAAYTTCAGCTTTGGG-3’</span></p>
<table border="1">
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font3">Table 2. PCR Condition</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">Condition</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">Temperature &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Duration &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cycle</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">Pre-denaturation Denaturation Annealing Extension</span></p>
<p><span class="font3">Final extension</span></p></td><td style="vertical-align:top;">
<p><span class="font3">95</span><span class="font4">°</span><span class="font3">C &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2 minutes &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-</span></p>
<p><span class="font3">95</span><span class="font4">°</span><span class="font3">C &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;45 seconds</span></p>
<p><span class="font3">55 </span><span class="font4">°</span><span class="font3">C &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 minute</span></p>
<p><span class="font3">72</span><span class="font4">°</span><span class="font3">C &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1 minute &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<sup>30</sup> cycles</span></p>
<p><span class="font3">72</span><span class="font4">°</span><span class="font3">C &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5 minutes</span></p></td></tr>
</table>
<h2><a name="bookmark16"></a><span class="font3" style="font-weight:bold;"><a name="bookmark17"></a>RESULTS AND DISCUSSION</span></h2>
<p><span class="font3">This study aims to identify pork DNA in beef (whole and milled) obtained from Pasar Pathuk and Pasar Kranggan using </span><span class="font3" style="font-style:italic;">polymerase chain reaction</span><span class="font3"> (PCR). This method has a high sensitivity to detect the smallest amount of DNA present in the samples of interest. In this study, two pairs of primers were used; MTCB </span><span class="font3" style="font-style:italic;">(mitochondrial cytochrome b)</span><span class="font3"> and P14.</span></p>
<p><span class="font3">DNA isolation was done using the FavorGen commercial kit </span><span class="font3" style="font-style:italic;">FavorGen® FavorPrep<sup>TM</sup> Tissue Genomic DNA Extraction Mini Kit</span><span class="font3">. DNA isolation consists of 3 main principles: cell destruction (lysis), separation of DNA from solid materials such as proteins and cellulose (extraction) and DNA purification (Nurhayati and Darmawati, 2017). Isolated DNA on fresh pork, fresh beef, whole and ground beef were visualized using gel electrophoresis as</span></p>
<p><span class="font3">a qualitative measure. The results of the visualization are shown in Figure 2.</span></p><img src="https://jurnal.harianregional.com/media/87975-2.jpg" alt="" style="width:427pt;height:162pt;">
<p><span class="font3">Figure 2. The result of fresh meat DNA isolation. K(+): Positive control (fresh pork); K(-): negative control (fresh beef); S1 - S7: fresh beef; S8 – S14: ground beef</span></p>
<p><span class="font3">Based on Figure 2, it is shown that the DNA quality of S8 – S14 samples was better compared to S1 – S7 samples. One factor that could explain this is the contaminants in the form of proteins or the remaining solution from the isolation kit (Kurniama et al. 2017 and Fatchiyah et al. 2011). Smears that appear on the bands are caused by at least two factors. The first factor is the remnants of solution from the isolation kit, the second factor is DNA degradation during isolation (Setiaputri et al. 2020 and Suparningtyas et al. 2018). Repeatedly washing DNA with ethanol could possibly clear the smear, therefore can be used as one method to purify DNA (Fatmawati et al. 2015). Samples were then further analyzed by Nanodrop as a quantitative measure to obtain DNA concentration; results are shown in Table 3.</span></p>
<p><span class="font3">T</span><span class="font3" style="text-decoration:underline;">able 3. Measurement of Quality and Concentration of DNA Isolat</span><span class="font3">es</span></p>
<table border="1">
<tr><td style="vertical-align:middle;">
<p><span class="font3" style="font-weight:bold;">Sample</span></p></td><td style="vertical-align:top;">
<p><span class="font3" style="font-weight:bold;">A260/A280 Ratio</span></p></td><td style="vertical-align:top;">
<p><span class="font3" style="font-weight:bold;">Concentration of DNA (ng/</span><span class="font4" style="font-weight:bold;">µ</span><span class="font3" style="font-weight:bold;">l)</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">Positive Control</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,746</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">103,07</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font3">Negative Control</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">1,993</span></p></td><td style="vertical-align:middle;">
<p><span class="font3">87,07</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S1</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">2,018</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">162,68</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S2</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,905</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">144,11</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S3</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,741</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">73,74</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S4</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,818</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">186,45</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S5</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,987</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">75,37</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S6</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,805</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">96,71</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S7</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,976</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">100,63</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S8</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,971</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">114,47</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S9</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,993</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">209,16</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S10</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,834</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">114,85</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S11</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,945</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">157,06</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S12</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,946</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">135,85</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S13</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,832</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">45,65</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3">S14</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">1,868</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3">114,90</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font3" style="font-weight:bold;">Average</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3" style="font-weight:bold;">1,807</span></p></td><td style="vertical-align:bottom;">
<p><span class="font3" style="font-weight:bold;">108,985</span></p></td></tr>
</table>
<p><span class="font3">The measurement of DNA concentration and purity is necessary to know the degree of contamination of the isolated DNA. DNA of good quality would have an A</span><span class="font1">260</span><span class="font3">/A</span><span class="font1">280 </span><span class="font3">ratio in the range of 1.8 to 2.0. A</span><span class="font1">260</span><span class="font3">/A</span><span class="font1">280 </span><span class="font3">ratio below 1.8 indicates that DNA contains protein contaminants. Failure to break down (lysis) cell components could be the cause of protein contamination (Pratama, 2015). A</span><span class="font1">260</span><span class="font3">/A</span><span class="font1">280</span></p>
<p><span class="font3">there is a certain amount of sample concentration that needs to be met to obtain high-quality amplicons (Mustafa et al., 2016 and Fatchiyah et al., 2011). The average A</span><span class="font1">260</span><span class="font3">/A</span><span class="font1">280 </span><span class="font3">ratio of 1.807 shown in Table 3 indicates that the purity of DNA isolated in this research was of good quality. Considering the good result of DNA concentration, samples were further analyzed by </span><span class="font3" style="font-style:italic;">polymerase chain reaction</span></p>
<p><span class="font3">ratio above 2.0 indicates that DNA contains</span></p>
<p><span class="font3">(PCR). Figure 3 shows PCR products using</span></p>
<p><span class="font3">remnants of the RNA.</span></p>
<p><span class="font3">MTCB primers. Based on the results, all</span></p>
<p><span class="font3">Measurement of DNA concentration</span></p>
<p><span class="font3">samples, control meat included, were all</span></p>
<div>
<p><span class="font3">is also necessary for the next step of the</span></p>
</div><br clear="all">
<div>
<p><span class="font3">successfully amplified on ~1140bp.</span></p>
<p><span class="font3">analysis, which is PCR. That is because</span></p><img src="https://jurnal.harianregional.com/media/87975-3.jpg" alt="" style="width:379pt;height:157pt;">
<p><span class="font3">Figure 3. PCR amplification results using MTCB primers electrified in 1.2% agarose gel 100 volts; M: marker 100 bp; K(+): positive control (fresh beef); K(-): negative control (ddH2O);</span></p>
<p><span class="font3">S1 - S7: fresh meat samples; S8 – S14: ground beef samples</span></p>
</div><br clear="all">
<p><span class="font3">Based on the results shown in Figure 3, it is shown that the positive control (fresh beef) was amplified and the negative control (ddH</span><span class="font1">2</span><span class="font3">O) was not amplified. This result indicates that the PCR machine worked as expected eliminating any technical error possibility. Most samples of ground beef (S1-S7) and fresh beef (S8-S14) were successfully amplified. Bands were shown to be sharp and bright indicating high specificity of primers to the target gene. However, below the bands, there were still smears. The appearance of</span></p>
<p><span class="font3">smears can be caused by the quantity of Mg++, dNTP, </span><span class="font3" style="font-style:italic;">Taq Polymerase,</span><span class="font3"> primer, and excess DNA templates. Another possibility is the inclusion of contaminants on the DNA template which inhibit the activity of Taq polymerase </span><span class="font3" style="font-style:italic;">(Fatchiyah et al., 2011).</span></p>
<p><span class="font3">All amplified samples have a length of ~1140 bp in correspondence to Naidu et.al. (2012) who explained that the target gene, </span><span class="font3" style="font-style:italic;">cytochrome</span><span class="font3"> b, has a sequence between 1140-1200 bp. It is then proven that 13 samples obtained from Pasar Pathuk and Pasar Kranggan contained meat derived</span></p>
<p><span class="font3">from mammals. There was one fresh beef sample (S5) which was not amplified using MTCB primers. Considering that S5 has a good A</span><span class="font1">260</span><span class="font3">/A</span><span class="font1">280 </span><span class="font3">ratio of 1,987 and DNA concentration of 75.37 ng/</span><span class="font4">µ</span><span class="font3">l, it is possible that this sample did not contain any mammalian meat; neither beef nor pork. Nevertheless, this is an offence on Law Number (No). 8 of 1999 on Consumer Protection, specifically on Chapter IV, Verse 8, item </span><span class="font3" style="font-style:italic;">i</span><span class="font3">: </span><span class="font3" style="font-style:italic;">“not attaching labels or making descriptions of products containing the name of the products, size, net weight/content, composition, instruction to use, date of manufacture, side effects, names and addresses of business contact persons, as well as other information for</span></p>
<h2><a name="bookmark18"></a><span class="font3" style="font-weight:bold;"><a name="bookmark19"></a>CONCLUSIONS</span></h2>
<p><span class="font3">Based on the results, it is concluded that fresh and ground beef obtained from milling services in Pasar Pathuk and Pasar Kranggan Yogyakarta is not contaminated by pork DNA. Further research is needed especially on the introduction of new samples from various locations. Moreover, investigation of numerous skin care products to uphold their halal status is also required.</span></p>
<h2><a name="bookmark20"></a><span class="font3" style="font-weight:bold;"><a name="bookmark21"></a>REFERENCES</span></h2>
<p><span class="font3">Quran Kemenag. Surah An-Nahl Verse 115 </span><a href="https://quran.kemenag.go.id/sura/16/115"><span class="font3">(</span><span class="font3" style="text-decoration:underline;">https://quran.kemenag.go.id/sura/16/</span></a><span class="font3" style="text-decoration:underline;"> </span><a href="https://quran.kemenag.go.id/sura/16/115"><span class="font3" style="text-decoration:underline;">115</span><span class="font3">)</span></a></p>
<p><span class="font3">Fatchiyah, E.L., Arumingtyas S., Widyarti, dan Rahayu, S. (2011). </span><span class="font3" style="font-style:italic;">Biologi Molekuler Prinsip Dasar Analisis</span><span class="font3">. Jakarta: Penerbit Erlangga</span></p>
<p><span class="font3">Fibriana, F., Widianti, T., Retnoningsih, A., dan Susanti. (2012). Deteksi Daging Babi pada Produk Bakso di Pusat Kota Salatiga Menggunakan Teknik </span><span class="font3" style="font-style:italic;">Polymerase &nbsp;&nbsp;&nbsp;Chain &nbsp;&nbsp;Reaction</span><span class="font3">.</span></p>
<p><span class="font3" style="font-style:italic;">Biosaintifika, 4(2), &nbsp;&nbsp;&nbsp;&nbsp;106-112.</span></p>
<p><span class="font3" style="font-style:italic;">use that according to the provision must be created.”</span></p>
<p><span class="font3">Samples were then further analyzed using PCR and P14 primers which target </span><span class="font3" style="font-style:italic;">PRE-1</span><span class="font3"> as specific molecular marker for pork. As shown in Figure 3, only positive control (pork meat) was amplified at ~481 bp while all 14 samples were not amplified using P14 primers. This result indicates that P14 primers have high specificity on detecting pork meat. However, it is shown on Figure 3 that the bands were not of good quality even on the positive control. The poor quality of DNA bands can be caused by duration and voltage implemented on electrophoresis (Mustollah, 2016) and Tilawah, 2019).</span></p>
<h2><a name="bookmark22"></a><span class="font3" style="font-weight:bold;"><a name="bookmark23"></a>ACKNOWLEDGEMENTS</span></h2>
<p><span class="font3">We would like to express our gratitude to all parties contributing to this research: Laboratory of Biochemistry and Biotechnology Universitas Ahmad Dahlan, Laboratory of Botany Universitas Ahmad Dahlan, Laboratory of Biotechnology Universitas Gadjah Mada, and all laboratory technicians involved in this research.</span></p>
<p><a href="https://doi.org/10.15294/biosaintifika.v4i2.3928"><span class="font3" style="text-decoration:underline;">https://doi.org/10.15294/biosaintifi</span></a><span class="font3" style="text-decoration:underline;"> </span><a href="https://doi.org/10.15294/biosaintifika.v4i2.3928"><span class="font3" style="text-decoration:underline;">ka.v4i2.3928</span></a></p>
<p><span class="font3">Kurniama, S.S., Mazieda, M.N., Listyorini, D., dan E. S. S. (2014). Optimization of DNA Isolation and Purification Technique From Chili Pepper (</span><span class="font3" style="font-style:italic;">Capsicum annum</span><span class="font3"> L.). Seminar Nasional XI Pendidikan Biologi FKIP UNS Biologi, Sains, Lingkungan, dan Pembelajarannya. </span><span class="font3" style="font-style:italic;">Pendidikan Biologi FKIP UNS, 11(1), 65-70</span></p>
<p><span class="font3">Mustafa, H., I, Rachmawati., dan Y, Udin.</span></p>
<p><span class="font3">(2016). Pengukuran Konsentrasi</span></p>
<p><span class="font3">dan Kemurnian DNA Genom Nyamuk </span><span class="font3" style="font-style:italic;">Anopheles barbirostris</span><span class="font3">. </span><span class="font3" style="font-style:italic;">Jurnal Vektor Penyakit, 10(1), 710.</span><a href="http://dx.doi.org/10.22435/vektorp.v10i1.6251.7-10"><span class="font3"> </span><span class="font3" style="text-decoration:underline;">10.22435/vektorp.v10i1.6251.7</span></a></p>
<p><a href="http://dx.doi.org/10.22435/vektorp.v10i1.6251.7-10"><span class="font3" style="text-decoration:underline;">-10</span></a></p>
<p><span class="font3">Mustollah, H. (2016). Analisa Profil Protein Gelatin Sapi dan Gelatin Babi </span><span class="font3" style="font-style:italic;">Gummy</span><span class="font3"> Vitamin C Menggunakan Metode SDS-Page (</span><span class="font3" style="font-style:italic;">Sodium Dodecyl Sulphate Poly Acrilamide</span><span class="font3"> Gel Elektrophoresis). &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Thesis.</span></p>
<p><a href="https://repository.uinjkt.ac.id/"><span class="font3" style="text-decoration:underline;">https://repository.uinjkt.ac.id/</span><span class="font3">.</span></a></p>
<p><span class="font3">Accessed on 18<sup>th</sup> of May 2021</span></p>
<p><span class="font3">Naidu, A., Fitak, R.R., Munguia-Vega, A., dan Culver, M. (2012). Novel Primer for Complete Mitochondrial Cytochorme </span><span class="font3" style="font-style:italic;">b</span><span class="font3"> Gene Sequencing in Mammals. </span><span class="font3" style="font-style:italic;">Molecular Ecology Resources, Volume 12, &nbsp;191-196.</span></p>
<p><span class="font3">DOI</span><a href="https://doi.org/10.1111/j.1755-0998.2011.03078.x"><span class="font3">:</span><span class="font3" style="text-decoration:underline;">10.1111/j.1755-</span></a><span class="font3" style="text-decoration:underline;"></span><a href="https://doi.org/10.1111/j.1755-0998.2011.03078.x"><span class="font3" style="text-decoration:underline;">0998.2011.03078.x</span></a></p>
<p><span class="font3">Nurhayati, B., dan Darmawati, S. (2017). </span><span class="font3" style="font-style:italic;">Biologi Sel dan Molekuler. </span><span class="font3">Kementrian Kesehatan Republik Indonesia</span></p>
<p><span class="font3">Pratama, P. (2015). Aplikas</span><span class="font3" style="font-style:italic;">i Real-Time PCR untuk Mendeteksi Bakteri Salmonella</span><span class="font3"> sp</span><span class="font3" style="font-style:italic;">. pada Hasil Perikanan</span><span class="font3">. Thesis. Bogor: Institut Pertanian Bogor</span></p>
<p><span class="font3">Redaksikibar, (2013). Pencemaran daging Haram di Yogyakarta.</span><a href="http://kibar-uk.org/2013/02/24/pencemaran-daging-haram-di-yogyakarta-bagian-1/"><span class="font3"> </span><span class="font3" style="text-decoration:underline;">http://kibar-</span></a><span class="font3" style="text-decoration:underline;">uk.org/2013/02/24/pencemaran-daging-haram-di-yogyakarta-bagian-1/</span><a href="http://kibar-uk.org/2013/02/24/pencemaran-daging-haram-di-yogyakarta-bagian-1/"><span class="font3">.</span></a><span class="font3"> Accessed on 25<sup>th</sup> of September 2020</span></p>
<p><span class="font3">Setiaputri, A.A., Barokah, G.R., Sahaba, Muh.A.B., Arbajayanti, R.D., Fabella, N., Pertiwi, R.M., Nurilmala, M., Nugraha, R., dan Abdullah, A. (2020). Perbandingan Metode Isolasi DNA Pada Produk</span></p>
<p><span class="font3">Perikanan Segar dan Olahan. </span><span class="font3" style="font-style:italic;">Jurnal Pengolahan Hasil Perikanan Indonesia, &nbsp;&nbsp;&nbsp;&nbsp;23(3), &nbsp;&nbsp;&nbsp;&nbsp;447-658.</span></p>
<p><a href="https://doi.org/10.17844/jphpi.v23i3.32314"><span class="font3" style="text-decoration:underline;">https://doi.org/10.17844/jphpi.v23i</span></a><span class="font3" style="text-decoration:underline;"> </span><a href="https://doi.org/10.17844/jphpi.v23i3.32314"><span class="font3" style="text-decoration:underline;">3.32314</span></a></p>
<p><span class="font3">Suparningtyas, J.F., Pramudyawardhani, O.D., Purwoko, D., dan Tajuddin, T. (2018). Analisis filogenetik beberapa klon karet dengan marka AFLP (</span><span class="font3" style="font-style:italic;">Amplified Fragment Length Polymorphism</span><span class="font3">). &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font3" style="font-style:italic;">Jurnal</span></p>
<p><span class="font3" style="font-style:italic;">Bioteknologi &amp;&nbsp;Biosains Indonesia, 5(1), &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;8-19.</span></p>
<p><a href="https://doi.org/10.29122/jbbi.v5i1.2544"><span class="font3" style="text-decoration:underline;">https://doi.org/10.29122/jbbi.v5i1.2</span></a><span class="font3" style="text-decoration:underline;"> </span><a href="https://doi.org/10.29122/jbbi.v5i1.2544"><span class="font3" style="text-decoration:underline;">544</span></a></p>
<p><span class="font3">Susilowati, T. (2019). Deteksi Kontaminan DNA Babi Pada Sampel Penggilingan Daging Di Pasar Surya Kota Surabaya Menggunakan </span><span class="font3" style="font-style:italic;">Real-Time Polymerase Chain Reaction</span><span class="font3">. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Thesis.</span></p>
<p><a href="https://digilib.uinsby.ac.id/"><span class="font3" style="text-decoration:underline;">https://digilib.uinsby.ac.id</span><span class="font3">.</span></a></p>
<p><span class="font3">Accessed on 20<sup>th</sup> of September 2020.</span></p>
<p><span class="font3">Tilawah, S., Sari, R., dan Apridamayanti, P. (2019). Optimasi Volume Dna Marker Dan Volume Dna Hasil Amplifikasi Gen Tetl Resistensi Antibiotik Tetrasiklin Dari Bakteri </span><span class="font3" style="font-style:italic;">Bacillus cereus</span><span class="font3"> Pada Pasien Ulkus Diabetik. </span><span class="font3" style="font-style:italic;">Jurnal Mahasiswa Farmasi Fakultas Kedokteran Untan, Volume 4. Nomor 1</span></p>
<p><span class="font3">Triasih, D., Heryanti, B.R., dan Kridasaksana, D. (2016). Kajian Tentang Perlindungan Hukum Bagi Konsumen Terhadap Produk Makanan Bersertifikat Halal. </span><span class="font3" style="font-style:italic;">Jurnal Dinamika Sosial Budaya, 18(2), 214-225.</span></p>
<p><a href="http://dx.doi.org/10.26623/jdsb.v18i2.571"><span class="font3" style="text-decoration:underline;">http://dx.doi.org/10.26623/jdsb.v18</span></a><span class="font3" style="text-decoration:underline;"> </span><a href="http://dx.doi.org/10.26623/jdsb.v18i2.571"><span class="font3">i2.571</span></a></p>
<p><span class="font0">28</span></p>