---
layout: full_article
title: "IN SILICO STUDY OF Hibiscus Sabdariffa Linn. ACTIVE COMPOUNDS IN GLP-1R: POTENTIAL AS ANTIDIABETIC DRUG"
author: "T. Andriani, N. Mawaddah, L. Erlina, R. K. Anggraeni, N. Ibrahim, M. Siagian"
categories: jchem
canonical_url: https://jurnal.harianregional.com/jchem/full-98556 
citation_abstract_html_url: "https://jurnal.harianregional.com/jchem/id-98556"
citation_pdf_url: "https://jurnal.harianregional.com/jchem/full-98556"  
comments: true
---

<p><span class="font4" style="font-weight:bold;">JURNAL KIMIA (</span><span class="font4" style="font-weight:bold;font-style:italic;">JOURNAL OF CHEMISTRY</span><span class="font4" style="font-weight:bold;">) 17 (2), JULI 2023 DOI: </span><a href="https://doi.org/10.24843/JCHEM.2023.v17.i02.p2"><span class="font4" style="font-weight:bold;">https://doi.org/10.24843/JCHEM.2023.v17.i02.p2</span></a></p>
<div>
<p><span class="font4" style="font-weight:bold;">p-ISSN 1907-9850</span></p>
<p><span class="font4" style="font-weight:bold;">e-ISSN 2599-2740</span></p>
</div><br clear="all">
<p><span class="font5" style="font-weight:bold;">IN SILICO STUDY OF </span><span class="font5" style="font-weight:bold;font-style:italic;">Hibiscus Sabdariffa</span><span class="font5" style="font-weight:bold;"> Linn. ACTIVE COMPOUNDS IN GLP-1R: POTENTIAL AS ANTIDIABETIC DRUG</span></p>
<p><span class="font5" style="font-weight:bold;">T. Andriani<sup>1</sup>, N. Mawaddah<sup>2</sup>*, L. Erlina<sup>3</sup>, R. K. Anggraeni<sup>2</sup>, N. Ibrahim<sup>1</sup>, M. Siagian<sup>1</sup></span></p>
<ul style="list-style:none;"><li>
<p><span class="font5" style="font-weight:bold;font-style:italic;"><sup>1</sup>Department of Medical Physiology and Biophysics, Faculty of Medicine, Universitas Indonesia, Jakarta 10430, Indonesia</span></p></li>
<li>
<p><span class="font5" style="font-weight:bold;font-style:italic;"><sup>2</sup>Master’s Program in Biomedical Sciences, Faculty of Medicine, Universitas Indonesia, Jakarta 10430, Indonesia <sup>3</sup>Department of Chemistry, Faculty of Medicine, Universitas Indonesia, Jakarta 10430, Indonesia</span></p></li></ul>
<p><span class="font5" style="font-weight:bold;font-style:italic;">*Email:</span><a href="mailto:nikmamawaddah28@gmail.com"><span class="font5" style="font-weight:bold;font-style:italic;"> nikmamawaddah28@gmail.com</span></a></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font5" style="font-weight:bold;"><a name="bookmark1"></a>ABSTRAK</span></h1>
<p><span class="font4">Diabetes melitus merupakan gangguan metabolisme yang ditandai dengan kondisi hiperglikemia akibat produksi insulin yang berkurang dan atau kondisi resistensi insulin. GLP-1 dapat meningkatkan sekresi, sensitivitas insulin, dan menurunkan kadar glukosa darah dengan cara mengikat reseptor GLP-1 (GLP-1R). Beberapa tanaman telah digunakan sebagai obat antidiabetes. Penelitian ini bertujuan untuk melihat bagaimana interaksi senyawa aktif dalam </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn. terhadap GLP-1R secara </span><span class="font4" style="font-style:italic;">in silico</span><span class="font4">. Penelitian dimulai dengan melakukan validasi internal reseptor GLP-1 (6ORV) kemudian dilakukan proses </span><span class="font4" style="font-style:italic;">docking</span><span class="font4"> terhadap </span><span class="font4" style="font-style:italic;">native ligand</span><span class="font4">. Hasil validasi reseptor 6ORV mempunyai nilai RMSD 2.14Å. Hasil </span><span class="font4" style="font-style:italic;">score docking</span><span class="font4"> 3 senyawa aktif </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn. terbaik adalah myricetin-3-arabinogalactoside (-10,64 kcal/mol), tetra-O-methyljeediflavanone (-9,95 kcal/mol), dan ethyl chlorogenate acid (-7,73 kcal/mol). Asam amino yang berperan terhadap afinitas pada sisi aktif ikatan ligan dengan reseptor adalah Trp 297, Trp 203, Met 204, Phe 230, Try 220, Lys 197. Temuan ini menunjukkan bahwa senyawa aktif </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn. dapat berikatan langsung dengan GLP-1R.</span></p>
<p><span class="font4" style="font-weight:bold;">Kata kunci</span><span class="font4">: Diabetes melitus, </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn., GLP-1R</span></p>
<h1><a name="bookmark2"></a><span class="font5" style="font-weight:bold;"><a name="bookmark3"></a>ABSTRACT</span></h1>
<p><span class="font4">Diabetes mellitus is a metabolic disorder characterized by hyperglycemia due to reduced insulin production and/or insulin resistance. GLP-1 can increase insulin secretion, improve insulin sensitivity, and lower the blood glucose levels by binding to the GLP-1 receptor (GLP-1R). Several plants have been used as antidiabetic drugs. This study aims to see the interactions of the active compounds in </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn. and GLP-1R </span><span class="font4" style="font-style:italic;">in silico</span><span class="font4">. The research started with internal validation of the receptor GLP-1 (6ORV), and then the native ligand was docked. The 6ORV receptor validation results have an RMSD value of 2.14Å. The results of docking scores of the 3 best active compounds </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn. are myricetin-3-arabinogalactoside (-10.64 kcal/mol), tetra-O-methyljeediflavanone (-9.95 kcal/mol), and ethyl chlorogenate acid (-7.73 kcal/mol). The amino acids that contribute to the affinity at the active site of the ligand bond with the receptor are Trp 297, Trp 203, Met 204, Phe 230, Try 220, Lys 197. These findings indicate that the active compound </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn. can bind directly to GLP-1R.</span></p>
<p><span class="font4" style="font-weight:bold;">Keywords</span><span class="font4">: Diabetes mellitus, </span><span class="font4" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font4"> Linn., GLP-1R</span></p>
<h1><a name="bookmark4"></a><span class="font5" style="font-weight:bold;"><a name="bookmark5"></a>INTRODUCTION</span></h1>
<p><span class="font5">Diabetes mellitus (DM) is a metabolic disorder characterized by hyperglycemia due to reduced insulin production, insulin resistance, or both.(Biessels &amp;&nbsp;Reagan, 2015) In 2019, the International Diabetes Federation (IDF) estimated that there were 463 million</span></p>
<p><span class="font5">individuals with DM, and this number is projected to increase to 578 million in 2030 and 700 million in 2045.(Daniels &amp;&nbsp;Mietlicki-Baase, 2019) Indonesia is included in the IDF Asia-Pacific region, with 19.5 million DM cases found in Indonesia in 2021.(International Diabetes Federation, 2021)</span></p>
<p><span class="font5">DM causes various complications, often associated with long-term damage, dysfunction, and failure of various organs.(Kodl &amp;&nbsp;Seaquist, 2008) The most common complications included persistent kidney failure (nephropathy), nerve injury (neuropathy), which increases the likelihood of ulcers and is the main cause of amputation of the lower extremities, eye disorders (retinopathy) which causes blindness, coronary heart failure, diseases involving peripheral arteries, cognitive impairment, and cerebrovascular disorders.(Saeedi et al., 2019) Some efforts are needed to treat patients appropriately in order to reduce the morbidity and mortality due to its complications.</span></p>
<p><span class="font5">Management of DM requires multidisciplinary action, which includes pharmacological and non-pharmacological therapy. The high incidence and prevalence of DM encourage researchers to improve and develop better therapy for DM patients that can reduce the risk of complications and increase the quality of life. Incretin-based therapy for DM is currently being developed.(Soelistijo, 2021) The effect of incretin is a response to release more insulin due to high postprandial blood glucose levels, which is responsible for 50-70% of total insulin secretion after oral glucose administration. The incretin effect is played by two hormones, namely GLP-1 and GIP.(Baggio &amp;&nbsp;Drucker, 2007)</span></p>
<p><span class="font5">Glucagon Like Peptide-1 (GLP-1) is one of the incretin hormones produced from the preproglucagon (PPG) gene and secreted by enteroendocrine L cells in response to the presence of glucose in the intestine.(Müller et al., 2019) GLP-1 shows various positive effects, such as increasing insulin secretion and sensitivity and lowering blood glucose levels.(Cho et al., 2014) Other beneficial effects of DM conditions include delayed gastric emptying, maintenance of pancreatic β cells (proliferation, neogenesis, antiapoptosis), inhibition of food intake and energy consumption, and function in increasing glucose uptake by muscle cells.(Fujiwara et al., 2019) Interestingly, GLP-1 also has antiinflammation and antioxidant activity, and also has a positive impact on other organs such as cardiovascular and brain, and organs affected by DM complications. GLP-1 mediates its effects on various organs by binding to the GLP-1 receptor (GLP-1R) which is expressed in its related organs. GLP-1 and GLP-1R bonds</span></p>
<p><span class="font5">will activate this receptor and produce certain cellular signaling and corresponding physiological responses.(Yildirim Simsir et al., 2018)</span></p>
<p><span class="font5">Glucagon Like Peptide-1 receptor agonist is a new class of therapy for DM. This type of drug is represented by Exenatide, Liraglutide, Exenatide LAR, Albiglutide, and Tasphoglutide. Despite its superiority that can naturally mimic human GLP-1, of course there are side effects that patients can experience due to its consumption, including causing nausea, vomiting, constipation, diarrhea, bloating, and other digestive disorders.(Trujillo et al., 2021)(Wharton et al., 2022) Besides that, these drugs are still rare and sold at relatively high prices.(Reid, 2012)</span></p>
<p><span class="font5">Currently, many studies have discussed the potential of new active compounds from natural ingredients, including those derived from plants to be developed as new drug. Indonesia is known as a country with high biodiversity, marked by the many plants that can be used in medicine, one of which is </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. or Rosella which is known to have the potential to prevent DM and its complications.(Guardiola &amp;&nbsp;Mach, 2014) </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. is rich in bioactive compounds such as anthocyanins, quercetin, other flavonoids, organic acids, and polysaccharides, which have several beneficial physiological effects, such as antioxidant, antibacterial, anti-inflammatory, anticholesterol and anti-diabetic effects.(Jabeur et al., 2017)</span></p>
<p><span class="font5">The active compounds of herbal medicines are believed to have lower side effects, high bioavailability, and low cytotoxic effects.(Umadevi et al., 2022) Candidates for a new drug in diabetes mellitus in the botanical sector must start with an </span><span class="font5" style="font-style:italic;">in silico</span><span class="font5"> study. </span><span class="font5" style="font-style:italic;">In silico </span><span class="font5">studies have been widely used to search for and design new potential drugs from natural materials and are also expected to optimize the research process and prevent losses </span><span class="font5" style="font-style:italic;">in vitro</span><span class="font5"> and </span><span class="font5" style="font-style:italic;">in vivo</span><span class="font5"> research steps in a precise, accurate, fast, and effective manner. The mechanism of antidiabetic effect of </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. could be through GLP-1 receptor activation. Therefore, this study aims to see how the interactions of the active compounds in </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. against GLP-1R </span><span class="font5" style="font-style:italic;">in silico</span><span class="font5"> are.</span></p>
<p><span class="font5" style="font-weight:bold;">MATERIALS AND METHOD</span></p>
<h1><a name="bookmark6"></a><span class="font5" style="font-weight:bold;"><a name="bookmark7"></a>Materials and Tools</span></h1>
<p><span class="font5">The materials used consisted of the three-dimensional structure of GLP-1R obtained through the Protein Data Bank website (PDB ID = 6ORV) and 39 compounds derived from the </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. plant. This research used MarvinSketch, AutoDockTools, Ligplot+, and PyMOL software. The Software was operated on Lenovo ThinkPad with i7-4600U processor, 8GB RAM, Windows 13, and 64-bit operating system.</span></p>
<h1><a name="bookmark8"></a><span class="font5" style="font-weight:bold;"><a name="bookmark9"></a>Ligands Preparation</span><span class="font5">.</span></h1>
<p><span class="font5">Ligands are the phytochemical constituents of </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. from a study by (Herranz-López et al., 2017) as many as 39 compounds were added with N2V as the control positive ligand. Two and threedimensional ligand structures were obtained from the Pubchem® database in .sdf format. Structures that were not found in the Pubchem® database were drawn manually using MarvinSketch. All structures were optimized in three dimensions and changed the format to .pdb using MarvinSketch. All these ligands were then given charge, set torsion and written as pdbqt format using AutoDockTools-1.5.6.</span></p>
<h1><a name="bookmark10"></a><span class="font5" style="font-weight:bold;"><a name="bookmark11"></a>Protein Preparation</span></h1>
<p><span class="font5">The three-dimensional structure of the GLP-1R was downloaded from the Protein Data Bank (PDB ID: 6ORV). Furthermore, the unused part including water molecules was removed, hydrogen nonpolar was added, gasteiger charged was given, and Grid box size and grid center coordinate were arranged. A grid box of 40 x 50 x 40 Å was generated around the center of the active site residues of receptor. This structure was then converted to pdbqt format. All these steps were conducted using AutoDockTools-1.5.6.</span></p>
<h1><a name="bookmark12"></a><span class="font5" style="font-weight:bold;"><a name="bookmark13"></a>Validation of Molecular Docking Protocol</span></h1>
<p><span class="font5">The main objective of molecular docking is to identify energetically favorable binding modes of ligands into the target receptor’s binding site.(Ece, 2018) Validity is determined by redocking the native ligand to a prepared target protein. The method is said to be valid if the Root Mean Square Deviation (RMSD) value obtained is ≤ 2Å.(Puratchikody et al., 2016) To ensure that the tested ligand binds in the ideal position for each ligand, the</span></p>
<p><span class="font5">binding site orientation is carried out using the blind docking method, and the results of all tested ligands show cavity with the highest affinity equal to the comparative ligands. Docking search parameter is used by the Lamarckian genetic algorithm with the number of genetic algorithms being 10 runs, population size is 150, the maximum number of energy evaluating is medium which is 2.500.000, the maximum number of generations is 27.000, spacing default is 0.375 Å, and the default docking parameter is used for running options.</span></p>
<p><span class="font5">The primary parameters used in the docking process are the free energy of binding (∆G), the inhibition constant (K</span><span class="font3">i</span><span class="font5">), amino acid residues, and the number of hydrogen bonds.(Kim et al., 2016) ∆G and K</span><span class="font3">i </span><span class="font5">scores determine ligand affinity to the receptor. The more negative the ∆G and the lower the K</span><span class="font3">i </span><span class="font5">scores indicate higher ligand affinity toward the active site of the receptor. All tested ligands are then compared with the result of control positive ligand to determine the potency of both tested ligands.(Natesan et al., 2012) The amino acid residues of ligands are then compared with amino acid residues of control positive ligand to assess the similarity of interaction between ligands and control positive ligand. More similar amino acid residues </span><span class="font6">indicate a </span><span class="font5">higher probability that the ligand will have a similar activity to the control positive ligand.(Pena Neshich et al., 2015) Visualization of twodimensional and three-dimensional ligandreceptor interactions was performed using Ligplot+ and PyMOL software.</span></p>
<h1><a name="bookmark14"></a><span class="font5" style="font-weight:bold;"><a name="bookmark15"></a>RESULTS AND DISCUSSION</span></h1>
<p><span class="font5">Molecular docking is a computational research method that aims to estimate the interaction and affinity of a ligand with a macromolecule (usually an enzyme or protein). A ligand and protein molecule are predicted by placement techniques in a specific area (an active site) to provide optimal results (Motiejunas &amp;&nbsp;Wade, 2006). The molecular docking method uses the AutoDock 4.2.6 software application. Docking validation is used successively for the x center (125.194), y center (109.599), and z center (81.738) with an RMSD value obtained from the method validation results of 2.14Å. Molecular docking results show an interaction between the active compounds </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. which</span></p>
<p><span class="font5">interact with GLP-1R and have an activating &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;interacting with GLP-1R are described in the</span></p>
<p><span class="font5">function (as an activator). The active &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;table below.</span></p>
<p><span class="font5">compounds </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn.</span></p>
<p><span class="font5" style="font-weight:bold;">Table 1. </span><span class="font5">The Results Of The Comparison Between ΔG And Ki Values From The Docking Of The Active Compounds </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. With The GLP-1R</span></p>
<table border="1">
<tr><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">No</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">Compound &nbsp;Name</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4" style="font-weight:bold;">Binding Energy (kcal/mol)</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">Inhibition Constant</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">1</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">1-O-caffeoylquinic acid</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-5.80</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">55.63 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">2</span></p></td><td style="vertical-align:top;">
<p><span class="font4">2-O-caffeoylhydroxycitric acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-4.19</span></p></td><td style="vertical-align:top;">
<p><span class="font4">847.27 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">2-O-trans-Feruloylhydroxitric acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-7.08</span></p></td><td style="vertical-align:top;">
<p><span class="font4">6.41 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">4</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3-Methylquercetin</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.83</span></p></td><td style="vertical-align:top;">
<p><span class="font4">52.93 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">5</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Coumaroylquinic acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.65</span></p></td><td style="vertical-align:top;">
<p><span class="font4">72.28 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">6</span></p></td><td style="vertical-align:top;">
<p><span class="font4">5-O-Caffeoylshikimic acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.58</span></p></td><td style="vertical-align:top;">
<p><span class="font4">81.63 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">7</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Caffeoylglucose</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-3.18</span></p></td><td style="vertical-align:top;">
<p><span class="font4">4.65 mM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">8</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Chlorogenic acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-4.88</span></p></td><td style="vertical-align:top;">
<p><span class="font4">263.31 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">9</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Chlorogenoquinone</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.99</span></p></td><td style="vertical-align:top;">
<p><span class="font4">40.85 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">10</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Cryptochlorogenic acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-4.60</span></p></td><td style="vertical-align:top;">
<p><span class="font4">424.20 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">11</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Cyanidine 3-sambubioside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.42</span></p></td><td style="vertical-align:top;">
<p><span class="font4">106.68 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">12</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Delphinidin 3-O-beta-D-sambubioside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.57</span></p></td><td style="vertical-align:top;">
<p><span class="font4">83.26 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">13</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Delphinidin 3-sambubioside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.43</span></p></td><td style="vertical-align:top;">
<p><span class="font4">104.28 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">14</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">Ethyl chlorogenate acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">-7.73</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">2.16 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">15</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Hibiscus acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-2.40</span></p></td><td style="vertical-align:top;">
<p><span class="font4">17.48 mM</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">16</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Hibiscus acid dimethylester</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">-3.60</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">2.29 mM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">17</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Hydroxycitric acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-2.61</span></p></td><td style="vertical-align:top;">
<p><span class="font4">12.19 mM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">18</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Kaempferol</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.74</span></p></td><td style="vertical-align:top;">
<p><span class="font4">61.85 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">19</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Kaempferol-3-Glucuronide</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.91</span></p></td><td style="vertical-align:top;">
<p><span class="font4">46.21 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">20</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Kaempferol-3-O-sambubioside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.91</span></p></td><td style="vertical-align:top;">
<p><span class="font4">46.65 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">21</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Kaempferol-3-O-rutinoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-6.98</span></p></td><td style="vertical-align:top;">
<p><span class="font4">7.62 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">22</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Leucoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.55</span></p></td><td style="vertical-align:top;">
<p><span class="font4">86.03 uM</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">23</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Methyl digallate-para</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-7.13</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">5.98 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">24</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Methyl diga &nbsp;llate-meta</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-7.30</span></p></td><td style="vertical-align:top;">
<p><span class="font4">4.48 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">25</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Methyl-epigallocatechin</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-7.65</span></p></td><td style="vertical-align:top;">
<p><span class="font4">2.47 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">26</span></p></td><td style="vertical-align:top;">
<p><span class="font4">N-Feruloyltyramine</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-6.47</span></p></td><td style="vertical-align:top;">
<p><span class="font4">18.04 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">27</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Myricetin</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.25</span></p></td><td style="vertical-align:top;">
<p><span class="font4">142.32 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">28</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">Myricetin-3-arabinogalactoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">-10.64</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">15.82 nM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">29</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Myricetin-3-glucoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.34</span></p></td><td style="vertical-align:top;">
<p><span class="font4">122.59 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">30</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Neochlorogenic acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.38</span></p></td><td style="vertical-align:top;">
<p><span class="font4">112.93 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">31</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Prodelphinidin-B3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.81</span></p></td><td style="vertical-align:top;">
<p><span class="font4">55.54 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">32</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Quercetin</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.62</span></p></td><td style="vertical-align:top;">
<p><span class="font4">75.38 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">33</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Quercetin 3-7-diglucuronide</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.75</span></p></td><td style="vertical-align:top;">
<p><span class="font4">60.49 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">34</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Quercetin 3-O-glucuronide</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-6.13</span></p></td><td style="vertical-align:top;">
<p><span class="font4">31.95 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">35</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Quercetin 3-rutinoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-7.12</span></p></td><td style="vertical-align:top;">
<p><span class="font4">6.09 uM</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">36</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Quercetin 3-sambubioside</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">-5.11</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">178.74 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">37</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Quercetin-3-Glucoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-5.53</span></p></td><td style="vertical-align:top;">
<p><span class="font4">89.01 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">38</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">Tetra-O-methyljeediflavanone</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">-9.95</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">50.48 nM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">39</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Hibiscus acid hydroxyethylesther</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-4.41</span></p></td><td style="vertical-align:top;">
<p><span class="font4">582.73 uM</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">40</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">Native Ligand</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">-12.02</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">1.53 M</span></p></td></tr>
</table>
<p><span class="font5">Bond-free energy is a vital aspect for assessing the interaction between the ligand and the receptor. This energy indicates the bond strength (affinity) between the tested ligand and the target protein. The more negative the value of the bond energy obtained, the more stable the complex formed is between GLP-1R (6ORV) and the active compound derived from </span><span class="font5" style="font-style:italic;">Hibiscus</span></p>
<p><span class="font5" style="font-style:italic;">sabdariffa</span><span class="font5"> Linn. Increasingly negative values indicate that the compound has an interaction in the form of increasing attraction between atoms. In contrast, the repulsive forces between particles become minimal so that the conformation of the compound obtained is more stable (Manna et al., 2017). The low bond energy value also indicates that less energy is</span></p>
<p><span class="font5">required to carry out the binding or interaction of a ligand with a receptor. Meanwhile, the inhibition constant is directly proportional to the number of doses needed to cause pharmacological activity. The smaller the inhibition constant, the smaller the required dose is (Vargas et al., 2018). Several factors affect the binding affinity between the ligand and the receptor. Electrostatic, van der Waals, and hydrophobic interactions are important factors besides other factors such as entropy, desolvation, flexibility of the receptor structure, and the structure of the water molecule at the bond site (Haider, 2010).</span></p>
<p><span class="font5">In this study, three active compounds were found and had the best docking results, namely ethyl chlorogenate acid with ∆G (-7.73 kcal/mol) and Ki (2.16 uM), myricetin-3-arabinogalactoside with ∆G (-10.64 kcal/mol) and Ki (15.82 nM), and tetra-O-methyljeediflavanone with ∆G (-9.95 kcal/mol) and Ki (50.48 nM). These three compounds have the closest free bond energy to the positive control ligand value and have the highest binding affinity among the other ligands. The analysis results also obtained amino acids which are thought to play a role in the affinity of the active compounds of </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa </span><span class="font5">Linn. to GLP-1R, as seen in Table 2.</span></p>
<p><span class="font5">The target binding sites of this protein are located at Trp 297, Trp 203, Met 204, Ala 200, Leu 201, Leu 217, Phe 230, Try145, Try148, Try220, Lys 197. The essential amino acid compounds contained in the positive control ligand, namely Myricetin-3-arabinogalactoside, Tetra-O-methyljeediflavanone, and Ethyl chlorogenate acid, which are the three active compounds with the best docking, </span><span class="font6">and they </span><span class="font5">are Trp 297, Trp 203, Met 204, Phe 230, Try 220, Lys 197. Even though it differs from the position of the positive control ligand hydrogen bond (N2V), the presence of hydrogen bonds in the active compound with the best docking value has a vital role in the affinity between the compound and the target protein because it forms electrostatic interactions (donor and acceptor hydrogen) (Muttaqin, 2019). The amino acid residue plays a role in activating GLP-1R to carry out its function. The determined distance between the bonds indicates that the active compound has bonded with amino acid residues. In general, bonds occur at a distance of about 5 Ǻ and not below 1 Ǻ because there will be repulsions between atoms so that it does not meet the general bonding requirements (Faridah et al., 2021).</span></p>
<table border="1">
<tr><td colspan="5" style="vertical-align:top;">
<p><span class="font5" style="font-weight:bold;">Table 2. </span><span class="font5">Comparison Between Amino Acid Residues And The Number Of Hydrogen Bonds</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Amino Acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Native Ligand</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Myricetin-3-arabinogalactoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Tetra-O-methyljeediflavanone</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Ethyl chlorogenate acid</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Trp 297</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Trp 203</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Met 204</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Ala 200</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Leu 201</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Leu 217</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Phe 230</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Try 145</span></p></td><td style="vertical-align:top;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">(H) 3.29 Å</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:top;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Try 148</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Try 220</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">(H)</span></p>
<p><span class="font4">3.09 Å</span></p></td><td style="vertical-align:top;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:top;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Lys 197</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">(H) 2.65 Å</span></p></td><td style="vertical-align:top;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">(H) 3.26 Å</span></p></td><td style="vertical-align:top;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Cys 226</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:bottom;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:bottom;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Cys 296</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Val 229</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Met 233</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">√</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font1" style="font-weight:bold;font-style:italic;">4</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Asp 198</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font0" style="font-weight:bold;">4</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Val 194</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">-</span></p></td></tr>
</table>
<p><span class="font5">The visualization of interactions between the top three ligands and GLP-1R used Ligplot and PyMOL. The comparison between the molecular interactions of the top three</span></p>
<p><span class="font5">ligands and the interactions of the positive control ligand molecules towards the same receptor is shown in Figures 1 and 2.</span></p>
<div><img src="https://jurnal.harianregional.com/media/98556-1.jpg" alt="" style="width:156pt;height:198pt;">
</div><br clear="all">
<div>
<p><span class="font6">(b)</span></p><img src="https://jurnal.harianregional.com/media/98556-2.jpg" alt="" style="width:190pt;height:201pt;">
</div><br clear="all">
<div>
<p><span class="font6">(c)</span></p><img src="https://jurnal.harianregional.com/media/98556-3.jpg" alt="" style="width:180pt;height:198pt;">
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/98556-4.jpg" alt="" style="width:168pt;height:217pt;">
<p><span class="font5" style="font-weight:bold;">Figure 1. </span><span class="font5">Results Of 2D Molecular Docking Visualization Of The Top Three Active Compounds And GLP-1R (a) Myricetin-3 arabinogalactoside And GLP-1R; (b) Tetra-O-methyljeediflavanone And GLP-1R; (c) Ethylchlorogenate acid And GLP-1R; (d) Native Ligand And GLP-1R</span></p>
</div><br clear="all">
<div>
<p><span class="font6">(a)</span></p><img src="https://jurnal.harianregional.com/media/98556-5.jpg" alt="" style="width:167pt;height:164pt;">
<p><span class="font6">(b)</span></p>
</div><br clear="all">
<div><img src="https://jurnal.harianregional.com/media/98556-6.jpg" alt="" style="width:172pt;height:147pt;">
</div><br clear="all">
<div>
<p><span class="font6">(c)</span></p><img src="https://jurnal.harianregional.com/media/98556-7.jpg" alt="" style="width:167pt;height:177pt;">
<p><span class="font5" style="font-weight:bold;">Figure 2. </span><span class="font5">Results Of 3D Molecular Docking Visualization Of The Top Three Active Compounds And GLP-1R (a) Myricetin-3 arabinogalactoside And GLP-1R; (b) Tetra-O-methyljeediflavanone And GLP-1R; (c) Ethylchlorogenate acid And GLP-1R; (d) Native Ligand And GLP-1R</span></p>
</div><br clear="all">
<div>
<p><span class="font6">(d)</span></p><img src="https://jurnal.harianregional.com/media/98556-8.jpg" alt="" style="width:171pt;height:175pt;">
</div><br clear="all">
<p><span class="font5">The compounds with the top three docking results were then examined for their physicochemical properties to see the potential of the compounds when developed into medicinal preparations. The physicochemical parameters of the tested compounds were adjusted according to the Lipinski rule. There are five rules: (1) if the test compound has a molecular weight of more than 500 Da, it will be difficult for the compound to penetrate the cell membrane; (2) a log P value greater than 5 indicates that the compound is increasingly lipophilic and binds very tightly to the membrane, making it challenging to identify targets and is also toxic; (3) However, if the log</span></p>
<p><span class="font5">P value is too small or negative, it is also not very good because it will be difficult for the compound to penetrate the lipid bilayer membrane; (4) Hydrogen bond donors and acceptors are the magnitudes of the hydrogen bond capacity; (5) If the hydrogen capacity is higher, the energy required in the absorption process is also higher. In general, the Lipinski rule of five (RO5) indicates the solubility of certain compounds in penetrating the membrane by passive diffusion (Brito, 2011). The test compounds are said to meet the requirements for the formation of oral preparations if there is no more than one violation of the Lipinski rules (Lipinski, 2004).</span></p>
<div>
<p><span class="font5" style="font-weight:bold;">Table 3. </span><span class="font5">Results Of The Lipinski Rule Of Five</span></p>
<table border="1">
<tr><td style="vertical-align:middle;">
<p><span class="font4">Ligan</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Molecular Weight (g/mol)</span></p></td><td style="vertical-align:top;">
<p><span class="font4">HydrogenB ond Acceptor</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Hydrogen Bond</span></p>
<p><span class="font4">Donor</span></p></td><td style="vertical-align:top;">
<p><span class="font4">LogP</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Description</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Myricetin-3-arabinogalactoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">612.49</span></p></td><td style="vertical-align:top;">
<p><span class="font4">17</span></p></td><td style="vertical-align:top;">
<p><span class="font4">11</span></p></td><td style="vertical-align:top;">
<p><span class="font4">-1.91</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Did not meet Lipinski RO5 requirements</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Tetra-O-methyljeediflavanone</span></p></td><td style="vertical-align:top;">
<p><span class="font4">614.60</span></p></td><td style="vertical-align:top;">
<p><span class="font4">11</span></p></td><td style="vertical-align:top;">
<p><span class="font4">3</span></p></td><td style="vertical-align:top;">
<p><span class="font4">4.31</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Did not meet Lipinski RO5 requirements</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font4">Ethyl chlorogenate acid</span></p></td><td style="vertical-align:top;">
<p><span class="font4">382.36</span></p></td><td style="vertical-align:top;">
<p><span class="font4">9</span></p></td><td style="vertical-align:top;">
<p><span class="font4">5</span></p></td><td style="vertical-align:top;">
<p><span class="font4">0.21</span></p></td><td style="vertical-align:bottom;">
<p><span class="font4">Met Lipinski RO5 requirements</span></p></td></tr>
</table>
</div><br clear="all">
<p><span class="font5">Based on Table 3, of all the top active compounds analyzed, there was one compound that met the Lipinski RO5 requirements and met the requirements for making oral preparations. Further research is needed to determine the pharmacokinetic profile of these active</span></p>
<p><span class="font5">compounds. Then the active compound must have a good ADME profile and be non-toxic, and the top three test compounds will be evaluated for their toxicity through the AdmetSAR website (Cheng et al., 2012).</span></p>
<p><span class="font5" style="font-weight:bold;">Table 4. </span><span class="font5">Toxicity Test Results</span></p>
<table border="1">
<tr><td rowspan="2" style="vertical-align:top;">
<p><span class="font4">Ligand</span></p></td><td colspan="3" style="vertical-align:middle;">
<p><span class="font4">Toxicities</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">AMES</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">Carcinogenesis</span></p></td><td style="vertical-align:middle;">
<p><span class="font4" style="font-weight:bold;">AOT</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Myricetin-3-arabinogalactoside</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">III</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Tetra-O-methyljeediflavanone</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">III</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Ethyl chlorogenate acid</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">-</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">III</span></p></td></tr>
</table>
<table border="1">
<tr><td style="vertical-align:top;"></td><td style="vertical-align:top;">
<p><span class="font5" style="font-weight:bold;">Table 5. </span><span class="font5">Ligand Pharmacokinetic Observations</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Ligand</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Pharmacokinetics</span></p>
<p><span class="font4" style="font-weight:bold;">Absorption &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Inhibitor CYP</span></p>
<p><span class="font4" style="font-weight:bold;">GI &nbsp;&nbsp;&nbsp;&nbsp;CYP1A2 &nbsp;&nbsp;CYP2CI9 &nbsp;CYP2C9 &nbsp;&nbsp;CYP2D6 &nbsp;&nbsp;CYP3A4</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Myricetin-3-arabinogalactoside</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Low &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font4">Tetra-O-</span></p>
<p><span class="font4">methyljeediflavanone</span></p></td><td style="vertical-align:top;">
<p><span class="font4">Low &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Yes &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font4">Ethyl chlorogenate acid</span></p></td><td style="vertical-align:middle;">
<p><span class="font4">Low &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No</span></p></td></tr>
</table>
<p><span class="font5">According to the results of the analysis, the top three active compounds are not carcinogenic, non-toxic, and included in the AOT III category, which means they are slightly toxic if ingested orally. Then the three observed ligands show that these ligands have low gastrointestinal absorption. Gastrointestinal absorption can be an indicator of whether oral administration of the compound will be effective or not. Interactions of the compounds with cytochrome P450 (CYP1A2, CYP2C19, CYP2C9, CYP2D6, CYP3A4) were identified. These iso-enzymes play a major role in drug elimination through metabolic biotransformation. If this enzyme is inhibited,</span></p>
<p><span class="font5">the drug or its metabolites may accumulate in the body, causing toxic effects (Yang et al., 2017). Myricetin-3-arabinogalactoside and Ethyl chlorogenate acid are predicted to be noninhibitors for one of the isoenzymes. However, Tetra-O-methyljeediflavanone turned out to be a CYP2C9 inhibitor.</span></p>
<p><span class="font5">This study shows the potential of the active compound </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. to interact with GLP-1R. This interaction might cause activation of the GLP-1R and initiate signal transduction and produce associated physiological responses. For example, pancreatic β cells activates several signals that have an impact on insulin secretion, increasing</span></p>
<p><span class="font5">proliferation, and preventing apoptosis of pancreatic β cells (Ma et al., 2014)(Kartinah et al., 2019). </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. has ability to maintain circulating GLP-1 levels to reach its target in the pancreas (Domínguez Avila et al., 2017). This </span><span class="font5" style="font-style:italic;">in silico</span><span class="font5"> study showed an additional beneficial effect of </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. whose active compounds can also act as a ligand independent variable which is predicted to have antidiabetic activity by binding to GLP-1R.</span></p>
<h1><a name="bookmark16"></a><span class="font5" style="font-weight:bold;"><a name="bookmark17"></a>CONCLUSION</span></h1>
<p><span class="font5">Based on the docking results of 39 test compounds on </span><span class="font5" style="font-style:italic;">Hibiscus sabdariffa</span><span class="font5"> Linn. three candidate compounds with the best binding affinity with GLP-1R were obtained, namely myricetin-3-arabinogalactoside, &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;tetra-O-</span></p>
<p><span class="font5">methyljeediflavanone, and ethyl chlorogenic acid. Based on visualization of the interaction of active compounds in GLP-1R, it can be concluded that amino acids play a role in the active site of the receptor, namely Trp 297, Trp 203, Met 204, Phe 230, Try 220, and Lys 197. However, the results of this study must be validated further through both </span><span class="font5" style="font-style:italic;">in vitro</span><span class="font5"> and </span><span class="font5" style="font-style:italic;">in vivo</span><span class="font5"> studies to elucidate the potential activity through GLP-1 and GLP-1R which produce the expected responses in diabetes mellitus management.</span></p>
<h1><a name="bookmark18"></a><span class="font5" style="font-weight:bold;"><a name="bookmark19"></a>REFERENCE</span></h1>
<p><span class="font5">Baggio, L. L., &amp;&nbsp;Drucker, D. J. 2007. Biology of Incretins: &nbsp;&nbsp;GLP-1 and GIP.</span></p>
<p><span class="font5" style="font-style:italic;">Gastroenterology</span><span class="font5">. &nbsp;132(6): &nbsp;2131–2157.</span></p>
<p><a href="https://doi.org/10.1053/j.gastro.2007.03.0"><span class="font5">https://doi.org/10.1053/j.gastro.2007.03.0</span></a><span class="font5"> 54</span></p>
<p><span class="font5">Biessels, G. J., &nbsp;&amp;&nbsp;Reagan, L. P. 2015.</span></p>
<p><span class="font5">Hippocampal insulin resistance and cognitive dysfunction. </span><span class="font5" style="font-style:italic;">Nature Reviews Neuroscience</span><span class="font5">. &nbsp;&nbsp;&nbsp;&nbsp;16(11): &nbsp;&nbsp;&nbsp;&nbsp;660–671.</span></p>
<p><a href="https://doi.org/10.1038/nrn4019"><span class="font5">https://doi.org/10.1038/nrn4019</span></a></p>
<p><span class="font5">Brito, M. A. de. 2011. Pharmacokinetic study with computational tools in the medicinal chemistry course. </span><span class="font5" style="font-style:italic;">Brazilian Journal of Pharmaceutical Sciences</span><span class="font5">. 47(4): 797–</span></p>
<p><span class="font5">805. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><a href="https://doi.org/10.1590/S1984-"><span class="font5">https://doi.org/10.1590/S1984-</span></a></p>
<p><span class="font5">82502011000400017</span></p>
<p><span class="font5">Cheng, F., Li, W., Zhou, Y., Shen, J., Wu, Z., Liu, G., Lee, P. W., &amp;&nbsp;Tang, Y. 2012. </span><span class="font5" style="font-style:italic;">admetSAR: A Comprehensive Source and Free Tool for Assessment of Chemical ADMET &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Properties</span><span class="font5">.</span></p>
<p><a href="https://doi.org/10.1021/ci300367a"><span class="font5">https://doi.org/10.1021/ci300367a</span></a></p>
<p><span class="font5">Cho, Y. M., Fujita, Y., &amp;&nbsp;Kieffer, T. J. 2014. Glucagon-Like Peptide-1: &nbsp;&nbsp;&nbsp;Glucose</span></p>
<p><span class="font5">Homeostasis and Beyond. </span><span class="font5" style="font-style:italic;">Annual Review of &nbsp;&nbsp;Physiology</span><span class="font5">. &nbsp;&nbsp;&nbsp;76(1): &nbsp;&nbsp;&nbsp;535–559.</span></p>
<p><a href="https://doi.org/10.1146/annurev-physiol-021113-170315"><span class="font5">https://doi.org/10.1146/annurev-physiol-021113-170315</span></a></p>
<p><span class="font5">Daniels, D., &amp;&nbsp;Mietlicki-Baase, E. G. 2019. Glucagon-like peptide 1 in the brain: Where is it coming from, where is it going? </span><span class="font5" style="font-style:italic;">Diabetes</span><span class="font5">. &nbsp;&nbsp;&nbsp;68(1): &nbsp;&nbsp;&nbsp;15–17.</span></p>
<p><a href="https://doi.org/10.2337/dbi18-0045"><span class="font5">https://doi.org/10.2337/dbi18-0045</span></a></p>
<p><span class="font5">Domínguez Avila, J. A., Rodrigo García, J., González Aguilar, G. A., &amp;&nbsp;de la Rosa, L. A. 2017. The Antidiabetic Mechanisms of Polyphenols Related to Increased Glucagon-Like Peptide-1 (GLP1) and Insulin Signaling. </span><span class="font5" style="font-style:italic;">Molecules</span><span class="font5">. 22(6): 1–16. </span><a href="https://doi.org/10.3390/molecules220609"><span class="font5">https://doi.org/10.3390/molecules220609</span></a><span class="font5"> 03</span></p>
<p><span class="font5">Ece, A. 2018. E-Pharmacophore Mapping Combined with Virtual Screening and Molecular Docking to Identify Potent and Selective Inhibitors of P90 Ribosomal S6 Kinase (RSK). </span><span class="font5" style="font-style:italic;">Turkish &nbsp;Journal of</span></p>
<p><span class="font5" style="font-style:italic;">Pharmaceutical Sciences</span><span class="font5">. 13(2): 241– 248. </span><a href="https://doi.org/10.4274/tjps.28290"><span class="font5">https://doi.org/10.4274/tjps.28290</span></a></p>
<p><span class="font5">Faridah, F., Sumaryono, W., Simanjuntak, P., &amp;&nbsp;Triwibowo, R. R. 2021. Analysis of Pancreatic Lipase Inhibitor Activity of Chlorogenic Acid Derivatives in Green Coffee Beans as Antiobesity using In Silico. </span><span class="font5" style="font-style:italic;">Jurnal Ilmu Kefarmasian Indonesia</span><span class="font5">. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;19(1): &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;125.</span></p>
<p><a href="https://doi.org/10.35814/jifi.v19i1.946"><span class="font5">https://doi.org/10.35814/jifi.v19i1.946</span></a></p>
<p><span class="font5">Fujiwara, Y., Eguchi, S., Murayama, H., Takahashi, Y., Toda, M., Imai, K., &amp;&nbsp;Tsuda, K. 2019. Relationship between diet/exercise and pharmacotherapy to enhance the GLP</span><span class="font2">‐</span><span class="font5">1 levels in type 2 diabetes. </span><span class="font5" style="font-style:italic;">Endocrinology, Diabetes &amp;&nbsp;Metabolism</span><span class="font5">. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2(3): &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1–14.</span></p>
<p><a href="https://doi.org/10.1002/edm2.68"><span class="font5">https://doi.org/10.1002/edm2.68</span></a></p>
<p><span class="font5">Guardiola, S., &amp;&nbsp;Mach, N. 2014. Therapeutic potential of Hibiscus sabdariffa: A review of the scientific evidence. </span><span class="font5" style="font-style:italic;">Endocrinología y Nutrición (English Edition)</span><span class="font5">. 61(5), 274– 295.</span></p>
<p><a href="https://doi.org/10.1016/j.endoen.2014.04"><span class="font5">https://doi.org/10.1016/j.endoen.2014.04</span></a><span class="font5">. 003</span></p>
<p><span class="font5">Haider, M. K. 2010. Computational Analysis of Protein-Ligand Interaction. In </span><span class="font5" style="font-style:italic;">Yuki Gosei Kagaku Kyokaishi/Journal of Synthetic Organic Chemistry</span><span class="font5">. University of York. </span><a href="https://doi.org/10.5059/yukigoseikyokais"><span class="font5">https://doi.org/10.5059/yukigoseikyokais</span></a><span class="font5"> hi.54.427</span></p>
<p><span class="font5">Herranz-López, M., Olivares-Vicente, M., Encinar, J., Barrajón-Catalán, E., Segura-Carretero, A., Joven, J., &amp;&nbsp;Micol, V. 2017. Multi-Targeted Molecular Effects of Hibiscus sabdariffa Polyphenols: An Opportunity for a Global Approach to Obesity. </span><span class="font5" style="font-style:italic;">Nutrients</span><span class="font5">. &nbsp;&nbsp;&nbsp;&nbsp;9(8): &nbsp;&nbsp;&nbsp;&nbsp;907.</span></p>
<p><a href="https://doi.org/10.3390/nu9080907"><span class="font5">https://doi.org/10.3390/nu9080907</span></a></p>
<p><span class="font5">International Diabetes Federation. 2021. International Diabetes Federation. In </span><span class="font5" style="font-style:italic;">Diabetes Research and Clinical Practice </span><span class="font5">(10th ed., Vol. 102, Issue 2). </span><a href="https://doi.org/10.1016/j.diabres.2013.10"><span class="font5">https://doi.org/10.1016/j.diabres.2013.10</span></a><span class="font5">. 013</span></p>
<p><span class="font5">Jabeur, I., Pereira, E., Barros, L., Calhelha, R. C., Soković, M., Oliveira, M. B. P. P., &amp;&nbsp;Ferreira, I. C. F. R. 2017. Hibiscus sabdariffa L. as a source of nutrients, bioactive compounds and colouring agents. </span><span class="font5" style="font-style:italic;">Food Research International</span><span class="font5">. 100: 717–723.</span></p>
<p><a href="https://doi.org/10.1016/j.foodres.2017.07"><span class="font5">https://doi.org/10.1016/j.foodres.2017.07</span></a><span class="font5">. 073</span></p>
<p><span class="font5">Kartinah, N. T., Fadilah, F., Ibrahim, E. I., &amp;&nbsp;Suryati, Y. 2019. The Potential of Hibiscus sabdariffa Linn in Inducing Glucagon-Like Peptide-1 via SGLT-1 and GLPR in DM Rats. </span><span class="font5" style="font-style:italic;">BioMed Research International</span><span class="font5">. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2019: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1–8.</span></p>
<p><a href="https://doi.org/10.1155/2019/8724824"><span class="font5">https://doi.org/10.1155/2019/8724824</span></a></p>
<p><span class="font5">Kim, J., Kwon, J., Kim, M., Do, J., Lee, D., &amp;&nbsp;Han, H. 2016. Computational protein– ligand docking and virtual drug screening with the &nbsp;AutoDock &nbsp;suite. </span><span class="font5" style="font-style:italic;">Polymer</span></p>
<p><span class="font5" style="font-style:italic;">Journal</span><span class="font5">. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;48(7): &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;829–834.</span></p>
<p><a href="https://doi.org/10.1038/pj.2016.37"><span class="font5">https://doi.org/10.1038/pj.2016.37</span></a></p>
<p><span class="font5">Kodl, C. T., &amp;&nbsp;Seaquist, E. R. 2008. Cognitive Dysfunction and Diabetes &nbsp;Mellitus.</span></p>
<p><span class="font5" style="font-style:italic;">Endocrine Reviews</span><span class="font5">. &nbsp;29(4): &nbsp;494–511.</span></p>
<p><a href="https://doi.org/10.1210/er.2007-0034"><span class="font5">https://doi.org/10.1210/er.2007-0034</span></a></p>
<p><span class="font5">Lipinski, C. A. 2004. Lead- and drug-like compounds: The rule-of-five revolution. </span><span class="font5" style="font-style:italic;">Drug Discovery Today: Technologies</span><span class="font5">. 1(4): &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;337–341.</span></p>
<p><a href="https://doi.org/10.1016/j.ddtec.2004.11.0"><span class="font5">https://doi.org/10.1016/j.ddtec.2004.11.0</span></a><span class="font5"> 07</span></p>
<p><span class="font5">Ma, X., Guan, Y., &amp;&nbsp;Hua, X. 2014. Glucagon-like peptide 1-potentiated insulin secretion and proliferation of pancreatic β-cells. </span><span class="font5" style="font-style:italic;">Journal of Diabetes</span><span class="font5">. &nbsp;6(5): &nbsp;394–402.</span></p>
<p><a href="https://doi.org/10.1111/1753-0407.12161"><span class="font5">https://doi.org/10.1111/1753-0407.12161</span></a></p>
<p><span class="font5">Manna, A., Laksitorini, M. D., Hudiyanti, D., &amp;&nbsp;Siahaan, P. 2017. Molecular Docking of Interaction between E-Cadherin Protein and Conformational Structure of Cyclic</span></p>
<p><span class="font5">Peptide ADTC3 (Ac-CADTPC-NH2) Simulated on 20 ns. </span><span class="font5" style="font-style:italic;">Jurnal Kimia Sains Dan &nbsp;&nbsp;&nbsp;Aplikasi</span><span class="font5">,. &nbsp;&nbsp;&nbsp;&nbsp;20(1): &nbsp;&nbsp;&nbsp;&nbsp;30–36.</span></p>
<p><a href="https://doi.org/10.14710/jksa.20.1.30-36"><span class="font5">https://doi.org/10.14710/jksa.20.1.30-36</span></a></p>
<p><span class="font5">Motiejunas, D., &amp;&nbsp;Wade, R. C. 2006. Structural, energetic, and dynamic aspects of ligandreceptor interactions. </span><span class="font5" style="font-style:italic;">Comprehensive Medicinal Chemistry II</span><span class="font5">. &nbsp;4: &nbsp;193–212.</span></p>
<p><a href="https://doi.org/10.1016/b0-08-045044-x/00250-9"><span class="font5">https://doi.org/10.1016/b0-08-045044-x/00250-9</span></a></p>
<p><span class="font5">Müller, T. D., Finan, B., Bloom, S. R., Alessio, D. D., Drucker, D. J., Flatt, P. R., &amp;&nbsp;Fritsche, A. 2019. Glucagon-like peptide 1 (GLP-1). </span><span class="font5" style="font-style:italic;">Molecular Metabolism</span><span class="font5">. 30(September): &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;72–130.</span></p>
<p><a href="https://doi.org/10.1016/j.molmet.2019.09"><span class="font5">https://doi.org/10.1016/j.molmet.2019.09</span></a><span class="font5">. 010</span></p>
<p><span class="font5">Muttaqin, F. Z. 2019. Molecular Docking and Molecular Dynamic Studies of Stilbene Derivative Compounds As &nbsp;Sirtuin-3</span></p>
<p><span class="font5">(Sirt3) Histone Deacetylase Inhibitor on Melanoma &nbsp;Skin Cancer and Their</span></p>
<p><span class="font5">Toxicities &nbsp;&nbsp;Prediction. &nbsp;&nbsp;</span><span class="font5" style="font-style:italic;">Journal of</span></p>
<p><span class="font5" style="font-style:italic;">Pharmacopolium</span><span class="font5">. &nbsp;&nbsp;&nbsp;2(2): &nbsp;&nbsp;&nbsp;112–121.</span></p>
<p><a href="https://doi.org/10.36465/jop.v2i2.489"><span class="font5">https://doi.org/10.36465/jop.v2i2.489</span></a></p>
<p><span class="font5">Natesan, S., Subramaniam, R., Bergeron, C., &amp;&nbsp;Balaz, S. 2012. Binding affinity prediction for ligands and receptors forming tautomers and ionization species: Inhibition of mitogen-activated protein kinase-activated protein kinase 2 (MK2). </span><span class="font5" style="font-style:italic;">Journal of Medicinal Chemistry</span><span class="font5">. 55(5): 2035–2047.</span></p>
<p><a href="https://doi.org/10.1021/jm201217q"><span class="font5">https://doi.org/10.1021/jm201217q</span></a></p>
<p><span class="font5">Pena Neshich, I., Nishimura, L., de Moraes, F., Salim, J., Villalta-Romero, F., Borro, L., Yano, I., Mazoni, I., Tasic, L., Jardine, J., &amp;&nbsp;Neshich, G. 2015. Computational Biology Tools for Identifying Specific Ligand Binding Residues for Novel Agrochemical and Drug Design. </span><span class="font5" style="font-style:italic;">Current Protein &amp;&nbsp;Peptide Science</span><span class="font5">. 16(8): 701– 717.</span></p>
<p><a href="https://doi.org/10.2174/13892037166661"><span class="font5">https://doi.org/10.2174/13892037166661</span></a><span class="font5"> 50505234923</span></p>
<p><span class="font5">Puratchikody, A., Sriram, D., Umamaheswari, A., &amp;&nbsp;Irfan, N. 2016. 3 </span><span class="font2">‐</span><span class="font5"> D structural interactions and quantitative structural toxicity studies of tyrosine derivatives intended for safe potent inflammation treatment. </span><span class="font5" style="font-style:italic;">Chemistry Central Journal</span><span class="font5">. 1– 19. &nbsp;</span><a href="https://doi.org/10.1186/s13065-016-"><span class="font5">https://doi.org/10.1186/s13065-016-</span></a></p>
<p><span class="font5">0169-9</span></p>
<p><span class="font5">Reid, T. 2012. </span><span class="font5" style="font-style:italic;">Choosing GLP-1 Receptor Agonists or DPP-4 Inhibitors: Weighing</span></p>
<p><span class="font5" style="font-style:italic;">the Clinical Trial Evidence</span><span class="font5">. 30(1): 3–12. </span><a href="https://diabetesjournals.org/clinical/articl"><span class="font5">https://diabetesjournals.org/clinical/articl</span></a><span class="font5"> e/30/1/3/35430/Choosing-GLP-1-Receptor-Agonists-or-DPP-4</span></p>
<p><span class="font5">Saeedi, P., Petersohn, I., Salpea, P., Malanda, B., Karuranga, S., Unwin, N., Colagiuri, S., Guariguata, L., Motala, A. A., Ogurtsova, K., Shaw, J. E., Bright, D., &amp;&nbsp;Williams, R. 2019. Global and regional diabetes prevalence estimates for 2019 and projections for 2030 and 2045: Results from the International Diabetes Federation Diabetes Atlas, 9th edition. </span><span class="font5" style="font-style:italic;">Diabetes Research and Clinical Practice</span><span class="font5">. </span><span class="font5" style="font-style:italic;">157</span><span class="font5">: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;107843.</span></p>
<p><a href="https://doi.org/10.1016/j.diabres.2019.10"><span class="font5">https://doi.org/10.1016/j.diabres.2019.10</span></a><span class="font5"> 7843</span></p>
<p><span class="font5">Soelistijo, S. 2021. Pedoman Pengelolaan dan Pencegahan Diabetes Melitus Tipe 2 Dewasa di Indonesia 2021. </span><span class="font5" style="font-style:italic;">Global Initiative &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;for &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Asthma</span><span class="font5">, &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;46.</span></p>
<p><a href="http://www.ginasthma.org"><span class="font5">www.ginasthma.org</span></a><span class="font5">.</span></p>
<p><span class="font5">Trujillo, J. M., Nuffer, W., &amp;&nbsp;Smith, B. A. 2021. </span><span class="font5" style="font-style:italic;">GLP-1 receptor agonists: an updated review of head-to-head clinical studies</span><span class="font5">. 1– 15.</span></p>
<p><a href="https://doi.org/10.1177/20420188219973"><span class="font5">https://doi.org/10.1177/20420188219973</span></a><span class="font5"> 20</span></p>
<p><span class="font5">Umadevi, P., Manivannan, S., Fayad, A. M., &amp;&nbsp;Shelvy, S. 2022. In silico analysis of phytochemicals as potential inhibitors of proteases involved in SARS-CoV-2 infection. </span><span class="font5" style="font-style:italic;">Journal of Biomolecular Structure and Dynamics</span><span class="font5">. 40(11): 5053– 5059.</span></p>
<p><a href="https://doi.org/10.1080/07391102.2020.1"><span class="font5">https://doi.org/10.1080/07391102.2020.1</span></a><span class="font5"> 866669</span></p>
<p><span class="font5">Vargas, J. A. R., Lopez, A. G., Piñol, M. C., &amp;&nbsp;Froeyen, M. 2018. Molecular docking study on the interaction between 2-substituted-4,5-difuryl Imidazoles with different protein target for antileishmanial activity. &nbsp;&nbsp;&nbsp;&nbsp;</span><span class="font5" style="font-style:italic;">Journal &nbsp;&nbsp;&nbsp;&nbsp;of Applied</span></p>
<p><span class="font5" style="font-style:italic;">Pharmaceutical Science</span><span class="font5">. &nbsp;8(3): &nbsp;14–22.</span></p>
<p><a href="https://doi.org/10.7324/JAPS.2018.8303"><span class="font5">https://doi.org/10.7324/JAPS.2018.8303</span></a></p>
<p><span class="font5">Wharton, S., Davies, M., Dicker, D., Lingvay, I., Mosenzon, O., Rubino, D. M., &amp;</span></p>
<p><span class="font5">Pedersen, S. D. 2022. Managing the gastrointestinal side effects of GLP-1 receptor agonists in obesity: recommendations for clinical practice. </span><span class="font5" style="font-style:italic;">Postgraduate Medicine</span><span class="font5">. 134(1): 14–19.</span></p>
<p><a href="https://doi.org/10.1080/00325481.2021.2"><span class="font5">https://doi.org/10.1080/00325481.2021.2</span></a><span class="font5"> 002616</span></p>
<p><span class="font5">Yang, H., Chaofeng, L., Lixia, S., Jie, L., Yingchun, C., Zhuang, W., Weihua, L., Guixia, L., &amp;&nbsp;Yun, T. 2017. AdmetSAR 2.0: &nbsp;web-service for prediction and</span></p>
<p><span class="font5">optimization of chemical ADMET properties. </span><span class="font5" style="font-style:italic;">Bioinformatics</span><span class="font5">. 33(16),: 1–7. </span><a href="https://doi.org/10.1093/bioinformatics/bt"><span class="font5">https://doi.org/10.1093/bioinformatics/bt</span></a><span class="font5"> y707/5085368</span></p>
<p><span class="font5">Yildirim Simsir, I., Soyaltin, U. E., &nbsp;&amp;</span></p>
<p><span class="font5">Cetinkalp, S. 2018. Glucagon like peptide-1 &nbsp;(GLP-1) likes Alzheimer’s disease.</span></p>
<p><span class="font5" style="font-style:italic;">Diabetes and Metabolic &nbsp;Synd</span><span class="font5">rome:</span></p>
<p><span class="font5">Clinical Research and Reviews. 12(3): 469–475.</span></p>
<p><a href="https://doi.org/10.1016/j.dsx.2018.03.002"><span class="font5">https://doi.org/10.1016/j.dsx.2018.03.002</span></a></p>
<p><span class="font5">128</span></p>