---
layout: full_article
title: "THE EFFECT OF THE D185 MUTATION ON THE STABILITY AND FUNCTIONALITY OF RUBISCO LIKE PROTEIN (RLP) FROM CHROMOHALOBACTER SALEXIGEN BKL 5"
author: "I. Sudarmanto, M. S. Rohman, W. T. Artama"
categories: jchem
canonical_url: https://jurnal.harianregional.com/jchem/full-108563 
citation_abstract_html_url: "https://jurnal.harianregional.com/jchem/id-108563"
citation_pdf_url: "https://jurnal.harianregional.com/jchem/full-108563"  
comments: true
---

<p><span class="font4" style="font-weight:bold;">JURNAL KIMIA (</span><span class="font4" style="font-weight:bold;font-style:italic;">JOURNAL OF CHEMISTRY</span><span class="font4" style="font-weight:bold;">) 18 (1), JANUARI 2024</span></p>
<p><span class="font4" style="font-weight:bold;">DOI: </span><a href="https://doi.org/10.24843/JCHEM.2024.v18.i01.p01"><span class="font4" style="font-weight:bold;">https://doi.org/10.24843/JCHEM.2024.v18.i01.p01</span></a></p>
<p><span class="font4" style="font-weight:bold;">p-ISSN 1907-9850</span></p>
<p><span class="font4" style="font-weight:bold;">e-ISSN 2599-2740</span></p>
<p><span class="font5" style="font-weight:bold;">EFFECT OF THE D185 MUTATION ON THE STABILITY AND FUNCTIONALITY OF RUBISCO LIKE PROTEIN (RLP) FROM </span><span class="font5" style="font-weight:bold;font-style:italic;">CHROMOHALOBACTER SALEXIGEN</span><span class="font5" style="font-weight:bold;"> BKL 5</span></p>
<p><span class="font5" style="font-weight:bold;">I. Sudarmanto <sup>1,2</sup>, M. S. Rohman<sup>2,3*</sup>, W. T. Artama<sup>2</sup></span><span class="font5"><sup>,</sup></span><span class="font5" style="font-weight:bold;"><sup>4</sup></span></p>
<ul style="list-style:none;"><li>
<p><span class="font5" style="font-weight:bold;font-style:italic;"><sup>1</sup>Chemistry Study Program, Department of Sciences, Institute Technology of Sumatera, Jl Terusan Ryacudu, WayHui, Lampung Selatan, 35365, Lampung</span></p></li>
<li>
<p><span class="font5" style="font-weight:bold;font-style:italic;"><sup>2</sup>Post Graduate Program of Biotechnology, Post Graduate Program, University of Gadjah Mada, Jl</span></p></li></ul>
<p><span class="font5" style="font-weight:bold;font-style:italic;">Teknika Utara, Catur Tunggal, Sleman, 55281, Yogyakarta</span></p>
<ul style="list-style:none;"><li>
<p><span class="font5" style="font-weight:bold;font-style:italic;"><sup>3</sup>Department of Agricultural Microbiology, Agriculture Faculty, University of Gadjah Mada, Karang Malang, Catur Tunggal, Sleman, 55281, Yogyakarta</span></p></li>
<li>
<p><span class="font5" style="font-weight:bold;font-style:italic;"><sup>4</sup>Veterinary Medicine Faculty, University of Gadjah Mada, Jl Fauna No 2, Karang Gayam, Catur</span></p></li></ul>
<p><span class="font5" style="font-weight:bold;font-style:italic;">Tunggal, Sleman, 55281, Yogyakarta</span></p>
<p><span class="font5" style="font-weight:bold;font-style:italic;">*Email:</span><a href="mailto:saifur@ugm.ac.id"><span class="font5" style="font-weight:bold;font-style:italic;"> saifur@ugm.ac.id</span></a></p><a name="caption1"></a>
<h1><a name="bookmark0"></a><span class="font5" style="font-weight:bold;"><a name="bookmark1"></a>ABSTRACT</span></h1>
<p><span class="font4">In the case of Rubisco Like Protein (RLP) </span><span class="font4" style="font-style:italic;">Chromohalobacter salexigen</span><span class="font4"> BKL 5 (RLP CS), which is halophilic, the presence of salt bridges (SB) is not too numerous but efficient enough to maintain stability in high salt stress. This study was to determine the effect of mutations SB at position 185 (D185/WT) on the stability and activity of RLP CS through an in-silico approach by replacing aspartic acid with glutamic acid and alanine (D185E and D185A). The methods used were Molecular Dynamics Simulations (MDS) and Molecular Docking (MD). The MDS was used to study molecular characteristics at the atomic level, while MD was for the interaction patterns of proteins and ligands. The results of the MDS analysis carried out at 10ns showed that the mutation at position 185 with alanine and glutamic acid changed the size/dimensional of RLP CS and affected its overall geometric structure. Interestingly, even though the structure has changed, the activity of the protein remains relatively constant, which is indicated by the results of MD, and has a relatively similar binding energy value of around -6.3 Kcal/mol.</span></p>
<p><span class="font4" style="font-weight:bold;">Keywords</span><span class="font4">: </span><span class="font4" style="font-style:italic;">Chromohalobacter salexigen</span><span class="font4"> BKL 5, molecular docking, molecular dynamic simulation, mutation, saltbridge</span></p>
<h1><a name="bookmark2"></a><span class="font5" style="font-weight:bold;"><a name="bookmark3"></a>INTRODUCTION</span></h1>
<p><span class="font5">Saltbridge (SB) is defined as the electrostatic interaction between two oppositely charged groups: anionic carboxylates from glutamic acid (E) or aspartic acid (D), and cationic ammonium from arginine (R) or lysine (K) (Bosshard et al., 2004; Musafia et al., 1995). Salt bridge interactions in proteins can provide structural and functional conformation and contribute to protein recognition, degradation, catalysis, and stability. (Albeck et al., 2000; Kumar &amp;&nbsp;Nussinov, 2002).</span></p>
<p><span class="font5">Mutation analysis is a useful procedure for estimating the contribution of several factors to the conformational stability of a protein. The effects of mutations on protein stability differ from site to site, depending on their environment within the structure (Takano et al., 1995; Takano et al., 2000). One approach that is often used to predict the effect of mutations on proteins is MDS. Molecular dynamics (MD) simulations can predict how individual atoms in</span></p>
<p><span class="font5">a protein or other molecular system will move over time, based on general models of physics governing interatomic interactions (Karplus and McCammon, 2002; Rácz et al., 2022). These simulations can capture a wide variety of important biomolecular processes, including conformational changes, ligand binding, and protein folding, revealing the positions of all atoms at femtosecond temporal resolution. Such simulations can also predict how biomolecules will respond at the atomic level to perturbations such as mutations, phosphorylation, protonation, or the addition or removal of ligands.</span></p>
<p><span class="font5">Simulations can also predict the behavior of proteins and other biomolecules in full atomic detail and at very fine temporal resolution. Substantial improvements in the speed, accuracy, and accessibility of simulations, together with the proliferation of experimental structural data, have increased the attractiveness of biomolecular simulations for experimentalists. Simulations have proven</span></p>
<p><span class="font5">valuable in deciphering the functional mechanisms of proteins and other biomolecules in uncovering the structural basis of disease and in the design and optimization of small molecules, peptides and proteins (Hollingsworth &amp;&nbsp;Dror, 2018).</span></p>
<p><span class="font5">The results of previous observations showed that the saltbridge in halophilic RLP only has half of its homologous RLP saltbridge which is mesophilic (unpublished). A more detailed observation of the RLP CS saltbridge shows that the saltbridge at position D185 is unique because it has the most interactions and is closest to the active site of the protein which is thought to have a major contribution in maintaining the stability (conformation) of RLP CS to function optimally. This study was conducted to determine how much influence the saltbridge has on the function and stability of RLP cs. Tests were carried out through the process of mutation of aspartic acid with other residues (glutamic acid and alanine) and analyzed using a computational approach, namely Molecular dynamic simulation and molecular docking methods. It is hoped that the results of this study can provide an overview of how the saltbridge influences the stability and activity of RLP CS.</span></p>
<p><span class="font5" style="font-weight:bold;">MATERIAL AND METHODS</span></p>
<h1><a name="bookmark4"></a><span class="font5" style="font-weight:bold;"><a name="bookmark5"></a>Material and Tool</span></h1>
<p><span class="font5">The 3D structure of RLP CS in the form of PDB files, called as Wild Type (WT). Protein sequences in FASTA format containing mutants at position 185, hereinafter referred to as D185E (asp mutation to glutamic acid) and D185A (aspartic acid mutation to alanine). The computational analysis was operated on ACER Nitro with i7 processor, 32 GB RAM, Windows 10, GPU GTX 1060 6 GB and 64-bit operating system</span></p>
<h1><a name="bookmark6"></a><span class="font5" style="font-weight:bold;"><a name="bookmark7"></a>Method</span></h1>
<p><span class="font5">The first step is identification and visualization of the areas/residues of amino acids to be mutated using the Chimera tool, followed by the following steps.</span></p>
<p><span class="font5" style="font-weight:bold;font-style:italic;">Homologation of mutational 3D structures</span></p>
<p><span class="font5">Mutations were carried out on the aspartic acid amino acid residue at position 185 (D185/WT) by replacing it with glutamic acid (D185E) and alanine (D185A). The selection of these two types of residues was based on</span></p>
<p><span class="font5">previous study which showed that RLP CS has significant differences in glutamic acid and alanine compared to its non-halophilic homologs (unpublished). Sequences containing mutant residues were homologated with AlphaFold to obtain a 3D structure in the form of PDB files (Miyazono &amp;&nbsp;Tanokura, 2022). The selected structures were validated with ERRAT and SAVE (Messaoudi et al., 2013).</span></p>
<p><span class="font5">The results of the homologation were analyzed for energy in the ground state with the SPDV Tool (Guex &amp;&nbsp;Peitsch, 1997) then superimposed with RLP WT to find out the difference in geometry in the form of RMSD using Chimera (Pettersen et al., 2004).</span></p>
<p><span class="font5" style="font-weight:bold;font-style:italic;">Molecular Dynamic Simulation</span></p>
<p><span class="font5">Molecular dynamics simulations can be used as a procedure to study molecular characteristics at the atomic level, such as protein conformation changes that occur. In this study protein molecular dynamics (MDS) simulations were carried out using the Gromacs-based method performed on the WebGro server (Oostenbrink et al., 2004; Abraham et al., 2015).</span></p>
<p><span class="font5">Proteins were prepared using the GROMOS96 43a1 force field. SPC was chosen as a model solvent (a triclinic water box with a size of 50 × 75 × 70 Å) for protein complexes. This system is neutralized by adding sodium or chlorine ions based on the total charge. To minimize the system before MD, the steepest descent (5000 steps) algorithm was applied. MD simulations were carried out in the presence of 0.15 M NaCl using constant temperature (300 K) and pressure (1.0 bar). The approximate number of frames per simulation is 1000. The simulation time is set to 10 ns. The parameters analyzed were RMSD, RMSF, Rg, SASA and the number of hydrogen bonds formed. Graph preparation and visualization using Originlab 2019 and GraPhadPrism 8.</span></p>
<p><span class="font5" style="font-weight:bold;font-style:italic;">Molecular Docking</span></p>
<p><span class="font5">To predict the interaction patterns of proteins and ligands, a molecular docking process was carried out. In this study the method used was Autodock Vina and the MGL Tool. The ligand used was RUBP whose structure was obtained from a web-based portal (Pubchem) in SDF format. The protein used is the PDB structure of the RLP WT file and the mutants prepared using the MGL Tool (Morris et al., 2009). All polar hydrogens were removed</span></p>
<p><span class="font5">from the ligand and a Gasteiger type charge was added. Thereafter the receptor files were prepared by adding hydrogen and a Kollman charge. Then all files are converted to pdbqt format for docking. The grid is arranged according to the structure in such a way as to cover the active site area of the RLP protein. After preparation of the ligand and receptor, docking was carried out using Autodock Vina (Walker &amp;&nbsp;Causgrove, 2009). The best docking position (the most negative binding energy value) was chosen for visualization. Visualization of docking results using LigPlot.</span></p>
<p><span class="font5" style="font-weight:bold;">RESULT AND DISCUSSION</span></p>
<h1><a name="bookmark8"></a><span class="font5" style="font-weight:bold;"><a name="bookmark9"></a>Visualization of SB at position 185 and Homologation of Mutant Structures</span></h1>
<p><span class="font5">Previous studies revealed that the aspartic acid (asp/D) residue at position 185 is unique because it has interactions with several other residues to produce a relatively large amount of saltbridge (SB) compared to other residues that make up SB. Visualization of SB</span></p>
<p><span class="font5">at position 185 is shown in </span><span class="font5" style="font-weight:bold;">Figure 1</span><span class="font5">. In the figure it can be seen that aspartic acid at position 185 binds to arginine (R) at 146,221 and 394 which forms a salt bridge which is important to maintain conformation, especially the active site area.</span></p>
<p><span class="font5">The results of the homologation with AlphaFold which have been validated (data not shown) are then analyzed for energy and compared with WT which shows that the two mutants are geometrically different (</span><span class="font5" style="font-weight:bold;">Table 1</span><span class="font5">). This is consistent with initial predictions that changing or even removing the saltbridge will affect the stability of the RLP CS (WT) protein, considering that the saltbridge is very important to maintain the structure of a protein. The small RMSD value of the superimposed results on the two mutants indicates that the two mutants are geometrically smaller than the WT structure of RLP CS and this is reinforced by the energy value which also changes. This indicates that mutations in the saltbridge area will reduce stability.</span></p>
<div><img src="https://jurnal.harianregional.com/media/108563-1.jpg" alt="" style="width:348pt;height:174pt;">
<p><span class="font4" style="font-weight:bold;">Figure 1. </span><span class="font4">Position of Aspartic Acid at Number 185 (A) and the Salt-bridge Interaction Formed (B)</span></p>
</div><br clear="all">
<div>
<p><span class="font3" style="font-weight:bold;">Table 1. </span><span class="font3">Characteristics of WT and Mutation Results</span></p>
</div><br clear="all">
<div>
<table border="1">
<tr><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">Protein &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RMSD</span></p></td><td style="vertical-align:top;">
<p><span class="font6">Δ</span><span class="font2" style="font-weight:bold;">G &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Binding affinity &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Volume active site</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font2" style="font-weight:bold;">Compare to WT</span></p></td><td style="vertical-align:bottom;">
<p><span class="font2" style="font-weight:bold;">KJ/mol &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;KCal/mol &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(SA) </span><span class="font4" style="font-weight:bold;">Å</span><span class="font2" style="font-weight:bold;"><sup>3</sup></span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2" style="font-weight:bold;">WT &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0</span></p></td><td style="vertical-align:middle;">
<p><span class="font2" style="font-weight:bold;">22.444.898 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-6,3 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;376.486</span></p></td></tr>
<tr><td style="vertical-align:middle;">
<p><span class="font2" style="font-weight:bold;">D185E &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0,403</span></p></td><td style="vertical-align:middle;">
<p><span class="font2" style="font-weight:bold;">20.364.660 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-6,2 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;364.192</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">D185A &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;0,685</span></p></td><td style="vertical-align:top;">
<p><span class="font2" style="font-weight:bold;">20.182.055 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-6,3 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;372.971</span></p></td></tr>
</table>
</div><br clear="all">
<h1><a name="bookmark10"></a><span class="font5" style="font-weight:bold;"><a name="bookmark11"></a>Molecular Dynamics</span></h1>
<p><span class="font5">RMSD is a parameter commonly used to determine the stability of a system (protein). RMSD can describe the conformational and geometric changes of a protein. The greater the RMSD value, the more unstable the protein will be and vice versa. The MDS analysis shows that the RMSD value of the mutant is greater than that of WT which illustrates that the conformation of the mutant protein changes and becomes unstable compared to WT (</span><span class="font5" style="font-weight:bold;">Figure 2b</span><span class="font5">). This happens because there is no salt bridge that allows the protein to lose its stability. Although the D185E mutant still has a saltbridge, its RMSD value is still greater than that of WT, indicating that aspartic acid is the best residue that can maintain the stability of the RLP protein in a halophilic state.</span></p>
<p><span class="font5">The conformational geometry of WT and mutant proteins can be observed by studying the radius of gyration (Rg), SASA, and hydrogen bond analysis (Shukla et al., 2018). The Rg of a protein describes the size of the protein globally, the higher the value, the more open the protein tends to be and vice versa. A small Rg value indicates that the protein is in a compact state while a large Rg value indicates that the protein has a smaller folding structure. In this study, both mutants showed smaller Rg values than their WT, especially the D185A mutant (</span><span class="font5" style="font-weight:bold;">Figure 2d</span><span class="font5">). This shows that D185A has smaller dimensions than WT and tends to be more folded than WT and D185E. This fact was further confirmed by observing the SASA parameters and the number of hydrogen bonds formed.</span></p>
<p><span class="font5">SASA is defined as protein accessibility to water (solvent). Lower SASA value determines a compact structure and vice versa. SASA can also describe folding patterns in proteins, where a small SASA value indicates that there is high folding. In this case, the WT and mutant proteins show different SASA patterns, the D185A area has the smallest value compared to D185E and WT. (</span><span class="font5" style="font-weight:bold;">Figure 2c)</span><span class="font5">. This occurs because the aspartic acid mutation with alanine (D185A) will remove the saltbridge so that the protein structure will fold in response to</span></p>
<p><span class="font5">hydrophobic amino acids in protein to water (solvent). At D185E the SASA value will also decrease because even though the saltbridge is still formed, the glutamic acid side chain is longer so that it will affect the distance the saltbridge is formed (longer).</span></p>
<p><span class="font5">The number of hydrogen bonds in a protein describes the atomic level interactions. It also validates the compactness or shape of the structure. The compact structure of proteins is generally advantageous for intramolecular hydrogen bond interactions. In order to predict how the mutation affects the structural stability, the intramolecular hydrogen bonds of WT and the mutant are shown in </span><span class="font5" style="font-weight:bold;">Figure 3</span><span class="font5">. It was observed that the WT protein and the mutant protein exhibited different numbers of hydrogen bonds. In general, D185A has fewer hydrogen bonds than WT and D185E. This strengthens the notion that the structure of D185A is more compact because fewer interactions involving hydrogen bonds are formed in its molecule.</span></p>
<p><span class="font5">The results of the comparison of Rg and SASA and the number of hydrogen bonds in the molecule against WT and mutants strengthen the notion that the mutation in D185 affects the conformation and geometry of RLP globally.</span></p>
<p><span class="font5">RMSF is a parameter that can describe the mobility/flexibility of individual amino acid residues. Rigid structures such as helices and sheets will have a high RMSF value while loops and turns will have a low RMSF value. Calculation of the RMSF value in the RLP WT and mutants shows that the RMSF value of the mutant has a lower value compared to the RLP WT (</span><span class="font5" style="font-weight:bold;">Figure 2a</span><span class="font5">). This fact strengthens the notion that mutant RLP has higher cohesiveness than WT which causes the amino acid residues to be more bound. Observation of the residues in the active site area (190-320) shows that the residue in that area has almost no difference between WT and its mutants which indicates that the active site area is conserved and stable.</span></p>
<div><img src="https://jurnal.harianregional.com/media/108563-2.jpg" alt="" style="width:406pt;height:467pt;">
</div><br clear="all">
<h1><a name="bookmark12"></a><span class="font5" style="font-weight:bold;"><a name="bookmark13"></a>Molecular Docking</span></h1>
<p><span class="font5">Molecular docking is performed to predict changes in active site residues after mutation by observing the interactions between the ligand and amino acid residues in the active site area and measuring the binding energy formed (Meng et al., 2011; Al-Karmalawy et al., 2021). The ligand used was Ribulose 1.5 Biphosphate as a substrate for the Rubsico Like Protein from </span><span class="font5" style="font-style:italic;">Chromohalobacter salexigen</span><span class="font5"> BKL 5 (RLP CS) protein analogue.</span></p>
<p><span class="font5">The docking results showed that there was no change in the interaction between the ligand and the active site of the protein which was characterized by similar binding energy values (</span><span class="font5" style="font-weight:bold;">Table 1</span><span class="font5">). This shows that the mutation</span></p>
<p><span class="font5">does not cause changes in protein activity. This fact occurs because the active site region which is responsible for protein activity has not changed which is confirmed by the results of analysis using CastP 3.0 (Tian et al., 2018) which indicates that the active site volume is not significantly different (</span><span class="font5" style="font-weight:bold;">Table 1</span><span class="font5">) which is also validated by the results of the analysis of the RMSF value showed that there was no change in the RMSF value of WT and mutants in the amino acid residues located at positions 190320 (area in the square line) (</span><span class="font5" style="font-weight:bold;">Figure 2a</span><span class="font5">). The interaction of the ligand and active site protein in WT (</span><span class="font5" style="font-weight:bold;">Figure 4) </span><span class="font5">used as example of docking illustration.</span></p>
<div>
<p><span class="font0" style="font-weight:bold;">NE</span></p><img src="https://jurnal.harianregional.com/media/108563-3.jpg" alt="" style="width:250pt;height:215pt;">
<p><span class="font1" style="font-weight:bold;">Figure 4. </span><span class="font1">RUBP Ligand Interactions on The Active Site of RLP CS. Visualization Using LigPIot</span></p>
</div><br clear="all">
<h1><a name="bookmark14"></a><span class="font5" style="font-weight:bold;"><a name="bookmark15"></a>CONCLUSION</span></h1>
<p><span class="font5">The aspartic acid (asp) mutation at position 185 which is responsible for the formation of saltbridge (SB) with alanine (ala) and glutamic acid (glu) changes the structure of the RLP CS globally as indicated by a change in Rg value of mutant. Interestingly, although the global structure of the mutation results changes, the volume of the active region tends to remain the same and the interaction of the ligand-amino acid residues at the active site does not change as indicated by the binding energy values of the RLP WT and RMSF values of the MDS results of the two mutants which are similar. Aspartic acid mutation with alanine causes RLP CS to lose SB at position 185 which causes decreased stability (rising RMSD) and tends to fold more (low SASA value), this is consistent with the theory that SB is important for maintaining protein structure/conformation and stability.</span></p>
<p><span class="font5" style="font-weight:bold;">ABBREVIATION</span></p>
<table border="1">
<tr><td style="vertical-align:top;">
<p><span class="font5">CS</span></p></td><td style="vertical-align:top;">
<p><span class="font5">: &nbsp;</span><span class="font5" style="font-style:italic;">Chromohalobacter salexigen</span></p>
<p><span class="font5">BKL 5</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font5">MD</span></p></td><td style="vertical-align:bottom;">
<p><span class="font5">: &nbsp;Molecular Docking</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font5">MDS</span></p></td><td style="vertical-align:bottom;">
<p><span class="font5">: &nbsp;Molecular Dynamic Simulation</span></p></td></tr>
<tr><td style="vertical-align:top;">
<p><span class="font5">RLP</span></p></td><td style="vertical-align:top;">
<p><span class="font5">: &nbsp;Rubisco Like Protein</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font5">RMSD</span></p></td><td style="vertical-align:bottom;">
<p><span class="font5">: &nbsp;Root Mean Square Deviation</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font5">RMSF</span></p></td><td style="vertical-align:bottom;">
<p><span class="font5">: &nbsp;Root Mean Square of Fluctuation</span></p></td></tr>
<tr><td style="vertical-align:bottom;">
<p><span class="font5">Rg</span></p></td><td style="vertical-align:bottom;">
<p><span class="font5">: &nbsp;Radius of Gyration</span></p></td></tr>
</table>
<p><span class="font5">RUBP &nbsp;: &nbsp;Ribulose Biposphate</span></p>
<p><span class="font5">SASA &nbsp;&nbsp;: &nbsp;Solvent Accessibilty Surface Area</span></p>
<p><span class="font5">SB &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: &nbsp;Saltbridge</span></p>
<p><span class="font5">WT &nbsp;&nbsp;&nbsp;: &nbsp;Wild Type</span></p>
<h1><a name="bookmark16"></a><span class="font5" style="font-weight:bold;"><a name="bookmark17"></a>ACKNOLEDGEMENT</span></h1>
<p><span class="font5">We thank the Institute Technology of Sumatera for awarding the scholarship and Hibah Bersaing Pasca Sarjana UGM for funding this study.</span></p>
<h1><a name="bookmark18"></a><span class="font5" style="font-weight:bold;"><a name="bookmark19"></a>REFERENCE</span></h1>
<p><span class="font5">Abraham, M.J., Murtola, T., Schulz, R., Páll, S., Smith, J.C., Hess, B., Lindahl, E. 2015. GROMACS: &nbsp;&nbsp;High Performance</span></p>
<p><span class="font5">Molecular Simulations Through MultiLevel Parallelism From Laptops to Supercomputers. </span><span class="font5" style="font-style:italic;">SoftwareX</span><span class="font5">. 1(2): 19– 25</span></p>
<p><span class="font5">Albeck, S., Unger, R., Schreiber, G. 2000. Evaluation of Direct and Cooperative Contributions Towards The Strength of Buried Hydrogen Bonds and Salt Bridges. </span><span class="font5" style="font-style:italic;">Journal of Molecular Biology</span><span class="font5">. </span><span class="font5" style="font-style:italic;">298</span><span class="font5">(3): 503–520</span></p>
<p><span class="font5">Al-Karmalawy, A.A., Dahab, M.A., Metwaly, A.M., Elhady, S.S., Elkaeed, E.B., Eissa, I.H., Darwish, K.M. 2021. Molecular Docking and Dynamics Simulation Revealed the Potential</span></p>
<p><span class="font5">Inhibitory Activity of ACEIs Against SARS-CoV-2 Targeting the hACE2 Receptor. </span><span class="font5" style="font-style:italic;">Frontiers in Chemistry</span><span class="font5">. </span><span class="font5" style="font-style:italic;">9</span></p>
<p><span class="font5">Bosshard, H. R., Marti, D. N., Jelesarov, I. 2004. Protein Stabilization by Salt Bridges: Concepts, Experimental Approaches and Clarification of Some Misunderstandings. </span><span class="font5" style="font-style:italic;">Journal of Molecular Recognition.</span><span class="font5"> 17(1): 1–16</span></p>
<p><span class="font5">Guex, N., Peitsch, M.C. 1997. SWISS-MODEL and the Swiss-Pdb Viewer: &nbsp;An</span></p>
<p><span class="font5">Environment for Comparative Protein Modeling. </span><span class="font5" style="font-style:italic;">Electrophoresis</span><span class="font5">. &nbsp;&nbsp;18(15):</span></p>
<p><span class="font5">2714–2723</span></p>
<p><span class="font5">Hollingsworth, S.A., Dror, R.O. 2018. Molecular Dynamics Simulation for All. </span><span class="font5" style="font-style:italic;">Neuron</span><span class="font5">. 99(6): 1129–1143</span></p>
<p><span class="font5">Karplus, M., McCammon, J.A. 2002. Molecular Dynamics &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Simulations &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;of</span></p>
<p><span class="font5">Biomolecules. </span><span class="font5" style="font-style:italic;">Nature Structural Biology</span><span class="font5">. </span><span class="font5" style="font-style:italic;">9</span><span class="font5">(9): 646–652</span></p>
<p><span class="font5">Kumar, S., Nussinov, R. 2002. Close-Range Electrostatic Interactions in Proteins. </span><span class="font5" style="font-style:italic;">ChemBioChem</span><span class="font5">. </span><span class="font5" style="font-style:italic;">3</span><span class="font5">(7): 604-610</span></p>
<p><span class="font5">Meng, X.Y., Zhang, H.X., Mezei, M., Cui, M. 2011. Molecular Docking: A Powerful Approach for Structure-Based Drug Discovery. </span><span class="font5" style="font-style:italic;">Current Computer Aided-Drug Design</span><span class="font5">. </span><span class="font5" style="font-style:italic;">7</span><span class="font5">(2): 146–157</span></p>
<p><span class="font5">Messaoudi, A., Belguith, H., Ben Hamida, J. 2013. Homology Modeling and Virtual Screening Approaches to Identify Potent Inhibitors of VEB-1 β-Lactamase. </span><span class="font5" style="font-style:italic;">Theoretical Biology and Medical Modelling</span><span class="font5">. </span><span class="font5" style="font-style:italic;">10</span><span class="font5">(1)</span></p>
<p><span class="font5">Miyazono, K., Tanokura, M. 2022. New Era in Structural Biology with The Alphafold Program. </span><span class="font5" style="font-style:italic;">Translational and Regulatory Sciences</span><span class="font5">. 4(2): 48–52</span></p>
<p><span class="font5">Morris, G.M., Huey, R., Lindstrom, W., Sanner, M.F., Belew, R. K., Goodsell, D. S., Olson, A.J. 2009. AutoDock4 and AutoDockTools4: Automated Docking with Selective Receptor Flexibility. </span><span class="font5" style="font-style:italic;">Journal of Computational Chemistry</span><span class="font5">. 30(16): 2785–2791</span></p>
<p><span class="font5">Musafia, B., Buchner, V., Arad, D. 1995. Complex Salt Bridges in Proteins: Statistical Analysis of Structure and Function. </span><span class="font5" style="font-style:italic;">Journal of Molecular Biology</span><span class="font5">. 254(4): 761–770</span></p>
<p><span class="font5">Oostenbrink, C., Villa, A., Mark, A.E., Van Gunsteren, W.F. 2004. A Biomolecular</span></p>
<p><span class="font5">Force Field Based on The Free Enthalpy of Hydration and Solvation: The GROMOS Force-Field Parameter Sets 53A5 and 53A6. </span><span class="font5" style="font-style:italic;">Journal of Computational Chemistry</span><span class="font5">. 25(13): 1656–1676</span></p>
<p><span class="font5">Pettersen, E.F., Goddard, T.D., Huang, C.C., Couch, G. S., Greenblatt, D.M., Meng, E.C., Ferrin, T.E. 2004. UCSF Chimera: A Visualization System for Exploratory Research and Analysis. </span><span class="font5" style="font-style:italic;">Journal of Computational &nbsp;&nbsp;Chemistry</span><span class="font5">. &nbsp;&nbsp;25(13):</span></p>
<p><span class="font5">1605–1612</span></p>
<p><span class="font5">Rácz, A., Mihalovits, L.M., Bajusz, D., Héberger, K., &amp;&nbsp;Miranda-Quintana,</span></p>
<p><span class="font5">R.A. 2022. Molecular Dynamics Simulations and Diversity Selection by Extended Continuous Similarity Indices. </span><span class="font5" style="font-style:italic;">Journal of Chemical Information and Modeling</span><span class="font5">. &nbsp;62(14):</span></p>
<p><span class="font5">3415–3425</span></p>
<p><span class="font5">Shukla, R., Shukla, H., Tripathi, T. 2018. Activity loss by H46A Mutation in Mycobacterium Tuberculosis Isocitrate Lyase is Due To Decrease in Structural Plasticity and Collective Motions of The Active Site. &nbsp;&nbsp;</span><span class="font5" style="font-style:italic;">Tuberculosis</span><span class="font5">.</span></p>
<p><span class="font5">108:143–150</span></p>
<p><span class="font5">Takano, K., Ogasahara, K., Kaneda, H., Yamagata, Y., Fujii, S., Kanaya, E., Kikuchi, M., Oobatake, M., Yutani, K. 1995. Contribution of Hydrophobic Residues to the Stability of Human Lysozyme: Calorimetric Studies and X-ray Structural Analysis of the Five Isoleucine to Valine Mutants. </span><span class="font5" style="font-style:italic;">Journal of Molecular Biology</span><span class="font5">. 254(1): 62–76</span></p>
<p><span class="font5">Takano, K., Tsuchimori, K., Yamagata, Y., Yutani, K. 2000. Contribution of Salt Bridges Near The Surface of A Protein to The Conformational Stability,. </span><span class="font5" style="font-style:italic;">Biochemistry</span><span class="font5">. 39(40): 12375–12381</span></p>
<p><span class="font5">Tian, W., Chen, C., Lei, X., Zhao, J., Liang, J. 2018. CASTp 3.0: Computed Atlas of Surface Topography of Proteins. </span><span class="font5" style="font-style:italic;">Nucleic Acids Research</span><span class="font5">. &nbsp;&nbsp;46(W1):</span></p>
<p><span class="font5">W363–W367</span></p>
<p><span class="font5">Walker, K. D., Causgrove, T. P. 2009. Contribution of Arginine-Glutamic Acid Salt Bridges to Helix Stability. </span><span class="font5" style="font-style:italic;">Journal of Molecular Modeling</span><span class="font5">. 15(10): 1213–1219</span></p>
<p><span class="font6">7</span></p>